<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Title</title>
	<link rel="stylesheet" href="css/igem-reset.css">
	<link rel="stylesheet" href="css/SBcss/style.css">
   	<link rel="stylesheet" href="css/styleSB.css">
    <link rel="stylesheet" href="css/DryLabPage.css">
	<style>/* Reset CSS */

		#sideMenu, #top_title, .patrollink, #firstHeading, #home_logo, #sideMenu { display:none; }
		#content { padding:0px; width:100%; margin-top:-7px; margin-left:0px; border:none;}
		body, html {background-color:#212529; width: 100%; height: 100%;}
		#bodyContent h1, #bodyContent h2, #bodyContent h3, #bodyContent h4, #bodyContent h5 { margin-bottom: 0px; }
		#bodyContent a[href ^="https://"], .link-https { padding-right:0px;}

    </style>
</head>

<body>
    
	<!-- NAVBAR SECTION -->

	<nav class="navbar fixed-top navbar-expand-lg navbar-light menu">
		<div class="container-fluid nav-container">
		  <a class="navbar-brand" href="#">
			<img src="assets/svgs/synbactory2.svg">
		  </a>
		  <div class="center-expand">
		</div>
		  <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		  <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
		    <ul class="navbar-nav">
		      <li class="nav-item">
			<a class="nav-link active" aria-current="page" href="#">Home</a>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Team
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Team.html">Members</a></li>
			  <li><a class="dropdown-item" href="Attributions.html">Attributions</a></li>
			  <li><a class="dropdown-item" href="Collaborations.html">Collaborations</a></li>
			  <li><a class="dropdown-item" href="Sponsors.html">Sponsors</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Project
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Description.html">Description</a></li>
			  <li><a class="dropdown-item" href="Design.html">Design</a></li>
			  <li><a class="dropdown-item" href="Notebook.html">Notebook</a></li>
			  <li><a class="dropdown-item" href="Contribution.html">Contributions</a></li>
			  <li><a class="dropdown-item" href="Engineering.html">Engineering Success</a></li>
			  <li><a class="dropdown-item" href="Implementation.html">Future Implementation</a></li>
			  <li><a class="dropdown-item" href="Safety.html">Safety</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Wet Lab
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Experiments.html">Experiments</a></li>
			  <li><a class="dropdown-item" href="Constructs.html">Constructs</a></li>
			  <li><a class="dropdown-item" href="Measurement.html">Measurement</a></li>
			  <li><a class="dropdown-item" href="LabNotebook.html">Lab Notebook</a></li>
			  <li><a class="dropdown-item" href="Results.html">Results</a></li>
			  <li><a class="dropdown-item" href="FutureWork.html">Future Work</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Parts
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <!-- <li><a class="dropdown-item" href="#">Overview</a></li> -->
			  <li><a class="dropdown-item" href="BasicParts.html">Basic</a></li>
			  <li><a class="dropdown-item" href="CompositeParts.html">Composite</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Dry Lab
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Model.html">Overview</a></li>
			  <li><a class="dropdown-item" href="EColi.html">E Coli</a></li>
			  <li><a class="dropdown-item" href="Cyanobacteria.html">Cyanobacteria</a></li>
			  <li><a class="dropdown-item" href="Coculture.html">Coculture</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Human Practices
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Human_Practices.html">iHP</a></li>
			  <li><a class="dropdown-item" href="#">Public Engagement</a></li>
			  <li><a class="dropdown-item" href="Communication.html">Education</a></li>
			</ul>
		      </li>
		      <li class="nav-item">
			<a class="nav-link" href="#">Judging Form</a>
		      </li>
		      <!-- <button type="button" class="rounded-pill btn-rounded">Settings</button> -->
		    </ul>
		  </div>
		</div>
	</nav>




    <!-- TITLE SECTION -->
    
    <div class="section-1">
        <h1><br><br><br><br>Cyanobacteria</h1>
    </div>
        


    <!-- WAVES SPACER -->
	<a id="top"></a>
    <div class="spacer layer1"></div>




    <!-- CONTENT + SIDENAV SECTION -->

    <div class="second-div container-fluid">
        <div class="row">


            <!-- SIDENAV -->
            <div class="col-lg-3 col-md-3">
                <div class="sidenav">
                    <nav>
                        <ul id="sidenav-id">
                            <li class="nav-item"><a class="nav-link" href="#1">Introduction</a></li>
                            <li class="nav-item"><a class="nav-link" href="#2">Modelling Cyanobacteria for Sucrose Production</a></li>
                            <li class="nav-item"><a class="nav-link" href="#3">Biomass vs CO2</a></li>
                            <li class="nav-item"><a class="nav-link" href="#4">Biomass vs Sucrose</a></li>
                            <li class="nav-item"><a class="nav-link" href="#5">Sucrose Production</a></li>
                            <li class="nav-item"><a class="nav-link" href="#6">Identifying Over Expressions and Knockouts</a></li>
			    			<li class="nav-item"><a class="nav-link" href="#7">References</a></li>
                        </ul>
                    </nav>
                </div>
            </div>




            <!-- CONTENT -->
            <div class="col-lg-9 col-md-9">
                <div class="content">



                    <!-- INTRODUCTION -->
                    <a id="1"></a>
                    <h1>Introduction</h1>
                    <p>Cyanobacteria is the green element, both figuratively and literally, of our project. Modelling its behaviour both in a pure culture and in the co culture matters a lot to the direction we put our attention to. We’ve used FBA and MoMA to predict the growth rates and sucrose production rates of cyanobacteria, both with and without salt stress. Dr.Costas Maranas was generous enough to provide the Genome scale metabolic model for Synechococcus elongatus UTEX 2973 from Mueller et al., 2017.</p>
                    <!-- END OF INTRODUCTION-->



                    <!-- MODELLING CYANO FOR SUCROSE PRODUCTION -->
                    <a id="2"></a>
                    <h1>Modelling Cyanobacteria for optimum sucrose production</h1>

                    <p>The first hurdle we encountered while modelling cyanobacteria was simulating salt stress. Literature says simulating something like salt stress alters a lot of reactions in the organism and not all of them are straight forward or even known, so we had to make a few assumptions to make the model work.</p>
                    <h2 class="h3"><strong>Assumptions:</strong></h2>
                    <ol class="a">
                        <li style="padding-left:1em;">sps and spp genes are a part of the UTEX 2973 Genome even though they weren’t part of the model *insert spp and sps reaction equations* </li>
                        <li style="padding-left:1em;">cscB exports are of constant efficiency </li>
                        <li style="padding-left:1em;">FBA is a steady-state analysis therefore the growth phase cannot be simulated (check <a href="Coculture.html">co-culture </a>for more information on this) </li>
                        <li style="padding-left:1em;">Coupling biomass and sucrose accumulation and restraining cscB export to less than 100% simulates salt stress </li>
                    </ol>
                    <br>
                    <h2 class="h3"><strong>Results: </strong></h2>
                    <ul class="b">
                        <li style="padding-left:1em;">Upon adding “SPP”, “SPS” and “cscB” and running FBA with an objective to maximise growth rate we obtained a maximum growth rate of <strong>0.3033 h-1</strong> and a sucrose production and transport flux of <strong>0</strong>. This value is very similar to the experimental growth rate of <strong>0.33 ± 0.05 h</strong> (paper insert) , which implies confidence in the model we’re using. </li>
                        <li style="padding-left:1em;">For the following conditions upon performing MoMA for optimised sucrose production we obtained a growth rate of <strong>0.2012 h−1</strong>  and a sucrose output flux of <strong>0.3079 mmol/gDW/h</strong>. </li>
                        <ul class="d">
                            <li style="padding-left:1em;">Conditions:</li>
                                <ul class="c">
                                    <li style="padding-left:1em;">Salt stress of <strong>150mM</strong> simulated by coupling sucrose accumulation and biomass production </li>
                                    <li style="padding-left:1em;">CO2 uptake rate of <strong>-12.2 mmol/gDW/h</strong> <a href="#">[27]</a> </li>
                                    <li style="padding-left:1em;">cscB efficiency of <strong>90%</strong> <a href="#">[12]</a> </li>
                                </ul>
                        </ul>
                    </ul>
                    <br>
                    <strong>Upon performing MoMA for a range of values of CO2 uptakes and cscB efficiencies the following result was obtained:</strong>
                    <br><br><br><br>

                    <!-- BIOMASS VS CO2-->
                    <a id="3"></a>
                    <h2 class="h2"> <strong>Biomass vs CO2</strong></h2>
                    <br>
					<div class="graph">
						<img src="assets/Graphs/Cyano/Basic/Biomass vs CO2.jpg" alt="Biomass vs CO_2 graph for different constant cscB efficiency values and a constant salt stress of 150mM. A linear pattern is observed" width=800>
						<p style="padding-top: 15px;"> Fig 1: Relationship between Growth rate and CO2 uptake rates when run at a salt conc. 150mM under varying cscB efficiencies</p>
					</div>
					<br>
                    <p><strong>Observations: </strong></p>
                    <ol class="a">
                        <li style="padding-left:1em;">Biomass production rates decrease with decreasing CO2 uptake rates for a constant cscB efficiency </li>
                        <li style="padding-left:1em;">Biomass increases for a decreasing cscB efficiency at a given CO2 intake value. </li>
                        <li style="padding-left:1em;">The linear nature of the graph is due to the fact that FBA is generally a linear model and not a non-linear model. The assumption that cscB has a constant efficiency is also a contributing factor </li>
                    </ol>
                    <p><strong>Inference: </strong> </p>
                    <p style="padding-left: 12%;"> Carbon flux through the biomass reaction decreases as the efficiency of cscB increases. Since the objective function is to maximise sucrose production we can safely say cscB acts as a sucrose sink and therefore increases flux through sucrose production reaction at higher efficiencies.</p>
                    <br><br><br>

                    <!-- BIOMASS VS SUCROSE-->
                    <a id="4"></a>
                    <h2 class="h2"> <strong>Biomass vs Sucrose</strong></h2>
                    <br>
					<div class="graph">
						<img src="assets/Graphs/Cyano/Basic/Biomass vs Sucrose.jpg" alt="Biomass vs Sucrose graph for different CO_2 uptake rates. We observe a linear pattern yet again where sucrose production increases with decrease in growth rates" width=800>
						<p style="padding-top: 15px;">Fig 2: Relationship between sucrose production and growth rate under a varying CO2 uptake rates for a salt concentration of 150mM</p>
					</div>
					<p><strong>Observations: </strong></p>
                    <ol class="a">
                        <li style="padding-left:1em;">Sucrose production rates decrease with increasing biomass production rates </li>
                        <li style="padding-left:1em;">Both increase with increase in CO2 uptake </li>
                        <li style="padding-left:1em;">The linear nature is again, due to the linear nature of FBA and the fact that both of them share the same carbon source i.e. CO2 </li>
                    </ol>
                    <p><strong>Inference: </strong></p>
                    <p style="padding-left: 12%;">Since the objective function is sucrose production and both biomass and sucrose share the same carbon source one can easily say they’ll be inversely proportional. Therefore, our aim is to direct carbon flux through both the reactions in a fixed ratio such that the organism lives comfortably and produces ample amounts of sucrose. </p>
                    <br><br><br>

                    <!-- SUCROSE PRODUCTION-->
                    <a id="5"></a>
                    <h2 class="h2"> <strong>Sucrose production</strong></h2>
                    <p>This is a graphical representation of all the results and relations we have obtained from our attempts till now. We have eliminated all the metabolic states that do not agree with our MoMA results. The colour bar shows the sucrose production fluxes.</p>
                    <p>Observations for this are similar to the previous ones. This just helps us identify the region of conditions that we should be trying to replicate in the lab. Clearly sucrose production is maximum in the region with high cscB efficiency and high CO2 uptake rates.</p>
                    <br>
					<div class="graph">
						<img src="assets/Graphs/Cyano/Basic/3_Variables.jpg" alt="A graph respresting varaition of sucrose production with increase in growth rates, CO_2 Uptake rates and cscB efficieny values" width=800>
						<p style="padding-top: 15px;">Fig 9: A graph showing the production of sucrose for varying CO2 uptake rates and cscB efficiencies <br> Note - the colour bar on the right shows the mmol/gDW/h values of sucrose</p>
					</div>
					<p>As you can see from the graph sucrose production is maximum when there is a higher intake of carbon dioxide (source) and a higher cscB efficiency (sink). Ideally we would be trying to re-create the above mentioned conditions for enhanced production of sucrose whilst maintaining a significant fraction of maximum growth rates, but amplifying something that is already at 90% efficiency (cscB) is hard and to some extend unnecessary. Therefore we went a step ahead to find out other possible ways to increase sucrose production without losing much growth rates.</p>
                    <!-- END OF MODELLING CYANO FOR SUCROSE PRODUCTION -->



                    <!-- OVER EXPRESSIONS AND KNOCKOUTS -->
                    <a id="6"></a>
                    <h1>Identifying Over-expressions and Knockouts for enhanced sucrose production</h1>
			        <p>We have used OptKnock to identify knockout targets and FSEOF to identify over-expression targets.</p>
			        <br><br>


                    <!-- TARGETS FOR OVER EXPRESSION-->
                    <h2 class="h2">Targets for Over-expression</h2>
			        <p>FSEOF is the go to algorithm to identify targets for amplification. Upon running FSEOF under the following conditions we obtained a few targets for over-expression.</p>
					<br>

					<h2 class="h3">Assumptions: </h2>
			        <ol class="a">
				        <li style="padding-left:1em;">Carbon Dioxide uptake rate is -12.2mmol/gDW/hr (Max value from literature) </li>
				        <li style="padding-left:1em;">We have neglected salt stress because we were getting insensible results because of it </li>
			        </ol>
					<br>

					<h2 class="h3">Sources of inaccuraies: </h2>
					<ol class="a">
						<li style="padding-left:1em;"> The model is not complete </li>
						<li style="padding-left:1em;"> The geens have no names</li>
						<li style="padding-left:1em;"> Some reactions have no gene-protein-relations. Gene-protein-relations (GPRs) are rules that bind reactions in the organism to particular genes or combination of genes. For example, if reaction R1 requires both genes G1 and G2 then in the absence of either one of the genes the reaction seizes to work</li>
					</ol>
					<br>

					<h2 class="h3">Results: </h2>
					<ol class="a"> 
						<li style="padding-left: 1em;"> We have obtained a total of 80 reactions that can be overexpressed</li>
						<li style="padding-left: 1em;"> We have obtained a total gene count of 87. This number is a bit misleading because if we were to over express all these 80 reactions we wouldn’t have to over express all the 87 genes because  some of the Gene-Protein-Relations do not require the entire set of genes that govern them</li>
						<li style="padding-left: 1em;"> These 87 genes are responsible for a total of 555 reactions in the organism. Few of them are the same reaction but in different parts of the organism, for example if the cytoplasm and periplasm both contain a reaction X, the reaction X is counted twice</li>
						<li style="padding-left: 1em;"> If we were to count X only once then we have 107 reactions  governed by these 87 genes but this is not of much importance because over expressing any gene will over express all its reactions whether it is in cytoplasm or periplasm</li>
						<li style="padding-left: 1em;"> 25 out of the 80 target reactions have no gene-protein-relations so we cannot test these out in the lab</li>
						<li style="padding-left: 1em;"> Although the sucrose production reaches the theoretical maximum, the biomass flux slowly dies as this happens therefore the higher is not always the better</li>
						<li style="padding-left: 1em;"> Almost all of them follow linear behaviour</li>					
					</ol>
					<br>
				
					<p>Given below are 4 graphs each with 5 reactions plotted against varying sucrose production fluxes. This is just to give a comparison of the various results we obtained. If you would like to download the entire set of interactive graphs representing these results <a>click here</a>. We also have graphs which depict the variations of biomass along with sucrose and these reactions (<a>click here</a>)</p>
					<br>
					<!-- Slideshow of graphs-->
					<div id="carouselExampleCaptions" class="carousel carousel-dark slide" data-bs-ride="carousel">
						<div class="carousel-indicators">
						  <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
						  <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="1" aria-label="Slide 2"></button>
						  <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="2" aria-label="Slide 3"></button>
						  <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="3" aria-label="Slide 4"></button>
						</div>
						<div class="carousel-inner">
							<div class="carousel-item active graph"> 
								<img src="assets/graphs/Cyano/FSEOF/Cyano_FSEOF_1-5.jpg" alt="Best 5 over-expression targets in one graph" width=800>
							</div>
							<div class="carousel-item graph">
								<img src="assets/Graphs/Cyano/FSEOF/Cyano_FSEOF_6-10.jpg" alt="Next best 5 over-expression targets in one graph" width=800>
							</div>
							<div class="carousel-item graph">
								<img src="assets/Graphs/Cyano/FSEOF/Cyano_FSEOF_11-15.jpg" alt="Third best 5 over-expression targets in one graph" width=800>
							</div>
							<div class="carousel-item graph">
								<img src="assets/Graphs/Cyano/FSEOF/Cyano_FSEOF_16-20.jpg" alt="Fourth best 5 over-expression targets in one graph" width=800>
							</div>
						</div>
						<button class="carousel-control-prev" type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide="prev">
						  <span class="carousel-control-prev-icon" aria-hidden="true"></span>
						  <span class="visually-hidden">Previous</span>
						</button>
						<button class="carousel-control-next" type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide="next">
						  <span class="carousel-control-next-icon" aria-hidden="true"></span>
						  <span class="visually-hidden">Next</span>
						</button>
						  </div>
						  <h3 id="slidetext"></h3>
			        <br><br><br>
					<!-- END OF SLIDESHOW -->
					
					<p>We searched through literature to find a way to represent the data we have collected, but we found no proven way to do so. Therefore this can utmost be considered as a qualitative analysis of potential targets for over expression as opposed to being a quantitative and conclusive result of targets for overexpression.</p>
					<p><strong> These still need in-laboratory proof for us to confirm these over expression targets.</strong></p>
					<p>Since we could not just leave the results as is, we ranked them in order of decreasing slope because we believe that the ones with higher slopes have a higher flux handling capacity i.e., they can hold more flux for the desired purpose. In terms of the analogy between FSEOF and traffic in a city, these are the roads that can take a good chunk of traffic away from other smaller roads and still not get jammed. </p>
					<p> These targets of overexpression are in terms of reactions, but in wet lab we can not selectively overexpress reactions. What we can overexpress in the lab are genes. So naturally, we looked into how other reactions in the model get affected if and when we overexpress one of the above genes.
						The table below shows how the genes responsible for the above reactions will affect other reactions in the organism. We have generated a <strong>reaction flux vs sucrose flux graph</strong> and a <strong>reaction vs sucrose vs biomass graph</strong> for each over expression target, all of which can be downloaded <a href="#">here</a>.
					</p>
					<p><strong>Note - the “fig x” in each row indicates which graph in the folder corresponds to the said reaction</strong></p>
					<br>
		
					<!-- Start of Table -->
					<h1>Over-expression Table</h1>
					<hr>
					<br>
					<div class="table">
						<!-- Headers -->
						<div class="table1-row table-header">
							<div class="cell">
								Target Reaction
							</div>
							<div class="cell">
								Gene-Protein-Relation
							</div>
							<div class="cell">
								Other reactions controlled by the gene
							</div>
						</div>
						<!--End of headers -->

						<!-- Other rows -->
						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Proton exchange (Fig 1)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>
					

						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Water transport pp (Fig 2)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_08900
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								Water transport ltr
							</div>
						</div>
						
						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Water transport ext (Fig 3)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
							 	-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								ChorismateL-glutamine aminotransferase (Fig 4)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_14170 or M744_09950
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								<strong>M744_14170</strong> - Xanthosine-5'-phosphateL-glutamine amido-ligase (AMP-forming) <br>
								<strong>M744_09950</strong> - NA
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Pyrophosphate phosphohydrolase (Fig 5)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_09680
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								Triphosphate phosphohydrolase
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Asp-tRNA(Asn)L-glutamine amido-ligase (ADP-forming) (Fig 6)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_02485 and M744_04675 and M744_05790
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								<strong>M744_02485</strong> - Glu-tRNA(Gln)L-glutamine amido-ligase(ADP-forming) <br>
								<strong>M744_04675</strong> - Glu-tRNA(Gln)L-glutamine amido-ligase(ADP-forming) <br>
								<strong>M744_05790</strong> - Glu-tRNA(Gln)L-glutamine amido-ligase(ADP-forming)
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								L-AsparaginetRNA(Asn) ligase (AMP-forming) (Fig 7)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_12600
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								L-Aspartateammonia ligase (AMP-forming) (Fig 8)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								ATPAMP phosphotransferase (Fig 9)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_05275
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Cytochrome b6/f complex periplasm (Fig 10)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_10480
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								Cytochrome b6/f complex, Plastoquinone Oxidized-plastocyanin oxidoreductase
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								NAD(P)H dehydrogenase (plastoquinone-8 & 3 protons) (periplasm) (Fig 11)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_06540 and M744_06535 and <br> M744_09900 and M744_09515 and <br> M744_10750 and M744_09885 and <br> M744_09890 and M744_07765 and <br> M744_09895 and M744_10740 <br> and M744_10745 and M744_06510 <br> and M744_05170 and M744_01630 and <br> M744_01625 and M744_03420 or <br> M744_09385 and M744_06535 and <br> M744_09900 and M744_09515 and <br> M744_10750 and M744_09885 and <br> M744_09890 and M744_07765 and <br> M744_09895 and M744_10740 and <br> M744_10745 and M744_06510 and <br> M744_05170 and M744_01630 and <br> M744_01625 and M744_03420
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								<strong>M744_06540, M744_06535, M744_09385, M744_06535</strong> - NAD(P)H dehydrogenase (plastoquinone-8 & 3 protons) (tilacoide) <br>

								<strong>M744_09900, M744_09515, M744_09885, M744_06510, M744_05170, M744_09900, M744_09515, M744_09885, M744_06510, M744_05170</strong>  - NAD(P)H dehydrogenase (plastoquinone-8 & 3 protons) (tilacoide), 
								Active co2 transporter facilitator (tilacoide), 
								Active co2 transporter facilitator (periplasm) <br>
								
								<strong>M744_10750, M744_09890, M744_07765, M744_09895, M744_10740, M744_10745, M744_10750, M744_09890, M744_07765, M744_09895, M744_10740, M744_10745</strong> - NADPHp-benzoquinone oxidoreductase,
								NADHquinone oxidoreductase, 
								NAD(P)H dehydrogenase (plastoquinone-8 & 3 protons) (tilacoide), 
								Active co2 transporter facilitator (tilacoide), 
								Active co2 transporter facilitator (periplasm) <br>
								
								
								<strong>M744_01630, M744_01625, M744_03420, M744_01630, M744_01625, M744_03420</strong>  - Bidirectional [NiFe] Hydrogenase,
								NAD(P)H dehydrogenase (plastoquinone-8 & 3 protons) (tilacoide), 
								Active co2 transporter facilitator (tilacoide), 
								Active co2 transporter facilitator (periplasm) <br>
							
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Cytochrome C Oxidase, Synechocystis (2 protons) (Fig 12)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_02050 or ( M744_03160 and M744_03155 and M744_03150 )
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								<strong>M744_02050, M744_03160, M744_03155, M744_03150</strong>  - cytochrome c oxidase, synechocystis (2 protons periplasm)
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								2,3,4,5-TetrahydrodipicolinateNAD+ oxidoreductase (Fig 13)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_05690
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								2,3,4,5-TetrahydrodipicolinateNADP+ oxidoreductase
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								2,3,4,5-TetrahydrodipicolinateNADP+ oxidoreductase (Fig 14)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_05690
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								2,3,4,5-TetrahydrodipicolinateNAD+ oxidoreductase
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Sedoheptulose 1,7-bisphosphate D-glyceraldehyde-3-phosphate-lyase (Fig 15)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_09355
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								D-Fructose 1-phosphate D-glyceraldehyde-3-phosphate-lyase, <br>
								D-Fructose-1,6-bisphosphate D-glyceraldehyde-3-phosphate-lyase
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								FerredoxinNADP+ oxidoreductase (Fig 16)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_11820
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Oxygen transport ext (Fig 17)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								ATP synthase (four protons for one ATP) (periplasm) (Fig 18)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_01360 and M744_01350 and M744_01345 and M744_01335 and M744_01340 and M744_01330 and M744_04715 and M744_04710 and M744_01355 and M744_01365
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								<strong>ALL</strong> - ATP synthetase(u)
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								2-Phospho-D-glycerate 2,3-phosphomutase (Fig 19)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_00620 and M744_06000 and M744_00535 and M744_08585
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								<strong>M744_00620</strong> - 3-Phospho-D-glyceroylphosphate 2,3-phosphomutase
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Carbonic acid hydro-lyase (Fig 21)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_09475 or M744_09655 or M744_09335 or M744_14040
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								<strong>M744_09335</strong> - Carbonic anhydrase
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Nitrate exchange (Fig 22)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								ATPnucleoside-diphosphate phosphatransferase (Fig 23)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_03750
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								D-Ribulose-5-phosphate 3-epimerase (Fig 24)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_13820
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Acetyl-CoAenzyme N6-(dihydrolipoyl)lysine S-acetyltransferase (Fig 25)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_11335
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								PyruvateNAD+ 2-oxidoreductase (CoA-acetylating)
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								L-Valinepyruvate aminotransferase (Fig 26)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_05030
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Sucrose exchange (Fig 27)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								SPS (Fig 28)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								SPP (Fig 29)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								cscB (Fig 30)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								UTP-glucose-1-phosphate uridylyltransferase (Fig 31)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								ATPnucleoside-diphosphate phosphate transferase (Fig 32)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_03750
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								L-Aspartate2-oxoglutarate aminotransferase (Fig 33)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_03485
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								L-Phenylalanine2-oxoglutarate aminotransferase,
								L-Tyrosine2-oxoglutarate aminotransferase,
								L-Cysteine2-oxoglutarate aminotransferase,
								4-Hydroxy-L-glutamate2-oxoglutarate aminotransferase								
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								L-Phenylalanine2-oxoglutarate aminotransferase (Fig 56)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_03485 or M744_12480
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								<strong>M744_03485</strong> - Same as above <br>

								<strong>M744_12480</strong> - L-Tyrosine2-oxoglutarate aminotransferase,
								LL-2,6-diaminoheptanedioate2-oxoglutarate aminotransferase
								
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								L-Tyrosine2-oxoglutarate aminotransferase (Fig 58)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_03485 or M744_12480
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								Same as above
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								(S)-Malate hydro-lyase (Fig 34)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_11660
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								L-HomoserineNAD+ oxidoreductase (Fig 35)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_05930
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								(3R)-3-Hydroxyhexanoyl-[acyl-carrier-protein]NADP+ oxidoreductase (Fig 36)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_13375 or M744_08910
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								<strong>M744_13375, M744_08910</strong> - (3R)-3-Hydroxybutanoyl-[acyl-carrier protein]NADP+ oxidoreductase,
								(3R)-3-Hydroxytetradecanoyl-[acyl-carrier-protein]NADP+,
								3oxoacyl-acp reductase (c180),
								(3R)-3-Hydroxyoctanoyl-[acyl-carrier-protein]NADP+ oxidoreductase,
								3-oxoacyl-[acyl-carrier-protein] reductase (n-C121),
								3-oxoacyl-[acyl-carrier-protein] reductase (n-C141),
								3-oxoacyl-[acyl-carrier-protein] reductase (n-C161),
								3-oxoacyl-[acyl-carrier-protein] reductase (n-C181)
								
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								(3R)-3-Hydroxybutanoyl-[acyl-carrier protein]NADP+ oxidoreductase (Fig 37)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_13375 or M744_08910
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								Same as above
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								(3R)-3-Hydroxydodecanoyl-[acyl-carrier-protein]NADP+ oxidoreductase (Fig 38)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_13375 or M744_08910
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
							 	ame as above
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								(3R)-3-Hydroxyoctanoyl-[acyl-carrier-protein]NADP+ oxidoreductase (Fig 39)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_13375 or M744_08910
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								Same as above
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								(3R)-3-Hydroxypalmitoyl-[acyl-carrier-protein]NADP+ oxidoreductase (Fig 42)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_13375 or M744_08910
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
							 	Same as above
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								L-Leucine2-oxoglutarate aminotransferase (Fig 40)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_09945 or M744_11540
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								<strong>M744_09945</strong> - ​​L-Valine2-oxoglutarate aminotransferase,
								L-Isoleucine2-oxoglutarate aminotransferase,
								4-amino-4-deoxychorismate pyruvate-lyase
								
								<strong>M744_11540</strong> - ​​L-Valine2-oxoglutarate aminotransferase,
								L-Isoleucine2-oxoglutarate aminotransferase
							</div>
						</div>



						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								3-Isopropylmalate hydro-lyase (Fig 41)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_06945 and M744_03470
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								<strong>M744_06945, M744_03470</strong> - (R)-2-Methylmalate hydro-lyase,
								2-Isopropylmalate hydro-lyase,
								3-Isopropylmalate hydro-lyase
								
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								L-Isoleucine2-oxoglutarate aminotransferase (Fig 49)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_09945 or M744_11540
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								Same as above
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								IMP 1,2-hydrolase (decyclizing) (Fig 43)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_01000
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								10-Formyltetrahydrofolate5'-phosphoribosyl-5-amino-4
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								5,6,7,8-TetrahydrofolateNADP+ oxidoreductase (Fig 44)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_07375
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								N2-Acetyl-L-ornithine2-oxoglutarate aminotransferase (Fig 45)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_02940 or M744_11990
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								<strong>M744_02490</strong> - S-Adenosyl-L-methionine8-amino-7-oxononanoate aminotransferase,
								N-Succinyl-L-2,6-diaminoheptanedioate2-ocoglytarate
								
								<strong>M744_11990</strong> - L-Alanine3-oxopropanoate aminotransferase,
								4-Aminobutanoate2-oxoglutarate aminotransferase
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								N-Succinyl-L-2,6-diaminoheptanedioate2-ocoglytarate (Fig 50)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_02940
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								Same as above
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								N-Acetyl-L-glutamate-5-semialdehydeNADP+ 5-oxidoreductase (Fig 46)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_09420
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								5,10-methylenetetrahydrofolate,FADH2dUMP C-methyltransferase (Fig 47)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								1-(5-Phospho-D-ribosyl)-5-amino-4-imidazolecarboxylate carboxy-lyase (Fig 48)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_00760 and M744_09450
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								sn-Glycerol-3-phosphateNADP+ 2-oxidoreductase (Fig 51)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_03610
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Phosphate exchange (Fig 52)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Orotidine-5'-phosphatepyrophosphate phosphoribosyltransferase (Fig 53)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_03215
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								(S)-Dihydroorotate amidohydrolase (Fig 54)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_00530 and M744_00520
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								L-1-Pyrroline-5-carboxylateNAD+ oxidoreductase (Fig 55) 
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_00515
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								3-OxopropanoateNAD+ oxidoreductase (decarboxylating), <br>
								AcetaldehydeNAD+ oxidoreductase,<br>
								beta-Aminopropion aldehydeNAD+ oxidoreductase,<br>						
								glycolaldehydeNAD+ oxidoreductase,<br>								
								D-GlyceraldehydeNAD+ oxidoreductase,<br>								
								4-AminobutyraldehydeNAD+ oxidoreductase,<br>								
								Propane-1,2-diolNAD+ oxidoreductase,<br>								
								PhenylacetaldehydeNAD+ oxidoreductase,<br>								
								4-aminobutanalNAD+ 1-oxidoreductase,<br>								
								midazole acetaldehydeNAD+ oxidoreductase,<br>								
								N4-AcetylaminobutanalNAD+ oxidoreductase,<br>
								Methylmalonate - semialdehyde dehydrogenase (propanol),<br>						
								Indole-3-acetaldehydeNAD+ oxidoreductase,<br>
								L-1-Pyrroline-5-carboxylateNAD+ oxidoreductase <br>
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								3-OxopropanoateNAD+ oxidoreductase (decarboxylating) (Fig 77)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_00515
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								Same as above
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Sulfate exchange (Fig 57)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Potassium exchange (Fig 59)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								diacylglycerol kinase (n-C160) (Fig 60)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_00975
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								diacylglycerol kinase (n-C120),<br>
								diacylglycerol kinase (n-C161),<br>								
								diacylglycerol kinase (n-C180),<br>								
								diacylglycerol kinase (n-C181),<br>
								diacylglycerol kinase (n-C160)	
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
							  diacylglycerol kinase (n-C161) (Fig 62)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_00975
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								Same as above
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								diacylglycerol kinase (n-C181) (Fig 78)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_00975
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								Same as above
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								diacylglycerol kinase (n-C180) (Fig 80)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_00975
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								Same as above
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								D-Glucosamine 1-phosphate 1,6-phosphomutase (Fig 61)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_05710
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								D-Ribose 1,5-phosphomutase
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Magnesium exchange (Fig 63)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								UDP-N-acetylmuramate NADP+ oxidoreductase (Fig 64)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_07780
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Fe2+ exchange (Fig 65)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Fe3+ exchange (Fig 66)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Calcium exchange (Fig 67)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Calcium (Ca+2) transport out via proton antiport (periplasm) (Fig 68)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_02980
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Sodium exchange (Fig 69)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Sodium transport out via proton antiport (cytoplasm to periplasm) (Fig 70)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								(M744_12695 or M744_10315) and M744_04470
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Co2+ exchange (Fig 71)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Molybdate exchange (Fig 72)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Mn2+ exchange (Fig 73)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Cu2+ exchange (Fig74)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Zinc exchange (Fig 75)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Nicotinate-nucleotide pyrophosphatase
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_11945
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								Phosphoribosyltransferase (Fig 76)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								-
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
					</div>


						<div class="table1-row">
							<div class="cell" data-title="Target Reaction">
								ATPpolyphosphate phosphotransferase (Fig 79)
							</div>
							<div class="cell" data-title="Gene-Protein-Relation">
								M744_08870
							</div>
							<div class="cell" data-title="Other reactions controlled by the gene">
								-
							</div>
						</div>
					</div>	

					<br><br><br>


					<!-- TARGETS FOR KNOCKOUTS-->
                    <h2 class="h2">Targets for Knockouts</h2>
					<P>We ran Optknock like we did for e coli, but we were unable to obtain any sensible result. All our runs gave a <strong>0 growth rate</strong>. We believe the problem is with the model, as it is still incomplete and not verified, and therefore have concluded nothing further can be done to produce Knockout results.</P>
					<br><br><br><br><br><br><br><br><br><br>
					<!-- REFERENCES -->
					<h1 id="7">References</h1>
					<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>		
                </div>
            </div>        
		</div>
    </div>



<!-- -----------------------END OF CONTECT ----------------------------------- -->
	
	<!-- SPONSORS BAR -->
	<div class="footer">
		<div class="footer-wrapper">
			<!-- <div id="not-sponsors" class="footer-wrapper">	 -->
				<div id="Team", class="footer-wrapper">
					<img class="teamlogo" src="assets/Logos/IISER-Pune-iGEM_logo.png" alt="iGEM IISER Pune Logo" width="170">
				</div>
				<div id="socials" class="vertical-stack">
					<div><h3>Team IISER Pune India</h3></div>
					<div id="logos" class ="footer-wrapper ">
						<a href="https://www.instagram.com/igem_iiserp/"><img class="footer-links-socials" src="assets/Logos/Instagram_logo.jpg" alt="Instagram" width="20"></a>
						<a href="https://www.linkedin.com/in/igem-iiser-pune-2021/"><img class="footer-links-socials" src="assets/Logos/linkedin_logo.png" alt="Linkedin" width="30"></a>
						<a href="https://twitter.com/igem_iiserpune"><img class="footer-links-socials" src="assets/Logos/Twitter_logo.png" alt="Twitter" width="35"></a>
						<a href="https://www.facebook.com/igemiiserpune"><img class="footer-links-socials" src="assets/Logos/Facebook_logo.png" alt="Facebook" width="25"></a>
						<a href="https://www.youtube.com/channel/UCyo6NMLvM58sSh_tnxht36w"><img class="footer-links-socials" src="assets/Logos/Youtube_logo.png" alt="Youtube" width="35"></a>
					</div>
				</div>
			<!-- </div> -->
			<div id="sponsors" class="footer-wrapper">
				<div id="main-sponsor", class="">
					<a href="https://www.sailife.com/"><img class="footer-links" src="assets/Logos/Sai_logo.jpg" alt="Sai Life Sciences" width="280"></a>
				</div>
				<div class="vertical-stack">
					<a href="https://sg.idtdna.com/pages"><img class="footer-links" src="assets/Logos/IDT_logo.png" alt="IDT" width="220"></a>
					<a href="https://www.geneious.com/"><img class="footer-links" src="assets/Logos/Geneious_logo.png" alt="Geneious" width="220"></a>
				</div>
				<div class="vertical-stack">
					<div id="horizontal stack">
						<a href="https://www.notion.so/"><img class="footer-links" src="assets/Logos/Notion_logo.png" alt="Notion" width="70"></a>
						<img src="assets/Logos/IISER Pune Logo.png" alt="IISER Pune" width="80" style="padding-left: 1em;">
					</div>
					<a href="https://www.zoho.com/"><img class="footer-links" src="assets/Logos/Zoho_logo.png" alt="Zoho" width="150"></a>
				</div>
			</div>	
		</div>
	</div>
	<!-- END OF SPONSOR BAR-->


<!-- ACTUAL SCROLLING BAR-->
	<div class="header"></div>
    <div class="line" id="scrollIndicator"></div>

    <!-- back to top button -->
    <a href="#top"><img src="assets/svgs/arrow-up-square-fill.svg" id="myBtn" title="Go to top" width="30" height="30"></a>

	
	<!-- LINKING MAIN JS FILES-->
    <script src="js/bootstrap-bundle.js"></script>
	<script src="js/main.js"></script>
	<script src="js/jquery.min.js"></script>


	<!-- Gets the screenheight of each page for Scrollbar? -->
	<script>
		var screenheight = window.innerHeight,
		    screenwidth = window.innerWidth;

	</script>


	<!-- Scrollbar stuff again, only @Namasi22 knows :) -->
	<script>
		var body = document.body,
    		html = document.documentElement;

		var pageheight = Math.max(body.scrollHeight, body.offsetHeight, 
                       		   html.clientHeight, html.scrollHeight, html.offsetHeight);
	</script>
	

	<!-- Scrollbar stuff again, only @Namasi22 knows :) -->
	<script>
		var backtotopheight=pageheight/4.65
	</script>


	<!-- Script for the slideshow, wherever needed -->	
	<script>
		var total = $('.item').length;
		var actualtotal = 4;
		var currentIndex = $('div.active').index() + 1;
		$('#slidetext').html(currentIndex + '/'  + actualtotal);

		// This triggers after each slide change
		$('.carousel').on('slid.bs.carousel', function () {
  			currentIndex = $('div.active').index() + 1;
 
  		// Now display this wherever you want
  		var text = currentIndex + '/' + actualtotal;
  			$('#slidetext').html(text);
		});
	</script>



	<!-- SCRIPT FOR SCROLL AWARE SIDEBAR HIGHLIGHT -->
	<!-- ------------------------------------------------------------------------------------------------------------- -->
	<!-- MAKE SURE YOU CHANGE THE NUMBER OF DIVISIONS -->
	<script>
		
			$(document).ready(function() {
				  $(window).scroll(function() {
					var scroll_top = $(window).scrollTop();
					var div_one_top = $("#1").offset().top;
					var div_two_top = $("#2").offset().top;
					var div_three_top = $("#3").offset().top;
					var div_four_top = $("#4").offset().top;
					var div_five_top = $("#5").offset().top;
					var div_six_top = $("#6").offset().top;
					var div_seven_top = $("#7").offset().top;
	
					if (scroll_top > div_one_top && scroll_top < div_two_top) {
						//Division 1
						if (!$('a[href="#1"]').hasClass('sidenav-active')) {
							$('a[href="#1"]').addClass('sidenav-active');
						}
	
						if ($('a[href="#2"]').hasClass('sidenav-active')) {
							$('a[href="#2"]').removeClass('sidenav-active');
						}
	
					} else if (scroll_top > div_two_top && scroll_top < div_three_top) {
						// Division 2
						if ($('a[href="#1"]').hasClass('sidenav-active')) {
							$('a[href="#1"]').removeClass('sidenav-active');
						}
	
						if ($('a[href="#3"]').hasClass('sidenav-active')) {
							$('a[href="#3"]').removeClass('sidenav-active');
						}
	
						if (!$('a[href="#2"]').hasClass('sidenav-active')) {
							$('a[href="#2"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_three_top && scroll_top < div_four_top) {
						//Division 3
						if ($('a[href="#2"]').hasClass('sidenav-active')) {
							$('a[href="#2"]').removeClass('sidenav-active');
						}
	
						if ($('a[href="#4"]').hasClass('sidenav-active')) {
							$('a[href="#4"]').removeClass('sidenav-active');
						} 
	
						if (!$('a[href="#3"]').hasClass('sidenav-active')) {
							$('a[href="#3"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_four_top && scroll_top < div_five_top) {
						// You are in div 4
						if ($('a[href="#3"]').hasClass('sidenav-active')) {
							$('a[href="#3"]').removeClass('sidenav-active');
						}
	
						if ($('a[href="#5"]').hasClass('sidenav-active')) {
							$('a[href="#5"]').removeClass('sidenav-active');
						} 
	
						if (!$('a[href="#4"]').hasClass('sidenav-active')) {
							$('a[href="#4"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_five_top && scroll_top < div_six_top) {
						// You are in div 5
						if ($('a[href="#4"]').hasClass('sidenav-active')) {
							$('a[href="#4"]').removeClass('sidenav-active');
						}
	
						if ($('a[href="#6"]').hasClass('sidenav-active')) {
							$('a[href="#6"]').removeClass('sidenav-active');
						} 
	
						if (!$('a[href="#5"]').hasClass('sidenav-active')) {
							$('a[href="#5"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_six_top && scroll_top < div_seven_top) {
						// You are in div 6
						if ($('a[href="#5"]').hasClass('sidenav-active')) {
							$('a[href="#5"]').removeClass('sidenav-active');
						}
	
						if ($('a[href="#7"]').hasClass('sidenav-active')) {
							$('a[href="#7"]').removeClass('sidenav-active');
						} 
	
						if (!$('a[href="#6"]').hasClass('sidenav-active')) {
							$('a[href="#6"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_seven_top) {
						// You are in div 7
						if ($('a[href="#6"]').hasClass('sidenav-active')) {
							$('a[href="#6"]').removeClass('sidenav-active');
						}
	
						if (!$('a[href="#7"]').hasClass('sidenav-active')) {
							$('a[href="#7"]').addClass('sidenav-active');
						}
					}
				});
			});
			
	</script>
</body>
