

<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Design</title>
	<link rel="stylesheet" href="css/igem-reset.css">
	<link rel="stylesheet" href="css/SBcss/style.css">
   	<link rel="stylesheet" href="css/styleSB.css">
    	<link rel="stylesheet" href="css/DryLabPage.css">
	<style>/* Reset CSS */

		#sideMenu, #top_title, .patrollink, #firstHeading, #home_logo, #sideMenu { display:none; }
		#content { padding:0px; width:100%; margin-top:-7px; margin-left:0px; border:none;}
		body, html {background-color:#212529; width: 100%; height: 100%;}
		#bodyContent h1, #bodyContent h2, #bodyContent h3, #bodyContent h4, #bodyContent h5 { margin-bottom: 0px; }
		#bodyContent a[href ^="https://"], .link-https { padding-right:0px;}

    </style>
<body>
    

	
	<!-- NAVBAR SECTION -->

	<nav class="navbar fixed-top navbar-expand-lg navbar-light menu">
		<div class="container-fluid nav-container">
		  <a class="navbar-brand" href="index.html">
			<img src="assets/svgs/synbactory2.svg">
		  </a>
		  <div class="center-expand">
		</div>
		  <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		  <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
		    <ul class="navbar-nav">
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Team
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Team.html">Members</a></li>
			  <li><a class="dropdown-item" href="Attributions.html">Attributions</a></li>
			  <li><a class="dropdown-item" href="Collaborations.html">Collaborations</a></li>
			  <li><a class="dropdown-item" href="Partnership.html">Partnership</a></li>
			  <li><a class="dropdown-item" href="Sponsors.html">Sponsors</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Project
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Description.html">Description</a></li>
			  <li><a class="dropdown-item" href="Design.html">Design</a></li>
			  <li><a class="dropdown-item" href="TeamNotebook/Team Notebook 11e9906b405543a29361e38a50c81024.html">Notebook</a></li>
			  <li><a class="dropdown-item" href="Contribution.html">Contributions</a></li>
			  <li><a class="dropdown-item" href="Engineering.html">Engineering Success</a></li>
			  <li><a class="dropdown-item" href="Implementation.html">Future Implementation</a></li>
			  <li><a class="dropdown-item" href="Safety.html">Safety</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Wet Lab
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Experiments.html">Experiments</a></li>
			  <li><a class="dropdown-item" href="Constructs.html">Constructs</a></li>
			  <li><a class="dropdown-item" href="Measurement.html">Measurement</a></li>
			  <li><a class="dropdown-item" href="LabNotebook.html">Lab Notebook</a></li>
			  <li><a class="dropdown-item" href="Results.html">Results</a></li>
			  <li><a class="dropdown-item" href="FutureWork.html">Future Work</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Parts
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <!-- <li><a class="dropdown-item" href="#">Overview</a></li> -->
			  <li><a class="dropdown-item" href="BasicParts.html">Basic</a></li>
			  <li><a class="dropdown-item" href="CompositeParts.html">Composite</a></li>
			</ul>
		      </li>
		      <li class="nav-item">
			<a class="nav-link" href="https://2021.igem.org/Team:IISER-Pune-India/Model">Model</a>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Human Practices
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Human_Practices.html">Integrated Human Practices</a></li>
			  <li><a class="dropdown-item" href="Inclusivity.html">Inclusivity and Accessibility</a></li>
			  <li><a class="dropdown-item" href="Communication.html">Public Engagement and Education</a></li>
			</ul>
		      </li>
		      <li class="nav-item">
			<a class="nav-link" href="https://igem.org/2021_Judging_Form?id=3971">Judging Form</a>
		      </li>
		      <!-- <button type="button" class="rounded-pill btn-rounded">Settings</button> -->
		    </ul>
		  </div>
		</div>
	</nav>



    <!-- TITLE SECTION -->
    
    <div class="section-1">
        <h1><br><br><br><br>Design</h1>
    </div>
        


    <!-- WAVES SPACER -->

    <div class="spacer layer1"></div>
	<a id="top"></a>



    <!-- CONTENT + SIDENAV SECTION -->

<style>
	.content h1, .content h2, .content h3, .content h4, .content h5, .content h6{
	color: white !important;
	font-family: ProximaNova !important;
}
</style>
    <div class="second-div container-fluid">
        <div class="row">
            <!-- SIDENAV -->
            <div class="col-lg-3 col-md-3">
                <div class="sidenav">
                    <nav>
                        <ul id="sidenav-id">
                            <li class="nav-item"><a class="nav-link" href="#1">Cyanobacteria</a></li>
                            <li class="nav-item"><a class="nav-link" href="#2">E Coli</a></li>
                            <li class="nav-item"><a class="nav-link" href="#3">Modelling</a></li>
                            <li class="nav-item"><a class="nav-link" href="#4">References</a></li>
                        </ul>
                    </nav>
                </div>
            </div>
            <!-- CONTENT -->
            <div class="col-lg-9 col-md-9">
                <div class="content">
                    <h1 id="1">Cyanobacteria - the autotrophic module </h1>
                    <hr><br><br><br>
		    <p><strong>Cyanobacteria - the autotrophic module</strong></p>
		    <p>Certain cyanobacteria strains, like <em>Synechococcus elongatus</em>, naturally accumulate sucrose intracellularly under salt stress[1] to counter the osmotic pressure of their saline environment. We aimed to engineer <em>S. elongatus</em> UTEX 2973, a fast-growing strain of cyanobacteria,[2] to continually produce and secrete the sucrose it accumulates. To accomplish this, we intended to express <em>cscB </em>in our cyanobacterium. <em>cscB,</em> native to <em>E. coli</em> W, is a gene that codes for sucrose permease, a symporter that can transport sucrose and protons across the membrane of bacterial cells.[3]</p>

		    <div style="display: flex; flex-direction: row; justify-content: center;"><img style="max-width: 600px;" src="assets/Design/Cyanobacteria Sucrose Production Pathway_white.png"></div>
<br>
<p id="desc">The metabolic pathway for sucrose production in Synechococcus elongatus. Figure adapted from Lin et al. (2020)[7]
</p>
<h4>Abbreviations:&nbsp;</h4>
<p style="text-align: justify;">ADP-Glu: adenosine diphosphate glucose;&nbsp;</p>
<p style="text-align: justify;">CBB cycle: Calvin-Benson-Bassham cycle;&nbsp;</p>
<p style="text-align: justify;">CscB: sucrose permease;&nbsp;</p>
<p style="text-align: justify;">F6P: fructose 6-phosphate;&nbsp;</p>
<p style="text-align: justify;">G1P: glucose 1-phosphate;&nbsp;</p>
<p style="text-align: justify;">G6P: glucose 6-phosphate;&nbsp;</p>
<p style="text-align: justify;">S6P: sucrose 6-phosphate;&nbsp;</p>
<p style="text-align: justify;">UDP-Glu: uridine diphosphate glucose;&nbsp;</p>
<p style="text-align: justify;"><em>glgA</em>: glycogen synthase;&nbsp;</p>
<p style="text-align: justify;"><em>glgC</em>: ADP-glucose pyrophosphorylase;&nbsp;</p>
<p style="text-align: justify;"><em>sps</em>: sucrose-phosphate synthase&nbsp;</p>
<p style="text-align: justify;"><em>spp</em>: sucrose-phosphate phosphatase;</p>

<br>

<p style="text-align: justify;">Once the symporter is present, it acts as a &lsquo;sucrose sink&rsquo;, continually transporting sucrose into the medium along the pH gradient, compelling the cyanobacterial cells to continually produce more sucrose to maintain their tonicity under salt stress.[3] This secreted sucrose would serve as a carbon source for the heterotrophic <em>E. coli</em> present in our co-culture setup.&nbsp;</p>
<p style="text-align: justify;">We planned to culture the cyanobacteria in an alkaline, carbon-free, saline medium, to enable it to fix gaseous carbon dioxide as sucrose, and secrete it out. To maximize the yield of sucrose generated, we wished to express <em>cscB</em> under a range of constitutively expressed, strong promoters - including <em>Pcpc560</em> (native to <em>Synechocystis</em> PCC 6803), J23119 (native to <em>E. coli</em>), <em>PcpcB-m6</em> (a synthetic mutant variant of the native <em>PcpcB</em>) - and native promoters <em>PpsbA2</em> and <em>PpsbA3</em>.[4][5][6]</p>
<p style="text-align: justify;">Since <em>S. elongatus</em> UTEX 2973 is not naturally transformable, it is typically engineered using bacterial conjugation via triparental mating.[2] We opted for a CRISPR/Cas 12a (also known as CRISPR/Cpf1) based mechanism to integrate<em> cscB</em> into the Neutral Site 1 of UTEX 2973&rsquo;s genome. The more commonly used Cas9 has been found to be toxic to cyanobacteria and hence cannot be used to modify it.[7]</p>
<p style="text-align: justify;">Additionally, we also wished to look into the possible overexpression of the rate-limiting enzymes of the sucrose production pathway, sucrose phosphate synthase (SPS) and sucrose phosphate phosphatase (SPP), coded for by <em>sps</em> and <em>spp</em> genes respectively.[8] A similar CRISPR/Cas 12a based approach would allow for the genomic integration of <em>sps</em> and <em>spp</em> into the Neutral Site 3 of UTEX 2973&rsquo;s genome.&nbsp;</p>
<p style="text-align: justify;">However, we were pressed for time due to limited lab access during the ongoing pandemic and were unable to carry out all the experiments we designed. For a proof of concept, we used the pre-engineered strains of <em>S. elongatus</em> UTEX 2973 kindly given to us by Prof. Pakrasi, at the Washington University at St. Louis.[7]</p>
<p style="text-align: justify;">We received the following strains from them:&nbsp;</p>
<p style="text-align: justify;"><em>S. elongatus</em> UTEX 2973 WT</p>
<p style="text-align: justify;"><em>S. elongatus</em> UTEX 2973-cscB</p>
<p style="text-align: justify;"><em>S. elongatus</em> UTEX 2973-cscB-sps</p>
<p style="text-align: justify;"><em>S. elongatus</em> UTEX 2973-cscB-sps-spp</p>
<p style="text-align: justify;">Due to limited time and the constraint of just a single incubator, we worked primarily with the first and second strains. The WT was chosen as a negative control for all our experiments.The reasoning behind our choice of sucrose-producing strain was the observation by Lin et al., 2020[7], that the <em>cscB</em> strain had the highest yields of sucrose. We do however plan on working with the other strains in future.</p>
<p style="text-align: justify;">In UTEX 2973-cscB, <em>cscB</em> was integrated into the Neutral Site 3 of the cyanobacterium&rsquo;s genome under a <em>lacUV5</em> promoter, using a suicide vector. In the latter two strains,<em> sps </em>and <em>spp</em> were respectively expressed under a <em>trc10</em> promoter, through a self-replicating RSF1010 based plasmid. They have achieved a maximum sucrose titer of 8.1 g/L over 5 days and a maximum daily productivity rate of 1.9 g/L/day in the 2973-cscB strain.[7]</p>

                    <h1 id="2">E. coli - the heterotrophic module </h1>
                    <hr><br><br>
                    <p><em>E. coli</em> is commonly used for industrial chemical biosynthesis. However, most commonly used strains of <em>E. coli</em> cannot naturally consume sucrose.[9] This module focuses on engineering <em>E. coli</em> to take up sucrose produced by <em>S. elongatus</em> in the co-culture and metabolize it to produce industrially important metabolites.&nbsp;</p>
<p style="text-align: justify;">Our initial goal was to produce succinic acid as a final product through<em> E. coli</em> under aerobic conditions, by engineering its metabolic pathways. The aerobic conditions would be facilitated by oxygen evolved by <em>S. elongatus</em>. However, we decided against succinate production as we would not have enough time to engineer <em>E. coli</em>&rsquo;s metabolic pathways from scratch, and also due to the inputs we received about the advantages of microaerobic conditions over aerobic and anaerobic conditions for metabolite production, from <a href="Human_Practices.html#yazd">Dr. Yazdani</a> and <a href="Human_Practices.html#amrit">Dr. Yazdani</a></p>
<br>
<p>As a proof of concept, we chose instead to work with <em>E. coli</em> KJK01, provided to us by Dr. Yazdani. This is a modified MG1655-based strain that can metabolize glucose and xylose to produce butanol.</p>
<p><em>E. coli</em> KJK01 produces optimal butanol yields under microaerobic conditions, and also produces equal quantities of ethanol and some amount of pyruvate. This strain would have to be modified to consume sucrose so that it may be cultivated alongside <em>S. elongatus</em> in our co-culture design.</p>
					


<h1 id="3">Sucrose Transport and Metabolism:</h1>
<hr><br><br>

<p><em>E. coli </em>W is one of few non-pathogenic wild-type strains that can naturally consume sucrose. It does so with the help of three crucial genes:11</p>
<ol>
<li><em>cscB</em>: codes for sucrose permease, a symporter that transports sucrose and protons across the bacterial cell membrane.</li>
<li><em>cscA</em>: codes for sucrose-6-phosphate hydrolase (invertase) that hydrolyses sucrose into glucose and fructose.</li>
<li><em>cscK</em>: codes for fructokinase that catalyzes the conversion of fructose to fructose-6-phosphate.</li>
</ol>
<p style="text-align: justify;">that the WT strain, lacking <em>cscB</em> can act as a negative control for all our experiments</p>
<br>
<div style="display: flex; flex-direction: row; justify-content: center;"><img style="max-width: 600px;" src="assets/Design/Sucrose utilization by Ecoli_white.png"></div>
<br>
<p id="desc">Sucrose consumption pathway in E. coli W. Figure adapted from Mohamed et al., (2019)[11]</p><br>
<p>Claudia Vickers <em>et al. </em>(2012) created pCSCX, a plasmid containing these three genes under a native bidirectional promoter.12 &nbsp;When transformed into KJK01, the plasmid would enable the bacterium to subsist on sucrose as its only available source of carbon. The transformed strain, pCSCX-KJK01 will be ready for a co-culture with <em>S. elongatus, </em>consuming the sucrose generated by it to produce butanol.&nbsp;</p>
<br>
<div style="display: flex; flex-direction: row; justify-content: center;"><img style="max-width: 600px;" src="assets/Design/pcscmap.jpg"></div>
<br>
<p id="desc">Map of the pCSCX plasmid adapted from Bruschi et al. (2012)[12]
</p>
<h1 id="4">Butanol production</h1>
<hr><br><br>

<p>The KJK01 strain uses the acetoacetyl-CoA reduction pathway to produce one molecule of butanol from two molecules of acetyl-CoA.10 The strain carries:</p>
<ul>
<li>From <em>Clostridium acetobutylicum:&nbsp;</em></li>
<ul>
<li><a href="https://metacyc.org/gene?orgid=META&amp;id=MONOMER-16803" target="_blank" rel="noopener">hbd</a> - codes for beta-hydroxybutyryl-CoA dehydrogenase which converts acetoacetyl-coA to (<em>S</em>)-3-hydroxybutanoyl-CoA.</li>
<li><a href="https://metacyc.org/gene?orgid=META&amp;id=CRTCLOS-MONOMER" target="_blank" rel="noopener">crt</a> - codes for 3-hydroxy-butyryl-CoA dehydratase that converts (<em>S</em>)-3-hydroxybutanoyl-CoA to <span style="color: #333333;">crotonyl-CoA.</span></li>
<li><a href="https://metacyc.org/gene?orgid=META&amp;id=G-12174" target="_blank" rel="noopener">adhE2</a> - codes for butyraldehyde/butanol dehydrogenase, which is a multifunctional enzyme with both alcohol dehydrogenase<span style="color: #333333;"> and acetaldehyde dehydrogenase activities.</span></li>
</ul>
</ul>
<ul>
<li>From <em>Treponema denticola</em></li>
<ul>
<li><a href="https://www.genome.jp/dbget-bin/www_bget?ec:1.3.1.38" target="_blank" rel="noopener">ter</a> - codes for trans-enoyl-coenzyme A (CoA) reductase that converts acyl-CoA to trans-2,3-dehydroacyl-CoA</li>
</ul>
<li>From <em>E.coli</em></li>
<ul>
<li><a href="https://www.ecocyc.org/gene?orgid=ECOLI&amp;id=EG11672" target="_blank" rel="noopener">atoB</a> - coding for acetyl-CoA acetyltransferase that converts acetyl-coA to acetoacetyl-coA.</li>
</ul>
</ul>
<p>These genes were integrated using CRISPR/Cas9 into the genome in a markerless fashion.[10]&nbsp;</p>
<p>Additionally, it has the following knockouts:&nbsp;</p>
<ul>
<li><a href="https://www.ecocyc.org/gene?orgid=ECOLI&amp;id=EG20173" target="_blank" rel="noopener">pta </a>- codes for an enzyme that catalyzes the conversion of acetyl-CoA to acetate.</li>
<li><a href="https://www.ecocyc.org/gene?orgid=ECOLI&amp;id=EG10031" target="_blank" rel="noopener">adhE</a> - codes for an enzyme that catalyzes the conversion of acetyl-CoA to ethanol.</li>
<li><a href="https://www.ecocyc.org/gene?orgid=ECOLI&amp;id=EG10330" target="_blank" rel="noopener">frdA</a> - codes for an enzyme that catalyzes the conversion of fumarate to succinate.</li>
<li><a href="https://www.ecocyc.org/gene?orgid=ECOLI&amp;id=G592" target="_blank" rel="noopener">ldhA</a> - codes for an enzyme that catalyzes the conversion of pyruvate to lactate.</li>
</ul>
<br>
<div style="display: flex; flex-direction: row; justify-content: center;"><img style="max-width: 550px;" src="assets/Design/butanol pathway.png"></div>
<br>
<p id="desc">Butanol production pathway in E. coli KJK01 with associated knockouts (represented by an X).Figure adapted from Abdelaal et al., 2019[10]</p>
<p>The above pathway is a Clostridial-based fermentative pathway for n-butanol production. Three of the genes are from <em>Clostridium</em>, with the exception of the first enzyme THL from <em>E. coli</em> itself and the intermediate enzyme TER from the <em>T. denticola</em>. These alternative enzymes from non-Clostridial sources were shown to be more efficient in n-butanol production via fermentation.10</p>
<p>In order to make this strain capable of consuming sucrose from the medium, we will express <em>cscB</em> gene, coding for the sucrose transporter, in it. This sucrose then will be converted to glucose and fructose via <em>cscA </em>and <em>cscK</em>, of which the former enters the butanol synthesis pathway in the engineered <em>E. coli</em> strain.</p>
<p><em>E. coli</em> KJK01 strain generated a yield of 5.4g/L butanol over 27 hours from glucose and we hope to get a similar yield with sucrose.[10] &nbsp;After sufficient assays confirming that pCSCX-KJK01 is capable of surviving with sucrose as its sole carbon source, it will be co-cultured with engineered S. elongatus to validate the co-culture.</p>
<p>You can read more about the constructs we made and the experiments we performed with these strains here and here.</p>
<br>
<h1 id="5">Succinate production</h1>
<hr><br><br>
<p style="text-align: justify;">Our original plan was to modify an <em>E. coli </em>straincapable of sucrose metabolism to produce succinate via the reductive branch of the TCA cycle, activated during anaerobic respiration. One of the main limiting factors of this process is the availability of NADH, with one mole of succinate requiring two moles of NADH.13 &nbsp;Thus four knockouts are made to get rid of pathways competing for NADH. An additional knockout is made to activate the glyoxylate pathway, so that succinate is now produced through two routes (see <em>Fig 4</em>). A non-native pyruvate carboxylase coding gene <em>pyc</em> is expressed to reduce pyruvate accumulation in the cell.[13]</p>
<p style="text-align: justify;">These modifications have previously been shown to produce a yield of 1.59 moles of succinate per mole glucose, 13 &nbsp;and 3.33 moles of succinate per mole sucrose.[14]</p>
<p style="text-align: justify;">The list of modifications is as follows:</p>
<ul>
<li>Knockout <em>ldhA</em></li>
<li>Knockout <em>adhE</em></li>
<li>Knockout <em>ack-pta</em></li>
<li>Knockout <em>iclR</em></li>
<li>Overexpress non-native <em>pyc</em></li>
</ul>
<p style="text-align: justify;">The final pathway after these modifications is as follows:</p>

<br>
<div style="display: flex; flex-direction: row; justify-content: center;"><img style="max-width: 600px; background-color: white;" src="assets/Design/image4.png"></div>
<p id="desc">Modified metabolic pathway in E. coli which produces succinate as its final product. X on a gene name represents a knockout of that gene.</p>
<br>

<h1 id="6">Modelling</h1>
                    <hr><br><br>
		    <p>Our modeling objective was primarily to optimize butanol production. This involved predicting individual growth rates and the production yields of our organisms, identifying key gene modification targets, and analyzing crucial co-culture parameters.&nbsp;</p>
<p>To begin modeling, we needed to make adjustments to the available models of <em>E. coli</em> and <em>S. elongatus </em>2973 - iML1515 and iSyu683 respectively- , to match the strains that we would be using in our co-culture. This involved adding reactions and knockouts that weren&rsquo;t already present in the base models. For <em>S. elongatus</em>, we additionally had to model salt stress, the condition responsible for sucrose production in the first place.&nbsp;</p>
<p>To simulate the growth rates, sucrose, and butanol production in monocultures, we planned to use two constraint-based analysis techniques - Flux Balance Analysis (FBA) and Minimum of Metabolic Adjustments (MoMA). Knowing how the individual strains react to various environmental conditions would also serve to validate the results of our co-culture modeling.</p>
<p>We then planned to analyze the co-culture to identify optimal parameter values for maximum butanol production. This would be done using two techniques - SteadyCom and dynamic modeling, which we performed in collaboration with Team Toulouse_INSA_UPS.&nbsp;</p>
<p>SteadyCom is used to analyze the steady-state behavior of co-cultures, microbiomes, and host-microbe interactions15. It provides results such as the biomass ratio of the constituent organisms at steady-state growth, and the fluxes of various reactions in the multi-species network. It would allow us to identify certain key metabolites that play an important role in the interaction between the organisms.&nbsp;</p>
<p>Dynamic modeling shows the time evolution of concentrations of metabolites and abundances of the organisms in the co-culture in the bioreactor setup.&nbsp;</p>
<p>We then identify key gene modifications in the individual organisms <em>E. coli</em> and <em>S. elongatus</em> that would allow us to achieve these optimal values. This would be done using OptKnock and Flux Scanning based on Enforced Objective Flux (FSEOF).</p>
<p>FSEOF is used to identify metabolic reactions involved in sucrose and butanol production and correspondingly find the genes that regulate them. These genes can be overexpressed to increase sucrose production in <em>S. elongatus</em> and butanol production in <em>E. coli.</em></p>
<p>OptKnock on the other hand is an algorithm that finds and suggests which set of reactions to remove from a metabolic network to obtain a mutant that will produce a particular target of interest at a higher rate than the wild-type strain.</p>
<p>The final results obtained from all the analyses would provide us with crucial bioreactor parameters and gene modifications in the individual organisms <em>E. coli</em> and <em>S. elongatus</em> that would allow us to achieve optimal butanol production, and inform us of productive avenues to explore in the lab.</p>
<br>
<h3 id="7">References</h3>
<p>
	<ol>
		<li>Klähn, S., & Hagemann, M. (2011). Compatible solute biosynthesis in cyanobacteria. Environmental microbiology, 13(3), 551-562.</li>
		<li>Yu, J., Liberton, M., Cliften, P. et al. Synechococcus elongatus UTEX 2973, a fast growing cyanobacterial chassis for biosynthesis using light and CO2. Sci Rep 5, 8132 (2015). <a href="https://doi.org/10.1038/srep08132" target="_blank" rel="noopener">https://doi.org/10.1038/srep08132</a></li>
		<li>Ducat, D. C., Avelar-Rivas, J. A., Way, J. C., &amp; Silver, P. A. (2012). Rerouting carbon flux to enhance photosynthetic productivity. Applied and environmental microbiology, 78(8), 2660&ndash;2668. <a href="https://doi.org/10.1128/AEM.07901-11" target="_blank" rel="noopener">https://doi.org/10.1128/AEM.07901-11</a></li>
		<li>Vasudevan R, Gale GAR, Schiavon AA, et al. CyanoGate: A Modular Cloning Suite for Engineering Cyanobacteria Based on the Plant MoClo Syntax. Plant Physiol. 2019;180(1):39-55. <a href="https://doi.org/10.1104/pp.18.01401" target="_blank" rel="noopener">https://doi.org/10.1104/pp.18.01401</a></li>
		<li>Sengupta, Annesha, Swati Madhu, and Pramod P. Wangikar. "A Library of tunable, portable, and inducer-free promoters derived from cyanobacteria." ACS Synthetic Biology 9.7 (2020): 1790-1801. <a href="https://doi.org/10.1021/acssynbio.0c00152" target="_blank" rel="noopener">https://doi.org/10.1021/acssynbio.0c00152</a></li>
		<li>Li S, Sun T, Xu C, Chen L, Zhang W. Development and optimization of genetic toolboxes for a fast-growing cyanobacterium Synechococcus elongatus UTEX 2973. Metab Eng. 2018 Jul;48:163-174. <a href="https://doi.org/10.1016/j.ymben.2018.06.002" target="_blank" rel="noopener">https://doi.org/10.1016/j.ymben.2018.06.002</a></li>
		<li>Lin, PC., Zhang, F. &amp; Pakrasi, H.B. Enhanced production of sucrose in the fast-growing cyanobacterium Synechococcus elongatus UTEX 2973. Sci Rep 10, 390 (2020). <a href="https://doi.org/10.1038/s41598-019-57319-5" target="_blank" rel="noopener">https://doi.org/10.1038/s41598-019-57319-5</a></li>
		<li>Ungerer, J., Pakrasi, H. Cpf1 Is A Versatile Tool for CRISPR Genome Editing Across Diverse Species of Cyanobacteria. Sci Rep 6, 39681 (2016). <a href="https://doi.org/10.1038/srep39681" target="_blank" rel="noopener">https://doi.org/10.1038/srep39681</a></li>
		<li>Sabri, S., Nielsen, L. K., &amp; Vickers, C. E. (2013). Molecular control of sucrose utilization in Escherichia coli W, an efficient sucrose-utilizing strain. Applied and environmental microbiology, 79(2), 478&ndash;487. <a href="https://doi.org/10.1128/AEM.02544-12" target="_blank" rel="noopener">https://doi.org/10.1128/AEM.02544-12</a></li>
		<li>Abdelaal, A. S., Jawed, K., &amp; Yazdani, S. S. (2019). CRISPR/Cas9-mediated engineering of Escherichia coli for n-butanol production from xylose in defined medium. Journal of Industrial Microbiology and Biotechnology, 46(7), 965-975. <a href="https://doi.org/10.1007/s10295-019-02180-8" target="_blank" rel="noopener">https://doi.org/10.1007/s10295-019-02180-8</a></li>
		<li>Mohamed, E.T., Mundhada, H., Landberg, J. et al. Generation of an E. coli platform strain for improved sucrose utilization using adaptive laboratory evolution. Microb Cell Fact 18, 116 (2019). <a href="https://doi.org/10.1186/s12934-019-1165-2" target="_blank" rel="noopener">https://doi.org/10.1186/s12934-019-1165-2</a></li>
		<li>Bruschi, M., Boyes, S. J., Sugiarto, H., Nielsen, L. K., &amp; Vickers, C. E. (2012). A transferable sucrose utilization approach for non-sucrose-utilizing Escherichia coli strains. Biotechnology advances, 30(5), 1001&ndash;1010. <a href="https://doi.org/10.1016/j.biotechadv.2011.08.019" target="_blank" rel="noopener">https://doi.org/10.1016/j.biotechadv.2011.08.019</a></li>
		<li>Ailen M. S&aacute;nchez, George N. Bennett, Ka-Yiu San, Novel pathway engineering design of the anaerobic central metabolic pathway in Escherichia coli to increase succinate yield and productivity, Metabolic Engineering, Volume 7, Issue 3, 2005, Pages 229-239, ISSN 1096-7176, <a href="https://doi.org/10.1016/j.ymben.2005.03.001" target="_blank" rel="noopener">https://doi.org/10.1016/j.ymben.2005.03.001</a></li>
		<li>Wang J, Zhu J, Bennett GN, San KY. Succinate production from different carbon sources under anaerobic conditions by metabolic engineered Escherichia coli strains. Metab Eng. 2011 May;13(3):328-35. Epub 2011 Mar 30. PMID: 21440082. <a href="https://doi.org/10.1016/j.ymben.2011.03.004" target="_blank" rel="noopener">https://doi.org/10.1016/j.ymben.2011.03.004</a></li>
		<li>Chan, S. H. J., Simons, M. N., & Maranas, C. D. (2017). SteadyCom: predicting microbial abundances while ensuring community stability. PLoS computational biology, 13(5), e1005539.</li>
	</ol>
</p>
				</div>
			</div>
        </div>
    </div>

<!-- SPONSORS BAR -->
	<div class="footer">
		<div class="footer-wrapper">
			<!-- <div id="not-sponsors" class="footer-wrapper">	 -->
				<div id="Team", class="footer-wrapper">
					<img class="teamlogo" src="assets/Logos/IISER-Pune-iGEM_logo.png" alt="iGEM IISER Pune Logo" width="170">
				</div>
				<div id="socials" class="vertical-stack">
					<div><h3>Team IISER Pune India</h3></div>
					<div id="logos" class ="footer-wrapper ">
						<a href="https://www.instagram.com/igem_iiserp/"><img class="footer-links-socials" src="assets/Logos/Instagram_logo.jpg" alt="Instagram" width="20"></a>
						<a href="https://www.linkedin.com/in/igem-iiser-pune-2021/"><img class="footer-links-socials" src="assets/Logos/linkedin_logo.png" alt="Linkedin" width="30"></a>
						<a href="https://twitter.com/igem_iiserpune"><img class="footer-links-socials" src="assets/Logos/Twitter_logo.png" alt="Twitter" width="35"></a>
						<a href="https://www.facebook.com/igemiiserpune"><img class="footer-links-socials" src="assets/Logos/Facebook_logo.png" alt="Facebook" width="25"></a>
						<a href="https://www.youtube.com/channel/UCyo6NMLvM58sSh_tnxht36w"><img class="footer-links-socials" src="assets/Logos/Youtube_logo.png" alt="Youtube" width="35"></a>
					</div>
				</div>
			<!-- </div> -->
			<div id="sponsors" class="footer-wrapper">
				<div id="main-sponsor", class="">
					<a href="https://www.sailife.com/"><img class="footer-links" src="assets/Logos/Sai_logo.jpg" alt="Sai Life Sciences" width="280"></a>
				</div>
				<div class="vertical-stack">
					<a href="https://sg.idtdna.com/pages"><img class="footer-links" src="assets/Logos/IDT_logo.png" alt="IDT" width="220"></a>
					<a href="https://www.geneious.com/"><img class="footer-links" src="assets/Logos/Geneious_logo.png" alt="Geneious" width="220"></a>
				</div>
				<div class="vertical-stack">
					<div id="horizontal stack">
						<a href="https://www.notion.so/"><img class="footer-links" src="assets/Logos/Notion_logo.png" alt="Notion" width="70"></a>
						<img src="assets/Logos/IISER Pune Logo.png" alt="IISER Pune" width="80" style="padding-left: 1em;">
					</div>
					<a href="https://www.zoho.com/"><img class="footer-links" src="assets/Logos/Zoho_logo.png" alt="Zoho" width="150"></a>
				</div>
			</div>	
		</div>
	</div>
	<!-- END OF SPONSOR BAR--> 

<!-- ACTUAL SCROLLING BAR-->
	<div class="header"></div>
    <div class="line" id="scrollIndicator"></div>


    <!-- back to top button -->
    <a href="#top"><img src="assets/svgs/arrow-up-square-fill.svg" id="myBtn" title="Go to top" width="30" height="30"></a>

	
	<!-- LINKING MAIN JS FILES-->
    <script src="js/bootstrap-bundle.js"></script>
	<script src="js/main.js"></script>
	<script src="js/jquery.min.js"></script>


	<!-- Gets the screenheight of each page for Scrollbar? -->
	<script>
		var screenheight = window.innerHeight,
		    screenwidth = window.innerWidth;

	</script>


	<!-- Scrollbar stuff again, only @Namasi22 knows :) -->
	<script>
		var body = document.body,
    		html = document.documentElement;

		var pageheight = Math.max( body.scrollHeight, body.offsetHeight, 
                       		   html.clientHeight, html.scrollHeight, html.offsetHeight );
	</script>


	<!-- To make the button appear -->
	<script>
		var backtotopheight=pageheight/4.65
	</script>


	<!-- SCRIPT FOR SCROLL AWARE SIDEBAR HIGHLIGHT -->
	<!-- ------------------------------------------------------------------------------------------------------------- -->
	<!-- MAKE SURE YOU CHANGE THE NUMBER OF DIVISIONS -->
	<script>
		
		$(document).ready(function() {
			$(window).scroll(function() {
				var scroll_top = $(window).scrollTop();
				var div_one_top = $("#1").offset().top;
				var div_two_top = $("#2").offset().top;
				var div_three_top = $("#3").offset().top;
				var div_four_top = $("#4").offset().top;
				var div_five_top = $("#5").offset().top;
				var div_six_top = $("#6").offset().top;
				var div_seven_top = $("#7").offset().top;
	
				if (scroll_top > div_one_top && scroll_top < div_two_top) {
					//Division 1
					if (!$('a[href="#1"]').hasClass('sidenav-active')) {
						$('a[href="#1"]').addClass('sidenav-active');
					}
	
					if ($('a[href="#2"]').hasClass('sidenav-active')) {
						$('a[href="#2"]').removeClass('sidenav-active');
					}
	
				} else if (scroll_top > div_two_top && scroll_top < div_three_top) {
					// Division 2
					if ($('a[href="#1"]').hasClass('sidenav-active')) {
						$('a[href="#1"]').removeClass('sidenav-active');
					}
	
					if ($('a[href="#3"]').hasClass('sidenav-active')) {
						$('a[href="#3"]').removeClass('sidenav-active');
					}
	
					if (!$('a[href="#2"]').hasClass('sidenav-active')) {
						$('a[href="#2"]').addClass('sidenav-active');
					}
				} else if (scroll_top > div_three_top && scroll_top < div_four_top) {
					//Division 3
					if ($('a[href="#2"]').hasClass('sidenav-active')) {
						$('a[href="#2"]').removeClass('sidenav-active');
					}
	
					if ($('a[href="#4"]').hasClass('sidenav-active')) {
						$('a[href="#4"]').removeClass('sidenav-active');
					} 
	
					if (!$('a[href="#3"]').hasClass('sidenav-active')) {
						$('a[href="#3"]').addClass('sidenav-active');
					}
				} else if (scroll_top > div_four_top && scroll_top < div_five_top) {
					// You are in div 4
					if ($('a[href="#3"]').hasClass('sidenav-active')) {
						$('a[href="#3"]').removeClass('sidenav-active');
					}
	
					if ($('a[href="#5"]').hasClass('sidenav-active')) {
						$('a[href="#5"]').removeClass('sidenav-active');
					} 
	
					if (!$('a[href="#4"]').hasClass('sidenav-active')) {
						$('a[href="#4"]').addClass('sidenav-active');
					}
				} else if (scroll_top > div_five_top && scroll_top < div_six_top) {
					// You are in div 5
					if ($('a[href="#4"]').hasClass('sidenav-active')) {
						$('a[href="#4"]').removeClass('sidenav-active');
					}
	
					if ($('a[href="#6"]').hasClass('sidenav-active')) {
						$('a[href="#6"]').removeClass('sidenav-active');
					} 
	
					if (!$('a[href="#5"]').hasClass('sidenav-active')) {
						$('a[href="#5"]').addClass('sidenav-active');
					}
				} else if (scroll_top > div_six_top && scroll_top < div_seven_top) {
					// You are in div 6
					if ($('a[href="#5"]').hasClass('sidenav-active')) {
						$('a[href="#5"]').removeClass('sidenav-active');
					}
	
					if ($('a[href="#7"]').hasClass('sidenav-active')) {
						$('a[href="#7"]').removeClass('sidenav-active');
					} 
	
					if (!$('a[href="#6"]').hasClass('sidenav-active')) {
						$('a[href="#6"]').addClass('sidenav-active');
					}
				} else if (scroll_top > div_seven_top) {
					// You are in div 7
					if ($('a[href="#6"]').hasClass('sidenav-active')) {
						$('a[href="#6"]').removeClass('sidenav-active');
					}
	
					if (!$('a[href="#7"]').hasClass('sidenav-active')) {
						$('a[href="#7"]').addClass('sidenav-active');
					}
				}
			});
		});
		
	</script>	

	
	<!-- Script for the slideshow, wherever needed -->
	<script>
		var total = $('.item').length;
		var actualtotal = 80;
		var currentIndex = $('div.active').index() + 1;
		$('#slidetext').html(currentIndex + '/'  + actualtotal);

		// This triggers after each slide change
		$('.carousel').on('slid.bs.carousel', function () {
  			currentIndex = $('div.active').index() + 1;
 
  		// Now display this wherever you want
  		var text = currentIndex + '/' + actualtotal;
  			$('#slidetext').html(text);
		});

	</script>
</body>
