<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Title</title>
	<link rel="stylesheet" href="css/igem-reset.css">
	<link rel="stylesheet" href="css/SBcss/style.css">
   	<link rel="stylesheet" href="css/styleSB.css">
    <link rel="stylesheet" href="css/DryLabPage.css">
	<style>/* Reset CSS */

		#sideMenu, #top_title, .patrollink, #firstHeading, #home_logo, #sideMenu { display:none; }
		#content { padding:0px; width:100%; margin-top:-7px; margin-left:0px; border:none;}
		body, html {background-color:#212529; width: 100%; height: 100%;}
		#bodyContent h1, #bodyContent h2, #bodyContent h3, #bodyContent h4, #bodyContent h5 { margin-bottom: 0px; }
		#bodyContent a[href ^="https://"], .link-https { padding-right:0px;}

    </style>
</head>

<!-- Superior styling KEKW-->
<style>
	.sidenav {
		height: auto;
		width: auto;
		padding-top: 150px;
		padding-left: 5px;
		padding-right: 10px;
	
		margin-top: 25%;
		margin-bottom: 20px;
		position: -webkit-sticky;
		position: sticky;
		top: 7%!important;
	}
</style>

<body>
    
	<!-- NAVBAR SECTION -->

	<nav class="navbar fixed-top navbar-expand-lg navbar-light menu">
		<div class="container-fluid nav-container">
		  <a class="navbar-brand" href="#">
			<img src="assets/svgs/synbactory2.svg">
		  </a>
		  <div class="center-expand">
		</div>
		  <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		  <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
		    <ul class="navbar-nav">
		      <li class="nav-item">
			<a class="nav-link active" aria-current="page" href="#">Home</a>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Team
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Team.html">Members</a></li>
			  <li><a class="dropdown-item" href="Attributions.html">Attributions</a></li>
			  <li><a class="dropdown-item" href="Collaborations.html">Collaborations</a></li>
			  <li><a class="dropdown-item" href="Partnership.html">Partnership</a></li>
			  <li><a class="dropdown-item" href="Sponsors.html">Sponsors</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Project
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Description.html">Description</a></li>
			  <li><a class="dropdown-item" href="Design.html">Design</a></li>
			  <li><a class="dropdown-item" href="Notebook.html">Notebook</a></li>
			  <li><a class="dropdown-item" href="Contribution.html">Contributions</a></li>
			  <li><a class="dropdown-item" href="Engineering.html">Engineering Success</a></li>
			  <li><a class="dropdown-item" href="Implementation.html">Future Implementation</a></li>
			  <li><a class="dropdown-item" href="Safety.html">Safety</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Wet Lab
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Experiments.html">Experiments</a></li>
			  <li><a class="dropdown-item" href="Constructs.html">Constructs</a></li>
			  <li><a class="dropdown-item" href="Measurement.html">Measurement</a></li>
			  <li><a class="dropdown-item" href="LabNotebook.html">Lab Notebook</a></li>
			  <li><a class="dropdown-item" href="Results.html">Results</a></li>
			  <li><a class="dropdown-item" href="FutureWork.html">Future Work</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Parts
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <!-- <li><a class="dropdown-item" href="#">Overview</a></li> -->
			  <li><a class="dropdown-item" href="BasicParts.html">Basic</a></li>
			  <li><a class="dropdown-item" href="CompositeParts.html">Composite</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Dry Lab
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Model.html">Overview</a></li>
			  <li><a class="dropdown-item" href="EColi.html">E Coli</a></li>
			  <li><a class="dropdown-item" href="Cyanobacteria.html">Cyanobacteria</a></li>
			  <li><a class="dropdown-item" href="Coculture.html">Coculture</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Human Practices
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Human_Practices.html">iHP</a></li>
			  <li><a class="dropdown-item" href="Inclusivity.html">Inclusivity</a></li>
			  <li><a class="dropdown-item" href="Communication.html">Education</a></li>
			</ul>
		      </li>
		      <li class="nav-item">
			<a class="nav-link" href="https://igem.org/2021_Judging_Form?id=3971">Judging Form</a>
		      </li>
		      <!-- <button type="button" class="rounded-pill btn-rounded">Settings</button> -->
		    </ul>
		  </div>
		</div>
	</nav>




    <!-- TITLE SECTION -->
    
    <div class="section-1">
        <h1><br><br><br><br>Overview</h1>
    </div>
        


    <!-- WAVES SPACER -->
	<a id="top"></a>
    <div class="spacer layer1"></div>




    <!-- CONTENT + SIDENAV SECTION -->

    <div class="second-div container-fluid">
        <div class="row">


            <!-- SIDENAV -->
            <div class="col-lg-3 col-md-3">

				<!-- Overview sidebar-->
                <div class="sidenav" id="overviewside">
                    <nav>
                        <ul id="sidenav-id">
                            <li class="nav-item"><a class="nav-link" href="#o1">Introduction</a></li>
                            <li class="nav-item"><a class="nav-link" href="#o2">Flux Balance Analysis</a></li>
                            <li class="nav-item"><a class="nav-link" href="#o3">Minimisation of Metabolic Adjustment</a></li>
                            <li class="nav-item"><a class="nav-link" href="#o4">Flux Scanning based on Enforced Objective Flux</a></li>
                            <li class="nav-item"><a class="nav-link" href="#o5"> OptKnock</a></li>
                            <li class="nav-item"><a class="nav-link" href="#o6">SteadyCom</a></li>
							<li class="nav-item"><a class="nav-link" href="#o7">Dynamic Modelling</a></li>
			    			<li class="nav-item"><a class="nav-link" href="#o8">References</a></li>
                        </ul>
                    </nav>
                </div>

				
				<!-- Cyano Sidebar-->
				<div class="sidenav" style="visibility: hidden;" id="cyanoside">
                    <nav>
                        <ul id="sidenav-id">
                            <li class="nav-item"><a class="nav-link" href="#c1">Introduction</a></li>
                            <li class="nav-item"><a class="nav-link" href="#c2">Modelling Cyanobacteria for Sucrose Production</a></li>
                            <li class="nav-item"><a class="nav-link" href="#c3">Biomass vs CO2</a></li>
                            <li class="nav-item"><a class="nav-link" href="#c4">Biomass vs Sucrose</a></li>
                            <li class="nav-item"><a class="nav-link" href="#c5">Sucrose Production</a></li>
                            <li class="nav-item"><a class="nav-link" href="#c6">Identifying Over Expressions and Knockouts</a></li>
			    			<li class="nav-item"><a class="nav-link" href="#c7">References</a></li>
                        </ul>
                    </nav>
                </div>

				<!-- E coli Sidebar-->
				<div class="sidenav" style="visibility: hidden;" id="ecoliside">
                    <nav>
                        <ul id="sidenav-id">
                            <li class="nav-item"><a class="nav-link" href="#e1">Introduction</a></li>
                            <li class="nav-item"><a class="nav-link" href="#e2">Model Construction</a></li>
                            <li class="nav-item"><a class="nav-link" href="#e3">Modelling E coli for optimum Butanol production</a></li>
							<li class="nav-item"><a class="nav-link" href="#e4">Results</a></li>
							<li class="nav-item"><a class="nav-link" href="#e5">OptKnock</a></li>
							<li class="nav-item"><a class="nav-link" href="#e6">FSEOF</a></li>
							<li class="nav-item"><a class="nav-link" href="#e7">References</a></li>
                        </ul>
                    </nav>
                </div>


				<!-- Co Culture Sidebar-->
				<div class="sidenav" style="visibility: hidden;"id="cocultureside">
                    <nav>
                        <ul id="sidenav-id">
                            <li class="nav-item"><a class="nav-link" href="#cc1">Introduction</a></li>
                            <li class="nav-item"><a class="nav-link" href="#cc2">Section 2</a></li>
                            <li class="nav-item"><a class="nav-link" href="#cc3">Section 3</a></li>
                            <li class="nav-item"><a class="nav-link" href="#cc4">Section 4</a></li>
                            <li class="nav-item"><a class="nav-link" href="#cc5">Section 5</a></li>
                            <li class="nav-item"><a class="nav-link" href="#cc6">References</a></li>
                        </ul>
                    </nav>
                </div>

            </div>



			
            <!-- CONTENT -->
            <div class="col-lg-9 col-md-9">
                <div class="content">

					<ul class="nav nav-tabs nav-fill" id="myTab" role="tablist">
						<li class="nav-item" role="presentation">
							<button class="nav-link active" id="overview-tab" data-bs-toggle="tab" data-bs-target="#overview" type="button" role="tab" aria-controls="overview" aria-selected="true">Overview</button>
						</li>
						<li class="nav-item" role="presentation">
							<button class="nav-link" id="cyano-tab" data-bs-toggle="tab" data-bs-target="#cyano" type="button" role="tab" aria-controls="cyano" aria-selected="false">Cyanobacteria</button>
						</li>
						<li class="nav-item" role="presentation">
							<button class="nav-link" id="ecoli-tab" data-bs-toggle="tab" data-bs-target="#ecoli" type="button" role="tab" aria-controls="ecoli" aria-selected="false">E. coli</button>
						</li>
						<li class="nav-item" role="presentation">
							<button class="nav-link" id="coculture-tab" data-bs-toggle="tab" data-bs-target="#Coculture" data-bs-target ="#cocultureside" type="button" role="tab" aria-controls="cyano" aria-selected="false">Co-culture</button>
						</li>
					</ul>

					<div class="tab-content" id="myTabContent">

						<!-------------------------------------------- OVERVIEW TAB ---------------------------------------------------->
						<div class="tab-pane fade show active" id="overview" role="tabpanel" aria-labelledby="overview-tab">
							<a id="o1"></a>

							<!-- INTRODUCTION -->
							<h1>Introduction</h1>
							<p>Mathematical modelling has become an integral part of almost all projects dealing with metabolic engineering. Information starting from the dynamics of our system, to what genetic modifications can further our project can be obtained from the right mathematical models. These models help as starting points for the wet-lab experiments as they give a rough idea of where to target your attention.</p>
							<p>SynBactory being heavily dependent on organism’s growths, behaviours and interactions, along with metabolic networks and its possible manipulations, is a project that can benefit a lot from mathematical models.</p>
							<strong>The main objectives were:</strong>
							<ol class="a">
								<li style="padding-left:1em">Predict growth rates of our individual organisms in pure cultures</li>
								<li style="padding-left:1em">Predict growth rates under different stresses</li>
								<li style="padding-left:1em">Identify the important interactive metabolites in the co culture</li>
								<li style="padding-left:1em">Identify targets to over express or knockout in each organism for improved yields of sucrose and butanol</li>
							</ol>
							<p>The core technique of our modelling attempts is a constraint based modelling technique called Flux Balance Analysis, in short FBA. We have used the COBRA package in MATLAB<a href="#ref1"><sup>1</sup></a> for almost all our models.</p>
							<a id="o2"></a>
							<br>


							<!-- FLUX BALANCE ANALYSIS -->
							
							<h1>Flux Balance Analysis</h1>
							<p>FBA is a steady state analysis algorithm that is widely used in metabolic predictions and optimisations. FBA assumes the organism to be in a metabolically steady state i.e., there is no accumulation of metabolites within the organism.<a href="#ref2"><sup>2</sup></a> To perform FBA we require a metabolic model of the organism. A metabolic model of an organism contains information on all (or most) reactions in an organism, the metabolites involved, the genes present in the organism, the Gene-Reaction-Relation, and a calculated biomass production reaction (growth rate reaction) that gives the growth rate when analysed.</p>
							<strong>The core elements of FBA are<a href="#ref2"><sup>2</sup></a>:</strong>
							<ol class="a">
								<li style="padding-left:1em">S Matrix or the stoichiometric matrix - This is a matrix of the order (number of metabolites) * (number of reactions)</li>
								<li style="padding-left:1em">The objective function or c - This is a column vector that holds the information to attain the aim of the analysis, for example biomass optimisation</li>								
								<li style="padding-left:1em">Lower bounds and upper bounds - This is a set of values that help constrain the solution space</li>
								<li style="padding-left:1em">Solution flux space - This is a set of fluxes that give the same objective function value</li>
							</ol>
							<p>In an organism, it is normal to have the same metabolites participating in multiple reactions, and therefore it is safe to assume the number of reactions will be more than the number of metabolites. This can be compared to a system of linear equations with more equations than variables. Without constraints the solution space is infinite, so we introduce constraints in the form of bounds.</p>
							<br>
							<strong>Equations of FBA and their solutions<a href="#ref2"><sup>2</sup></a></strong>
							<strong>Constraints:</strong>
							<ul>
								<li style="padding-left:1em;">Sv = 0</li>
								<li style="padding-left:1em;">Lower Bounds</li>
								<li style="padding-left:1em;">Upper Bounds</li>
							</ul>
							<br>
							<strong>Objective Function<a href="#ref2"><sup>2</sup></a>:</strong>
							<ul>
								<li style="padding-left:1em;">C(transpose)v</li>
							</ul>
							<br>
							<strong>Objective<a href="#ref2"><sup>2</sup></a>:</strong>
							<ul>
								<li style="padding-left:1em;">Minimise or Maximise Objective function</li>
							</ul>
							<br>
							<strong>Solution<a href="#ref2"><sup>2</sup></a>:</strong>
							<ul>
								<li style="padding-left:1em;">Plausible flux distributions for the above mentioned objective function</li>
							</ul>
							<p>If one has perturbed the metabolic network of the organism, then simple maximisation of growth rate does not give results that agree with experiments. MoMA is a widely used algorithm to identify solution sets that agree most with real life values.</p>
							<a id="o3"></a>
							<br>


							<!-- MoMA -->
							<h1>Minimisation of Metabolic Adjustments</h1>
							<p>MoMA is an extension of Flux Balance Analysis that imposes more constraints to make the model as close to reality as possible, albeit it isn’t the best at doing so. The logic of MoMA is very simple - minimise the euclidean distance (L2 norm) between the wild-type flux distribution and the flux distribution of the modified organism in the flux vector space<a href="#ref3"><sup>3</sup></a>.</p>
							<br>
							<p><strong>The algorithm<a href="#ref3"><sup>3</sup></a>:</strong></p>
							<ol class="a">
								<li style="padding-left:1em;">Find the wild-type/base strain flux distribution using FBA for maximized growth rates</li>
								<li style="padding-left:1em;">Simulate the deletion, over-expression or any stress</li>
								<li style="padding-left:1em;">Impose an additional constraint that minimizes the L2 norm of the solution and the wild type flux; it would look something like this - ||v_wild - v_mod||^2 is minimised.</li>
								<li style="padding-left:1em;">Run FBA multiple times to find such a v_mod.</li>
							</ol>
							<br>
							<p>Fascinating thing is this algorithm quite literally translates into the name of the technique.</p>
							<p>Here we know what the modifications are that we need to make in order to get the strain we need, but what if we need to find further modifications that would help increase the yield of the product, simple FBA or MoMA will not suffice for this purpose. This is where we start using techniques like FSEOF and OptKnock.</p>
							<a id="o4"></a>
							<br>


							<!-- FSEOF -->
							<h1>Flux Scanning based on Enforced Objective Flux (FSEOF)</h1>
							<p>This algorithm was developed to identify gene targets that can be amplified for increased production of selected metabolites or even increased growth rates. We have used this technique to identify the reactions and further genes that we can over-express to increase sucrose production in cyanobacteria and butanol production in E. coli.</p>
							<p>To understand FSEOF imagine the whole organism as a big city, the reactions as roads, and the flux through the reactions as the traffic in each road. Now consider evacuating this city as our objective, then the roads which lead to the exits will start carrying more traffic. If we can widen these roads we can in theory get more people out. If evacuating this city was another way of saying - maximise the production of a particular metabolite, then widen the roads can be considered as amplifying the corresponding reactions.</p>
							<br>
							<p>In the form of an algorithm it would look something like this<a href="#ref4"><sup>4</sup></a>:</p>
							<ol class="a">
								<li style="padding-left:1em;">Find maximum growth under media conditions using FBA </li>
								<li style="padding-left:1em;">Find the maximum production of the metabolite using FBA </li>
								<li style="padding-left:1em;">Constraint the production of the metabolite to some fraction of the maximum metabolite production value </li>
								<li style="padding-left:1em;">Run FBA for maximum growth rate </li>
								<li style="padding-left:1em;">Calculate the product of the metabolite production value for a given constraint and the corresponding growth rate value. This would help rank the target later on </li>
								<li style="padding-left:1em;">Plot how the flux values of each reaction changes as we increase the fractional value of the constraint on the metabolite </li>
							</ol>
							<br>
							<p>Since FBA and all of its extensions work with reactions and not genes we will have to trace the reactions back to its controlling genes and see if those genes are responsible for any other process that could end up killing the organism or hindering its growth if amplified.</p>
							<a id="o5"></a>
							<br>


							<!-- OptKnock -->
							<h1>OptKnock</h1>
							<p>OptKnock is an algorithm that find and suggests which set of reactions to remove from a metabolic network to obtain a mutant that will produce a particular target of interest at a higher rate than the wild-type strain<a href="#ref5"><sup>5</sup></a>. The logic is pretty simple -</p>
							<ol class="a">
								<li style="padding-left:1em;">You find the set of reactions that you want to get knockouts from </li>
								<li style="padding-left:1em;">Change their bounds to 0 (this is how you simulate deletion in models) either one by one or in pairs or triplets </li>
								<li style="padding-left:1em;">Run FBA for maximised growth </li>
								<li style="padding-left:1em;">Once you have a set of reactions which when deleted increases the production of the target of interest you trace them back to their respective genes </li>
								<li style="padding-left:1em;">Once you’ve identified the genes you eliminate all the synthetic lethals from the set. Synthetic lethals are essential genes that was required by the organism to live </li>
							</ol>
							<br>
							<p>Once you’ve eliminated all the synthetic lethal you have a set of reactions and their corresponding genes which can be knocked out for enhanced production of the metabolite of interest.</p>
							<a id="o6"></a>
							<br>


							<!-- STEADYCOM-->
							<h1>SteadyCom</h1>
							<p>SteadyCom is an algorithm used to analyze the steady state behaviour of co-cultures, microbiomes and host-microbe interactions. Similar to FBA, SteadyCom assumes that there is no accumulation of metabolites and only looks at the fluxes of reactions. Apart from this, the crux of SteadyCom is the assumption that - at steady state (if a steady state exists) the growth rates of all the organisms is the same, so that one organism does not dominate and take over the entire population. Based on the constraints in the model, SteadyCom tries to maximize this community growth rate, and provides results including the biomass ratios of the organisms at steady state growth and fluxes of various reactions of the multi-species metabolic network.</p>
							<p><strong>The Logic:</strong></p>
							<ol class="a">
								<li style="padding-left: 1em;">Create a joint model which has all the reactions of every organism’s GSM and a separate community space to allow for the transfer of common extracellular metabolites between the separate extracellular compartments of the individual models</li>
								<li style="padding-left: 1em;">The algorithm predicts steady state fluxes which maximizes the community growth rate, such that all organisms grow at this rate</li>
								<li style="padding-left: 1em;">The fluxes of reactions out of individual organisms are multiplied by their biomass, so that abundances of organisms are taken into account in the interactions</li>
							</ol>
							<a id="o7"></a>
							<br>

							
							<!-- D-FBA-->
							<h1>Dynamic Modelling</h1>
							<br>
							<p><strong>Method</strong></p>
							<p>Dynamic modelling here involves using fluxes obtained from FBA to form and solve ordinary differential equations that predict the concentrations of metabolites and biomasses in the bioreactor over time. Essentially, these fluxes act as the first order time derivatives in the system of differential equations.</p>
							<p>When we optimize a model using FBA, we get the fluxes of all the reactions that allow the model to optimize its objective function. So, given an initial concentration of a metabolite, the FBA flux gives the derivative of the concentration at that point. Using this derivative, we can calculate the concentration of the metabolite at the next time step, and use this as the new initial concentration. This process can be repeated iteratively within the required time frame.</p>
							<p><strong>The algorithm: </strong></p>
							<ol class="a">
								<li style="padding-left: 1em;"> Set the flux restrictions of the cyanobacteria uptake rates, of co2 and light, based on concentration of these present in the bioreactor available for uptake</li>
								<li style="padding-left: 1em;"> Optimize the cyanobacteria model with sucrose production as its objective using FBA. This will naturally make the biomass flux of the cyanobacteria set to its lower bound 0. Biomass flux set to zero, implies that the rate of growth of biomass is zero. Hence, the maximized sucrose flux obtained from this result is the theoretical maximum sucrose yield possible from the model within the given constraints</li>
								<li style="padding-left: 1em;"> It is known that reaching the theoretical maximum yields of sucrose production is not very biologically feasible. So it is assumed that the salt stress provided to force cyanobacteria to produce sucrose, allows it to export sucrose out at some fraction of the maximum sucrose yield calculated above. This is referred to as the percentage productivity of sucrose flux in the cyanobacteria</li>
								<li style="padding-left: 1em;"> This productivity rate, x% of max sucrose flux, is now set as the lower bound for the sucrose export reaction in the cyanobacteria model, and the objective function is changed to biomass flux. An FBA is performed to maximize the biomass flux given this new constraint of sucrose production</li>
								<li style="padding-left: 1em;"> The final results from this FBA are taken as the fluxes of the cyanobacteria model</li>
								<li style="padding-left: 1em;"> The same process is repeated for the E. coli model, where the available sucrose in the bioreactor comes from the cyanobacteria flux calculated above. Butanol production is first optimized to find the maximum flux possible, then as above, some productivity for this reaction is set and biomass is optimized</li>
								<li style="padding-left: 1em;"> All the fluxes from both the models provide the time derivatives of the corresponding metabolites, which are integrated and plotted</li>
							</ol>
							<a id="o8"></a>
							<br><a id="ref1"></a><br><a id="ref2"></a><br><a id="ref3"></a><br>


							<!-- References -->
							<h2 class="h3"> References</h2>
							<ol class="a">
								<li style="padding-left: 1em;" id="ref4"> Heirendt, L., Arreckx, S., Pfau, T. et al. Creation and analysis of biochemical constraint-based models using the COBRA Toolbox v.3.0. Nat Protoc 14, 639–702 (2019). <a href="https://doi.org/10.1038/s41596-018-0098-2">https://doi.org/10.1038/s41596-018-0098-2</a></li> 
								<li style="padding-left: 1em;" id="ref5"> Orth, J. D., Thiele, I., & Palsson, B. Ø. (2010). What is flux balance analysis?. Nature biotechnology, 28(3), 245–248.<a href="https://doi.org/10.1038/nbt.1614">https://doi.org/10.1038/nbt.1614</a></li> 
								<li style="padding-left: 1em;"> 25 Segrè, D., Vitkup, D., & Church, G. M. (2002). Analysis of optimality in natural and perturbed metabolic networks. Proceedings of the National Academy of Sciences of the United States of America, 99(23), 15112–15117. <a href="https://doi.org/10.1073/pnas.232349399">https://doi.org/10.1073/pnas.232349399</a></li> 
								<li style="padding-left: 1em;"> Choi, H. S., Lee, S. Y., Kim, T. Y., & Woo, H. M. (2010). In silico identification of gene amplification targets for improvement of lycopene production. Applied and environmental microbiology, 76(10), 3097–3105. <a href="https://doi.org/10.1128/AEM.00115-10">https://doi.org/10.1128/AEM.00115-10</a></li> 
								<li style="padding-left: 1em;"> Burgard, A. P., Pharkya, P., & Maranas, C. D. (2003). Optknock: a bilevel programming framework for identifying gene knockout strategies for microbial strain optimization. Biotechnology and bioengineering, 84(6), 647–657. <a href="https://doi.org/10.1002/bit.10803">https://doi.org/10.1002/bit.10803</a></li> 
							</ol>
							<br><br>
						</div>
						<!-- END OF CONTENT -->


						<!-- SCROLL AWARE HIGHLIGHT-->
						<script>
				
							$(document).ready(function() {
								$(window).scroll(function() {
									var scroll_top = $(window).scrollTop();
									var div_one_top = $("#o1").offset().top;
									var div_two_top = $("#o2").offset().top;
									var div_three_top = $("#o3").offset().top;
									var div_four_top = $("#o4").offset().top;
									var div_five_top = $("#o5").offset().top;
									var div_six_top = $("#o6").offset().top;
									var div_seven_top = $("#o7").offset().top;
									var div_eight_top =$("#o8").offset().top;
					
									if (scroll_top > div_one_top && scroll_top < div_two_top) {
										//Division 1
										if (!$('a[href="#1"]').hasClass('sidenav-active')) {
											$('a[href="#1"]').addClass('sidenav-active');
										}
						
										if ($('a[href="#2"]').hasClass('sidenav-active')) {
											$('a[href="#2"]').removeClass('sidenav-active');
										}
					
									} else if (scroll_top > div_two_top && scroll_top < div_three_top) {
										// Division 2
										if ($('a[href="#1"]').hasClass('sidenav-active')) {
											$('a[href="#1"]').removeClass('sidenav-active');
										}
					
										if ($('a[href="#3"]').hasClass('sidenav-active')) {
											$('a[href="#3"]').removeClass('sidenav-active');
										}
						
										if (!$('a[href="#2"]').hasClass('sidenav-active')) {
											$('a[href="#2"]').addClass('sidenav-active');
										}
									} else if (scroll_top > div_three_top && scroll_top < div_four_top) {
										//Division 3
										if ($('a[href="#2"]').hasClass('sidenav-active')) {
											$('a[href="#2"]').removeClass('sidenav-active');
										}
					
										if ($('a[href="#4"]').hasClass('sidenav-active')) {
											$('a[href="#4"]').removeClass('sidenav-active');
										} 
					
										if (!$('a[href="#3"]').hasClass('sidenav-active')) {
											$('a[href="#3"]').addClass('sidenav-active');
										}
									} else if (scroll_top > div_four_top && scroll_top < div_five_top) {
									// You are in div 4
										if ($('a[href="#3"]').hasClass('sidenav-active')) {
											$('a[href="#3"]').removeClass('sidenav-active');
										}
						
										if ($('a[href="#5"]').hasClass('sidenav-active')) {
											$('a[href="#5"]').removeClass('sidenav-active');
										} 
					
										if (!$('a[href="#4"]').hasClass('sidenav-active')) {
											$('a[href="#4"]').addClass('sidenav-active');
										}
									} else if (scroll_top > div_five_top && scroll_top < div_six_top) {
										// You are in div 5
										if ($('a[href="#4"]').hasClass('sidenav-active')) {
											$('a[href="#4"]').removeClass('sidenav-active');
										}
						
										if ($('a[href="#6"]').hasClass('sidenav-active')) {
											$('a[href="#6"]').removeClass('sidenav-active');
										} 
						
										if (!$('a[href="#5"]').hasClass('sidenav-active')) {
											$('a[href="#5"]').addClass('sidenav-active');
										}
									} else if (scroll_top > div_six_top && scroll_top < div_seven_top) {
										// You are in div 6
										if ($('a[href="#5"]').hasClass('sidenav-active')) {
											$('a[href="#5"]').removeClass('sidenav-active');
										}
					
										if ($('a[href="#7"]').hasClass('sidenav-active')) {
											$('a[href="#7"]').removeClass('sidenav-active');
										} 
						
										if (!$('a[href="#6"]').hasClass('sidenav-active')) {
											$('a[href="#6"]').addClass('sidenav-active');
										}
									} else if (scroll_top > div_seven_top && scroll_top < div_eight_top) {
										// You are in div 7
										if ($('a[href="#6"]').hasClass('sidenav-active')) {
											$('a[href="#6"]').removeClass('sidenav-active');
										}
						
										if (!$('a[href="#7"]').hasClass('sidenav-active')) {
											$('a[href="#7"]').addClass('sidenav-active');
										}
										if ($('a[href="#8"]').hasClass('sidenav-active')) {
											$('a[href="#8"]').removeClass('sidenav-active');
										}
									} else if (scroll_top > div_eight_top) {
										if ($('a[href="#7"]').hasClass('sidenav-active')) {
											$('a[href="#7"]').removeClass('sidenav-active');
										}
						
										if (!$('a[href="#8"]').hasClass('sidenav-active')) {
											$('a[href="#8"]').addClass('sidenav-active');
										}
									}
								});
							});
						</script>
						<!-- END OF SCROLL AWARE HIGHLIGHT -->

						<!------------------------------------ END OF OVERVIEW TAB ------------------------------------------>





						<!------------------------------------- CYANO TAB ---------------------------------------------------->
						<div class="tab-pane fade show " id="cyano" role="tabpanel" aria-labelledby="cyano-tab">
							<!-- INTRODUCTION -->
							<a id="c1"></a>
							<h1>Introduction</h1>
							<p>Cyanobacteria is the green element, both figuratively and literally, of our project. Modelling its behaviour both in a pure culture and in the co culture matters a lot to the direction we put our attention to. We’ve used FBA and MoMA to predict the growth rates and sucrose production rates of cyanobacteria, both with and without salt stress. Dr.Costas Maranas was generous enough to provide the Genome scale metabolic model for Synechococcus elongatus UTEX 2973 from Mueller et al., 2017<a href="#ref1"><sup>1</sup></a>.</p>
							<br>
							<a id="c2"></a>
							<!-- END OF INTRODUCTION-->
		
		
		
							<!-- MODELLING CYANO FOR SUCROSE PRODUCTION -->
							<h1>Modelling Cyanobacteria for optimum sucrose production</h1>
							<p>The first hurdle we encountered while modelling cyanobacteria was simulating salt stress. Literature says simulating something like salt stress alters a lot of reactions in the organism and not all of them are straight forward or even known, so we had to make a few assumptions to make the model work.</p>
							<h2 class="h3"><strong>Assumptions:</strong></h2>
							<ol class="a">
								<li style="padding-left:1em;">sps and spp genes are a part of the UTEX 2973 Genome even though they weren’t part of the model *insert spp and sps reaction equations* </li>
								<li style="padding-left:1em;">cscB exports are of constant efficiency </li>
								<li style="padding-left:1em;">FBA is a steady-state analysis therefore the growth phase cannot be simulated (check <a href="Coculture.html">co-culture </a>for more information on this) </li>
								<li style="padding-left:1em;">Coupling biomass and sucrose accumulation and restraining cscB export to less than 100% simulates salt stress </li>
							</ol>
							<br>
							<h2 class="h3"><strong>Results: </strong></h2>
							<ul class="b">
								<li style="padding-left:1em;">Upon adding “SPP”, “SPS” and “cscB” and running FBA with an objective to maximise growth rate we obtained a maximum growth rate of <strong>0.3033 h-1</strong> and a sucrose production and transport flux of <strong>0</strong>. This value is very similar to the experimental growth rate of <strong>0.33 ± 0.05 h</strong> Abernathy et al.<a href="#ref2"><sup>2</sup></a> , which implies confidence in the model we’re using. </li>
								<li style="padding-left:1em;">For the following conditions upon performing MoMA for optimised sucrose production we obtained a growth rate of <strong>0.2012 h−1</strong>  and a sucrose output flux of <strong>0.3079 mmol/gDW/h</strong>. </li>
								<ul class="d">
									<li style="padding-left:1em;">Conditions:</li>
										<ul class="c">
											<li style="padding-left:1em;">Salt stress of <strong>150mM</strong><a href="#ref4"><sup>4</sup></a> simulated by coupling sucrose accumulation and biomass production </li>
											<li style="padding-left:1em;">CO2 uptake rate of <strong>-12.2 mmol/gDW/h</strong> <a href="#ref2"><sup>2</sup></a> </li>
											<li style="padding-left:1em;">cscB efficiency of <strong>90%</strong> <a href="#ref3"><sup>3</sup></a> </li>
										</ul>
								</ul>
							</ul>
							<br>
							<strong>Upon performing MoMA for a range of values of CO2 uptakes and cscB efficiencies the following result was obtained:</strong>
							<a id="c3"></a>               
							<br>
							<!-- END OF MODELLING -->
		
							<!-- BIOMASS VS CO2-->
							<h1> Biomass vs CO2</h1>
							<br>
							<div class="graph">
								<img src="assets/Graphs/Cyano Graphs/Basic/Biomass vs CO2.jpg" alt="Biomass vs CO_2 graph for different constant cscB efficiency values and a constant salt stress of 150mM. A linear pattern is observed" width=800>
								<p style="padding-top: 15px;"> Fig 1: Relationship between Growth rate and CO2 uptake rates when run at a salt conc. 150mM under varying cscB efficiencies</p>
							</div>
							<br>
							<p><strong>Observations: </strong></p>
							<ol class="a">
								<li style="padding-left:1em;">Biomass production rates decrease with decreasing CO2 uptake rates for a constant cscB efficiency </li>
								<li style="padding-left:1em;">Biomass increases for a decreasing cscB efficiency at a given CO2 intake value. </li>
								<li style="padding-left:1em;">The linear nature of the graph is due to the fact that FBA is generally a linear model and not a non-linear model. The assumption that cscB has a constant efficiency is also a contributing factor </li>
							</ol>
							<p><strong>Inference: </strong> </p>
							<p style="padding-left: 12%;"> Carbon flux through the biomass reaction decreases as the efficiency of cscB increases. Since the objective function is to maximise sucrose production we can safely say cscB acts as a sucrose sink and therefore increases flux through sucrose production reaction at higher efficiencies.</p>
							<a id="c4"></a>
							<br>
							<!-- END OF BIOMASS VS CO2-->
		
		
		
							<!-- BIOMASS VS SUCROSE-->
							<h1> Biomass vs Sucrose</h1>
							<br>
							<div class="graph">
								<img src="assets/Graphs/Cyano Graphs/Basic/Biomass vs Sucrose.jpg" alt="Biomass vs Sucrose graph for different CO_2 uptake rates. We observe a linear pattern yet again where sucrose production increases with decrease in growth rates" width=800>
								<p style="padding-top: 15px;">Fig 2: Relationship between sucrose production and growth rate under a varying CO2 uptake rates for a salt concentration of 150mM</p>
							</div>
							<p><strong>Observations: </strong></p>
							<ol class="a">
								<li style="padding-left:1em;">Sucrose production rates decrease with increasing biomass production rates </li>
								<li style="padding-left:1em;">Both increase with increase in CO2 uptake </li>
								<li style="padding-left:1em;">The linear nature is again, due to the linear nature of FBA and the fact that both of them share the same carbon source i.e. CO2 </li>
							</ol>
							<p><strong>Inference: </strong></p>
							<p style="padding-left: 12%;">Since the objective function is sucrose production and both biomass and sucrose share the same carbon source one can easily say they’ll be inversely proportional. Therefore, our aim is to direct carbon flux through both the reactions in a fixed ratio such that the organism lives comfortably and produces ample amounts of sucrose. </p>
							<a id="c5"></a>
							<br>
							<!-- END OF BIOMASS VS SUCROSE -->
		
							<!-- SUCROSE PRODUCTION-->
							<h1> Sucrose production</h1>
							<p>This is a graphical representation of all the results and relations we have obtained from our attempts till now. We have eliminated all the metabolic states that do not agree with our MoMA results. The colour bar shows the sucrose production fluxes.</p>
							<p>Observations for this are similar to the previous ones. This just helps us identify the region of conditions that we should be trying to replicate in the lab. Clearly sucrose production is maximum in the region with high cscB efficiency and high CO2 uptake rates.</p>
							<br>
							<div class="graph">
								<img src="assets/Graphs/Cyano Graphs/Basic/3_Variables.jpg" alt="A graph respresting varaition of sucrose production with increase in growth rates, CO_2 Uptake rates and cscB efficieny values" width=800>
								<p style="padding-top: 15px;">Fig 9: A graph showing the production of sucrose for varying CO2 uptake rates and cscB efficiencies <br> Note - the colour bar on the right shows the mmol/gDW/h values of sucrose</p>
							</div>
							<p>As you can see from the graph sucrose production is maximum when there is a higher intake of carbon dioxide (source) and a higher cscB efficiency (sink). Ideally we would be trying to re-create the above mentioned conditions for enhanced production of sucrose whilst maintaining a significant fraction of maximum growth rates, but amplifying something that is already at 90% efficiency (cscB) is hard and to some extend unnecessary. Therefore we went a step ahead to find out other possible ways to increase sucrose production without losing much growth rates.</p>
							<a id="c6"></a>
							<br>
							<!-- END OF MODELLING CYANO FOR SUCROSE PRODUCTION -->
		
		
		
							<!-- OVER EXPRESSIONS AND KNOCKOUTS -->
							<h1>Identifying Over-expressions and Knockouts for enhanced sucrose production</h1>
							<p>We have used OptKnock to identify knockout targets and FSEOF to identify over-expression targets.</p>
							<br><br>
		
		
							<!-- TARGETS FOR OVER EXPRESSION-->
							<h2 class="h2">Targets for Over-expression</h2>
							<p>FSEOF is the go to algorithm to identify targets for amplification. Upon running FSEOF under the following conditions we obtained a few targets for over-expression.</p>
							<br>
		
							<h2 class="h3">Assumptions: </h2>
							<ol class="a">
								<li style="padding-left:1em;">Carbon Dioxide uptake rate is -12.2mmol/gDW/hr (Max value from literature) </li>
								<li style="padding-left:1em;">We have neglected salt stress because we were getting insensible results because of it </li>
							</ol>
							<br>
		
							<h2 class="h3">Sources of inaccuraies: </h2>
							<ol class="a">
								<li style="padding-left:1em;"> The model is not complete </li>
								<li style="padding-left:1em;"> The geens have no names</li>
								<li style="padding-left:1em;"> Some reactions have no gene-protein-relations. Gene-protein-relations (GPRs) are rules that bind reactions in the organism to particular genes or combination of genes. For example, if reaction R1 requires both genes G1 and G2 then in the absence of either one of the genes the reaction seizes to work</li>
							</ol>
							<br>
		
							<h2 class="h3">Results: </h2>
							<ol class="a"> 
								<li style="padding-left: 1em;"> We have obtained a total of 80 reactions that can be overexpressed</li>
								<li style="padding-left: 1em;"> We have obtained a total gene count of 87. This number is a bit misleading because if we were to over express all these 80 reactions we wouldn’t have to over express all the 87 genes because  some of the Gene-Protein-Relations do not require the entire set of genes that govern them</li>
								<li style="padding-left: 1em;"> These 87 genes are responsible for a total of 555 reactions in the organism. Few of them are the same reaction but in different parts of the organism, for example if the cytoplasm and periplasm both contain a reaction X, the reaction X is counted twice</li>
								<li style="padding-left: 1em;"> If we were to count X only once then we have 107 reactions  governed by these 87 genes but this is not of much importance because over expressing any gene will over express all its reactions whether it is in cytoplasm or periplasm</li>
								<li style="padding-left: 1em;"> 25 out of the 80 target reactions have no gene-protein-relations so we cannot test these out in the lab</li>
								<li style="padding-left: 1em;"> Although the sucrose production reaches the theoretical maximum, the biomass flux slowly dies as this happens therefore the higher is not always the better</li>
								<li style="padding-left: 1em;"> Almost all of them follow linear behaviour</li>					
							</ol>
							<br>
						
							<p>Given below are 4 graphs each with 5 reactions plotted against varying sucrose production fluxes. This is just to give a comparison of the various results we obtained. If you would like to download the entire set of interactive graphs representing these results <a>click here</a>. We also have graphs which depict the variations of biomass along with sucrose and these reactions (<a>click here</a>)</p>
							<br>
							<!-- Slideshow of graphs-->
							<div id="carouselExampleCaptions" class="carousel slide" data-bs-ride="carousel">
								<div class="carousel-indicators" style="bottom:-50px;">
								  <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
								  <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="1" aria-label="Slide 2"></button>
								  <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="2" aria-label="Slide 3"></button>
								  <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="3" aria-label="Slide 4"></button>
								</div>
								<div style= "margin-bottom:50px; display:flex; flex-direction:row; align-items: center;" class="carousel-inner">
									<div class="carousel-item active graph"> 
										<img src="assets/graphs/Cyano Graphs/FSEOF/Cyano_FSEOF_1-5.jpg" alt="Best 5 over-expression targets in one graph" width=800>
									</div>
									<div class="carousel-item graph">
										<img src="assets/Graphs/Cyano Graphs/FSEOF/Cyano_FSEOF_6-10.jpg" alt="Next best 5 over-expression targets in one graph" width=800>
									</div>
									<div class="carousel-item graph">
										<img src="assets/Graphs/Cyano Graphs/FSEOF/Cyano_FSEOF_11-15.jpg" alt="Third best 5 over-expression targets in one graph" width=800>
									</div>
									<div class="carousel-item graph">
										<img src="assets/Graphs/Cyano Graphs/FSEOF/Cyano_FSEOF_16-20.jpg" alt="Fourth best 5 over-expression targets in one graph" width=800>
									</div>
								</div>
								<button class="carousel-control-prev" type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide="prev">
								  <span class="carousel-control-prev-icon" aria-hidden="true"></span>
								  <span class="visually-hidden">Previous</span>
								</button>
								<button class="carousel-control-next" type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide="next">
								  <span class="carousel-control-next-icon" aria-hidden="true"></span>
								  <span class="visually-hidden">Next</span>
								</button>
								  </div>
								  <div style="margin-top: 20px;display:flex; flex-direction:row; justify-content: center;">
								  <h3 id="slidetext"></h3>
							</div>
							<br><br><br>
							<!-- END OF SLIDESHOW -->
							
							<p>We searched through literature to find a way to represent the data we have collected, but we found no proven way to do so. Therefore this can utmost be considered as a qualitative analysis of potential targets for over expression as opposed to being a quantitative and conclusive result of targets for overexpression.</p>
							<p><strong> These still need in-laboratory proof for us to confirm these over expression targets.</strong></p>
							<p>Since we could not just leave the results as is, we ranked them in order of decreasing slope because we believe that the ones with higher slopes have a higher flux handling capacity i.e., they can hold more flux for the desired purpose. In terms of the analogy between FSEOF and traffic in a city, these are the roads that can take a good chunk of traffic away from other smaller roads and still not get jammed. </p>
							<p> These targets of overexpression are in terms of reactions, but in wet lab we can not selectively overexpress reactions. What we can overexpress in the lab are genes. So naturally, we looked into how other reactions in the model get affected if and when we overexpress one of the above genes.
								The table below shows how the genes responsible for the above reactions will affect other reactions in the organism. We have generated a <strong>reaction flux vs sucrose flux graph</strong> and a <strong>reaction vs sucrose vs biomass graph</strong> for each over expression target, all of which can be downloaded <a href="#">here</a>.
							</p>
							<p><strong>Note - the “fig x” in each row indicates which graph in the folder corresponds to the said reaction</strong></p>
							<br>
				
							<!-- Start of Table -->
							<h1>Over-expression Table</h1>
							<hr>
							<br>
							<div class="table">
								<!-- Headers -->
								<div class="table1-row table-header">
									<div class="cell">
										Target Reaction
									</div>
									<div class="cell">
										Gene-Protein-Relation
									</div>
									<div class="cell">
										Other reactions controlled by the gene
									</div>
								</div>
								<!--End of headers -->
		
								<!-- Other rows -->
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Proton exchange (Fig 1)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
							
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Water transport pp (Fig 2)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_08900
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										Water transport ltr
									</div>
								</div>
								
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Water transport ext (Fig 3)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										 -
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										ChorismateL-glutamine aminotransferase (Fig 4)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_14170 or M744_09950
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										<strong>M744_14170</strong> - Xanthosine-5'-phosphateL-glutamine amido-ligase (AMP-forming) <br>
										<strong>M744_09950</strong> - NA
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Pyrophosphate phosphohydrolase (Fig 5)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_09680
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										Triphosphate phosphohydrolase
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Asp-tRNA(Asn)L-glutamine amido-ligase (ADP-forming) (Fig 6)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_02485 and M744_04675 and M744_05790
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										<strong>M744_02485</strong> - Glu-tRNA(Gln)L-glutamine amido-ligase(ADP-forming) <br>
										<strong>M744_04675</strong> - Glu-tRNA(Gln)L-glutamine amido-ligase(ADP-forming) <br>
										<strong>M744_05790</strong> - Glu-tRNA(Gln)L-glutamine amido-ligase(ADP-forming)
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										L-AsparaginetRNA(Asn) ligase (AMP-forming) (Fig 7)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_12600
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										L-Aspartateammonia ligase (AMP-forming) (Fig 8)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										ATPAMP phosphotransferase (Fig 9)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_05275
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Cytochrome b6/f complex periplasm (Fig 10)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_10480
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										Cytochrome b6/f complex, Plastoquinone Oxidized-plastocyanin oxidoreductase
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										NAD(P)H dehydrogenase (plastoquinone-8 & 3 protons) (periplasm) (Fig 11)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_06540 and M744_06535 and <br> M744_09900 and M744_09515 and <br> M744_10750 and M744_09885 and <br> M744_09890 and M744_07765 and <br> M744_09895 and M744_10740 <br> and M744_10745 and M744_06510 <br> and M744_05170 and M744_01630 and <br> M744_01625 and M744_03420 or <br> M744_09385 and M744_06535 and <br> M744_09900 and M744_09515 and <br> M744_10750 and M744_09885 and <br> M744_09890 and M744_07765 and <br> M744_09895 and M744_10740 and <br> M744_10745 and M744_06510 and <br> M744_05170 and M744_01630 and <br> M744_01625 and M744_03420
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										<strong>M744_06540, M744_06535, M744_09385, M744_06535</strong> - NAD(P)H dehydrogenase (plastoquinone-8 & 3 protons) (tilacoide) <br>
		
										<strong>M744_09900, M744_09515, M744_09885, M744_06510, M744_05170, M744_09900, M744_09515, M744_09885, M744_06510, M744_05170</strong>  - NAD(P)H dehydrogenase (plastoquinone-8 & 3 protons) (tilacoide), 
										Active co2 transporter facilitator (tilacoide), 
										Active co2 transporter facilitator (periplasm) <br>
										
										<strong>M744_10750, M744_09890, M744_07765, M744_09895, M744_10740, M744_10745, M744_10750, M744_09890, M744_07765, M744_09895, M744_10740, M744_10745</strong> - NADPHp-benzoquinone oxidoreductase,
										NADHquinone oxidoreductase, 
										NAD(P)H dehydrogenase (plastoquinone-8 & 3 protons) (tilacoide), 
										Active co2 transporter facilitator (tilacoide), 
										Active co2 transporter facilitator (periplasm) <br>
										
										
										<strong>M744_01630, M744_01625, M744_03420, M744_01630, M744_01625, M744_03420</strong>  - Bidirectional [NiFe] Hydrogenase,
										NAD(P)H dehydrogenase (plastoquinone-8 & 3 protons) (tilacoide), 
										Active co2 transporter facilitator (tilacoide), 
										Active co2 transporter facilitator (periplasm) <br>
									
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Cytochrome C Oxidase, Synechocystis (2 protons) (Fig 12)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_02050 or ( M744_03160 and M744_03155 and M744_03150 )
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										<strong>M744_02050, M744_03160, M744_03155, M744_03150</strong>  - cytochrome c oxidase, synechocystis (2 protons periplasm)
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										2,3,4,5-TetrahydrodipicolinateNAD+ oxidoreductase (Fig 13)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_05690
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										2,3,4,5-TetrahydrodipicolinateNADP+ oxidoreductase
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										2,3,4,5-TetrahydrodipicolinateNADP+ oxidoreductase (Fig 14)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_05690
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										2,3,4,5-TetrahydrodipicolinateNAD+ oxidoreductase
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Sedoheptulose 1,7-bisphosphate D-glyceraldehyde-3-phosphate-lyase (Fig 15)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_09355
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										D-Fructose 1-phosphate D-glyceraldehyde-3-phosphate-lyase, <br>
										D-Fructose-1,6-bisphosphate D-glyceraldehyde-3-phosphate-lyase
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										FerredoxinNADP+ oxidoreductase (Fig 16)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_11820
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Oxygen transport ext (Fig 17)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										ATP synthase (four protons for one ATP) (periplasm) (Fig 18)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_01360 and M744_01350 and M744_01345 and M744_01335 and M744_01340 and M744_01330 and M744_04715 and M744_04710 and M744_01355 and M744_01365
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										<strong>ALL</strong> - ATP synthetase(u)
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										2-Phospho-D-glycerate 2,3-phosphomutase (Fig 19)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_00620 and M744_06000 and M744_00535 and M744_08585
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										<strong>M744_00620</strong> - 3-Phospho-D-glyceroylphosphate 2,3-phosphomutase
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Carbonic acid hydro-lyase (Fig 21)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_09475 or M744_09655 or M744_09335 or M744_14040
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										<strong>M744_09335</strong> - Carbonic anhydrase
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Nitrate exchange (Fig 22)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										ATPnucleoside-diphosphate phosphatransferase (Fig 23)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_03750
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										D-Ribulose-5-phosphate 3-epimerase (Fig 24)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_13820
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Acetyl-CoAenzyme N6-(dihydrolipoyl)lysine S-acetyltransferase (Fig 25)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_11335
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										PyruvateNAD+ 2-oxidoreductase (CoA-acetylating)
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										L-Valinepyruvate aminotransferase (Fig 26)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_05030
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Sucrose exchange (Fig 27)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										SPS (Fig 28)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										SPP (Fig 29)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										cscB (Fig 30)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										UTP-glucose-1-phosphate uridylyltransferase (Fig 31)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										ATPnucleoside-diphosphate phosphate transferase (Fig 32)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_03750
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										L-Aspartate2-oxoglutarate aminotransferase (Fig 33)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_03485
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										L-Phenylalanine2-oxoglutarate aminotransferase,
										L-Tyrosine2-oxoglutarate aminotransferase,
										L-Cysteine2-oxoglutarate aminotransferase,
										4-Hydroxy-L-glutamate2-oxoglutarate aminotransferase								
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										L-Phenylalanine2-oxoglutarate aminotransferase (Fig 56)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_03485 or M744_12480
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										<strong>M744_03485</strong> - Same as above <br>
		
										<strong>M744_12480</strong> - L-Tyrosine2-oxoglutarate aminotransferase,
										LL-2,6-diaminoheptanedioate2-oxoglutarate aminotransferase
										
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										L-Tyrosine2-oxoglutarate aminotransferase (Fig 58)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_03485 or M744_12480
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										Same as above
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										(S)-Malate hydro-lyase (Fig 34)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_11660
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										L-HomoserineNAD+ oxidoreductase (Fig 35)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_05930
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										(3R)-3-Hydroxyhexanoyl-[acyl-carrier-protein]NADP+ oxidoreductase (Fig 36)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_13375 or M744_08910
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										<strong>M744_13375, M744_08910</strong> - (3R)-3-Hydroxybutanoyl-[acyl-carrier protein]NADP+ oxidoreductase,
										(3R)-3-Hydroxytetradecanoyl-[acyl-carrier-protein]NADP+,
										3oxoacyl-acp reductase (c180),
										(3R)-3-Hydroxyoctanoyl-[acyl-carrier-protein]NADP+ oxidoreductase,
										3-oxoacyl-[acyl-carrier-protein] reductase (n-C121),
										3-oxoacyl-[acyl-carrier-protein] reductase (n-C141),
										3-oxoacyl-[acyl-carrier-protein] reductase (n-C161),
										3-oxoacyl-[acyl-carrier-protein] reductase (n-C181)
										
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										(3R)-3-Hydroxybutanoyl-[acyl-carrier protein]NADP+ oxidoreductase (Fig 37)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_13375 or M744_08910
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										Same as above
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										(3R)-3-Hydroxydodecanoyl-[acyl-carrier-protein]NADP+ oxidoreductase (Fig 38)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_13375 or M744_08910
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										 ame as above
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										(3R)-3-Hydroxyoctanoyl-[acyl-carrier-protein]NADP+ oxidoreductase (Fig 39)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_13375 or M744_08910
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										Same as above
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										(3R)-3-Hydroxypalmitoyl-[acyl-carrier-protein]NADP+ oxidoreductase (Fig 42)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_13375 or M744_08910
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										 Same as above
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										L-Leucine2-oxoglutarate aminotransferase (Fig 40)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_09945 or M744_11540
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										<strong>M744_09945</strong> - ​​L-Valine2-oxoglutarate aminotransferase,
										L-Isoleucine2-oxoglutarate aminotransferase,
										4-amino-4-deoxychorismate pyruvate-lyase
										
										<strong>M744_11540</strong> - ​​L-Valine2-oxoglutarate aminotransferase,
										L-Isoleucine2-oxoglutarate aminotransferase
									</div>
								</div>
		
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										3-Isopropylmalate hydro-lyase (Fig 41)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_06945 and M744_03470
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										<strong>M744_06945, M744_03470</strong> - (R)-2-Methylmalate hydro-lyase,
										2-Isopropylmalate hydro-lyase,
										3-Isopropylmalate hydro-lyase
										
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										L-Isoleucine2-oxoglutarate aminotransferase (Fig 49)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_09945 or M744_11540
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										Same as above
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										IMP 1,2-hydrolase (decyclizing) (Fig 43)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_01000
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										10-Formyltetrahydrofolate5'-phosphoribosyl-5-amino-4
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										5,6,7,8-TetrahydrofolateNADP+ oxidoreductase (Fig 44)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_07375
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										N2-Acetyl-L-ornithine2-oxoglutarate aminotransferase (Fig 45)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_02940 or M744_11990
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										<strong>M744_02490</strong> - S-Adenosyl-L-methionine8-amino-7-oxononanoate aminotransferase,
										N-Succinyl-L-2,6-diaminoheptanedioate2-ocoglytarate
										
										<strong>M744_11990</strong> - L-Alanine3-oxopropanoate aminotransferase,
										4-Aminobutanoate2-oxoglutarate aminotransferase
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										N-Succinyl-L-2,6-diaminoheptanedioate2-ocoglytarate (Fig 50)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_02940
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										Same as above
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										N-Acetyl-L-glutamate-5-semialdehydeNADP+ 5-oxidoreductase (Fig 46)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_09420
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										5,10-methylenetetrahydrofolate,FADH2dUMP C-methyltransferase (Fig 47)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										1-(5-Phospho-D-ribosyl)-5-amino-4-imidazolecarboxylate carboxy-lyase (Fig 48)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_00760 and M744_09450
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										sn-Glycerol-3-phosphateNADP+ 2-oxidoreductase (Fig 51)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_03610
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Phosphate exchange (Fig 52)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Orotidine-5'-phosphatepyrophosphate phosphoribosyltransferase (Fig 53)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_03215
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										(S)-Dihydroorotate amidohydrolase (Fig 54)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_00530 and M744_00520
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										L-1-Pyrroline-5-carboxylateNAD+ oxidoreductase (Fig 55) 
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_00515
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										3-OxopropanoateNAD+ oxidoreductase (decarboxylating), <br>
										AcetaldehydeNAD+ oxidoreductase,<br>
										beta-Aminopropion aldehydeNAD+ oxidoreductase,<br>						
										glycolaldehydeNAD+ oxidoreductase,<br>								
										D-GlyceraldehydeNAD+ oxidoreductase,<br>								
										4-AminobutyraldehydeNAD+ oxidoreductase,<br>								
										Propane-1,2-diolNAD+ oxidoreductase,<br>								
										PhenylacetaldehydeNAD+ oxidoreductase,<br>								
										4-aminobutanalNAD+ 1-oxidoreductase,<br>								
										midazole acetaldehydeNAD+ oxidoreductase,<br>								
										N4-AcetylaminobutanalNAD+ oxidoreductase,<br>
										Methylmalonate - semialdehyde dehydrogenase (propanol),<br>						
										Indole-3-acetaldehydeNAD+ oxidoreductase,<br>
										L-1-Pyrroline-5-carboxylateNAD+ oxidoreductase <br>
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										3-OxopropanoateNAD+ oxidoreductase (decarboxylating) (Fig 77)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_00515
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										Same as above
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Sulfate exchange (Fig 57)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Potassium exchange (Fig 59)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										diacylglycerol kinase (n-C160) (Fig 60)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_00975
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										diacylglycerol kinase (n-C120),<br>
										diacylglycerol kinase (n-C161),<br>								
										diacylglycerol kinase (n-C180),<br>								
										diacylglycerol kinase (n-C181),<br>
										diacylglycerol kinase (n-C160)	
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
									  diacylglycerol kinase (n-C161) (Fig 62)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_00975
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										Same as above
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										diacylglycerol kinase (n-C181) (Fig 78)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_00975
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										Same as above
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										diacylglycerol kinase (n-C180) (Fig 80)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_00975
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										Same as above
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										D-Glucosamine 1-phosphate 1,6-phosphomutase (Fig 61)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_05710
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										D-Ribose 1,5-phosphomutase
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Magnesium exchange (Fig 63)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										UDP-N-acetylmuramate NADP+ oxidoreductase (Fig 64)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_07780
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Fe2+ exchange (Fig 65)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Fe3+ exchange (Fig 66)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Calcium exchange (Fig 67)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Calcium (Ca+2) transport out via proton antiport (periplasm) (Fig 68)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_02980
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Sodium exchange (Fig 69)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Sodium transport out via proton antiport (cytoplasm to periplasm) (Fig 70)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										(M744_12695 or M744_10315) and M744_04470
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Co2+ exchange (Fig 71)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Molybdate exchange (Fig 72)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Mn2+ exchange (Fig 73)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Cu2+ exchange (Fig74)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Zinc exchange (Fig 75)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Nicotinate-nucleotide pyrophosphatase
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_11945
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										Phosphoribosyltransferase (Fig 76)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										-
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
							</div>
		
		
								<div class="table1-row">
									<div class="cell" data-title="Target Reaction">
										ATPpolyphosphate phosphotransferase (Fig 79)
									</div>
									<div class="cell" data-title="Gene-Protein-Relation">
										M744_08870
									</div>
									<div class="cell" data-title="Other reactions controlled by the gene">
										-
									</div>
								</div>
							</div>	
		
							<br><br><br>
		
		
							<!-- TARGETS FOR KNOCKOUTS-->
							<h2 class="h2">Targets for Knockouts</h2>
							<P>We ran Optknock like we did for e coli, but we were unable to obtain any sensible result. All our runs gave a <strong>0 growth rate</strong>. We believe the problem is with the model, as it is still incomplete and not verified, and therefore have concluded nothing further can be done to produce Knockout results.</P>
							<a id="c7"></a>
							<br><a id="ref1"></a><br><a id="ref2"></a><br><a id="ref3"></a><br>
							<!-- REFERENCES -->
							<h2 class="h3">References</h2>
							<ol class="a">
								<li style="padding-left: 1em;" id="ref4"> Mueller, T., Ungerer, J., Pakrasi, H. et al. Identifying the Metabolic Differences of a Fast-Growth Phenotype in Synechococcus UTEX 2973. Sci Rep 7, 41569 (2017). <a href="https://doi.org/10.1038/srep41569">https://doi.org/10.1038/srep41569</a></li>
								<li style="padding-left: 1em;">Abernathy, M.H., Yu, J., Ma, F. et al. Deciphering cyanobacterial phenotypes for fast photoautotrophic growth via isotopically nonstationary metabolic flux analysis. Biotechnol Biofuels 10, 273 (2017). <a href="https://doi.org/10.1186/s13068-017-0958-y">https://doi.org/10.1186/s13068-017-0958-y</a></li>
								<li style="padding-left: 1em;">Song K, Tan X, Liang Y, Lu X. The potential of Synechococcus elongatus UTEX 2973 for sugar feedstock production. Appl Microbiol Biotechnol. 2016 Sep;100(18):7865-75. Epub 2016 Apr 14. PMID: 27079574. <a href="https://doi.org/10.1007/s00253-016-7510-z">https://doi.org/10.1007/s00253-016-7510-z</a></li>
								<li style="padding-left: 1em;">Lin, PC., Zhang, F. & Pakrasi, H.B. Enhanced production of sucrose in the fast-growing cyanobacterium Synechococcus elongatus UTEX 2973. Sci Rep 10, 390 (2020). <a href="https://doi.org/10.1038/s41598-019-57319-5">https://doi.org/10.1038/s41598-019-57319-5</a></li>
							</ol>
							<br><br>
						</div>
						<!-- END OF CONTENT -->



						<!-- SCROLL AWARE HIGHLIGHT -->
						<script>
		
							$(document).ready(function() {
								$(window).scroll(function() {
									var scroll_top = $(window).scrollTop();
									var div_one_top = $("#c1").offset().top;
									var div_two_top = $("#c2").offset().top;
									var div_three_top = $("#c3").offset().top;
									var div_four_top = $("#c4").offset().top;
									var div_five_top = $("#c5").offset().top;
									var div_six_top = $("#c6").offset().top;
									var div_seven_top = $("#c7").offset().top;
					
									if (scroll_top > div_one_top && scroll_top < div_two_top) {
										//Division 1
										if (!$('a[href="#1"]').hasClass('sidenav-active')) {
											$('a[href="#1"]').addClass('sidenav-active');
										}
						
										if ($('a[href="#2"]').hasClass('sidenav-active')) {
											$('a[href="#2"]').removeClass('sidenav-active');
										}
					
									} else if (scroll_top > div_two_top && scroll_top < div_three_top) {
										// Division 2
										if ($('a[href="#1"]').hasClass('sidenav-active')) {
											$('a[href="#1"]').removeClass('sidenav-active');
										}
					
										if ($('a[href="#3"]').hasClass('sidenav-active')) {
											$('a[href="#3"]').removeClass('sidenav-active');
										}
						
										if (!$('a[href="#2"]').hasClass('sidenav-active')) {
											$('a[href="#2"]').addClass('sidenav-active');
										}
									} else if (scroll_top > div_three_top && scroll_top < div_four_top) {
										//Division 3
										if ($('a[href="#2"]').hasClass('sidenav-active')) {
											$('a[href="#2"]').removeClass('sidenav-active');
										}
					
										if ($('a[href="#4"]').hasClass('sidenav-active')) {
											$('a[href="#4"]').removeClass('sidenav-active');
										} 
					
										if (!$('a[href="#3"]').hasClass('sidenav-active')) {
											$('a[href="#3"]').addClass('sidenav-active');
										}
									} else if (scroll_top > div_four_top && scroll_top < div_five_top) {
									// You are in div 4
										if ($('a[href="#3"]').hasClass('sidenav-active')) {
											$('a[href="#3"]').removeClass('sidenav-active');
										}
						
										if ($('a[href="#5"]').hasClass('sidenav-active')) {
											$('a[href="#5"]').removeClass('sidenav-active');
										} 
					
										if (!$('a[href="#4"]').hasClass('sidenav-active')) {
											$('a[href="#4"]').addClass('sidenav-active');
										}
									} else if (scroll_top > div_five_top && scroll_top < div_six_top) {
										// You are in div 5
										if ($('a[href="#4"]').hasClass('sidenav-active')) {
											$('a[href="#4"]').removeClass('sidenav-active');
										}
						
										if ($('a[href="#6"]').hasClass('sidenav-active')) {
											$('a[href="#6"]').removeClass('sidenav-active');
										} 
						
										if (!$('a[href="#5"]').hasClass('sidenav-active')) {
											$('a[href="#5"]').addClass('sidenav-active');
										}
									} else if (scroll_top > div_six_top && scroll_top < div_seven_top) {
										// You are in div 6
										if ($('a[href="#5"]').hasClass('sidenav-active')) {
											$('a[href="#5"]').removeClass('sidenav-active');
										}
					
										if ($('a[href="#7"]').hasClass('sidenav-active')) {
											$('a[href="#7"]').removeClass('sidenav-active');
										} 
						
										if (!$('a[href="#6"]').hasClass('sidenav-active')) {
											$('a[href="#6"]').addClass('sidenav-active');
										}
									} else if (scroll_top > div_seven_top) {
										// You are in div 7
										if ($('a[href="#6"]').hasClass('sidenav-active')) {
											$('a[href="#6"]').removeClass('sidenav-active');
										}
						
										if (!$('a[href="#7"]').hasClass('sidenav-active')) {
											$('a[href="#7"]').addClass('sidenav-active');
										}
									}
								});
							});
								
						</script>
						<!--  END OF SCROLL AWARE HIGHLIGHT -->

						<!-------------------------------------- END OF CYANO TAB ------------------------------------------>




						<!----------------------------------------- E COLI TAB -------------------------------------------->
						<div class="tab-pane fade" id="ecoli" role="tabpanel" aria-labelledby="ecoli-tab">
							<!-- INTRODUCTION -->
							<a id="e1"></a>					
							<h1>Introduction</h1>
							<p>The economic value of our co-culture heavily depends on butanol production by E. coli, making it important to model its growth and butanol production. We used Flux Balance Analysis (FBA) to study the growth dynamics and butanol production in E.coli. We also aimed to optimize the strain by identifying gene targets for overexpression and knockouts in order to improve butanol yields using OptKnock and FSEOF.</p>
							<a id="e2"></a>
							<br>
							<!-- END OF INTRODUCTION -->


							

							<!-- MODEL CONSTRUCTION-->
							<h1>Model Construction</h1>
							<p>We aimed to replicate the KJK01 butanol-producing strain in silico that we received from Dr. ‪Syed Shams Yazdani‬ at ICGEB, Delhi.</p>
							<p>We used the iML1515 model from the BIGG database<a href="#ref1"><sup>1</sup></a> which is a metabolic flux model of E.coli MG1655 that contains 1516 genes. This model was the base strain that we further modified to replicate the KJK01 strain. All modifications were made using the COBRA Toolbox in MATLAB and all optimization problems were solved using the gurobi solver<a href="ref2"><sup>2</sup></a>.</p>
							<p>We first deleted the reactions corresponding to the following genes  that were deleted in the KJK01 strain<a href="#ref3"><sup>3</sup></a>, by setting the fluxes through those reactions to zero. The reactions corresponding to the genes in the model were taken from the EcoCyc database<a href="#ref4"><sup>4</sup></a>.</p>
							<ul class="b">
								<li style="padding-left: 1em;"><a href="https://www.ecocyc.org/gene?orgid=ECOLI&id=EG20173">pta</a> - codes for an enzyme that catalyses the conversion of acetyl-CoA to acetate. </li>
								<li style="padding-left: 1em;"><a href="https://www.ecocyc.org/gene?orgid=ECOLI&id=EG10031">adhE</a> - codes for an enzyme that catalyses the conversion of acetyl-CoA to ethanol. </li>
								<li style="padding-left: 1em;"><a href="https://www.ecocyc.org/gene?orgid=ECOLI&id=EG10330">frdA</a> - codes for an enzyme that catalyzes the conversion of fumarate to succinate. </li>
								<li style="padding-left: 1em;"><a href="https://www.ecocyc.org/gene?orgid=ECOLI&id=G592">ldhA</a> - codes for an enzyme that catalyses the conversion of pyruvate to lactate. </li>
							</ul>
							<p>These genes catalyze reactions that direct flux away from the butanol producing pathway and produce acetate,  ethanol, succinate and lactate respectively, which impact butanol yields. Thus their deletion will reroute flux back into the butanol producing pathway and increase yields.</p>
							<p>We then added reactions catalyzed by the heterologous genes expressed in E.coli in order to construct a butanol-producing pathway in the KJK01 strain. We added the reactions using the MetaCyc<a href="#ref5"><sup>5</sup></a> database and the KEGG<a href="#ref6"><sup>6</sup></a><sup>,</sup><a href="#ref7"><sup>7</sup></a><sup>,</sup><a href="#ref8"><sup>8</sup></a> database.</p>
							<p>These were reactions catalyzed by the following genes:</p>
							<ul class="b">
								<li style="padding-left: 1em;">From Clostridium acetobutylicum: </li>
								<ul class="c"> 
									<li style="padding-left: 1em;"><a href="https://metacyc.org/gene?orgid=META&id=MONOMER-16803">hbd</a> - codes for beta-hydroxybutyryl-CoA dehydrogenase which converts acetoacetyl-coA to (S)-3-hydroxybutanoyl-CoA</li>
									<li style="padding-left: 1em;"><a href="https://metacyc.org/gene?orgid=META&id=CRTCLOS-MONOMER">crt</a> - codes for 3-hydroxy-butyryl-CoA dehydratase that converts (S)-3-hydroxybutanoyl-CoA to  crotonyl-CoA</li>
									<li style="padding-left: 1em;"><a href="https://metacyc.org/gene?orgid=META&id=G-12174">adhE2</a> - codes for butyraldehyde/butanol dehydrogenase, which is a multifunctional enzyme with both alcohol dehydrogenase and acetaldehyde dehydrogenase activities.</li>
								</ul>
								<li style="padding-left: 1em;">From Treponema denticola</li>
								<ul class="c">
									<li style="padding-left: 1em;"><a href="https://www.genome.jp/dbget-bin/www_bget?ec:1.3.1.38">ter</a> - codes for trans-enoyl-coenzyme A (CoA) reductase that converts acyl-CoA to trans-2,3-dehydroacyl-CoA</li>
								</ul>
								<li style="padding-left: 1em;">From E.coli</li>
								<ul class="c">
									<li style="padding-left: 1em;"><a href="https://www.ecocyc.org/gene?orgid=ECOLI&id=EG11672">atoB</a> - coding for acetyl-CoA acetyltransferase that converts acetyl-coA to acetoacetyl-coA.</li>
								</ul>
							</ul>
							<p>We also added the reactions catalyzed by the cscA (invertase), cscB (sucrose permease), cscK (fructokinase) genes from E.coli W to allow the strain to be able to consume and metabolize sucrose<a>9</a>.</p>
							<p>However, we noticed that some of the heterologous gene reactions corresponding to the hbd, crt, cscK, and atoB genes already existed in the base iML1515 strain, and upon consultation with our advisors, we decided to consider the flux through those reactions as overexpressions of the corresponding genes.</p>
							<a id="e3"></a>
							<br>
							<!-- end of model construction -->



							<!-- Modelling E Coli For Optimum Butanol -->
							<h1>Modelling E coli for Optimum Butanol production</h1>
							<p>After adding the relevant genes and reactions to our model, the next step was to enforce additional flux through the reactions corresponding to the genes hbd, crt, atoB and cscK. While these genes were added in KJK01, in a metabolic model, we do not add duplicate reactions but rather increase flux through the existing reaction. The issue here is that fluxes through individual reactions are usually not available in literature.</p>
							<p>On suggestion from our PI, Indumathy, we used a computational way to create a model with suitable over expression levels. She suggested proceeding as follows:</p>
							<ol class="a">
								<li style="padding-left: 1em;"> Add all the reactions necessary for butanol synthesis</li>
								<li style="padding-left: 1em;">Make the appropriate deletions and set realistic bounds for our exchange reactions</li>
								<li style="padding-left: 1em;">Optimize the model for growth and obtain the base value fluxes for all the reactions that were to be overexpressed <br> 
									Some important fluxes are mentioned in the table below: <br><br>

								<!-- Table -->
								<div class="table">
								<!-- Headers -->
									<div class="table1-row table-header">
										<div class="cell">
											Reaction in the model corresponding to the reaction of the inserted gene
										</div>
										<div class="cell">
											Base flux value
										</div>
									</div>
									<!--End of headers -->

									<!-- Other rows -->
									<div class="table1-row">
										<div class="cell" data-title="Reaction in the model corresponding to the reaction of the inserted gene">
											HBD
										</div>
										<div class="cell" data-title="Base flux value">
											12.0665
										</div>
									</div>

									<div class="table1-row">
										<div class="cell" data-title="Reaction in the model corresponding to the reaction of the inserted gene">
											CRT 1
										</div>
										<div class="cell" data-title="Base flux value">
											12.0665
										</div>
									</div>

									<div class="table1-row">
										<div class="cell" data-title="Reaction in the model corresponding to the reaction of the inserted gene">
											CRT 2
										</div>
										<div class="cell" data-title="Base flux value">
											12.0665
										</div>
									</div>

									<div class="table1-row">
										<div class="cell" data-title="Reaction in the model corresponding to the reaction of the inserted gene">
											cscK
										</div>
										<div class="cell" data-title="Base flux value">
											20
										</div>
									</div>

									<div class="table1-row">
										<div class="cell" data-title="Reaction in the model corresponding to the reaction of the inserted gene">
											Biomass
										</div>
										<div class="cell" data-title="Base flux value">
											0.2238
										</div>
									</div>

									<div class="table1-row">
										<div class="cell" data-title="Reaction in the model corresponding to the reaction of the inserted gene">
											Butanol exchange
										</div>
										<div class="cell" data-title="Base flux value">
											0
										</div>
									</div>
								</div>
								<!-- END OF TABLE -->
								</li>
								<br>
								<li class="padding-left: 1em;">We then developed a nested loop designed as follows:</li>
								<ul class="c">
									<li style="padding-left: 1em;"> We force flux in the range 12 to 15 (with a step size of 0.5) through the hbd reaction</li>
									<li style="padding-left: 1em;"> For each flux value of hbd we had a loop forcing flux in the same range through the reaction CRT1</li>
									<li style="padding-left: 1em;"> For each flux value for CRT1 we had a loop forcing the same range of fluxes through the CRT 2 reaction</li>
									<li style="padding-left: 1em;"> For each flux value of CRT2 reaction, we had a loop forcing flux in the range 20 to 24 (for a step size of 1) through the cscK gene reaction</li>
								</ul>
								<li style="padding-left: 1em;"> We then optimize the model with growth as the objective function for each set of flux values and check for growth and butanol production</li>
							</ol>
							<br>
							<p>We thus generated a table (<a>Supplementary material [ecoli_ovrxps_fluxes_v2]</a>) of enforced flux distributions with their effects on growth and butanol production.</p>
							<br>
							<div class="graph">
								<img src="assets/Graphs/E coli graphs/optimizing_model_for_butanol_2.png" alt=" A graph showing the overexpression of the genes and the growth and butanol production resulting from the overexpression" width=800>
								<p style="padding-top: 15px;">Figure 1: A graph showing the overexpression of the genes and the growth and butanol production resulting from the overexpression. Note that we have plotted hbd as a representative of the hbd, crt_2 and atoB reactions since they all have the same fluxes through them. CRT 1 is not shown as it does not directly participate in butanol synthesis. The selected data point has been pointed out. The flux values are as follows; hbd - 14, crt_1 - 12, crt_2 - 14, atoB - 14, cscK - 20, biomass - 0.136 , butanol flux - 2.</p>
							</div>
							<br>
							<p>Out of the 12,005 expected data points, only 133 showed viable growth. We observed that the fluxes through the reactions of the genes hbd, crt_2, and atoB were the same whenever growth was observed, which is to be expected given that these fractions follow one after the other in the metabolic pathway. We set a cutoff of 60% for the biomass reaction and selected the datapoint showing the highest butanol production. This lead to the following flux distribution for the reactions to be overexpressed:</p>
							<!-- Table -->
							<div class="table" style="padding-left: 5%;">
								<!-- Headers -->
								<div class="table1-row table-header">
									<div class="cell">
										HBD
									</div>
									<div class="cell">
										CRT 1
									</div>
									<div class="cell">
										CRT 2
									</div>
									<div class="cell">
										atoB
									</div>
									<div class="cell">
										cscK
									</div>
								</div>
								<!-- END OF HEADER -->
								<div class="table1-row">
									<div class="cell" data-title="HBD">
										14
									</div>
									<div class="cell" data-title="CRT 1">
										12
									</div>
									<div class="cell" data-title="CRT 2">
										14
									</div>
									<div class="cell" data-title="atoB">
										14
									</div>
									<div class="cell" data-title="cscK">
										20
									</div>
								</div>
							</div>
							<br>
							<!-- END OF TABLE -->
							<p>At this value of overexpression we observed a growth of 0.136 and butanol production of 2.</p>
							<p>The final model with the overexpression fluxes is available here <a>(Supplementary material)</a></p>
							<br> <br>

							<h2 class="h2">Flux Variability Analysis for Overexpression Targets</h2>
							<p>We used flux variability analysis (FVA)<a href="#ref11"><sup>11</sup></a> to find the range of possible fluxes through the reactions of the crt_1, crt_2, atoB, cscK genes and the butanol exchange reaction.</p>
							<p>FVA can be used to find the minimum and maximum fluxes for any reaction for a particular state of the metabolic network. The FVA for these reactions were performed in anaerobic conditions, when growth was set to 60% of its optimal value.</p>
							<br>
							<!-- Table -->
							<div class="table" style="padding-left: 5%;">
								<!-- Headers -->
								<div class="table1-row table-header">
									<div class="cell">
										Reaction Name
									</div>
									<div class="cell">
										Minimum Flux
									</div>
									<div class="cell">
										Maximum Flux
									</div>
								</div>
								<!-- END OF HEADER -->

								<div class="table1-row">
									<div class="cell" data-title="Reaction Name">
										ECHOAH1 (CRT2) (by crt gene)
									</div>
									<div class="cell" data-title="Minimum Flux">
										18.4663 
									</div>
									<div class="cell" data-title="Maximum Flux">
										19.0552
									</div>	
								</div>
								
								<div class="table1-row">
									<div class="cell" data-title="Reaction Name">
										ECOAH2 (CRT1)  (by crt gene)
									</div>
									<div class="cell" data-title="Minimum Flux">
										-0.5411
									</div>
									<div class="cell" data-title="Maximum Flux">
										0.0478
									</div>	
								</div>

								<div class="table1-row">
									<div class="cell" data-title="Reaction Name">
										ACACT1r (by adhE gene)
									</div>
									<div class="cell" data-title="Minimum Flux">
										18.4663
									</div>
									<div class="cell" data-title="Maximum Flux">
										19.0552
									</div>	
								</div>

								<div class="table1-row">
									<div class="cell" data-title="Reaction Name">
										Butanol exchange reaction
									</div>
									<div class="cell" data-title="Minimum Flux">
										19.0074
									</div>
									<div class="cell" data-title="Maximum Flux">
										19.0074
									</div>	
								</div>

								<div class="table1-row">
									<div class="cell" data-title="Reaction Name">
										HACD1 (by hbd gene)
									</div>
									<div class="cell" data-title="Minimum Flux">
										18.4663
									</div>
									<div class="cell" data-title="Maximum Flux">
										19.0552
									</div>	
								</div>

								<div class="table1-row">
									<div class="cell" data-title="Reaction Name">
										HEX7 (by cscK gene)
									</div>
									<div class="cell" data-title="Minimum Flux">
										0
									</div>
									<div class="cell" data-title="Maximum Flux">
										25.8536
									</div>	
								</div>
							</div>
							<!-- END OF TABLE -->
							<br>
							<p>Thus we can see what optimal overexpression fluxes are required to maximize butanol yields, even when growth is not optimal.</p>
							<a id="e4"></a>
							<br>
							<!-- END OF MODELLING FOR BUTANOL -->


							<!-- RESULTS -->
							<h1> Results</h1>
							<h2 class="h3">Growth Rate v/s Sucrose</h2>
							<div class="graph">
								<img src="assets/Graphs/E coli graphs/growthratevsucrose.jpg" alt="A graph representing the relation between varying sucrose uptaek values and growth rates. We obsereve a linear relation between the two, where growth rate increases with increase in sucrose uptake value for both aerobic and anaerobic scenario. The aerobic system has a steeper rise " width =800>
								<p>Fig 12: Relationship between sucrose uptake and growth rate.</p>
							</div>
							<br>
							<p><strong> Observations: </strong></p>
							<ol class="a">
								<li style="padding-left: 1em;"> Growth rate increases linearly with sucrose uptake rate</li>
								<li style="padding-left: 1em;"> Growth rate is higher under aerobic conditions for all sucrose uptake rates as compared to anaerobic conditions</li>
							</ol>
							<p><strong>Inferences</strong> </p>
							<p style="padding-left: 12%;">Growth rate increases linearly with sucrose uptake rate as expected, since the carbon flux from sucrose metabolism will be directed to the biomass reaction, leading to an increase in growth rate. This confirms that the cscA, cscB, cscK reactions added to the model work in the metabolic network and the model can grow on sucrose as a carbon source.</p>
							<br><br><br>

							<h2 class="h3">Butanol Production vs Growth Rate</h2>
							<p>The objective function was first set to maximize the growth rate of the model. Butanol production was then set as the objective function for different percentages of the optimal growth rate at a sucrose uptake rate of 10 mmol/gDW/h to obtain the theoretical maximum yield of butanol if E.coli were growing at X% of its optimal growth rate.</p>
							<div class="graph">
								<img src="assets/Graphs/E coli graphs/maxbutvgrowth.jpg" alt="A graph showing the relation between butanol production and growth rate. Growth rate decreases linearly with increase in butanol production, for both aerobic and anaerobic systems. The line for aerobic system has a steeper decline than the anaerobic system" width=800>
								<p>Figure 3: Relationship between maximum butanol production and growth rate.</p>
							</div>
							<br>
							<p><strong>Observations: </strong></p>
							<ol class="a">
								<li style="padding-left: 1em;"> The maximal theoretical yield of butanol production decreases as growth rate increases</li>
								<li style="padding-left: 1em;"> Maximal butanol production is much higher for anaerobic conditions as compared to aerobic conditions</li>
								<li style="padding-left: 1em;"> Optimal growth rate (100% on the graph) leads to zero butanol production in both aerobic and anaerobic conditions</li>
							</ol>
							<p><strong>Inferences: </strong></p>
							<p style="padding-left: 12%;"> If E.coli grows at its optimal growth rate, butanol production is zero as would be expected, since it is a secondary metabolite that takes carbon flux away from the growth and has no role in the physiology of the cell as it is a heterologous compound in the E.coli metabolic network</p>
							<p style="padding-left: 12%;">As the growth rate is decreased from its optimal value, the maximal theoretical yield of butanol increases implying that carbon flux is rerouted to the butanol producing pathway in the network.</p>
							<p style="padding-left: 12%;"> This confirms that the butanol production pathway in the model is functional. Butanol production happens under anaerobic or microaerobic conditions<a href="#ref10"><sup>10</sup></a>. We wanted to thus test the effect of oxygenation on butanol production in the model, and as expected found that butanol production was significantly higher in anaerobic conditions as compared to aerobic conditions at all growth rates.</p>
							<br><br><br>

							<h2 class="h3">Butanol Production vs Sucrose Uptake</h2>
							<div class="graph">
								<img src="assets/Graphs/E coli graphs/butanolvsucrose.jpg", alt="A grpah depicting the relation between the maximum butanol production given a sucrose uptaek rate. We observe a linear pattern with the max butanol production increasing with increase in sucrose uptake values for both aerobic and anaerobic systems. The anaerobic line has a steeper rise." width=800>
								<p>Figure 4: Relationship between maximum butanol production and sucrose uptake when growth is 60% of its optimal value</p>
							</div>
							<br>
							<p><strong>Observations:</strong></p>
							<ol class="a">
								<li style="padding-left: 1em;"> Maximum butanol production increases linearly with increasing sucrose uptake</li>
								<li style="padding-left: 1em;"> Maximum butanol production is higher for anaerobic conditions as compared to aerobic conditions at all sucrose uptake rates</li>
							</ol>
							<p><strong>Inferences</strong></p>
							<p style="padding-left: 12%;">Increasing sucrose uptake rates lead to an increase in butanol production when butanol is the objective function. However we expect newly engineered strains that haven’t evolved for very long to optimize growth as an objective instead of butanol production. Thus there is a tradeoff between growth and butanol production, which we imitated by setting the growth rate of E.coli to 60% of the optimal growth rate and setting the objective function as butanol production. This shows that even at low growth rates, the extra flux from sucrose uptake can be used to increase butanol production over other metabolites in the model.</p>
							<br><br><br>

							<h2 class="h3">Butanol Production vs Oxygen Uptake</h2>
							<p>Our co-culture modelling results indicated that butanol production in the co-culture is optimum i microaerobic conditions as compared to anaerobic conditions and we wanted to test whether that would hold true for a monoculture as well, by testing the maximum butanol production at varying oxygen uptake rates, when growth was set to 60% of the optimal value</p>
							<div class="graph">
								<img src="assets/Graphs/E coli graphs/butanolvoxygen.jpg" alt="A graph depicting the relation between butanol production and oxygen uptaek rates. We observe a linear relation between the two, where as the oxygen uptaek value increases, the butanol production value decreases" width=800>
								<p>Figure 5: Relationship between maximum butanol production and oxygen uptake when growth is 60% of its optimal value</p>
							</div>
							<br>
							<p><strong>Observations: </strong></p>
							<ol class="a">
								<li style="padding-left: 1em;"> As oxygen uptake rates increase, maximum butanol production decreases</li>
							</ol>
							<p><strong>Inferences</strong></p>
							<p style="padding-left: 12%;">In a mono-culture, there is a slight decrease in maximal butanol production in microaerobic as compared to anaerobic conditions</p>
							<br><br><br>

							<h2 class="h2">Side Products</h2>
							<p>Abdelaal el. al<a href="#ref3"><sup>3</sup></a> observed pyruvate, ethanol and butyrate to be the major side products in the KJK01 butanol producing strain, which divert flux away from the butanol production pathway and reduce yields. Pyruvate was the major side product which indicated a redox imbalance, while ethanol was produced in equal quantities even after the deletion of the endogenous alcohol dehydrogenase gene (adhE).</p>
							<p>We wanted to estimate the maximal side product formation at different growth levels in anaerobic and aerobic conditions and see whether our results matched with those reported by Abdelaal el. al<a href="#ref3"><sup>3</sup></a>.</p>
							<p>We used flux variability analysis (FVA)<a href="#ref11"><sup>11</sup></a> to find the maximum possible fluxes through the side product reactions under a growth rate constraint. FVA can be used to find the minimum and maximum fluxes for any reaction for a particular state of the metabolic network. For example, it can be used to find the minimum and maximum fluxes for a reaction such that X% of the optimal growth rate is maintained.</p>
							<p>We obtained the optimal growth rates for aerobic and anaerobic conditions and for a particular percentage of that growth rate, FVA was performed for the exchange reactions of butanol, ethanol, pyruvate, butyrate and sucrose uptake to find the maximal fluxes through those reactions. The minimum fluxes for these reactions will be zero since all the carbon flux will be directed toward growth.</p>
							<br>
							<div class="graph">
								<img src="assets/Graphs/E coli graphs/sideproductsanaerobicblack.jpg" alt="" width=800>
								<p>Fig 13: Maximal side product formation as a function of growth in anaerobic conditions</p>
							</div>
							<br><br>
							<div class="graph">
								<img src="assets/Graphs/E coli graphs/sideproductsaerobicblack.jpg" alt="" width=800>
								<p>Fig 14: Maximal side product formation as a function of growth in aerobic conditions</p>
							</div>
							<br>
							<p><strong>Observations: </strong></p>
							<ol class="a">
								<li style="padding-left: 1em;"> In anaerobic and aerobic conditions, pyruvate production is consistently higher than the other side products</li>
								<li style="padding-left: 1em;"> Butyrate seems to be almost equally produced with butanol (particularly in aerobic conditions) whereas in both anaerobic and aerobic conditions, ethanol production is zero</li>
								<li style="padding-left: 1em;"> Sucrose consumption in both anaerobic and aerobic conditions is quite similar</li>
							</ol>
							<p><strong>Inferences</strong></p>
							<p style="padding-left: 12%;">Consistent with the observations of Abdelaal el. al<a href="#ref3"><sup>3</sup></a>, pyruvate was a major byproduct which indicates a redox imbalance in the network. Ethanol was zero, which is to be expected since there is only one gene (adhE) that produces ethanol from acetyl-coA, whose flux we set to zero to duplicate the KJK01 strain knockout of adhE. Given that the flux through pyruvate and butyrate production is equal or greater than that of butanol, these present alternate possible targets to increase butanol production</p>
							<a id="e5"></a>
							<br>
							<!-- END OF RESULTS -->


							<!-- OptKnock -->
							<h1>OptKnock</h1>
							<p>We ran OptKnock on our base model without the knockouts but with the overexpressions to identify the possible knockouts that could help increase butanol production. Our constructed model had 4 knockouts, so for OptKnock, we set the total number of knockouts to 5. Since the metabolic pathway for butanol production is linked to the TCA cycle in E. coli, we decided to look for knockouts in this cycle. Thus, we looked for knockouts in the glycolytic pathway plus the TCA cycle. We were hoping to find the same deletions as in our constructed model, plus some additional ones. We selected the top 5 knockouts. The results are summarized in the table below</p>
							<br>					
							<!-- Table -->
							<h1> Optknock Table </h1> 
							<hr><br>
							<div class="table">
								<!-- Header -->
								<div class="table1-row table-header">
									<div class="cell">
										S. No.
									</div>
									<div class="cell">
										Knockouts suggested by OptKnock
									</div>
									<div class="cell">
										Enzymes
									</div>
									<div class="cell">
										Growth
									</div>
									<div class="cell">
										Butanol production
									</div>
								</div>
								<!-- END OF HEADER -->

								<div class="table1-row">
									<div class="cell" data-title="S. No.">
										1
									</div>
									<div class="cell" data-title="Knockouts suggested by OptKnock">
										ACKr, ALCD2x, LDH_D, TPI
									</div>
									<div class="cell" data-title="Enzymes">
										Acetate kinase, Alcohol dehydrogenase (ethanol), D-lactate dehydrogenase, Triose-phosphate isomerase
									</div>
									<div class="cell" data-title="Growth">
										0.17
									</div>
									<div class="cell" data-title="Butanol Production">
										18.60
									</div>
								</div>

								<div class="table1-row">
									<div class="cell" data-title="S. No.">
										2
									</div>
									<div class="cell" data-title="Knockouts suggested by OptKnock">
										ACALD, FRD3, ACKr, LDH_D, TPI
									</div>
									<div class="cell" data-title="Enzymes">
										Acetaldehyde dehydrogenase (acetylating), Fumarate reductase, Acetate kinase, D-lactate dehydrogenase, Triose-phosphate isomerase
									</div>
									<div class="cell" data-title="Growth">
										0.17
									</div>
									<div class="cell" data-title="Butanol Production">
										18.60
									</div>
								</div>

								<div class="table1-row">
									<div class="cell" data-title="S. No.">
										3
									</div>
									<div class="cell" data-title="Knockouts suggested by OptKnock">
										ACALD, PTAr, ALCD2x, LDH_D, TPI
									</div>
									<div class="cell" data-title="Enzymes">
										Acetaldehyde dehydrogenase (acetylating), Phosphotransacetylase, Alcohol dehydrogenase (ethanol), D-lactate dehydrogenase, Triose-phosphate isomerase
									</div>
									<div class="cell" data-title="Growth">
										0.17
									</div>
									<div class="cell" data-title="Butanol Production">
										18.60
									</div>
								</div>

								<div class="table1-row">
									<div class="cell" data-title="S. No.">
										4
									</div>
									<div class="cell" data-title="Knockouts suggested by OptKnock">
										FRD2, PTAr, ALCD2x, LDH_D, TPI
									</div>
									<div class="cell" data-title="Enzymes">
										Fumarate reductase, Phosphotransacetylase, Alcohol dehydrogenase (ethanol), D-lactate dehydrogenase, Triose-phosphate isomerase
									</div>
									<div class="cell" data-title="Growth">
										0.17
									</div>
									<div class="cell" data-title="Butanol Production">
										18.60
									</div>
								</div>

								<div class="table1-row">
									<div class="cell" data-title="S. No.">
										5
									</div>
									<div class="cell" data-title="Knockouts suggested by OptKnock">
										ACALD, FRD2, PTAr, LDH_D, TPI
									</div>
									<div class="cell" data-title="Enzymes">
										Acetaldehyde dehydrogenase (acetylating), Fumarate reductase, Phosphotransacetylase, D-lactate dehydrogenase, Triose-phosphate isomerase
									</div>
									<div class="cell" data-title="Growth">
										0.17
									</div>
									<div class="cell" data-title="Butanol Production">
										18.60
									</div>
								</div>
							</div>
							<!-- END OF TABLE -->
							<br><br>
							<p>We observed that all the deletions (except ‘ACLD19’) we made in our constructed model showed up in some or the other combination of the OptKnock results. We observed a growth rate of 0.17 and butanol production of 18.6 in our suggested deletions. Furthermore, D-lactate dehydrogenase (LDH_D) and Triose-phosphate isomerase (TPI) were suggested in all the top 5 OptKnock deletions. While the suggested reactions have been knocked out in our constructed model, TPI is a new deletion suggested by OptKnock. TPI catalyzes the conversion of Dihydroxyacetone phosphate (DHAP) to 3-Phosphoglyceraldehyde (PGAL). It isn’t clear how this helps in butanol production yet. Experimental validation might be needed.</p>
							<a id="e6"></a>
							<br>
							<!-- END OF OPTKNOCK -->

							<!-- FSEOF -->
							<h1>FSEOF</h1>
							<p>We ran FSEOF on our constructed model without the overexpressions to identify reactions whose fluxes increased with increase in the enforced flux through the butanol exchange reactions. The fluxes through a total of 65 reactions showed a strict increase with increase in enforced butanol flux. Some of the reactions showing large increase in flux include proton transfer, CO2 exchange, etc. but also reactions of genes like pyruvate dehydrogenase, pyruvate kinase, Acyl-CoA dehydrogenase, 3-hydroxyacylCoA dehydratase, 3-hydroxyacylCoA dehydrogenase. Apart from the transport and exchange reactions of butanol, we did not observe any increase in flux in the reactions added by us. The full list of reactions is available </p>
							<a id ="e7"></a>

							<br><a id="ref1"></a><br><a id="ref2"></a><br><a id="ref3"></a><br>
							<!-- END OF FSEOF -->


							<!-- REFRENCES-->
							<h2 class="h2">References</h2>
							<ol class="a">
								<li style="padding-left: 1em;" id="ref4"> King, Z. A., Lu, J., Dräger, A., Miller, P., Federowicz, S., Lerman, J. A., ... & Lewis, N. E. (2016). BiGG Models: A platform for integrating, standardizing and sharing genome-scale models. Nucleic acids research, 44(D1), D515-D522.</li>
								<li style="padding-left: 1em;" id="ref5"> Gurobi Optimization, L.. (2021). Gurobi Optimizer Reference Manual.</li>
								<li style="padding-left: 1em;" id="ref6"> Abdelaal, A. S., Jawed, K., & Yazdani, S. S. (2019). CRISPR/Cas9-mediated engineering of Escherichia coli for n-butanol production from xylose in defined medium. Journal of Industrial Microbiology and Biotechnology, 46(7), 965-975.</li>
								<li style="padding-left: 1em;" id="ref7"> Keseler, I. M., Collado-Vides, J., Santos-Zavaleta, A., Peralta-Gil, M., Gama-Castro, S., Muñiz-Rascado, L., ... & Karp, P. D. (2010). EcoCyc: a comprehensive database of Escherichia coli biology. Nucleic acids research, 39(suppl_1), D583-D590.</li>
								<li style="padding-left: 1em;" id="ref8"> Caspi, R., Altman, T., Billington, R., Dreher, K., Foerster, H., Fulcher, C. A., ... & Karp, P. D. (2014). The MetaCyc database of metabolic pathways and enzymes and the BioCyc collection of Pathway/Genome Databases. Nucleic acids research, 42(D1), D459-D471.</li>
								<li style="padding-left: 1em;" id="ref9"> Kanehisa, M., & Goto, S. (2000). KEGG: kyoto encyclopedia of genes and genomes. Nucleic acids research, 28(1), 27-30.</li>
								<li style="padding-left: 1em;" id="ref10"> Kanehisa, M. (2019). Toward understanding the origin and evolution of cellular organisms. Protein Science, 28(11), 1947-1951.</li>
								<li style="padding-left: 1em;" id="ref11"> Kanehisa, M., Furumichi, M., Sato, Y., Ishiguro-Watanabe, M., & Tanabe, M. (2021). KEGG: integrating viruses and cellular organisms. Nucleic acids research, 49(D1), D545-D551.</li>
								<li style="padding-left: 1em;"> Bruschi, M., Boyes, S. J., Sugiarto, H., Nielsen, L. K., & Vickers, C. E. (2012). A transferable sucrose utilization approach for non-sucrose-utilizing Escherichia coli strains. Biotechnology advances, 30(5), 1001-1010.</li>
								<li style="padding-left: 1em;"> Wu, P., Wang, G., Wang, G., Børresen, B. T., Liu, H., & Zhang, J. (2016). Butanol production under microaerobic conditions with a symbiotic system of Clostridium acetobutylicum and Bacillus cereus. Microbial cell factories, 15(1), 1-11.</li>						
								<li style="padding-left: 1em;"> Mahadevan, R., & Schilling, C. H. (2003). The effects of alternate optimal solutions in constraint-based genome-scale metabolic models. Metabolic engineering, 5(4), 264-276.</li>
							</ol>
							<!-- END OF REFERECNES -->
						</div>
						<!-- END OF CONTENT -->									


						<!-- SCROLL AWARE HIHGLIGHT -->
						<script>
		
							$(document).ready(function() {
								$(window).scroll(function() {
									var scroll_top = $(window).scrollTop();
									var div_one_top = $("#e1").offset().top;
									var div_two_top = $("#e2").offset().top;
									var div_three_top = $("#e3").offset().top;
									var div_four_top = $("#e4").offset().top;
									var div_five_top = $("#e5").offset().top;
									var div_six_top = $("#e6").offset().top;
									var div_seven_top = $("#e7").offset().top;
					
									if (scroll_top > div_one_top && scroll_top < div_two_top) {
										//Division 1
										if (!$('a[href="#1"]').hasClass('sidenav-active')) {
											$('a[href="#1"]').addClass('sidenav-active');
										}
					
										if ($('a[href="#2"]').hasClass('sidenav-active')) {
											$('a[href="#2"]').removeClass('sidenav-active');
										}
					
									} else if (scroll_top > div_two_top && scroll_top < div_three_top) {
										// Division 2
										if ($('a[href="#1"]').hasClass('sidenav-active')) {
											$('a[href="#1"]').removeClass('sidenav-active');
										}
					
										if ($('a[href="#3"]').hasClass('sidenav-active')) {
											$('a[href="#3"]').removeClass('sidenav-active');
										}
					
										if (!$('a[href="#2"]').hasClass('sidenav-active')) {
											$('a[href="#2"]').addClass('sidenav-active');
										}
									} else if (scroll_top > div_three_top && scroll_top < div_four_top) {
										//Division 3
										if ($('a[href="#2"]').hasClass('sidenav-active')) {
											$('a[href="#2"]').removeClass('sidenav-active');
										}
					
										if ($('a[href="#4"]').hasClass('sidenav-active')) {
											$('a[href="#4"]').removeClass('sidenav-active');
										} 
					
										if (!$('a[href="#3"]').hasClass('sidenav-active')) {
											$('a[href="#3"]').addClass('sidenav-active');
										}
									} else if (scroll_top > div_four_top && scroll_top < div_five_top) {
										// You are in div 4
										if ($('a[href="#3"]').hasClass('sidenav-active')) {
											$('a[href="#3"]').removeClass('sidenav-active');
										}
					
										if ($('a[href="#5"]').hasClass('sidenav-active')) {
											$('a[href="#5"]').removeClass('sidenav-active');
										} 
					
										if (!$('a[href="#4"]').hasClass('sidenav-active')) {
											$('a[href="#4"]').addClass('sidenav-active');
										}
									} else if (scroll_top > div_five_top && scroll_top < div_six_top) {
										// You are in div 5
										if ($('a[href="#4"]').hasClass('sidenav-active')) {
											$('a[href="#4"]').removeClass('sidenav-active');
										}
					
										if ($('a[href="#6"]').hasClass('sidenav-active')) {
											$('a[href="#6"]').removeClass('sidenav-active');
										} 
					
										if (!$('a[href="#5"]').hasClass('sidenav-active')) {
											$('a[href="#5"]').addClass('sidenav-active');
										}
									} else if (scroll_top > div_six_top && scroll_top < div_seven_top) {
										// You are in div 6
										if ($('a[href="#5"]').hasClass('sidenav-active')) {
											$('a[href="#5"]').removeClass('sidenav-active');
										}
					
										if ($('a[href="#7"]').hasClass('sidenav-active')) {
											$('a[href="#7"]').removeClass('sidenav-active');
										} 
					
										if (!$('a[href="#6"]').hasClass('sidenav-active')) {
											$('a[href="#6"]').addClass('sidenav-active');
										}
									} else if (scroll_top > div_seven_top) {
										// You are in div 7
										if ($('a[href="#6"]').hasClass('sidenav-active')) {
											$('a[href="#6"]').removeClass('sidenav-active');
										}
					
										if (!$('a[href="#7"]').hasClass('sidenav-active')) {
											$('a[href="#7"]').addClass('sidenav-active');
										}
									}
								});
							});
							
						</script>
						<!-- END OF SCROLL AWARE HIGHLIGHT-->

						<!-------------------------------------- END OF E . COLI TAB --------------------------------------->




						<!------------------------------------------- CO CULTURE TAB ------------------------------------------------->
						<div class="tab-pane fade" id="Coculture" role="tabpanel" aria-labelledby="coculture-tab">
							<h1 id="cc1">Introduction</h1>
							<hr><br>
							<p>The co-culture aspect makes this project unique and allows us to bring the properties of cyanobacteria and E. coli together to form the biomanufacturing unit. Although the organisms have been modelled individually, the co-culture brings forth new properties and interactions that may offer additional insights and alternative techniques to optimize the production process.</p>
							<br>
							<p>Two techniques have been used to model the co-culture - SteadyCom and Dynamic Modelling. SteadyCom assumes a community steady state that allows us to predict the stability of the coo-culture and long-term interactions between the organisms. The dynamic modelling algorithm, developed by the INSA-UPS Toulouse iGEM Team, allows us to track the development of the co-culture over time. Together, these two models built on slightly different assumptions can offer a wide range of insights.</p>
					
							<h1 id="cc2">SteadyCom</h1>
							<hr><br>
							
							<h3>Building the Community Model of S. elongatus 2973 and E coli</h3><br>
							<p>SteadyCom uses a joint community model which is made by putting together the GSM’s of the individual species in the community. The individual models used to build the community model of our co culture were the modified versions of iML1515 and the TUEX 2973 model that were built to match KJK01 and the sucrose producing cyanobacteria. All the steadycom simulations including the creation of the joint model was done using the COBRA toolbox on MATLAB. During the process of creating the joint model however, we found that due to lack of documentation on SteadyCom and standardisation of GSM models, we faced many issues. We eventually troubleshot our issues after experimenting with joint models of just E coli and cyanobacteria and figuring out the intricacies of the joint model. This led to us deciding to create a checklist that we believe would greatly help future users of SteadyCom on COBRA navigate their way through creating joint community models and troubleshooting. You can read more about that on our contributions page here: <a href="Contribution.html">Contributions</a>.</p>
							<br>

							<p>Butanol is a secondary metabolite, so when just growth rate is optimized for in SteadyCom, butanol would not be produced as it would be taking away carbon flux from the growth pathways. In the actual co-culture, the induction by IPTG would force carbon flux through the butanol production pathway to produce butanol. To simulate this we set a lower bound for the butanol exchange reaction which would constrain the model to have flux through the butanol production pathway in E coli. The particular value for the lower bound was obtained through FBA at different percentages of optimal growth rate. You can read about the exact method we used to set the lower bound here: <a href="#4">Appendix</a>. The higher the percentage of optimal growth, the lower the butanol productivity.</p>
							<br>

							<p>The carbon dioxide uptake rate of the cyanobacteria was set at -12.2mmol/gDW/hour. Salt stress was simulated by setting the sucrose produced by cyanobacteria as per the results of FBA at 90% cscB efficiency. The butanol production for E coli was set at 60% of optimal growth. Oxygen exchange between the two species was allowed but excess oxygen was not provided to the culture.</p>
							<br>

							<p>Results:<br><br></p><br>

							<h3>Identifying Key metabolites controlling interactions in the Co-culture</h3>
							<br>
							<p>We knew that sucrose secreted by the cyanobacteria would play an important role in the interaction between the two species of the co culture as there is no other carbon source available for the E coli. Apart from sucrose, we ran SteadyCom FVA to identify other metabolites that could play crucial roles in the interactions of the consortium. SteadyCom FVA (Flux Variability Analysis) calculates the maximum and minimum values of flux through a set of reactions while upholding the optimal growth rate at steady state. By running FVA on the set of shuttle reactions of metabolites between the individual species and the community space, we identified those metabolites that are required to be taken up or given out by each species to maintain optimal community growth rate. </p> <br>
							<div style="overflow-x:auto; overflow-y: auto; max-height: 400px;">
								<table data-table-theme="dark zebra">
									<thead><tr><th>Metabolite</th><th>Min Flux Through E coli</th><th>Max Flux Through E coli</th><th>Min Flux Through Cyanobacteria</th><th>Max Flux Through Cyanobacteria</th></tr>
									</thead>
									<tbody>
										<tr><td>Carbon Dioxide</td><td>2.015443068</td><td>2.035662892</td><td>-16.26443278</td><td>-16.25798452</td></tr>
										<tr><td>2-Oxoglutarate</td><td>0</td><td>0.002524686123</td><td>0</td><td>0.0003069649685</td></tr>
										<tr><td>L-Glutamine</td><td>0</td><td>0</td><td>0</td><td>0</td></tr>
										<tr><td>Calcium</td><td>-0.0002535389119</td><td>-0.0002527589119</td><td>-0.0008734307167</td><td>-0.0008730069524</td></tr>
										<tr><td>L-Alanine</td><td>0</td><td>0.00351260678</td><td>0</td><td>0</td></tr>
										<tr><td>Copper</td><td>-0.00003453577675</td><td>-0.00003442960011</td><td>-0.0005822871444</td><td>-0.000582004635</td></tr>
										<tr><td>Zinc</td><td>-0.00001661027106</td><td>-0.00001655922939</td><td>-0.0005822871444</td><td>-0.000582004635</td></tr>
										<tr><td>L-Lysine</td><td>0</td><td>0.001346499266</td><td>0</td><td>0</td></tr>
										<tr><td>Cobalt</td><td>-0.000001217611619</td><td>-0.00000121401975</td><td>-0.000629578617</td><td>-0.000629273163</td></tr>
										<tr><td>L-Arginine</td><td>0</td><td>0.001392930275</td><td>0</td><td>0</td></tr>
										<tr><td>Spermidine</td><td>0</td><td>0.0004583827288</td><td>0</td><td>0</td></tr>
										<tr><td>D-Fructose</td><td>0</td><td>0</td><td>0</td><td>0</td></tr>
										<tr><td>Pyruvate</td><td>0</td><td>0.004367024646</td><td>0</td><td>0.0005116099794</td></tr>
										<tr><td>Acetate</td><td>0</td><td>0.00734454145</td><td>0</td><td>0.0007674099084</td></tr>
										<tr><td>Potassium</td><td>-0.009508031033</td><td>-0.009478726281</td><td>-0.03275365187</td><td>-0.03273776072</td></tr>
										<tr><td>L-Proline</td><td>0</td><td>0.001775603427</td><td>0</td><td>0</td></tr>
										<tr><td>Citrate</td><td>0</td><td>0.002154398825</td><td>0</td><td>0.000255805484</td></tr>
										<tr><td>L-Leucine</td><td>0</td><td>0.001405042712</td><td>0</td><td>0</td></tr>
										<tr><td>Ferrous</td><td>-82.06087069</td><td>0.01653009055</td><td>-0.00144900646</td><td>-0.001448303442</td></tr>
										<tr><td>Cyanate</td><td>0</td><td>0</td><td>0</td><td>0</td></tr>
										<tr><td>Manganese</td><td>-0.00003365910169</td><td>-0.00003355550589</td><td>-0.000586352314</td><td>-0.0005860678322</td></tr>
										<tr><td>Hydrogen</td><td>0</td><td>0</td><td>0</td><td>0.02517654629</td></tr>
										<tr><td>H+ (Proton)</td><td>-81.61311372</td><td>0.4666523263</td><td>0</td><td>0.067976675</td></tr>
										<tr><td>Glycine</td><td>0</td><td>0.00619155825</td><td>0</td><td>0</td></tr>
										<tr><td>Magnesium</td><td>-0.0004225644492</td><td>-0.0004212648532</td><td>-0.005271671263</td><td>-0.005269789665</td></tr>
										<tr><td>Sulfate</td><td>-0.01566625436</td><td>-0.01222838389</td><td>-0.05225928733</td><td>-0.0514761878</td></tr>
										<tr><td>L-Glutamate</td><td>0</td><td>0.002244165443</td><td>0</td><td>0.0003069581424</td></tr>
										<tr><td>L-Histidine</td><td>0</td><td>0.00141117827</td><td>0</td><td>0</td></tr>
										<tr><td>Sodium</td><td>0</td><td>0</td><td>-0.0007330855771</td><td>-0.0007327299045</td></tr>
										<tr><td>D-Glucose</td><td>0</td><td>0</td><td>0</td><td>0</td></tr>
										<tr><td>Molybdate</td><td>-0.0000003428730538</td><td>-0.00000033992553</td><td>-0.0005864103878</td><td>-0.0005861258779</td></tr>
										<tr><td>Ammonium</td><td>-0.5348762349</td><td>-0.5244517244</td><td>0</td><td>0</td></tr>
										<tr><td>Oxygen (O2)</td><td>-21.97422024</td><td>-1.452113566</td><td>21.9447781</td><td>21.97422024</td></tr>
										<tr><td>Succinate</td><td>0</td><td>0.002992220591</td><td>0</td><td>0.0003837110398</td></tr>
										<tr><td>L-Serine</td><td>0</td><td>0.00375767237</td><td>0</td><td>0</td></tr>
										<tr><td>Nitrate</td><td>0</td><td>0</td><td>-2.449057669</td><td>-2.447438726</td></tr>
										<tr><td>Nickel</td><td>-0.00001573345878</td><td>-0.00001568513517</td><td>0</td><td>0</td></tr>
										<tr><td>Putrescine</td><td>0</td><td>0.001615799119</td><td>0</td><td>0</td></tr>
										<tr><td>Sucrose</td><td>-0.4133693967</td><td>-0.4124306025</td><td>0.4131517121</td><td>0.4133693967</td></tr>
										<tr><td>Urea </td><td>0</td><td>0.005212255223</td><td>0</td><td>0</td></tr>
										<tr><td>Phosphate</td><td>-0.04850780168</td><td>-0.04684202939</td><td>-0.06041753101</td><td>-0.06036884354</td></tr>
										<tr><td>Ferric</td><td>-0.0173100254</td><td>82.06008939</td><td>-0.001319437891</td><td>-0.001318797737</td></tr>
										<tr><td>L-Malate</td><td>0</td><td>0.003231598238</td><td>0</td><td>0.0003837110398</td></tr>
										<tr><td>Fumarate</td><td>0</td><td>0.003231598238</td><td>0</td><td>0.0003837110399</td></tr>
										<tr><td>Water (H20)</td><td>2.651858043</td><td>43.69752008</td><td>-13.54056988</td><td>-13.50658154</td></tr>
									</tbody>
								</table>
							</div>
							<br>

							<p>Results: Apart from sucrose, two other metabolites that were required to be given out by one organism and taken in by the other to grow optimally were carbon dioxide and oxygen. Carbon dioxide is given out by E coli and taken in by the cyanobacteria and is already an integral patt of the production process acting as the main carbon source for the entire process. Oxygen given out by the cyanobacteria could act as an important source for e coli, who prefer aerobic conditions for growth. </p>
							<p>Apart from these, there were other metabolites such as sulfate, magnesium, phosphate, etc that both bacteria require to be supplied in the medium. This would imply that a shortage of these nutrients could lead to competition between the two species and de-stability of the co culture. </p>
							<br>

							<h3>Varying sucrose productivity of the cyanobacteria</h3>  
							<br>

							<p>Hypothesis: Increasing sucrose productivity of cyanobacteria will result in the increase of butanol yield from the co culture.</p> 
							<br>

							<p>Currently according to the salt stress model, cyanobacteria is producing about 0.3 mmol of sucrose per hour per gDW. But we decided to vary the sucrose productivity of cyanobacteria and model its effects on the butanol yield of the co culture. This was done by changing the flux of sucrose given out by cyanobacteria per gDW.</p>
							<br>

							INSERT GRAPH 1,2,3 here
							<br>
							
							<div style="display:flex; flex-direction: column; justify-content:space-around;">
								<div style="margin: 10px; display:flex; flex-direction: row; flex-wrap: wrap; justify-content:space-around;"><img src="assets/Graphs/SteadyCom Graphs/Steadycomgraph1.png" width="400px" height="400px"><img src="assets/Graphs/SteadyCom Graphs/Steadycomgraph2.png" width="400px" height="400px"></div>
								<div style="margin: 10px; display:flex; justify-content:center;"><img src="assets/Graphs/SteadyCom Graphs/Steadycomgraph3.png" width="400px" height="400px"></div>
							</div>
							<br>

							<p>Observations: 
								<br>
								As the sucrose productivity of cyanobacteria increases:
								<ol class="a">
									<li style="padding-left:1em;padding-top: 0.3em;">The max growth rate of the co culture comes down.</li>
									<li style="padding-left:1em;padding-top: 0.3em;">The relative abundance of the cyanobacteria in the co culture goes down and that of the E coli goes up.</li>
									<li style="padding-left:1em;padding-top: 0.3em;">The butanol produced per gDW of the co culture initially increases but after an optimal value of sucrose, it comes down</li>
								</ol>
							</p>
							<br>

							<p>Conclusion: For current levels of oxygen and Co2 uptake rates, there exists an optimum of sucrose productivity from cyanobacteria which gives a maximum butanol yield per gDW of the co culture. This is due to the fact that as sucrose productivity of the cyanobacteria increases, its relative abundance in the co culture also comes down. So for very high values of sucrose productivity (which is per gDW of the cyanobacteria), there is very little cyanobacteria in the culture and hence not a lot of sucrose for the E. coli to take up and produce butanol. A more detailed explanation of this result is given in the appendix. </p> <br>
							

							<h3>Varying butanol productivity/ % of optimal growth rate for E coli</h3>
							<p>Hypothesis: Increase in the butanol productivity of E coli will result in an increase of the butanol yield of the co culture.</p>
							<p>Apart from the sucrose productivity of Cyanobacteria, the butanol productivity of E coli is another important factor that will determine the amount of butanol produced by the co culture. As per how induction by IPTG is simulated in the model, a higher butanol productivity is correlated with the e coli growing at a lower percentage of its optimal/maximum growth rate at the given conditions. In reality, increasing butanol productivity could be achieved by a higher IPTG concentration, or by increasing the efficiency of the butanol pathway. This would redirect carbon flux away from growth and towards producing butanol. </p>
							<br>
							
							Insert Graphs 7,8,9
							<br>
							
							<div style="margin: 10px; display:flex; justify-content:center;"><img src="assets/Graphs/SteadyCom Graphs/Steadycomgraph7.png" width="400px" height="400px"></div><br>
							<p>Observations: <br>
								As butanol productivity of E coli increases:
								<ol class="a">
									<li style="padding-left:1em;padding-top: 0.3em;">The growth rate of the co-culture remains the same</li>
									<li style="padding-left:1em;padding-top: 0.3em;">The butanol produced per gDW of the co culture increases</li>
									<li style="padding-left:1em;padding-top: 0.3em;">The value of optimum sucrose productivity did not vary significantly at different levels of butanol productivity</li>
								</ol>
							</p>
							<p>Conclusions:<br>The maximum specific growth rate of the co culture is primarily controlled by the cyanobacteria hence will majorly only depend on the Co2 uptake rate and sucrose productivity. When butanol productivity of the E coli was increased, its relative abundance in the co culture decreased. This means that since there was less E coli in the co culture, it could take up more sucrose per gDW such that it produced more butanol while still growing at the growth rate of the cyanobacteria.</p>
							<p>Increasing the butanol productivity of e coli does increase the butanol productivity of the co culture. This is as each the E coli are now diverting more flux into the butanol producing pathway and hence are converting a higher fraction of the sucrose given out by the cyanobacteria into butanol. However, this comes at the cost of decreasing relative abundance of E coli in the co culture. While running steadycom, it was not possible for the E coli to grow below around 50% of its optimal growth rate. From which we can infer that the butanol productivity cannot be arbitrarily increased as after a point the co culture would not be able to maintain a steady state. </p>
							
							<div style="margin: 10px; display:flex; flex-direction: row; flex-wrap: wrap; justify-content:space-around;">
								<img src="assets/Graphs/SteadyCom Graphs/Steadycomgraph8.png" width="400px" height="400px">
								<img src="assets/Graphs/SteadyCom Graphs/Steadycomgraph9.png" width="400px" height="400px">
							</div>
							<br>
					

							<h3>Varying Oxygen levels to test aerobic vs micro aerobic vs anaerobic</h3>
							<br>


							<p>Hypothesis: Decreasing oxygen levels will result in an increase of the butanol yield of the co culture.</p>
							<p>As seen in the FVA results above, oxygen was one of the important metabolites being exchanged by the cyanobacteria and E coli. It is to be noted that cyanobacteria are producing more oxygen than the E coli need and the co culture as a whole would have a positive production of oxygen (see appendix for more). This means that our simulations would be in an aerobic environment as far as the E coli are concerned. However, based on discussions with experts and FBA results for E coli, we realised that anaerobic or even microaerobic conditions might result in a higher butanol yield. Thus we decided to look at how lowering oxygen levels in the co culture would affect butanol yield. We did this by setting decreasing bounds to how much oxygen the E coli could take up per gDW.</p>
							
							
							<!-- SLIDER 5  -->
							<div class="rangeImage" id="rangeImage5">
							</div>
							<br><br>


							<div class="range-slider">
								<input type="range" min="1" max="6" step="1" value="1" class="myRange" id="myRange5">
							</div>
							<div class="rangeOutput" id="rangeValue5"></div><div class="rangeOutput">Butanol Productivity
							</div>
							<br><br>
							
							<h4>Image description:
								<b><div id="ImageDesc5">
								</div>
							</h4></b><br><br>
					
							<div style="margin: 10px; display:flex; flex-direction: row; flex-wrap: wrap; justify-content:space-around;"><img src="assets/Graphs/SteadyCom Graphs/Steadycomgraph11.png" width="400px" height="400px"><img src="assets/Graphs/SteadyCom Graphs/Steadycomgraph12.png" width="400px" height="400px"></div><br>
							<p>Observations: <br>
								As the oxygen levels in the co culture environment is brought down:
								<ol class="a">
									<li style="padding-left:1em;padding-top: 0.3em;">The maximum possible butanol yield of the co culture goes up</li>
									<li style="padding-left:1em;padding-top: 0.3em;">The sucrose productivity value for which butanol yield of the co culture is maximum initially decreases then increases </li>
								</ol>

								<p>However at very low oxygen levels, close to anaerobic conditions:</p>
								<ol class="a">
									<li style="padding-left:1em;padding-top: 0.3em;">The E coli cannot grow at low sucrose productivity of the cyanobacteria, including the current productivity predicted by our salt stress</li>
									<li style="padding-left:1em;padding-top: 0.3em;">For sucrose productivity in the range 0.3 - 0.5 mmol/hr/gDW, the butanol yields do not change much with a further decrease in oxygen levels</li>
								</ol>

								<ol class="a">
									<li style="padding-left:1em;padding-top: 0.3em;"></li>
									<li style="padding-left:1em;padding-top: 0.3em;"></li>
								</ol>
							</p>

							<div style="margin: 10px; display:flex; justify-content:center;">
								<img src="assets/Graphs/SteadyCom Graphs/Steadycomgraph13.png" width="400px" height="400px">
							</div><br>


							<p>Conclusions: 
								<br><br>
								Butanol yield does increase as the oxygen levels are brought down, however at very low levels of oxygen, the E coli is not able to sustain growth with the current levels of sucrose production from the cyanobacteria. Further, the butanol yield at low oxygen levels does not increase significantly unless sucrose productivity of the cyanobacteria is increased to a very large extent. <br>
								Thus we can say that using oxygen quenchers to bring the oxygen levels down is a good idea upto a certain extent. This suggests that a microaerobic condition would be best for the butanol yield. This matches what was told to us by Dr. Yazdani, who said that at aerobic conditions, the E coli would divert most of its flux towards growth and hence butanol yield would be low, whereas at anaerobic conditions the E coli would grow too slowly.
								<br>
								This is also supported by the fact that at very low oxygen levels, the E coli cannot grow below 90% of its maximum growth rate. The only way for the E coli to divert more flux towards butanol  would be to further lower its biomass to take up more sucrose. 
							</p>


							<h3>Increasing CO2 uptake rate of cyanobacteria</h3>
							<br>


							<p>Hypothesis: Increasing the Co2 uptake efficiency of cyanobacteria will increase the butanol yield of the co culture.</p>
							<p>In a meeting with Dr Malathy, Dr. Dube and Dr. Lobo, we were told to look into increasing the Co2 uptake efficiency of the cyanobacteria from a perspective of making our production process gaining advantage over others and interesting stakeholders. We decided to model the effect of this on the butanol yield as well. We expected that if CO2 uptake efficiency increases, cyanobacteria biomass and sucrose would be produced at a higher rate, resulting in butanol being produced at a higher rate as well. This was done by increasing the max Co2 allowed for the cyanobacteria to take in per hour.</p>
							<br>



							<!-- SLIDER 6 -->
							<div class="rangeImage" id="rangeImage6">
							</div>
							<br><br>


							<div class="range-slider">
								<input type="range" min="1" max="6" step="1" value="1" class="myRange"id="myRange6">
							</div>

							<div class="rangeOutput" id="rangeValue6"></div><div class="rangeOutput">
								Sucrose Productivity
							</div>
							<br><br>
							
							<h4>
								Image description:
								<b><div id="ImageDesc6">
								</div>
							</h4></b><br><br>
							
							<p>Observations:
								<ol class="a">
									<li style="padding-left:1em;padding-top: 0.3em;">As the Co2 uptake rate of the cyanobacteria is increased, the butanol yield of the co culture given in mmol/h/gDW increases</li>
									<li style="padding-left:1em;padding-top: 0.3em;">The co culture is stable at much higher rates of sucrose productivity of cyanobacteria</li>
								</ol>
							</p>
							

							<p>
								Conclusion: 
								<br>

								Increasing the Co2 uptake efficiency of cyanobacteria will result not only in more Co2 being captured by the co culture every hour, it will also increase the butanol being produced per hour by the co culture. This is as a higher CO2 uptake rate would result in a higher sucrose productivity (as is also predicted from FBA and MOMA) which would naturally result in an increased butanol yield.
							</p>
							<br>


							<h1 id="3">Dynamic Modelling</h1>
							<hr><br>

							<h3>Varying sucrose productivity of the cyanobacteria</h3>
							<br>
							
							<p>Hypothesis: As SteadyCom predicts there to be an optimum sucrose productivity which gives the maximum butanol yield, the dynamic model is expected to give similar results.<br><br>Observations: The dynamic simulation was performed at aerobic conditions for 60% butanol productivity of E. coli. Initially as sucrose productivity of cyanobacteria was increased, the total butanol yield also increased. However, when increasing sucrose productivity beyond 70%, the butanol yield starts decreasing. <br>
								<!-- SLIDER 2 VARYING SUCROSE PROD -->
								<div class="rangeImage" id="rangeImage2"></div><br><br>
								<div class="range-slider">
									<input type="range" min="1" max="8" step="1" value="1" class="myRange"id="myRange2">
								</div>
								<div class="rangeOutput" id="rangeValue2"></div><div class="rangeOutput">Sucrose Productivity</div><br>   
								<br>
								<h4>Image description:
									<b><div id="ImageDesc2">
									</div>
								</h4></b>
								<br><br>
								
								Conclusion: This can be explained because the total amount of sucrose produced is both proportional to the biomass of cyanobacteria and the sucrose productivity of the cyanobacteria. But, increasing the sucrose productivity, diverts carbon flux away from growth. This causes the biomass of cyanobacteria to grow slower, but each individual is producing sucrose more efficiently. Since such a trade-off exists, the optimum sucrose yield ends up being around 70% sucrose productivity of cyanobacteria, which in turn gives maximum butanol yield.
							</p>
							<br>


							<h3>Varying butanol productivity of E. coli</h3>
							<br>


							<p>Hypothesis: As butanol efficiency increases, total butanol yield will increase.
								<br><br>
								Observation: The dynamic simulation was done at 60% sucrose productivity from cyanobacteria, at aerobic conditions. The simulations suggest that as butanol productivity increases, varying from 10% to 95%, the butanol yield keeps increasing. <br>
								<!-- SLIDER 1  -->
								<div class="rangeImage" id="rangeImage1"></div><br><br>
								<div class="range-slider">
									<input type="range" min="1" max="8" step="1" value="1" class="myRange" id="myRange1">
								</div>
								<div class="rangeOutput" id="rangeValue1"></div><div class="rangeOutput">Butanol Productivity</div><br>   
								<br>
								<h4>Image description:
									<b><div id="ImageDesc1">
									</div>
								</h4></b><br><br>
								Conclusion: Methods to increase butanol productivity will help increase butanol yield. However, it is not biologically realistic to reach 95% butanol yield as shown in the simulation. This would divert too much flux away from growth of the E. coli, which is not feasible in real life. 
							</p>

							<h3>Varying Oxygen levels to test aerobic vs micro aerobic vs anaerobic</h3>
							<br>
							<p>Hypothesis: SteadyCom suggests that micro aerobic conditions are ideal for butanol productivity. The results are expected to match in the dynamic model as well.<br><br>
								Observation: The simulations were run at 60% productivity of sucrose and butanol aerobically. The oxygen uptake flux of E. coli was already well below the oxygen produced by the cyanobacteria. This is because at the sucrose flux available to the E. coli, high oxygen consumption was not required for optimal growth and productivity. <br>However, as we decrease the oxygen uptake of E. coli, we notice that initially butanol productivity increases. This could be explained by the fact that E. coli starts switching some of its flux to a different anaerobic pathway, where growth rate is low. This in turn implies that there is more carbon flux available to produce butanol efficiently. <br>This trend does not sustain as oxygen uptake is further decreased. This could be explained because as oxygen uptake is decreased, the growth rate of the E coli goes down significantly which brings down the total butanol yield.<br><br>
								<!-- SLIDER 3 -->
								<div class="rangeImage" id="rangeImage3"></div><br><br>
								<div class="range-slider">
									<input type="range" min="1" max="11" step="1" value="1" class="myRange" id="myRange3">
								</div>
								<div class="rangeOutput" id="rangeValue3"></div><div class="rangeOutput">Oxygen Uptake Rate</div><br>   
								<br>
								<h4>Image description:
									<b><div id="ImageDesc3"></div>
								</h4></b>
								<br><br>
								Conclusion: Micro aerobic conditions are the most efficient for butanol production in the given co-culture setting. 
							</p>


							<h3>Effect of CO2 </h3>
							<br>
							<p>Hypothesis: As co2 input increases in the bioreactor, the productivity of the entire consortium as a system is improved. Increasing the productivity of co2 sequestration by cyanobacteria should have similar effects.<br><br>
								Observation: As co2 input is increased, the butanol yield, as well as the growth rates of both the organisms increase.<br>Similar effects are observed as co2 uptake rate is increased in the cyanobacteria.<br><br>
								<!-- SLIDER 4 -->
								<div class="rangeImage" id="rangeImage4"></div><br><br>
								<div class="range-slider">
									<input type="range" min="1" max="10" step="1" value="1" class="myRange" id="myRange4">
								</div>
								<div class="rangeOutput" id="rangeValue4"></div><div class="rangeOutput">CO2 Uptake Rate</div><br>   
								<br>
								<h4>Image description:
									<b><div id="ImageDesc4"></div>
								</h4></b>
								<br><br>
								Conclusion: Increasing co2 uptake can be done in two ways - increasing the flow of input co2 into the bioreactor and increasing efficiency of the cyanobacteria to take up available co2. They are both good at increasing the yield of butanol.
								<br>
							</p>


							<h3>Summary of results Co-culture</h3><br>
							<p>Although the two models begin with slightly different assumptions, the analysis of the co-culture arrives at similar conclusions. This boosts confidence in the accuracy of the results. 
								<ol class="a">
									<li style="padding-left:1em;padding-top: 0.3em;">There exists an optimum sucrose productivity of the cyanobacteria for which the butanol yield of the co-culture is maximized.</li>
									<li style="padding-left:1em;padding-top: 0.3em;">Increasing the butanol productivity of the E. coli will increase the butanol yield of the co-culture.</li>
									<li style="padding-left:1em;padding-top: 0.3em;">The oxygen produced by cyanobacteria is in excess of the oxygen required by the E. coli in the co-culture.</li>
									<li style="padding-left:1em;padding-top: 0.3em;">Removing the oxygen produced by the cyanobacteria to provide micro-aerobic conditions for the E. coli to grow increases the butanol yield of the co-culture.</li>
									<li style="padding-left:1em;padding-top: 0.3em;">Optimizing the CO2 uptake by the cyanobacteria increases the butanol yield of the co-culture. </li>
								</ol>
							</p>


							<h1 id="4">Appendix</h1><hr><br>
							<h3>Simulating IPTG Induction for the production of Butanol</h3><br>
							<p>
								Similar to how it was done in FBA, IPTG was induced by setting a bound for E coli to grow at a certain percentage of its optimal growth rate and diverting flux through the butanol production pathway. However due to the nature of the SteadCom algorithm, this is not as simple to do as in FBA. One cannot optimize for butanol within steadycom as the objective function is set to be the community growth rate, and further the sucrose intake by E coli, on which the optimal growth rate depends, varies. The method we used to overcome the problem was the following:
								<ol class="a">
									<li style="padding-left:1em;padding-top: 0.3em;">SteadyCom is run on the joint model while simulating salt stress but not IPTG. The butanol production in this case is usually 0 as growth is being optimized</li>
									<li style="padding-left:1em;padding-top: 0.3em;">Sucrose intake by the E coli per gDW from the SteadyCom result is used as the lower bound for an FBA with the E coli model</li>
									<li style="padding-left:1em;padding-top: 0.3em;">Growth rate is set to a percent of optimal at this level of sucrose intake and butanol is optimized for using FBA</li>
									<li style="padding-left:1em;padding-top: 0.3em;">Butanol flux gotten from FBA in this manner is set as a lower bound in the joint model and SteadyCom is run once again</li>
									<li style="padding-left:1em;padding-top: 0.3em;">This would lead to the production of butanol even though community growth rate is still being optimized for</li>
									<li style="padding-left:1em;padding-top: 0.3em;">The true percent of optimal growth rate the E coli is growing at can be found by comparing the results from the final SteadyCom run to FBA</li>
									<li style="padding-left:1em;padding-top: 0.3em;">This can be done at different percentages of optimal growth (relating inversely to butanol productivity) as well as different levels of sucrose productivity, oxygen, etc.</li>
								</ol>

							</p>


							<h3>Detailed explanation for optimal sucrose productivity</h3><br>
							<p>For high values of sucrose productivity, the max specific growth rate values of the co culture are low. This is as the cyanobacteria cannot grow fast while producing high amounts of sucrose per gDW. In turn, due to the steady state conditions, the E coli is also constrained to grow at a low growth rate. (In reality, if the E coli do grow at a higher growth rate, they will quickly deplete the sucrose in the medium.) To maintain this condition, the relative abundance of the E coli in the co culture must be higher such that per gDW of E coli, there is not much sucrose available and the E coli is constrained to grow at a lower growth rate. <br>
								This makes sense biologically as well. Initially the E coli will grow fast and deplete the sucrose until it has to slow its growth rate down, but this results in it having a high abundance at steady state.<br><br>
								As sucrose productivity increases the actual butanol produced per gDW of E coli is decreasing, but its relative biomass in the co culture is increasing. These two competing effects lead to an optimum of sucrose productivity for the butanol productivity per gDW of the co culture. <br><br>
							</p>


							<!-- Slideshow of graphs-->
							<div id="carouselExampleCaptions" class="carousel slide" data-bs-ride="carousel">
								<div class="carousel-indicators" style="bottom:-50px;">
									<button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
									<button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="1" aria-label="Slide 2"></button>
									<button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="2" aria-label="Slide 3"></button>
									<button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="3" aria-label="Slide 4"></button>
									<button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="4" aria-label="Slide 5"></button>
									<button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="5" aria-label="Slide 6"></button>
								</div>
								<div style= "margin-bottom:50px; display:flex; flex-direction:row; align-items: center;" class="carousel-inner">
									<div class="carousel-item active graph"> 
										<img src="assets/graphs/SteadyCom Graphs/Steadycomgraph1.png" alt="" width=800>
									</div>
									<div class="carousel-item graph">
										<img src="assets/graphs/SteadyCom Graphs/Steadycomgraph2.png" alt="" width=800>
									</div>
									<div class="carousel-item graph">
										<img src="assets/graphs/SteadyCom Graphs/Steadycomgraph3.png" alt="" width=800>
									</div>
									<div class="carousel-item graph">
										<img src="assets/graphs/SteadyCom Graphs/Steadycomgraph4.png" alt="" width=800>
									</div>
									<div class="carousel-item graph">
										<img src="assets/graphs/SteadyCom Graphs/Steadycomgraph5.png" alt="" width=800>
									</div>
									<div class="carousel-item graph">
										<img src="assets/graphs/SteadyCom Graphs/Steadycomgraph6.png" alt="" width=800>
									</div>
								</div>
								<button class="carousel-control-prev" type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide="prev">
									<span class="carousel-control-prev-icon" aria-hidden="true"></span>
									<span class="visually-hidden">Previous</span>
								</button>
								<button class="carousel-control-next" type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide="next">
									<span class="carousel-control-next-icon" aria-hidden="true"></span>
									<span class="visually-hidden">Next</span>
								</button>
							</div>
							<div style="margin-top: 20px;display:flex; flex-direction:row; justify-content: center;">
							<h3 id="slidetext"></h3>
							</div>
							<h3>Allowing Oxygen into the Co culture</h3><br>
							<p>The two following graphs show net oxygen flux per gDW of the co culture in two different conditions. The first allows only exchange of oxygen between the two species but extra oxygen is not allowed from the environment. The second allows both exchange of oxygen as well as for the E coli to take up more oxygen from the environment if it requires as well. </p>
							<br>
						</div>

						<!-- SCROLL AWARE HIGHLIGHT -->

						<!-- END OF SCROLL AWARE HIGHLIGHT -->

						<!------------------------------------- END OF CO CULTURE TAB----------------------------------------->


					</div>
				</div>
				<!-- END OF CONTENT -->


			</div>
		</div>
	</div>

	

	<!-- -----------------------END OF CONTENT ----------------------------------- -->
	



	<!--------------------------------------------- SPONSORS BAR ------------------------------------------------------->
	<div class="footer">
		<div class="footer-wrapper">
			<!-- <div id="not-sponsors" class="footer-wrapper">	 -->
				<div id="Team", class="footer-wrapper">
					<img class="teamlogo" src="assets/Logos/IISER-Pune-iGEM_logo.png" alt="iGEM IISER Pune Logo" width="170">
				</div>
				<div id="socials" class="vertical-stack">
					<div><h3>Team IISER Pune India</h3></div>
					<div id="logos" class ="footer-wrapper ">
						<a href="https://www.instagram.com/igem_iiserp/"><img class="footer-links-socials" src="assets/Logos/Instagram_logo.jpg" alt="Instagram" width="20"></a>
						<a href="https://www.linkedin.com/in/igem-iiser-pune-2021/"><img class="footer-links-socials" src="assets/Logos/linkedin_logo.png" alt="Linkedin" width="30"></a>
						<a href="https://twitter.com/igem_iiserpune"><img class="footer-links-socials" src="assets/Logos/Twitter_logo.png" alt="Twitter" width="35"></a>
						<a href="https://www.facebook.com/igemiiserpune"><img class="footer-links-socials" src="assets/Logos/Facebook_logo.png" alt="Facebook" width="25"></a>
						<a href="https://www.youtube.com/channel/UCyo6NMLvM58sSh_tnxht36w"><img class="footer-links-socials" src="assets/Logos/Youtube_logo.png" alt="Youtube" width="35"></a>
					</div>
				</div>
			<!-- </div> -->
			<div id="sponsors" class="footer-wrapper">
				<div id="main-sponsor", class="">
					<a href="https://www.sailife.com/"><img class="footer-links" src="assets/Logos/Sai_logo.jpg" alt="Sai Life Sciences" width="280"></a>
				</div>
				<div class="vertical-stack">
					<a href="https://sg.idtdna.com/pages"><img class="footer-links" src="assets/Logos/IDT_logo.png" alt="IDT" width="220"></a>
					<a href="https://www.geneious.com/"><img class="footer-links" src="assets/Logos/Geneious_logo.png" alt="Geneious" width="220"></a>
				</div>
				<div class="vertical-stack">
					<div id="horizontal stack">
						<a href="https://www.notion.so/"><img class="footer-links" src="assets/Logos/Notion_logo.png" alt="Notion" width="70"></a>
						<img src="assets/Logos/IISER Pune Logo.png" alt="IISER Pune" width="80" style="padding-left: 1em;">
					</div>
					<a href="https://www.zoho.com/"><img class="footer-links" src="assets/Logos/Zoho_logo.png" alt="Zoho" width="150"></a>
				</div>
			</div>	
		</div>
	</div>
	<!--------------------------------------------- END OF SPONSOR BAR--------------------------------------------->

	
	<!--------------------------------------------- ACTUAL SCROLLING BAR--------------------------------------------->
	<div class="header"></div>
	<div class="line" id="scrollIndicator"></div>

	<!-- Gets the screenheight of each page for Scrollbar? -->
	<script>
		var screenheight = window.innerHeight,
		    screenwidth = window.innerWidth;

	</script>


	<!-- Scrollbar stuff again, only @Namasi22 knows :) -->
	<script>
		var body = document.body,
    		html = document.documentElement;

			var pageheight = Math.max(body.scrollHeight, body.offsetHeight, 
                       		   html.clientHeight, html.scrollHeight, html.offsetHeight);
	</script>
	

	<!-- Scrollbar stuff again, only @Namasi22 knows :) -->
	<script>
		var backtotopheight=pageheight/4.65
	</script>


	<!--------------------------------------------- back to top button --------------------------------------------->
	<a href="#top"><img src="assets/svgs/arrow-up-square-fill.svg" id="myBtn" title="Go to top" width="30" height="30"></a>

						
	<!--------------------------------------------- LINKING MAIN JS FILES--------------------------------------------->
	<script src="js/bootstrap-bundle.js"></script>
	<script src="js/main.js"></script>
	<script src="js/jquery.min.js"></script>

	
	<!----------------------------------------Sidenav list activation script -------------------------------------------------->
	<script>
		document.getElementById("myTab").addEventListener('click', sidebar);

		function sidebar(){
			if($('#cyano').hasClass('active')) {
				$('#ecoliside').css({visibility: "hidden"});
				$('#cocultureside').css({visibility: "hidden"});
				$('#overviewside').css({visibility: "hidden"});
				$('#cyanoside').css({visibility: "visible"});
				console.log('cyano')
			}
			else if($('#ecoli').hasClass('active')) {
				$('#cyanoside').css({visibility: "hidden"});
				$('#cocultureside').css({visibility: "hidden"});
				$('#overviewside').css({visibility: "hidden"});
				$('#ecoliside').css({visibility: "visible"});
				console.log('cyanoq')
			}
			else if($('#Coculture').hasClass('active')) {
				$('#cyanoside').css({visibility: "hidden"});
				$('#ecoliside').css({visibility: "hidden"});
				$('#overviewside').css({visibility: "hidden"});
				$('#cocultureside').css({visibility: "visible"});
				console.log('cyanof')
			}
			else if($('#overview').hasClass('active')) {
				$('#ecoliside').css({visibility: "hidden"});
				$('#cocultureside').css({visibility: "hidden"});
				$('#cyanoside').css({visibility: "hidden"});
				$('#overviewside').css({visibility: "visible"});
				console.log('cyawno')
			}
		}
	</script>
	<!----------------------------------------- End of Sidenav list activation script --------------------------------------->
	
	
	
	<!--------------------------------------------- Script for the slideshow, wherever needed --------------------------------------------->	
	<script>
		var total = $('.item').length;
		var actualtotal = 4;
		var currentIndex = $('div.active').index() + 1;
		$('#slidetext').html(currentIndex + '/'  + actualtotal);

		// This triggers after each slide change
		$('.carousel').on('slid.bs.carousel', function () {
  			currentIndex = $('div.active').index() + 1;
 
  		// Now display this wherever you want
  		var text = currentIndex + '/' + actualtotal;
  			$('#slidetext').html(text);
		});
	</script>


	<!----------------------------------------- SLIDERS FOR CO CULTURE --------------------------------------------->
	
	<!-- slider 1 -->
	<script>
		var slider1 = document.getElementById("myRange1");
		var output1 = document.getElementById("rangeValue1");
		var slideimage1 = document.getElementById("rangeImage1");
		var imageDesc1 = document.getElementById("ImageDesc1");
		const imagearray1 = ['95','90','80','70','60','40','20','10'];
		const descarray1 = ['me','me when','me when the','poopoo','peepee','6','7','8'];
		const igemarraybut1 = ['https://2021.igem.org/wiki/images/5/5e/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--butanol--10_.png', 'https://2021.igem.org/wiki/images/f/f7/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--butanol--20_.png', 'https://2021.igem.org/wiki/images/d/d3/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--butanol--40_.png', 'https://2021.igem.org/wiki/images/6/6a/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--butanol--60_.png', 'https://2021.igem.org/wiki/images/1/1a/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--butanol--70_.png', 'https://2021.igem.org/wiki/images/0/05/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--butanol--80_.png', 'https://2021.igem.org/wiki/images/0/0e/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--butanol--90_.png', 'https://2021.igem.org/wiki/images/f/f5/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--butanol--95_.png'];
		const igemarraycoc1 = ['https://2021.igem.org/wiki/images/3/30/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--culture--10_.png', 'https://2021.igem.org/wiki/images/1/1a/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--culture--20_.png', 'https://2021.igem.org/wiki/images/d/d2/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--culture--40_.png', 'https://2021.igem.org/wiki/images/2/21/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--culture--60_.png', 'https://2021.igem.org/wiki/images/1/1d/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--culture--70_.png', 'https://2021.igem.org/wiki/images/1/18/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--culture--80_.png', 'https://2021.igem.org/wiki/images/9/9b/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--culture--90_.png', 'https://2021.igem.org/wiki/images/5/54/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--culture--95_.png'];
		const igemarraysuc1 = ['https://2021.igem.org/wiki/images/6/6a/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--sucrose--10_.png', 'https://2021.igem.org/wiki/images/8/8a/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--sucrose--20_.png', 'https://2021.igem.org/wiki/images/1/1e/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--sucrose--40_.png', 'https://2021.igem.org/wiki/images/7/79/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--sucrose--60_.png', 'https://2021.igem.org/wiki/images/e/e9/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--sucrose--70_.png', 'https://2021.igem.org/wiki/images/6/65/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--sucrose--80_.png', 'https://2021.igem.org/wiki/images/b/bd/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--sucrose--90_.png', 'https://2021.igem.org/wiki/images/8/8f/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_butanol_productivity--sucrose--95_.png'];
		const correctorder1 = ['8', '7', '6', '5', '4', '3', '2', '1'];
		output1.innerHTML = slider1.value;
		output1.innerHTML = '95';
		imageDesc1.innerHTML = 'me'
	  	slideimage1.innerHTML = '<div><img src="assets/Graphs/Dynamic%20Graphs/co2/input%20to%20bioreactor/culture/181.png" width="450px" height="450px"></div>' + '<div class="rangeImageRow2"><img src="assets/Graphs/Dynamic%20Graphs/co2/input%20to%20bioreactor/sucrose/181.png" width="450px" height="450px">' + '<img src="assets/Graphs/Dynamic%20Graphs/co2/input%20to%20bioreactor/butanol/181.png" width="450px" height="450px"></div>' + '"assets/Graphs/Dynamic%20Graphs/co2/input%20to%20bioreactor/sucrose/181.png"';
		
		
		slider1.oninput = function() {
	  		output1.innerHTML = imagearray1[this.value - 1] + '%';
	  		imageDesc1.innerHTML = igemarraycoc1[correctorder1[this.value-1]-1]
	  		slideimage1.innerHTML = '<div><img src="' + igemarraycoc1[correctorder1[this.value-1]-1] + '" width="450px" height="450px"></div>' + '<div class="rangeImageRow2"><img src="' + igemarraysuc1[correctorder1[this.value-1]-1] + '" width="450px" height="450px">' + '<img src="' + igemarraybut1[correctorder1[this.value-1]-1] + '" width="450px" height="450px"></div>' + 'img src="' + igemarraysuc1[correctorder1[this.value-1]-1] + '" width="450px" height="450px"';
		}
	</script>



	<!-- slider 2 -->
	<script>
		var slider2 = document.getElementById("myRange2");
		var output2 = document.getElementById("rangeValue2");
		var slideimage2 = document.getElementById("rangeImage2");
		var imageDesc2 = document.getElementById("ImageDesc2");
		const imagearray2 = ['95','90','80','70','60','40','20','10'];
		const descarray2 = ['me','me when','me when the','poopoo','peepee','6','7','8'];
		const igemarraybut2 = ['https://2021.igem.org/wiki/images/7/78/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--butanol--10_.png', 'https://2021.igem.org/wiki/images/9/9e/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--butanol--20_.png', 'https://2021.igem.org/wiki/images/2/2b/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--butanol--40_.png', 'https://2021.igem.org/wiki/images/5/5a/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--butanol--60_.png', 'https://2021.igem.org/wiki/images/2/21/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--butanol--70_.png', 'https://2021.igem.org/wiki/images/a/a8/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--butanol--80_.png', 'https://2021.igem.org/wiki/images/8/82/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--butanol--90_.png', 'https://2021.igem.org/wiki/images/d/df/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--butanol--95_.png'];
		const igemarraycoc2 = ['https://2021.igem.org/wiki/images/6/6f/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--culture--10_.png', 'https://2021.igem.org/wiki/images/c/c6/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--culture--20_.png', 'https://2021.igem.org/wiki/images/0/07/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--culture--40_.png', 'https://2021.igem.org/wiki/images/1/1e/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--culture--60_.png', 'https://2021.igem.org/wiki/images/6/63/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--culture--70_.png', 'https://2021.igem.org/wiki/images/8/89/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--culture--80_.png', 'https://2021.igem.org/wiki/images/7/72/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--culture--90_.png', 'https://2021.igem.org/wiki/images/0/0a/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--culture--95_.png'];
		const igemarraysuc2 = ['https://2021.igem.org/wiki/images/a/a3/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--sucrose--10_.png', 'https://2021.igem.org/wiki/images/5/57/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--sucrose--20_.png', 'https://2021.igem.org/wiki/images/2/24/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--sucrose--40_.png', 'https://2021.igem.org/wiki/images/f/fc/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--sucrose--60_.png', 'https://2021.igem.org/wiki/images/e/ef/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--sucrose--70_.png', 'https://2021.igem.org/wiki/images/4/4d/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--sucrose--80_.png', 'https://2021.igem.org/wiki/images/e/e7/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--sucrose--90_.png', 'https://2021.igem.org/wiki/images/1/1b/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_sucrose_productivity--sucrose--95_.png'];
		const correctorder2 = ['8', '7', '6', '5', '4', '3', '2', '1'];
		output2.innerHTML = slider2.value;
		output2.innerHTML = '95%';
		imageDesc2.innerHTML = 'me'
		slideimage2.innerHTML = '<div><img src="assets/Graphs/Dynamic%20Graphs/co2/input%20to%20bioreactor/culture/181.png" width="450px" height="450px"></div>' + '<div class="rangeImageRow2"><img src="assets/Graphs/Dynamic%20Graphs/co2/input%20to%20bioreactor/sucrose/181.png" width="450px" height="450px">' + '<img src="assets/Graphs/Dynamic%20Graphs/co2/input%20to%20bioreactor/butanol/181.png" width="450px" height="450px"></div>' + '"assets/Graphs/Dynamic%20Graphs/co2/input%20to%20bioreactor/sucrose/181.png"';
		
		slider2.oninput = function() {
			output2.innerHTML = imagearray2[this.value - 1] + '%';
			imageDesc2.innerHTML = igemarraycoc2[correctorder2[this.value-1]-1]
			slideimage2.innerHTML = '<div><img src="' + igemarraycoc2[correctorder2[this.value-1]-1] + '" width="450px" height="450px"></div>' + '<div class="rangeImageRow2"><img src="' + igemarraysuc2[correctorder2[this.value-1]-1] + '" width="450px" height="450px">' + '<img src="' + igemarraybut2[correctorder2[this.value-1]-1] + '" width="450px" height="450px"></div>' + 'img src="' + igemarraysuc2[correctorder2[this.value-1]-1] + '" width="450px" height="450px"';
		}
	</script>



	<!-- slider 3 -->
	<script>
		var slider3 = document.getElementById("myRange3");
		var output3 = document.getElementById("rangeValue3");
		var slideimage3 = document.getElementById("rangeImage3");
		var imageDesc3 = document.getElementById("ImageDesc3");
		const imagearray3 = ['-4','-3','-2','-1.75','-1.25','-1.5','-1','-0.75','-0.5','-0.25','0'];
		const descarray3 = ['me','me when','me when the','poopoo','peepee','6','7','8','0','0','0','0','0','0'];
		const igemarraybut3 = ['https://2021.igem.org/wiki/images/f/f0/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--butanol---0.25.png', 'https://2021.igem.org/wiki/images/7/7b/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--butanol---0.5.png', 'https://2021.igem.org/wiki/images/c/c3/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--butanol---0.75.png', 'https://2021.igem.org/wiki/images/1/15/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--butanol---1.25.png', 'https://2021.igem.org/wiki/images/8/84/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--butanol---1.5.png', 'https://2021.igem.org/wiki/images/c/c8/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--butanol---1.75.png', 'https://2021.igem.org/wiki/images/9/93/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--butanol---1.png', 'https://2021.igem.org/wiki/images/9/98/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--butanol---2.png', 'https://2021.igem.org/wiki/images/8/8b/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--butanol---3.png', 'https://2021.igem.org/wiki/images/0/07/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--butanol---4.png', 'https://2021.igem.org/wiki/images/4/4b/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--butanol--0.png'];
		const igemarraycoc3 = ['https://2021.igem.org/wiki/images/a/a7/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--culture---0.25.png', 'https://2021.igem.org/wiki/images/0/06/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--culture---0.5.png', 'https://2021.igem.org/wiki/images/8/87/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--culture---0.75.png', 'https://2021.igem.org/wiki/images/8/82/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--culture---1.25.png', 'https://2021.igem.org/wiki/images/a/a4/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--culture---1.5.png', 'https://2021.igem.org/wiki/images/6/61/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--culture---1.75.png', 'https://2021.igem.org/wiki/images/e/e9/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--culture---1.png', 'https://2021.igem.org/wiki/images/c/c4/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--culture---2.png', 'https://2021.igem.org/wiki/images/a/ac/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--culture---3.png', 'https://2021.igem.org/wiki/images/5/51/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--culture---4.png', 'https://2021.igem.org/wiki/images/5/50/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--culture--0.png'];
		const igemarraysuc3 = ['https://2021.igem.org/wiki/images/c/cf/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--sucrose---0.25.png', 'https://2021.igem.org/wiki/images/2/24/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--sucrose---0.5.png', 'https://2021.igem.org/wiki/images/0/0a/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--sucrose---0.75.png', 'https://2021.igem.org/wiki/images/7/7e/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--sucrose---1.25.png', 'https://2021.igem.org/wiki/images/f/f5/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--sucrose---1.5.png', 'https://2021.igem.org/wiki/images/a/a2/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--sucrose---1.75.png', 'https://2021.igem.org/wiki/images/7/72/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--sucrose---1.png', 'https://2021.igem.org/wiki/images/b/b0/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--sucrose---2.png', 'https://2021.igem.org/wiki/images/4/49/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--sucrose---3.png', 'https://2021.igem.org/wiki/images/8/82/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--sucrose---4.png', 'https://2021.igem.org/wiki/images/a/ac/T--IISER-Pune-India--Graphs--Dynamic_Graphs--varying_o2_uptake--sucrose--0.png'];
		const correctorder3 = ['10', '9', '8', '6', '5', '4', '7', '3', '2', '1', '11'];
		output3.innerHTML = slider3.value;
		output3.innerHTML = '-4';
		imageDesc3.innerHTML = 'me'
		slideimage3.innerHTML = '<div><img src="assets/Graphs/Dynamic%20Graphs/co2/input%20to%20bioreactor/culture/181.png" width="450px" height="450px"></div>' + '<div class="rangeImageRow2"><img src="assets/Graphs/Dynamic%20Graphs/co2/input%20to%20bioreactor/sucrose/181.png" width="450px" height="450px">' + '<img src="assets/Graphs/Dynamic%20Graphs/co2/input%20to%20bioreactor/butanol/181.png" width="450px" height="450px"></div>' + '"assets/Graphs/Dynamic%20Graphs/co2/input%20to%20bioreactor/sucrose/181.png"';
		
		
		slider3.oninput = function() {
		  output3.innerHTML = imagearray3[this.value - 1] + '%';
		  imageDesc3.innerHTML = igemarraycoc3[correctorder3[this.value-1]-1]
		  slideimage3.innerHTML = '<div><img src="' + igemarraycoc3[correctorder3[this.value-1]-1] + '" width="450px" height="450px"></div>' + '<div class="rangeImageRow2"><img src="' + igemarraysuc3[correctorder3[this.value-1]-1] + '" width="450px" height="450px">' + '<img src="' + igemarraybut3[correctorder3[this.value-1]-1] + '" width="450px" height="450px"></div>' + 'img src="' + igemarraysuc3[correctorder3[this.value-1]-1] + '" width="450px" height="450px"';
		}
	</script>



	<!-- slider 4 -->
	<script>
		var slider4 = document.getElementById("myRange4");
		var output4 = document.getElementById("rangeValue4");
		var slideimage4 = document.getElementById("rangeImage4");
		var imageDesc4 = document.getElementById("ImageDesc4");
		const imagearray4 = ['181','121','101','81','61','41','31','21','11','5'];
		const descarray4 = ['me','me when','me when the','poopoo','peepee','6','7','8'];
		const igemarraybut4 = ['https://2021.igem.org/wiki/images/9/97/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--butanol--101.png', 'https://2021.igem.org/wiki/images/1/1e/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--butanol--11.png', 'https://2021.igem.org/wiki/images/3/35/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--butanol--121.png', 'https://2021.igem.org/wiki/images/6/6c/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--butanol--181.png', 'https://2021.igem.org/wiki/images/d/d5/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--butanol--21.png', 'https://2021.igem.org/wiki/images/1/12/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--butanol--31.png', 'https://2021.igem.org/wiki/images/3/32/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--butanol--41.png', 'https://2021.igem.org/wiki/images/0/05/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--butanol--5.png', 'https://2021.igem.org/wiki/images/f/f5/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--butanol--61.png', 'https://2021.igem.org/wiki/images/9/9f/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--butanol--81.png'];
		const igemarraycoc4 = ['https://2021.igem.org/wiki/images/2/2e/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--culture--101.png', 'https://2021.igem.org/wiki/images/c/c1/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--culture--11.png', 'https://2021.igem.org/wiki/images/b/bc/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--culture--121.png', 'https://2021.igem.org/wiki/images/f/f1/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--culture--181.png', 'https://2021.igem.org/wiki/images/a/ac/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--culture--21.png', 'https://2021.igem.org/wiki/images/8/8d/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--culture--31.png', 'https://2021.igem.org/wiki/images/2/21/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--culture--41.png', 'https://2021.igem.org/wiki/images/5/51/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--culture--5.png', 'https://2021.igem.org/wiki/images/5/50/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--culture--61.png', 'https://2021.igem.org/wiki/images/d/d1/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--culture--81.png'];
		const igemarraysuc4 = ['https://2021.igem.org/wiki/images/7/7c/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--sucrose--101.png', 'https://2021.igem.org/wiki/images/5/5a/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--sucrose--11.png', 'https://2021.igem.org/wiki/images/4/4b/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--sucrose--121.png', 'https://2021.igem.org/wiki/images/f/f0/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--sucrose--181.png', 'https://2021.igem.org/wiki/images/9/94/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--sucrose--21.png', 'https://2021.igem.org/wiki/images/9/91/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--sucrose--31.png', 'https://2021.igem.org/wiki/images/1/13/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--sucrose--41.png', 'https://2021.igem.org/wiki/images/6/6b/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--sucrose--5.png', 'https://2021.igem.org/wiki/images/8/81/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--sucrose--61.png', 'https://2021.igem.org/wiki/images/7/7d/T--IISER-Pune-India--Graphs--Dynamic_Graphs--co2--input_to_bioreactor--sucrose--81.png'];
		const correctorder4 = ['4', '3', '1', '10', '9', '7', '6', '5', '2', '8'];
		output4.innerHTML = slider4.value;
		output4.innerHTML = '181';
		imageDesc4.innerHTML = 'me'
		slideimage4.innerHTML = '<div><img src="assets/Graphs/Dynamic%20Graphs/co2/input%20to%20bioreactor/culture/181.png" width="450px" height="450px"></div>' + '<div class="rangeImageRow2"><img src="assets/Graphs/Dynamic%20Graphs/co2/input%20to%20bioreactor/sucrose/181.png" width="450px" height="450px">' + '<img src="assets/Graphs/Dynamic%20Graphs/co2/input%20to%20bioreactor/butanol/181.png" width="450px" height="450px"></div>' + '"assets/Graphs/Dynamic%20Graphs/co2/input%20to%20bioreactor/sucrose/181.png"';
		slider4.oninput = function() {
		output4.innerHTML = imagearray4[this.value - 1] + '%';
		imageDesc4.innerHTML = igemarraycoc4[correctorder4[this.value-1]-1]
		slideimage4.innerHTML = '<div><img src="' + igemarraycoc4[correctorder4[this.value-1]-1] + '" width="450px" height="450px"></div>' + '<div class="rangeImageRow2"><img src="' + igemarraysuc4[correctorder4[this.value-1]-1] + '" width="450px" height="450px">' + '<img src="' + igemarraybut4[correctorder4[this.value-1]-1] + '" width="450px" height="450px"></div>' + 'img src="' + igemarraysuc4[correctorder4[this.value-1]-1] + '" width="450px" height="450px"';
		}
	</script>



	<!-- slider 5 -->
	<script>
		var slider5 = document.getElementById("myRange5");
		var output5 = document.getElementById("rangeValue5");
		var slideimage5 = document.getElementById("rangeImage5");
		var imageDesc5 = document.getElementById("ImageDesc5");
		const imagearray5 = ['95','90','80','70','60','40','20','10'];
		const descarray5 = ['me','me when','me when the','poopoo','peepee','6','7','8'];
		const igemarray5 = ['https://2021.igem.org/wiki/images/5/5a/T--IISER-Pune-India--Graphs--SteadyCom_Graphs--Steadycomgraph10_aaerobic.png', 'https://2021.igem.org/wiki/images/5/5b/T--IISER-Pune-India--Graphs--SteadyCom_Graphs--Steadycomgraph10_b8.png', 'https://2021.igem.org/wiki/images/7/7c/T--IISER-Pune-India--Graphs--SteadyCom_Graphs--Steadycomgraph10_c4.png', 'https://2021.igem.org/wiki/images/5/55/T--IISER-Pune-India--Graphs--SteadyCom_Graphs--Steadycomgraph10_d2.png', 'https://2021.igem.org/wiki/images/7/7b/T--IISER-Pune-India--Graphs--SteadyCom_Graphs--Steadycomgraph10_e1.png', 'https://2021.igem.org/wiki/images/b/b7/T--IISER-Pune-India--Graphs--SteadyCom_Graphs--Steadycomgraph10_f0.png'];
		const correctorder5 = ['1', '2', '3', '4', '5', '6'];
		output5.innerHTML = slider5.value;
		output5.innerHTML = '1';
		imageDesc5.innerHTML = 'me'
	  	slideimage5.innerHTML = '<div><img src="https://2021.igem.org/wiki/images/5/5a/T--IISER-Pune-India--Graphs--SteadyCom_Graphs--Steadycomgraph10_aaerobic.png" width="450px" height="450px"></div>';
	

		slider5.oninput = function() {
	  		output5.innerHTML = imagearray5[this.value - 1] + '%';
	  		imageDesc5.innerHTML = igemarray5[this.value-1];
	  		slideimage5.innerHTML = '<div><img src="' + igemarray5[this.value-1] + '" width="450px" height="450px"></div>';
		}
	</script>



	<!-- slider 6 -->
	<script>
		var slider6 = document.getElementById("myRange6");
		var output6 = document.getElementById("rangeValue6");
		var slideimage6 = document.getElementById("rangeImage6");
		var imageDesc6 = document.getElementById("ImageDesc6");
		const imagearray6 = ['95','90','80','70','60','40','20','10'];
		const descarray6 = ['me','me when','me when the','poopoo','peepee','6','7','8'];
		const igemarray6 = ['https://2021.igem.org/wiki/images/4/46/T--IISER-Pune-India--Graphs--SteadyCom_Graphs--Steadycomgraph14_a.png', 'https://2021.igem.org/wiki/images/c/cb/T--IISER-Pune-India--Graphs--SteadyCom_Graphs--Steadycomgraph14_b.png', 'https://2021.igem.org/wiki/images/6/62/T--IISER-Pune-India--Graphs--SteadyCom_Graphs--Steadycomgraph14_c.png', 'https://2021.igem.org/wiki/images/a/a3/T--IISER-Pune-India--Graphs--SteadyCom_Graphs--Steadycomgraph14_d.png', 'https://2021.igem.org/wiki/images/2/25/T--IISER-Pune-India--Graphs--SteadyCom_Graphs--Steadycomgraph14_e.png', 'https://2021.igem.org/wiki/images/a/af/T--IISER-Pune-India--Graphs--SteadyCom_Graphs--Steadycomgraph14_f.png'];
		const correctorder6 = ['1', '2', '3', '4', '5', '6'];
		output6.innerHTML = slider6.value;
		output6.innerHTML = '1';
		imageDesc6.innerHTML = 'me'
		slideimage6.innerHTML = '<div><img src="https://2021.igem.org/wiki/images/4/46/T--IISER-Pune-India--Graphs--SteadyCom_Graphs--Steadycomgraph14_a.png" width="450px" height="450px"></div>';
		
		
		slider6.oninput = function() {
			output6.innerHTML = imagearray6[this.value - 1] + '%';
			imageDesc6.innerHTML = igemarray6[this.value-1];
			slideimage6.innerHTML = '<div><img src="' + igemarray6[this.value-1] + '" width="450px" height="450px"></div>';
		}
	</script>


</body>






