

<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Experiments</title>
	<link rel="stylesheet" href="css/igem-reset.css">
	<link rel="stylesheet" href="css/SBcss/style.css">
   	<link rel="stylesheet" href="css/styleSB.css">
    	<link rel="stylesheet" href="css/DryLabPage.css">
	<style>/* Reset CSS */

		#sideMenu, #top_title, .patrollink, #firstHeading, #home_logo, #sideMenu { display:none; }
		#content { padding:0px; width:100%; margin-top:-7px; margin-left:0px; border:none;}
		body, html {background-color:#212529; width: 100%; height: 100%;}
		#bodyContent h1, #bodyContent h2, #bodyContent h3, #bodyContent h4, #bodyContent h5 { margin-bottom: 0px; }
		#bodyContent a[href ^="https://"], .link-https { padding-right:0px;}

		
    </style>
<body>
    

	
	<style>
		#pageheader{
			font-size: 40vmin !important;
		}
	</style>

	<!-- NAVBAR SECTION -->

	<nav class="navbar fixed-top navbar-expand-lg navbar-light menu">
		<div class="container-fluid nav-container">
		  <a class="navbar-brand" href="index.html">
			<img src="assets/svgs/synbactory2.svg">
		  </a>
		  <div class="center-expand">
		</div>
		  <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		  <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
		    <ul class="navbar-nav">
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Team
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Team.html">Members</a></li>
			  <li><a class="dropdown-item" href="Attributions.html">Attributions</a></li>
			  <li><a class="dropdown-item" href="Collaborations.html">Collaborations</a></li>
			  <li><a class="dropdown-item" href="Partnership.html">Partnership</a></li>
			  <li><a class="dropdown-item" href="Sponsors.html">Sponsors</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Project
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Description.html">Description</a></li>
			  <li><a class="dropdown-item" href="Design.html">Design</a></li>
			  <li><a class="dropdown-item" href="https://possible-trapezoid-297.notion.site/11e9906b405543a29361e38a50c81024?v=af10ed5ac15145e8a267a68ed8eb2005">Notebook</a></li>
			  <li><a class="dropdown-item" href="Contribution.html">Contributions</a></li>
			  <li><a class="dropdown-item" href="Engineering.html">Engineering Success</a></li>
			  <li><a class="dropdown-item" href="Implementation.html">Future Implementation</a></li>
			  <li><a class="dropdown-item" href="Safety.html">Safety</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Wet Lab
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Experiments.html">Experiments</a></li>
			  <li><a class="dropdown-item" href="Constructs.html">Constructs</a></li>
			  <li><a class="dropdown-item" href="Measurement.html">Measurement</a></li>
			  <li><a class="dropdown-item" href="LabNotebook.html">Lab Notebook</a></li>
			  <li><a class="dropdown-item" href="Results.html">Results</a></li>
			  <li><a class="dropdown-item" href="FutureWork.html">Future Work</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Parts
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <!-- <li><a class="dropdown-item" href="#">Overview</a></li> -->
			  <li><a class="dropdown-item" href="BasicParts.html">Basic</a></li>
			  <li><a class="dropdown-item" href="CompositeParts.html">Composite</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Dry Lab
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Model.html">Overview</a></li>
			  <li><a class="dropdown-item" href="EColi.html"><i>E. coli</i></a></li>
			  <li><a class="dropdown-item" href="Cyanobacteria.html">Cyanobacteria</a></li>
			  <li><a class="dropdown-item" href="Coculture.html">Coculture</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Human Practices
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Human_Practices.html">Integrated Human Practices</a></li>
			  <li><a class="dropdown-item" href="Inclusivity.html">Inclusivity and Accessibility</a></li>
			  <li><a class="dropdown-item" href="Communication.html">Public Engagement and Education</a></li>
			</ul>
		      </li>
		      <li class="nav-item">
			<a class="nav-link" href="https://igem.org/2021_Judging_Form?id=3971">Judging Form</a>
		      </li>
		      <!-- <button type="button" class="rounded-pill btn-rounded">Settings</button> -->
		    </ul>
		  </div>
		</div>
	</nav>



    <!-- TITLE SECTION -->
    
    <div class="section-1" >
        <h1 id="pageheader"><br><br><br><br>Experiments</h1>
    </div>
        


    <!-- WAVES SPACER -->

    <div class="spacer layer1"></div>
	<a id="top"></a>



    <!-- CONTENT + SIDENAV SECTION -->

	<!-- STYLING FOR FONTS -->
	<style>
		.content h1, .content h2, .content h4, .content h5, .content h6{
			color: white !important;
			font-family: ProximaNova !important;
		}

		.content h3 {
			color: #ffd73a !important;
			font-family: ProximaNova !important;
		}
		#reference {
			color: #fff !important;
		}

		.sidenav {
			height: auto;
			width: auto;
			padding-top: 150px;
			padding-left: 5px;
			padding-right: 10px;
		
			margin-top: 25%;
			margin-bottom: 20px;
			position: -webkit-sticky;
			position: sticky;
			top: 9.5% ;
		}
	</style>

	
	<!-- END OF STYLING FOR FONTS -->

    <div class="second-div container-fluid">
        <div class="row">


			<!-- SIDE NAV BAR -->
			<div class="col-lg-3 col-md-3">

				<div class="tab-content sidenav" id="SidenavTabs" style="top: -28%">

					<!-- overview Sidebar -->
					<div class="tab-pane fade show active overview-wet" id="overview-wet-side" role="tabpanel" aria-labelledby="overview-wet-tab"> 
						
						<div class="sidenav">
							<nav>
								<ul id="sidenav-id">
								</ul>
							</nav>
						</div>
					</div>



					<!-- Ecoli sidebar-->
					<div class="tab-pane fade ecoli-wet" id="ecoli-wet-side" role="tabpanel" aria-labelledby="ecoli-wet-tab"> 
						
						<div class="sidenav">
							<nav>
								<ul id="sidenav-id">
									<li class="nav-item"><a class="nav-link" href="#e1">Glycerol stocks of KJK01</a></li>
									<li class="nav-item"><a class="nav-link" href="#e2">Genomic DNA extraction</a></li>
									<li class="nav-item"><a class="nav-link" href="#e3">Confirmation of butanol producing genes</a></li>
									<li class="nav-item"><a class="nav-link" href="#e4">Gradient PCR</a></li>
									<li class="nav-item"><a class="nav-link" href="#e5">NMR confirmation of butanol production</a></li>
									<li class="nav-item"><a class="nav-link" href="#e6">Making KJK01 cells electrocompetent</a></li>
									<li class="nav-item"><a class="nav-link" href="#e7">Transformation of KJK01 with pCSCX plasmid</a></li>
									<li class="nav-item"><a class="nav-link" href="#e8">Glycerol stocks of pCSCX-KJK01</a></li>
									<li class="nav-item"><a class="nav-link" href="#e9">MIC Assay for KJK01</a></li>
									<li class="nav-item"><a class="nav-link" href="#e10">Growth curve in LB</a></li>
									<li class="nav-item"><a class="nav-link" href="#e11">Sucrose Assay</a></li>
								</ul>
							</nav>
						</div>
					</div>
						
					<!-- Cyano Sidebar-->
					<div class="tab-pane fade cyano-wet" id="cyano-wet-side" role="tabpanel" aria-labelledby="cyano-wet-tab"> 
						
						<div class="sidenav">
							<nav>
								<ul id="sidenav-id">
									<li class="nav-item"><a class="nav-link" href="#c1">Preparation of BG-11</a></li>
									<li class="nav-item"><a class="nav-link" href="#c2">Revival of <i>S. elongatus</i> strains from plates</a></li>
									<li class="nav-item"><a class="nav-link" href="#c3">Making DMSO and Glycerol Stocks</a></li>
									<li class="nav-item"><a class="nav-link" href="#c4">Revival of <i>S. elongatus</i> strains from glycerol or DMSO stocks</a></li>
									<li class="nav-item"><a class="nav-link" href="#c5">Growth Curves of <i>S. elongatus</i> under salt stress</a></li>
									<li class="nav-item"><a class="nav-link" href="#c6">Sucrose assay</a></li>
									<li class="nav-item"><a class="nav-link" href="#c8">References</a></li>
								</ul>
							</nav>
						</div>
					</div>

					<!-- Coculture Sidebar-->
					<div class="tab-pane fade coculture-wet" id="coculture-wet-side" role="tabpanel" aria-labelledby="coculture-wet-tab"> 
						
						<div class="sidenav">
							<nav>
								<ul id="sidenav-id">
									<li class="nav-item"><a class="nav-link" href="#cc1">Transformation of MG1655 with pKD4</a></li>
									<li class="nav-item"><a class="nav-link" href="#cc2">Preparation of CoBG-11</a></li>
									<li class="nav-item"><a class="nav-link" href="#cc3">Growth curve of KJK01 and pCSCX-KJK01 in BG-11 + sucrose</a></li>
									<li class="nav-item"><a class="nav-link" href="#cc4">Sucrose assay</a></li>
									<li class="nav-item"><a class="nav-link" href="#cc5">References</a></li>
								</ul>
							</nav>
						</div>
					</div>
				</div>
			</div>
			<!-- END OF SIDENAV BAR -->
           



            <!-- CONTENT -->
            <div class="col-lg-9 col-md-9">
                <div class="content">

					<!--------------------------------- STYLING FOR THE TABS ------------------------------------>
					<style>
						.content .nav-link{
							/* how it looks by default */
							color:white !important;
							font-size: 1.5rem;
						}
						.content .nav-link:hover{
							color:black !important;
							background-color: #28c79b;
						}
						.content .nav-link.active{
							color: white;
							background-color: #1f5e4c;
						}
						.content .nav-item .nav-link,
						content .nav-tabs .nav-link {
							-webkit-transition: all 250ms ease 0s;
							-moz-transition: all 250ms ease 0s;
							-o-transition: all 250ms ease 0s;
							-ms-transition: all 250ms ease 0s;
							transition: all 250ms ease 0s;
						}
					</style>
					<!--------------------------------- END OF STYLING FOR THE TABS ------------------------------------>

					<!-- Tabs -->
					<ul class="nav nav-tabs nav-fill" id="content-wet-sec-tabs" role="tablist">
						<li class="nav-item" role="presentation">
							<button class="nav-link active" id="overview-wet-tab"data-bs-toggle="tab" data-bs-target=".overview-wet" type="button" role="tab" aria-controls="overview-wet" aria-selected="true">Overview</button>
						</li>
						<li class="nav-item" role="presentation">
							<button class="nav-link" id="ecoli-wet-tab" data-bs-toggle="tab" data-bs-target=".ecoli-wet" type="button" role="tab" aria-controls="ecoli-wet" aria-selected="true"><i>E. coli</i></button>
						</li>
						<li class="nav-item" role="presentation">
							<button class="nav-link" id="cyano-wet-tab" data-bs-toggle="tab" data-bs-target=".cyano-wet" type="button" role="tab" aria-controls="cyano-wet" aria-selected="false">Cyanobacteria</button>
						</li>
						<li class="nav-item" role="presentation">
							<button class="nav-link" id="coculture-wet-tab" data-bs-toggle="tab" data-bs-target=".coculture-wet"  type="button" role="tab" aria-controls="coculture-wet" aria-selected="false">Co-culture</button>
						</li>
					<a id="e1"></a> <a id="c1"></a> <a id="cc1"></a>
					</ul>
					<!-- End of Tabs -->


					<div class="tab-content" id="myTabContent">

						<!--------------------------------------- OVERVIEW --------------------------------------------->
						<div class="tab-pane fade show active overview-wet" id="overview-wet" role="tabpanel" aria-labelledby="overview-wet-tab">
							<a id="o1"></a>
							<br><br><br>
							<p>Our Wet Lab work was geared towards observing the production of our final product from a co-culture of <i>E. coli</i> and <i>S. elongatus</i> UTEX 2973. To that end, we broke it down into four steps:</p>
							<ol class="a">
								<li style="padding-left: 1em;">Engineering our cyanobacteria to secrete sucrose under salt stress;</li>
								<li style="padding-left: 1em;">Engineering our <i>E. coli</i> to produce our final product (butanol or succinate);</li>
								<li style="padding-left: 1em;">Engineering the <i>E. coli</i> to consume sucrose;</li>
								<li style="padding-left: 1em;">Growing a co-culture of the two</li>
							</ol>
							<br>

							<p>Then came the devastating second wave of COVID-19, cutting our lab time down from seven months to two. We were incredibly fortunate, in that situation, to benefit from the help of Dr Himadri Pakrasi and Dr Shams Yazdani. Dr Pakrasi’s lab provided us with sucrose-secreting strains they had previously engineered - UTEX 2973-<i>cscB</i>, 2973-cscB-<i>spp</i> and 2973-cscB-<i>spp</i>-<i>spp</i>. <i>S. elongatus</i>’s freshwater strains naturally produce sucrose as an osmoprotectant, and the presence of <i>cscB</i> (coding for sucrose permease) leads to that sucrose being secreted into the growth medium<sup><a href="#oref1">[1]</a></sup>. The genes <i>spp</i> and <i>spp</i> upregulate the sucrose production pathway. This was a huge relief for us, as UTEX 2973 is not naturally transformable. Genes have to be introduced via triparental mating (a 9-day-long protocol involving conjugation) and incorporated into the genome via a homologous recombination strategy, such as CRISPR. Dr Yazdani’s lab lent us his <i>E. coli</i> KJK01 strain - an IPTG-inducible butanol-producing derivative of MG1655.<sup><a href="#oref2">[2]</a></sup> The genes controlling the butanol-producing pathway have been markerlessly integrated into the genome, making it easy to work with for the purposes of our co-culture.</p>
							<br>

							<p>Our immediate work in the lab, then, was to test out the strains we’d received and to set up a co-culture. We used NMR to assay for the presence of butanol in the LB medium of the KJK01. Following confirmation of butanol production, we made the KJK01 cells electrocompetent and transformed them with pCSCX - a plasmid containing genes for sucrose uptake and digestion.<sup><a href="#oref3">[3]</a></sup> pCSCX is marked with Kanamycin and Ampicillin, the former of which is also shared by UTEX-2973, making it useful for ensuring the purity of our co-culture. For the cyanobacteria, we used Megazyme’s kit for a colorimetric sucrose assay to establish a standard curve of fluorescence versus sucrose concentration, and used it to quantify the sucrose concentration in supernatant aliquots taken at regular intervals.<sup><a href="#oref4">[4]</a></sup> The same assay and standard curve were used to quantify sucrose consumption in KJK01 grown in Co-BG11 with sucrose. Co-BG11 is a variant of the standard BG11 medium for cyanobacteria, and is free of any other carbon source; this variant is optimised for co-cultures with <i>E. coli</i>.<sup><a href="#oref5">[5]</a></sup></p>
							<br>

							<p>Reviving cyanobacteria was a difficult task and involved setting up lights in our incubator, acquiring a lux meter and making many, many mistakes as it is a relatively new model organism. We maintained a record of everything we learn from our mentors at the Wangikar lab and are publishing it as a contribution to future teams working on cyanobacteria.</p>
							<br>
						
							<p>We intend in future work to re-engineer the cyanobacteria to express the sucrose production pathway and the sucrose permease gene under strong native promoters, and to choose a promoter that responds well to high CO<sub>2</sub> conditions for our final application. This is principally on the advice of Dr Shireesh Srivastava, who has observed in the course of his work that native promoters often lead to higher and more consistent levels of expression with less detriment to the host. We further intend to find optimal conditions - light, carbon dioxide, reductants to remove oxygen evolved by cyanobacteria, temperature and so forth - allow for a stable and productive co-culture.</p>
							<br>

							<p>For our E. coli KJK01 strain, we plan to characterize the bidirectional promoter present in our pcscX plasmid. This will help us understand the flux through the csc gene products and their importance for our strain. We further plan to increase the butanol tolerance of our strain based on studies by Reyes et al.<sup><a href="#oref6">[6]</a></sup> and Boyarskiy et al.<sup><a href="#oref7">[7]</a></sup> For further information on this check out our <a href="#Implementation.html">proposed implementation</a>. We will also perform our growth curves with this modified strain. We also plan to incorporate our drylab results into our wetlab by knocking out and overexpressing the genes suggested by OptKnock and FSEOF respectively. We further plan to optimize our co culture using the results from our modified strains.</p>
							
							
							<a id="o2"></a><br><a id="oref1"></a><br><a id="oref2"></a><br><a id="oref3"></a><br>


							<h3 id="reference">References</h3>
							<hr>
							<ol class="a">
								<li style="padding-left: 1em;" id="oref4">Lin, P. C., Zhang, F., & Pakrasi, H. B. (2020). Enhanced production of sucrose in the fast-growing cyanobacterium <i>Synechococcus elongatus</i> UTEX 2973. Scientific reports, 10(1), 1-8.</li>
								<li style="padding-left: 1em;" id="oref5">Abdelaal, A. S., Jawed, K., & Yazdani, S. S. (2019). CRISPR/Cas9-mediated engineering of Escherichia coli for n-butanol production from xylose in defined medium. Journal of Industrial Microbiology and Biotechnology, 46(7), 965-975.</li>
								<li style="padding-left: 1em;" id="oref6">Bruschi, M., Boyes, S. J., Sugiarto, H., Nielsen, L. K., & Vickers, C. E. (2012). A transferable sucrose utilization approach for non-sucrose-utilizing Escherichia coli strains. Biotechnology advances, 30(5), 1001–1010. <a href="https://doi.org/10.1016/j.biotechadv.2011.08.019">https://doi.org/10.1016/j.biotechadv.2011.08.019</a></li>
								<li style="padding-left: 1em;" id="oref7"><a href="https://www.megazyme.com/sucrose-d-glucose-assay-kit">Megazyme's sucrose assay kit</a></li>
								<li style="padding-left: 1em;">Zhang, L., Chen, L., Diao, J., Song, X., Shi, M., & Zhang, W. (2020). Construction and analysis of an artificial consortium based on the fast-growing cyanobacterium <i>Synechococcus elongatus</i> UTEX 2973 to produce the platform chemical 3-hydroxypropionic acid from CO 2. Biotechnology for biofuels, 13(1), 1-14.</li>
								<li style="padding-left: 1em;">Reyes, L. H., Almario, M. P., & Kao, K. C. (2011). Genomic library screens for genes involved in n-butanol tolerance in Escherichia coli. PloS one, 6(3), e17678.</li>
								<li style="padding-left: 1em;">Sergey Boyarskiy, Stephanie Davis López, Niwen Kong, Danielle Tullman-Ercek,Transcriptional feedback regulation of efflux protein expression for increased tolerance to and production of n-butanol,Metabolic Engineering,Volume 33,2016,Pages 130-137,ISSN 1096-7176</li>
							</ol>
							<br><br>
						</div>
						<!----------------------------------- END OF OVERVIEW ------------------------------------------->

						
						<!------------------------------------- START OF E COLI------------------------------------------>
						<div class="tab-pane fade ecoli-wet" id="ecoli-wet" role="tabpanel" aria-labelledby="ecoli-wet-tab">
							
							<br><br><br>
							<p>The experiments below are presented in the order in which we performed them.</p>
							<!-- GLYCEROL STOCKS OF KJK01 -->
							<br><br><br>
							<h2>Glycerol stocks of KJK01</h2>
							<br><br>
							<h3> Objective </h3>
							<p>To prepare glycerol stocks of the <i>E. coli</i> KJK01 strain.</p>
							<br>
							<h3>Working Principle<sup><a href="#eref1">[1]</a></sup></h3>
							<p>Bacterial cultures in the form of agar plates or liquid cultures are not suitable for long term storage at low temperatures such as 4&#176. Long term storage of bacteria is best at temperatures around -80&#176. However, agar plates and liquid cultures cannot be stored at such temperatures, as the water crystallizes in and around the bacterial cells, rupturing and killing them. Glycerol, an anti-freezing agent, is added to the media to store samples at such temperatures.</p>
							<br>
							<h3>Key Reagents/Apparatus</h3>
							<ul class="b">
								<li style="padding-left: 1em;"> Autoclaved 1X LB</li>
								<li style="padding-left: 1em;"> Autoclaved 50% glycerol</li>
								<br>
								<li style="padding-left: 1em;">Cryovials</li> 
							</ul>
							<br>
							<h3>Protocol</h3>
							<p> Can be found <a href="https://www.protocols.io/view/glycerol-stock-preparation-byyppxvn">here</a></p>
							<br><br>

							<div class="graph">
								<img src="assets/WetlabPhotos/Glycerol_stocks_3.jpeg" alt="Flash freezing the prepared KJK01 stocks" width=535>
								<br>
								<p id="desc">Flash freezing the prepared KJK01 stocks</p>
							</div>
							<a id="e2"></a><br><hr width="50%"><br><br>
							<!-- END OF GLYCEROL STOCKS OF KJK01-->



							<!-- GDNA EXTRACTION -->
							<!-- Likhith needs to add a ref -->
							<h2 class="h2">Genomic DNA extraction</h2>
							<br><br>
							<h3>Objective</h3>
							<p>To extract the genomic DNA of <i>E. coli</i> KJK01 to verify the presence of the <i>hbd-crt</i> butanol production pathway genes. </p>
							<br>

							<h3>Working principle<sup><a href="#eref2">[2]</a></sup></h3>
							<p>To extract the genomic DNA from the <i>E. coli</i> cells, the cell membrane is first lysed using SDS to release cellular contents. Proteins and RNA from the cells are destroyed using Proteinase K and RNase A respectively. This leaves behind only DNA, which is precipitated in isopropanol. </p>
							<p>Once the gDNA has been extracted, it can be run on an agarose gel to verify its presence.</p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;">Sodium dodecyl sulfate (SDS) </li>
								<li style="padding-left: 1em;">Proteinase K </li>
								<li style="padding-left: 1em;">RNase A </li>
								<li style="padding-left: 1em;">Isopropanol </li>
								<li style="padding-left: 1em;">Ethanol </li>
								<li style="padding-left: 1em;">Phenol-chloroform </li>
								<li style="padding-left: 1em;">Double distilled water (MilliQ water)</li>
								<li style="padding-left: 1em;">Sodium Acetate </li>								
								<li style="padding-left: 1em;">TAE buffer </li>
								<li style="padding-left: 1em;">Ethidium Bromide </li>
								<li style="padding-left: 1em;">DNA Ladder </li>
								<br>
								<li style="padding-left: 1em;">Centrifuge</li>
								<li style="padding-left: 1em;">Thermocycler </li>
								<li style="padding-left: 1em;">Gel Electrophoresis Machine </li>
								<li style="padding-left: 1em;">Gel imager </li>
								<li style="padding-left: 1em;">NanoDrop </li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<p> Can be found <a href="http://dx.doi.org/10.17504/protocols.io.bxeupjew">here</a></p>
							<a id="e3"></a><br><hr width="50%"><br><br>
							<!--END OF GDNA-->




							<!-- BUTANOL PRODUCING GENES -->
							<h2>Confirmation of butanol producing genes</h2>
							<br><br>
							<h3>Objective</h3>
							<p>To confirm the presence of butanol producing genes in KJK01 strain using PCR and gel electrophoresis. </p>
							<br>

							<h3>Working principle<sup><a href="#eref3">[3]</a></sup></h3>
							<p>The butanol producing ability of KJK01 is due to the presence of the cassette of genes inserted into the genome of the strain. Since the cassette has been added as one single component, verification of any component from the cassette will verify the presence of the butanol producing genes in KJK01.</p>
							<p>Primers against the <i>hbd-crt</i> genes (sequence provided by Dr Yazdani) were used to check for the presence of these 2 genes from the genomic DNA of KJK01. An amplicon of length of ~750bp from the PCR would verify the presence of <i>hbd-crt</i> genes.</p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;">Pfu polymerase </li>
								<li style="padding-left: 1em;">Pfu buffer </li>
								<li style="padding-left: 1em;">dNTP mix </li>
								<li style="padding-left: 1em;">Primers </li>
								<li style="padding-left: 1em;">Agarose </li>
								<li style="padding-left: 1em;">1X TAE </li>
								<li style="padding-left: 1em;">EtBr solution </li>
								<br>
								<li style="padding-left: 1em;">Thermocycler </li>
								<li style="padding-left: 1em;">Gel electrophoresis apparatus </li>
								<li style="padding-left: 1em;">Gel dock </li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<p> Can be found <a href="https://www.protocols.io/view/pcr-and-gel-electropheresis-bxempjc6">here</a></p>
							<a id="e4"></a><br><hr width="50%"><br><br>
							<!-- END OF BUTANOL PRODUCING GENES-->	
							



							<!-- GPCR -->
							<h2 class="h2">Gradient PCR</h2>
							<br><br>
							<h3>Objective</h3>
							<p>To perform a gradient PCR to find the best annealing temperature as well as the best amount of DMSO to add to obtain proper amplification of the gene of interest. </p>
							<br>

							<h3>Working principle<sup><a href="#eref4">[4]</a></sup></h3>
							<p>Thermocyclers used for PCR reactions can maintain different tubes at different temperatures in the same run. This is exploited to vary the annealing temperature and observe which of the set temperatures is best suited for annealing and thus gives best amplification. This allows for a single PCR run to give us results for multiple annealing temperatures.</p>
							<p>DMSO prevents the renaturation of single strands of DNA. Thus, it helps in amplification of the gene of interest.<sup><a href="#eref4">[4]</a></sup> An optimum amount can be found by adding different amounts to tubes in gradient PCR.</p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;">Pfu polymerase </li>
								<li style="padding-left: 1em;">Pfu buffer </li>
								<li style="padding-left: 1em;">dNTP mix </li>
								<li style="padding-left: 1em;">Primers </li>
								<li style="padding-left: 1em;">Agarose </li>
								<li style="padding-left: 1em;">1X TAE </li>
								<li style="padding-left: 1em;">EtBr solution </li>
								<li style="padding-left: 1em;">DMSO</li>
								<br>
								<li style="padding-left: 1em;">Thermocycler </li>
								<li style="padding-left: 1em;">Gel electrophoresis apparatus </li>
								<li style="padding-left: 1em;">Gel dock </li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<p> Can be found <a href="https://dx.doi.org/10.17504/protocols.io.by6kpzcw">here</a></p>
							<a id="e5"></a><br><hr width=50%><br><br>
							<!-- END OF GPCR -->




							<!-- NMR -->
							<h2 class="h2">NMR confirmation of butanol production</h2>
							<br><br>
							<h3>Objective</h3>
							<p>To confirm production of butanol by <i>E. coli</i> KJK01 using NMR spectroscopy </p>
							<br>

							<h3>Working principle <a href="#eref5"><sup>[5]</sup></a></h3>
							<p>Nuclear magnetic resonance (NMR) spectroscopy, is a spectroscopic technique to observe local magnetic fields around atomic nuclei based on their absorption of electromagnetic radiation in the radio-frequency region 4 to 900 MHz. The principle behind NMR is that many nuclei have a spin and all nuclei are electrically charged. If an external magnetic field is applied, energy transfer is possible between the ground state to a higher energy level (generally a single energy gap). The energy transfer takes place at a wavelength that corresponds to appropriate radio frequencies and when the spin returns to its base level, energy is emitted at the same frequency. The signal that matches this transfer is measured and processed in order to yield an NMR spectrum for the nucleus concerned. Fields are highly characteristic of individual compounds. NMR also provides detailed information about the structure, dynamics, reaction state, and chemical environment of molecules.</p>
							<p>We used proton NMR for the detection of butanol in the supernatant of our <i>E. coli</i> KJK01 cultures, alongside PCR confirmation of butanol producing genes.</p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;">Supernatants of IPTG induced <i>E. coli</i> KJK01 2&#176 culture </li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<ol class="a">
								<li style="padding-left: 1em;">Inoculate 1&#176 culture of KJK01 in 10 mL 1X LB and leave it overnight in an incubator at 37&#176C and 180 rpm.</li>
								<li style="padding-left: 1em;">The next morning, inoculate 2&#176 cultures of KJK01 in 1X LB and 1X TB and place in the incubator at 37&#176C and 180 rpm until it hits 0.6 OD<sub>600</sub>.</li>
								<ul class="c">
									<li style="padding-left: 1em;">We inoculated 6 2&#176 cultures (3 LB and 3 TB) to confirm if KJK01 grew better in LB or TB (The secondary culture was originally cultured in TB expecting better growth<a href="#eref3"><sup>[3]</sup></a>)</li>
								</ul>
								<li style="padding-left: 1em;"> At OD 0.6, induce cultures with appropriate concentrations of IPTG and leave it in the incubator overnight at 37&#176C and 180 rpm.</li>
								<ul class="c">
									<li style="padding-left: 1em;">The 6 cultures are:</li>
									<ul class="d">
										<li style="padding-left: 1em;">100 µL 1&#176 culture + 10 mL 1X LB + 0 IPTG </li> 
										<li style="padding-left: 1em;">100 µL 1&#176 culture + 10 mL 1X LB + 1 µL (0.1 mM) IPTG </li> 
										<li style="padding-left: 1em;">100 µL 1&#176 culture + 10 mL 1X LB + 2 µL (0.2 mM) IPTG </li> 
										<li style="padding-left: 1em;">100 µL 1&#176 culture + 10 mL 1X TB + 0 IPTG </li> 
										<li style="padding-left: 1em;">100 µL 1&#176 culture + 10 mL 1X TB + 1 µL (0.1 mM) IPTG </li> 
										<li style="padding-left: 1em;">100 µL 1&#176 culture + 10 mL 1X TB + 2 µL (0.2 mM) IPTG </li> 
									</ul>
								</ul>
								<li style="padding-left: 1em;"> The next morning, draw 1 mL sample from each culture and centrifuge at 4&#176 and 5000 rpm for 15 minutes.</li>
								<li style="padding-left: 1em;">Decant the solution and submit supernatant for NMR analysis.</li>
							</ol>
							<br>
							<p>Soumya Sahoo, a PhD student at the NMR Lab of IISER Pune helped us with the NMR analysis of our samples.</p>
							<a id="e6"></a><br><hr width="50%"><br><br>
							<!--END OF NMR -->




							<!-- ELECTROCOMPETENT -->
							<h2 class="h2">Making KJK01 cells electrocompetent</h2>
							<br><br>
							<h3>Objective</h3>
							<p> To prepare KJK01 electrocompetent cells </p>
							<br>

							<h3>Working principle</h3>
							<p>To transform any cells, they need to be competent. Only competent cells can take up DNA. Competent cells can be chemically or electrically stimulated to open up the cell pores and take up DNA. The preparation of cells for chemical or electrical transformation differs in protocol, and the cells competent for the same are called chemically competent and electrocompetent cells, respectively.</p>
							<p> Electrocompetent cells can be transformed using an electrical pulse. When electro-competent cells are pulsed, the pulse will rearrange the lipid bilayer to form a gap for the intake of the plasmid <sup><a href="#eref7">[7]</a></sup>.</p>
							<p>The preparation of electrocompetent cells involves washing the grown cells serially using water and glycerol multiple times. These washes are important to remove all the salts surrounding the cells. The salts would otherwise cause sparking during the electrical pulse, damaging the apparatus and the cell wall. Competent cells are finally mixed in dilute glycerol solution, flash frozen and stored at -80&#176.</p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;"> Autoclaved 2X LB </li>
								<li style="padding-left: 1em;"> Autoclaved 10% glycerol </li>
								<br>
								<li style="padding-left: 1em;">Centrifuge</li>
								<li style="padding-left: 1em;">Liq. Nitrogen </li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<p> Can be found <a href="https://www.protocols.io/view/making-electro-competent-cells-bxxhppj6">here</a></p>
							<a id="e7"></a><br><hr width="50%"><br><br>
							<!--END OF ELECTROCOMPETENT -->



							<!-- KJK01 PCSCX -->
							<h2 class="h2">Transformation of KJK01 with pCSCX plasmid</h2>
							<br><br>
							<h3>Objective</h3>
							<p>To transform <i>E. coli</i> KJK01 with plasmid pCSCX</p>
							<br>

							<h3>Working principle <sup><a href="#eref7">[7]</a></sup></h3>
							<p>Transformation via electroporation is a physical transformation method that uses an electrical pulse to create temporary pores in cell membranes through which substances like nucleic acids can pass into cells. Host cells and nucleic acids are suspended in a conductive solution, and an electrical circuit is closed around the mixture. An electrical pulse at an optimized voltage lasting a few microseconds to a millisecond is discharged through the cell suspension, which disturbs the phospholipid bilayer of the membrane and results in the formation of temporary pores. The electric potential across the cell membrane simultaneously rises to allow charged molecules like DNA to be driven across the membrane through the pores. This method can transform a large number of cells in a short period of time as it is easy and rapid. </p>
							<br>
							<p>The pCSCX gene contains <i>cscABK</i> genes from <i>E. coli</i> W that help in transport and metabolism of sucrose. <i>E. coli</i> strains that have been transformed with this plasmid have shown to be able to survive with sucrose as their sole carbon source<sup><a href="eref8">[8]</a></sup>. We hope to replicate these results.</p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;">Electrocompetent cells </li>
								<li style="padding-left: 1em;">Plasmid DNA </li>
								<li style="padding-left: 1em;">2X LB </li>
								<li style="padding-left: 1em;">MilliQ water </li>
								<br>
								<li style="padding-left: 1em;">BioRad Pulser </li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<p> Can be found <a href="https://www.protocols.io/view/transformation-of-e-coli-with-plasmid-byz3px8n">here</a></p>
							<br><br>

							<!-- SLIDE SHOW -->
							<div id="transformationslides" class="carousel slide" data-bs-ride="carousel">
								<div class="carousel-indicators" style="bottom:-50px;">
								<button type="button" data-bs-target="#transformationslides" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
								<button type="button" data-bs-target="#transformationslides" data-bs-slide-to="1" aria-label="Slide 2"></button>
								<button type="button" data-bs-target="#transformationslides" data-bs-slide-to="2" aria-label="Slide 3"></button>
								<button type="button" data-bs-target="#transformationslides" data-bs-slide-to="3" aria-label="Slide 4"></button>
								</div>
								<div style= "margin-bottom:50px; display:flex; flex-direction:row; align-items: center;" class="carousel-inner">
									<div class="carousel-item active graph"> 
										<img src="assets/WetlabPhotos/Tranformation_1.jpeg" alt="Best 5 over-expression targets in one graph" width=500>
									</div>
									<div class="carousel-item graph">
										<img src="assets/WetlabPhotos/Transformation_2.jpeg" alt="Next best 5 over-expression targets in one graph" width=500>
									</div>
									<div class="carousel-item graph">
										<img src="assets/WetlabPhotos/Transformation_3.jpeg" alt="Third best 5 over-expression targets in one graph" width=500>
									</div>
									<div class="carousel-item graph">
										<img src="assets/WetlabPhotos/Tranformation_4.jpeg" alt="Fourth best 5 over-expression targets in one graph" width=500>
									</div>
								</div>
								<button class="carousel-control-prev" type="button" data-bs-target="#transformationslides" data-bs-slide="prev">
								<span class="carousel-control-prev-icon" aria-hidden="true"></span>
								<span class="visually-hidden">Previous</span>
								</button>
								<button class="carousel-control-next" type="button" data-bs-target="#transformationslides" data-bs-slide="next">
								<span class="carousel-control-next-icon" aria-hidden="true"></span>
								<span class="visually-hidden">Next</span>
								</button>
							</div>
							<!-- END OF SLIDESHOW -->
							<a id="e8"></a><br><hr width=50%><br><br>
							<!--END OF KJK01 PCSCX -->



							<!-- STOCKS OF KJK01 + PCSCX -->
							<h2 class="h2">Glycerol stocks of pCSCX-KJK01</h2>
							<br><br>
							<h3>Objective</h3>
							<p>To prepare glycerol stocks of the <i>E. coli</i> KJK01 strain transformed with the <i>cscABK</i> gene containing pCSCX plasmid. </p>
							<br>

							<h3>Working principle</h3>
							<p>Bacterial cultures in the form of agar plates or liquid cultures are not suitable for long term storage at low temperatures such as 4&#176. Long term storage of bacteria is best at temperatures around -80&#176. However, agar plates and liquid cultures cannot be stored at such temperatures, as the water crystallizes in and around the bacterial cells, rupturing and killing them. Glycerol, an anti-freezing agent, is added to the media to store samples at such temperatures.<sup><a href="#eref1">[1]</a></sup></p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;">Autoclaved 1X LB </li>
								<li style="padding-left: 1em;">Autoclaved 50% glycerol </li>
								<br>
								<li style="padding-left: 1em;">Cryovials </li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<p> Can be found <a href="https://www.protocols.io/view/glycerol-stock-preparation-byyppxvn">here</a></p>
							<br><br>

							<div id="transformedstocksslides" class="carousel slide" data-bs-ride="carousel">
								<div class="carousel-indicators" style="bottom:-50px;">
								<button type="button" data-bs-target="#transformedstocksslides" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
								<button type="button" data-bs-target="#transformedstocksslides" data-bs-slide-to="1" aria-label="Slide 2"></button>
								</div>
								<div style= "margin-bottom:50px; display:flex; flex-direction:row; align-items: center;" class="carousel-inner">
									<div class="carousel-item active graph"> 
										<img src="assets/WetlabPhotos/Transformed_stocks_1.jpeg" alt="Best 5 over-expression targets in one graph" width=280>
									</div>
									<div class="carousel-item graph">
										<img src="assets/WetlabPhotos/Transformed_stocks_2.jpeg" alt="Next best 5 over-expression targets in one graph" width=500>
									</div>
								</div>
								<button class="carousel-control-prev" type="button" data-bs-target="#transformedstocksslides" data-bs-slide="prev">
								<span class="carousel-control-prev-icon" aria-hidden="true"></span>
								<span class="visually-hidden">Previous</span>
								</button>
								<button class="carousel-control-next" type="button" data-bs-target="#transformedstocksslides" data-bs-slide="next">
								<span class="carousel-control-next-icon" aria-hidden="true"></span>
								<span class="visually-hidden">Next</span>
								</button>
							</div>
							
							<a id="e9"></a><br><hr width="50%"><br><br>
							<!--END OF STOCKS OF KJK01 + PCSCX -->
							
							

							<!-- MIC KJK01 -->
							<h2 class="h2">MIC Assay for KJK01</h2>
							<br><br>
							<h3>Objective</h3>
							<p>To determine the minimum inhibitory concentration (MIC) of butanol and ethanol for <i> E. coli</i> KJK01 strain. </p>
							<br>

							<h3>Working principle</h3>
							<p>Minimum inhibitory concentration (MIC) is defined as the lowest concentration of an antimicrobial that will inhibit the visible growth of a microorganism after overnight incubation. <sup><a href="#eref9">[9]</a></sup> Determination of MIC involves obtaining a growth curve of the strain at varying concentrations of the antimicrobial and identifying the least concentration at which growth is inhibited.</p>
							<p>This experiment will help us calculate the optimum extraction frequency of products from the bioreactor medium in order to maximise yield from the cells.</p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;">MG1655 2&#176 culture (Control strain) </li>
								<li style="padding-left: 1em;">KJK01 2&#176 cultures (uninduced and induced with IPTG) </li>
								<li style="padding-left: 1em;">1X LB </li>
								<li style="padding-left: 1em;">MilliQ </li>
								<br>
								<li style="padding-left: 1em;">96-well plates </li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<p> Can be found <a href="https://www.protocols.io/view/minimum-inhibitory-concentration-of-butanol-for-e-byz8px9w">here</a></p>
							<br><br>

							<!-- SLIDE SHOW -->
							<div id="ecolimicslides" class="carousel slide" data-bs-ride="carousel">
								<div class="carousel-indicators" style="bottom:-50px;">
								<button type="button" data-bs-target="#ecolimicslides" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
								<button type="button" data-bs-target="#ecolimicslides" data-bs-slide-to="1" aria-label="Slide 2"></button>
								<button type="button" data-bs-target="#ecolimicslides" data-bs-slide-to="2" aria-label="Slide 3"></button>
								<button type="button" data-bs-target="#ecolimicslides" data-bs-slide-to="3" aria-label="Slide 4"></button>
								</div>
								<div style= "margin-bottom:50px; display:flex; flex-direction:row; align-items: center;" class="carousel-inner">
									<div class="carousel-item active graph"> 
										<img src="assets/WetlabPhotos/MIC_1.jpeg" alt="Best 5 over-expression targets in one graph" width=280>
									</div>
									<div class="carousel-item graph">
										<img src="assets/WetlabPhotos/MIC_2.jpeg" alt="Next best 5 over-expression targets in one graph" width=500>
									</div>
									<div class="carousel-item graph">
										<img src="assets/WetlabPhotos/MIC_3.jpeg" alt="Third best 5 over-expression targets in one graph" width=500>
									</div>
									<div class="carousel-item graph">
										<img src="assets/WetlabPhotos/MIC_4.jpeg" alt="Fourth best 5 over-expression targets in one graph" width=500>
									</div>
								</div>
								<button class="carousel-control-prev" type="button" data-bs-target="#ecolimicslides" data-bs-slide="prev">
								<span class="carousel-control-prev-icon" aria-hidden="true"></span>
								<span class="visually-hidden">Previous</span>
								</button>
								<button class="carousel-control-next" type="button" data-bs-target="#ecolimicslides" data-bs-slide="next">
								<span class="carousel-control-next-icon" aria-hidden="true"></span>
								<span class="visually-hidden">Next</span>
								</button>
							</div>

							<!-- END OF SLIDE SHOW -->

							<a id="e10"></a><br><hr width="50%"><br><br>
							<!--END OF OF MIC KJK01 -->



							<!-- GROWTH CURVE -->
							<h2 class="h2">Growth Curve in LB</h2>
							<br><br>
							<h3>Objective</h3>
							<p>To obtain growth curves of KJK01 and pCSCX-KJK01 in LB and to characterize their growth </p>
							<br>

							<h3>Working principle<sup><a href="#eref10">[10]</a></sup></h3>
							<p>When bacteria are inoculated into a liquid medium and the cell population is counted at intervals, it is possible to plot a typical bacterial growth curve that shows the growth of cells over time. It shows four distinct phases of growth. </p>
							<ul class="b">
								<li style="padding-left: 1em;">Lag phase: Slow growth or lack of growth due to physiological adaptation of cells to culture conditions or dilution of exoenzymes due to initial low cell densities. </li>
								<li style="padding-left: 1em;">Log or exponential phase: Optimal growth rates, during which cell numbers double at discrete time intervals known as the mean generation time. </li>
								<li style="padding-left: 1em;">Stationary phase: Growth (cell division) and death of cells counterbalance each other resulting in no net increase in cell numbers. The reduced growth rate is usually due to a lack of nutrients and/or a buildup of toxic waste constituents. </li>
								<li style="padding-left: 1em;">Decline or death phase: Death rate exceeds growth rate resulting in a net loss of viable cells. </li>
							</ul>
							<br>
							<p>This is one of the simplest methods used to analyze trends in growth because it uses a spectrophotometer to track changes in the optical density (OD) over time. In other words, as the number of cells in a sample increases, the transmission of light through the sample will decrease.</p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;">2&#176 cultures of KJK01 and pCSCX-KJK01 in LB </li>
								<br>
								<li style="padding-left: 1em;">Photospectrometer </li>
								<li style="padding-left: 1em;">Quartz/plastic (disposable) cuvette </li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<p> Can be found <a href="https://www.protocols.io/view/growth-rate-determination-of-e-coli-bxa2pige">here</a></p>
							<a id="e11"></a><br><hr width="50%"><br><br>
							<!--END OF GROWTH CURVE -->


							<!-- SUCROSE ASSAY -->
							<h2>Sucrose Assay</h2>
							<br><br>

							<h3>Objective</h3>
							<p>To measure the consumption of sucrose by <i>E. coli</i> KJK01 and pCSCX-KJK01</p>
							<br>
							<p>Same as for <i>S. elongatus</i>, except we look for decrease in sucrose concentration as opposed to increase. Link to sucrose assay for <i>S. elongatus</i> can be found <a href="#c6">here</a>.</p>
							<br>

							<h3>Working Principle</h3>
							<p>The sucrose assay is a colorimetric assay<sup><a href="#eref11">[11]</a></sup> to quantify sucrose concentration in cell supernatants. Supernatant samples from the cultures are taken at regular frequencies. A decreasing trend in the absorbance of successive samples after treatment is indicative of decreasing sucrose concentration, i.e, net sucrose consumption has been taking place.</p>
							<br>

							<ul class="b">
								<li style="padding-left: 1em;">Distilled water </li>
								<li style="padding-left: 1em;">D-Glucose standard solution (5 mL, 1.0 mg/mL) in 0.2% (w/v) benzoic acid </li>
								<li style="padding-left: 1em;">GOPOD Reagent Enzymes. Glucose oxidase plus peroxidase and 4-aminoantipyrine. Freeze-dried powder </li>
								<li style="padding-left: 1em;">GOPOD Reagent Buffer. p-hydroxybenzoic acid and sodium azide (0.095% w/v) </li>
								<li style="padding-left: 1em;">β-Fructosidase (invertase) solution </li>
								<li style="padding-left: 1em;">Acetate buffer</li>
								<br>
								<li style="padding-left: 1em;">96 well plate </li>
								<li style="padding-left: 1em;">1 mL Eppendorf tubes </li>
								<li style="padding-left: 1em;">Spectrophotometer set at 510 nm </li>
								<li style="padding-left: 1em;">Vortex mixer </li>
								<li style="padding-left: 1em;">Thermostated water bath </li>
							</ul>
							<br><a id="eref1"></a><br><a id="eref2"></a><br><a id="eref3"></a>


							<!-- References -->
							<hr>
							<h3 id="reference">Reference</h3>
							<ol class="a">
								<li style="padding-left: 1em;" id="eref4"><a href="https://www.addgene.org/protocols/create-glycerol-stock/">https://www.addgene.org/protocols/create-glycerol-stock/</a></li>
								<li style="padding-left: 1em;" id="eref5"> Surzycki, Stefan (2000). Basic Techniques in Molecular Biology || Preparation of Genomic DNA from Bacteria. , 10.1007/978-3-642-56968-5(Chapter 4), 79–100. doi:10.1007/978-3-642-56968-5_4 </li>
								<li style="padding-left: 1em;" id="eref6">Abdelaal, A. S., Jawed, K., & Yazdani, S. S. (2019). CRISPR/Cas9-mediated engineering of Escherichia coli for n-butanol production from xylose in defined medium. Journal of Industrial Microbiology and Biotechnology, 46(7), 965-975.</li>
								<li style="padding-left: 1em;" id="eref7">Gopalakrishnan, Guruprasad. (2012). Re: What exactly is the function of DMSO in PCR? . Retrieved from: <a href="https://www.researchgate.net/post/What_exactly_is_the_function_of_DMSO_in_PCR/500ad731e4f076bb14000011/citation/download">research gate</a></li>
								<li style="padding-left: 1em;" id="eref8"><a href="https://www.sciencedirect.com/topics/materials-science/nuclear-magnetic-resonance-spectroscopy">https://www.sciencedirect.com/topics/materials-science/nuclear-magnetic-resonance-spectroscopy</a></li>
								<li style="padding-left: 1em;" id="eref9"><a href="https://www.goldbio.com/articles/article/Introduction-to-Competent-Cells">https://www.goldbio.com/articles/article/Introduction-to-Competent-Cells</a></li>
								<li style="padding-left: 1em;" id="eref10"><a href="https://www.thermofisher.com/in/en/home/references/gibco-cell-culture-basics/transfection-basics/transfection-methods/electroporation.html">https://www.thermofisher.com/in/en/home/references/gibco-cell-culture-basics/transfection-basics/transfection-methods/electroporation.html</a> </li>
								<li style="padding-left: 1em;" id="eref11">Bruschi, M., Boyes, S. J., Sugiarto, H., Nielsen, L. K., & Vickers, C. E. (2012). A transferable sucrose utilization approach for non-sucrose-utilizing Escherichia coli strains. Biotechnology advances, 30(5), 1001–1010. <a href="https://doi.org/10.1016/j.biotechadv.2011.08.019">https://doi.org/10.1016/j.biotechadv.2011.08.019</a> </li>
								<li style="padding-left: 1em;"> Andrews J. M. (2001). Determination of minimum inhibitory concentrations. The Journal of antimicrobial chemotherapy, 48 Suppl 1, 5–16. <a href="https://doi.org/10.1093/jac/48.suppl_1.5">https://doi.org/10.1093/jac/48.suppl_1.5</a></li>
								<li style="padding-left: 1em;">Microbial Growth. (2021, January 4). Oregon State University <a href="https://bio.libretexts.org/@go/page/10667"></a>https://bio.libretexts.org/@go/page/10667</a>.</li>
							</ol>
							<br><br>
						</div>
						<!-------------------------------------- END OF E COLI ------------------------------------------>

					

						<!-------------------------------- START OF CYANOBACTERIA --------------------------------------->
						<div class="tab-pane fade cyano-wet" id="cyano-wet" role="tabpanel" aria-labelledby="cyano-wet-tab">
							<br><br><br>
							<p>The experiments below are presented in the order in which we performed them.</p>
							<br><br>
							<!-- Preparation of BG-11-->
							<h2>Preparation of BG-11</h2>
							<br><br>
							<h3>Objective</h3>
							<p>To prepare BG-11 media for culturing of cyanobacteria </p>
							<br>

							<h3>Working principle</h3>
							<p>BG-11 media is a mix of macronutrients and micronutrients without essentially a consumable carbon source. Since this media is used for culturing cyanobacteria which are capable of photosynthesizing, a consumable carbon source is not required. </p>
							<p>BG-11 mix has: sodium nitrate, dipotassium hydrogen phosphate, magnesium sulphate heptahydrate, calcium chloride dihydrate, citric acid, sodium carbonate, sodium EDTA, ferric ammonium citrate and micronutrients (trace metals) mix.</p>
							<p>Adjusting the pH of the media according to the optimal growth pH of the cyanobacteria is crucial for BG-11. It also so happens that post autoclaving the pH might change. In such cases it is always good to check the pH of an aliquot of sterile media before using.</p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;">BG-11 powder </li>
								<li style="padding-left: 1em;">Trace metal mix </li>
								<li style="padding-left: 1em;">1N HCl </li>
								<li style="padding-left: 1em;">1N NaOH </li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<ul class="b">
								<li style="padding-left: 1em;">For liquid media can be found <a href="https://dx.doi.org/10.17504/protocols.io.by6qpzdw">here</a></p></li>
								<li style="padding-left: 1em;">For Media agar can be found <a href="https://dx.doi.org/10.17504/protocols.io.by6rpzd6">here</a></li>
							</ul>
							<p> Can be found 
							<a id="c2"></a><br><hr width=50%><br><br>
							<!-- End of Preparation of BG-11-->



							<!-- Revival of cyanobacteria strains -->
							<!-- to be edited - protocol not link-->
							<h2>Revival of <i>S. elongatus</i> strains from plates</h2>
							<br><br>
							<h3>Objective</h3>
							<p>To revive UTEX 2973 (wild type, <i>cscB</i>, <i>cscB</i>-<i>spp</i>, <i>cscB</i>-sps-<i><i>spp</i></i>) from agar plates. </p>
							<br>

							<h3>Working principle</h3>
							<p>S. elongatus is often shipped in BG-11 agar plates patched with large amounts of cell culture. They can also be stored as cultures streaked on plates within a lab at 4oC. Cultures are revived by picking up the cells (a single colony or scraped biomass) and mixing them into liquid BG-11 medium and growing them in the incubator under low light conditions.</p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;">BG-11 medium </li>
								<br>
								<li style="padding-left: 1em;">Antibiotic markers (Kanamycin and Gentamycin)</li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<p> Can be found <a href="https://www.protocols.io/view/revival-of-s-elongatus-from-plates-bzbsp2ne">here</a></p>
							<a id="c3"></a><br><hr width=50%><br><br>
							<!-- End of Revival of cyanobacteria strains -->



							<!-- Making glycerol and DMSO stocks -->
							<!-- to be edited - protocol not link -->
							<h2>Making DMSO and Glycerol Stocks</h2>
							<br><br>
							<h3>Objective</h3>
							<p>To prepare glycerol and DMSO stocks of the cyanobacteria UTEX 2973 wild type and strain containing the <i>cscB</i> gene.</p>
							<br>

							<h3>Working principle</h3>
							<p>Bacterial cultures in the form of agar plates or liquid cultures are not suitable for long term storage at low temperatures such as 4oC. Long-term storage of bacteria is best at temperatures around -80oC. However, agar plates and liquid cultures cannot be stored at such temperatures, as the water crystallizes in and around the bacterial cells, rupturing and killing them. Glycerol or DMSO (anti-freezing agents) is added to the media to store samples at such temperatures. </p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;">UTEX 2973 WT and <i>cscB</i> 100 mL cultures </li>
								<li style="padding-left: 1em;">BG-11 Media (with micronutrients) </li>
								<li style="padding-left: 1em;">7% DMSO </li>
								<li style="padding-left: 1em;">25% glycerol</li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<p> Protocol for DMSO stock be found <a href="https://www.protocols.io/view/dmso-stock-preparation-bzbup2nw">here</a> and that for glycerol stock can be found <a href="https://www.protocols.io/view/glycerol-stock-preparation-for-s-elongatus-bzb4p2qw">here</a></p>
							<a id="c4"></a><br><hr width=50%><br><br>
							<!-- End of Making glycerol and DMSO stocks-->



							<!--Making plates from DMSO stocks-->
							<!-- Protocol not link -->
							<h2>Revival of <i>S. elongatus</i> strains from glycerol or DMSO stocks</h2>
							<br><br>
							<h3>Objective</h3>
							<p>To revive liquid cultures from glycerol or DMSO stocks of <i>S. elongatus</i> strains. </p>
							<br>

							<h3>Working principle</h3>
							<p>Stocks are more permanent methods of storing bacteria than plates. These stocks can be recovered by pouring them into small volumes of BG-11 and growing under low light levels. As the culture grows, it can be passaged into higher volumes and appropriate antibiotics can be added.</p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;">BG-11 </li>
								<li style="padding-left: 1em;">Kanamycin and Gentamycin </li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<p> Can be found <a href="https://www.protocols.io/view/s-elongatus-stock-revival-bzcxp2xn">here</a></p>
							<a id="c5"></a><br><hr width=50%><br><br>
							<!-- End of Making plates from DMSO stocks -->



							<!-- Cyanobacteria growth curve under salt stress -->
							<!-- protocol not link -->
							<h2>Growth Curves of <i>S. elongatus</i> under salt stress </h2>
							<br><br>
							<h3>Objective</h3>
							<p>To perform growth curves of UTEX 2973 and to validate sucrose production with respect to the Optical Density at 600 nm. </p>
							<br>

							<h3>Working principle</h3>
							<p>When liquid media is inoculated with bacteria and the cell population is counted at intervals, it is possible to plot a typical bacterial growth curve that shows the increase in the number of cells over time. Such growth curves show four distinct phases of growth:<sup><a href="#cref1">[1]</a></sup></p>
							<br>

							<p>Lag phase: There is a slow growth or lack of growth due to the physiological adaptation of cells to culture conditions or the dilution of exoenzymes (due to initial low cell densities).
								Log or exponential phase: Optimal growth rates are seen in this phase. Cell numbers double at discrete time intervals known as the mean generation time.
								Stationary phase: During this phase, the growth (cell division) and death of cells occur at the same rate, resulting in the number of cells being constant. The reduced growth rate is usually due to a lack of nutrients and/or a buildup of toxic waste constituents.
								Decline or death phase: Here, the death rate exceeds the growth rate, resulting in a net loss of viable cells.
							</p>
							<br>

							<p>This is one of the simplest methods used to analyze trends in growth because it uses a spectrophotometer to track changes in the optical density (OD) over time. In other words, as the number of cells in a sample increases, the transmission of light through the sample will decrease.<sup><a href="#cref2">[2]</a></sup></p>
							<br>

							<p>Growth curves for certain freshwater cyanobacterial species are carried out under salt stress to account for sucrose production in the particular strain. This is because sucrose is naturally produced intracellularly in these strains to balance the osmotic pressure of a saline environment.<sup><a HREF="#cref3">[3]</a></sup></p>
							<br>

							<p>This experiment was carried out in two iterations: one at 0.04% of carbon dioxide -  i.e, ambient carbon dioxide from the atmosphere - and the other with bicarbonate added to be equivalent to 0.6% of carbon dioxide. Find our results for these iterations <a href="#Results.html">here</a>.</p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;">2&#176 cyanobacteria cultures </li>
								<li style="padding-left: 1em;">5 mL of 150 mM NaCl </li>
								<li style="padding-left: 1em;">Distilled water </li>
								<br>
								<li style="padding-left: 1em;">Photospectrometer </li>
								<li style="padding-left: 1em;">Quartz/plastic (disposable) cuvette </li>
								<li style="padding-left: 1em;">1 mL microcentrifuge tubes </li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<p> The protocols for this experiment are <a href="https://www.protocols.io/view/growth-curves-of-s-elongatus-under-salt-stress-and-bzcap2se">here</a> (for the first iteration) and <a href="https://www.protocols.io/view/growth-curves-of-s-elongatus-under-salt-stress-and-bzc2p2ye">here</a> (for the second iteration).</p>
							<a id="c6"></a><br><hr width=50%><br><br>
							<!-- End of Cyanobacteria growth curve under salt stress -->



							<!-- Sucrose assay -->
							<!-- do i put this here or in co culture -->
							<h2>Sucrose assay </h2>
							<br><br>
							<h3>Objective</h3>
							<p>To measure the production of sucrose by <i>S. elongatus</i>  UTEX 2973 WT and cscB strains.</p>
							<br>

							<h3>Working principle</h3>
							<p>The sucrose assay is a colorimetric assay<sup><a href="#cref4">[4]</a></sup> to quantify sucrose concentration in cell supernatants. Supernatant samples from the cultures are taken at regular frequencies. An increasing trend in the absorbance of successive samples after treatment is indicative of increasing sucrose concentration, i.e, net sucrose production has been taking place. </p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;">Distilled water </li>
								<li style="padding-left: 1em;">D-Glucose standard solution (5 mL, 1.0 mg/mL) in 0.2% (w/v) benzoic acid </li>
								<li style="padding-left: 1em;">GOPOD Reagent Enzymes. Glucose oxidase plus peroxidase and 4-aminoantipyrine. Freeze-dried powder </li>
								<li style="padding-left: 1em;">GOPOD Reagent Buffer. p-hydroxybenzoic acid and sodium azide (0.095% w/v) </li>
								<li style="padding-left: 1em;">β-Fructosidase (invertase) solution </li>
								<li style="padding-left: 1em;">Acetate buffer</li>
								<br>
								<li style="padding-left: 1em;">96 well plate </li>
								<li style="padding-left: 1em;">1 mL Eppendorf tubes </li>
								<li style="padding-left: 1em;">Spectrophotometer set at 510 nm </li>
								<li style="padding-left: 1em;">Vortex mixer </li>
								<li style="padding-left: 1em;">Thermostated water bath </li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<p> Can be found <a href="https://www.protocols.io/view/megazyme-sucrose-d-glucose-assay-kit-k-sucgl-by5xpy7n">here</a></p>
							<a id="c7"></a><br><a id="cref1"></a><br><a id="cref2"></a><br><a id="cref3"></a>
							<!-- End of Sucrose assay -->



							<!-- REFERENCES -->
							<hr>
							<h3 id="reference">References</h3>
							<ol class="a">
								<li style="padding-left: 1em;" id="cref4">Microbial Growth. (2021, January 4). Oregon State University. <a href="https://bio.libretexts.org/@go/page/10667">https://bio.libretexts.org/@go/page/1066</a></li>
								<li style="padding-left: 1em;"><a href="https://microbenotes.com/bacterial-growth-curve-protocol">https://microbenotes.com/bacterial-growth-curve-protocol</a></li>
								<li style="padding-left: 1em;">Po-Cheng Lin, Fuzhong Zhang & Himadri B. Pakrasi; Enhanced production of sucrose in the fast-growing cyanobacterium <i>Synechococcus elongatus</i> UTEX 2973</li>
								<li style="padding-left: 1em;"><a href="https://www.megazyme.com/documents/Assay_Protocol/K-SUCGL_DATA.pdf"> https://www.megazyme.com/documents/Assay_Protocol/K-SUCGL_DATA.pdf</a> </li>
							</ol>
							<!-- END OF REFERECNES -->
						</div>
						<!-------------------------------- END OF CYANOBACTERIA ------------------------------------------->


						
						<!----------------------------------- START OF CO CULTURE ------------------------------------------>
						<div class="tab-pane fade coculture-wet" id="coculture-wet" role="tabpanel" aria-labelledby="coculture-wet-tab">
							<br><br><br>
							<p>THe experiments below are presented in the order in which we performed them.</p>
							<br><br>
							<!-- Transformation of KJK01 with pKD4 -->
							<!-- complete -->
							<h2>Transformation of KJK01 with pKD4 </h2>
							<br><br>
							<h3>Objective</h3>
							<p>To transform <i>E. coli</i> KJK01 with plasmid pKD4 </p>
							<br>

							<h3>Working principle<sup><a href="#ccref1">[1]</a></sup></h3>
							<p>Transformation via electroporation is a physical transformation method. Host cells and nucleic acids are suspended in a conductive solution, and an electrical circuit is closed around the mixture. An electrical pulse at an optimized voltage lasting a few microseconds to a millisecond is discharged through the cell suspension. This disturbs the phospholipid bilayer of the membrane and results in the formation of temporary pores through which substances like nucleic acids can pass into cells. The electric potential across the cell membrane simultaneously rises to allow charged molecules like DNA to be driven across the membrane through the pores. This method can transform a large number of cells in a short period of time as it is easy and rapid.</p>
							<br>
							<p>pKD4 is a blank plasmid containing Amp and Kan/Neo resistance genes<sup><a href="#ccref2">[2]</a></sup>. We aim to transform KJK01 strain with this plasmid as a negative control for our BG-11 + sucrose growth curve experiments. If this strain does not grow in BG-11 + sucrose, it shows that  any sucrose consuming properties that have been conferred upon the KJK01 strain is solely via the pCSCX plasmid. It also minimizes the risk of contamination of the KJK01 cultures.</p>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;">Electrocompetent cells </li>
								<li style="padding-left: 1em;">Plasmid DNA </li>
								<li style="padding-left: 1em;">2X LB </li>
								<li style="padding-left: 1em;">MilliQ water </li>
								<br>
								<li style="padding-left: 1em;">BioRad Pulser </li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<p> Can be found <a href="https://www.protocols.io/view/transformation-of-e-coli-with-plasmid-byz3px8n">here</a></p>
							<a id="cc2"></a><br><hr width=50%><br><br>
							<!-- End of Transformation of MG1655 with pKD4 -->



							<!-- Preparation of CoBG-11 -->
							<!-- protocol not linked -->
							<h2>Preparation of CoBG-11 </h2>
							<br><br>
							<h3>Objective</h3>
							<p>To prepare CoBG-11 media for coculture experiments of cyanobacteria and <i>E. coli</i> </p>
							<br>

							<h3>Working principle</h3>
							<p>CoBG-11 media is a mix of BG-11 with some salts whose concentrations have been optimized for <i> E. coli</i> growth by Zhang et al.<sup><a href="#ccref3">[3]</a></sup> </p>
							<br>

							<p>CoBG-11 has the following composition:</p>
							
							<ol class="a">
								<li style="padding-left: 1em;">150 mM NaCl </li>
								<li style="padding-left: 1em;">4 mM NH<sub>4</sub>Cl </li>
								<li style="padding-left: 1em;">3 g/L 2-[[1,3-dihydroxy-2-(hydroxymethyl) propan-2-yl] amino] ethanesulfonic acid (TES</li>
							</ol>
							<br>
							<p> The pH value is adjusted with NaOH to 8.3.</p>
							<br>

							<p>It is not recommended to use TESna as a replacement for TES as in the event that the pH of your solution is higher than 8.3, you will need to add HCl to reduce the pH. This, in turn, adds more chloride to the medium.</p>
							<br>

							<p>This is not a concern while preparing CoBG-11 in bulk, as only a small amount of HCl might be needed and this will not affect the chloride concentration significantly.</p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;">BG-11 medium </li>
								<li style="padding-left: 1em;">150mM NaCl </li>
								<li style="padding-left: 1em;">4 mM NH<sub>4</sub>Cl </li>
								<li style="padding-left: 1em;">3 g/L (TES) </li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<p> Can be found <a href="">here</a></p>
							<a id="cc3"></a><br><hr width=50%><br><br>
							<!-- End of Preparation of CoBG-11 -->
							


							<!-- Growth curve of KJK01 and pCSCX-KJK01 in BG-11 + sucrose -->
							<!-- Links to pages need to be fixed -->
							<h2>Growth curve of KJK01 and pCSCX-KJK01 in BG-11 + sucrose </h2>
							<br><br>
							<h3>Objective</h3>
							<p> To obtain growth curves of KJK01 and pCSCX-KJK01 in BG-11 + sucrose and characterizing their growth, as well as determining the rate of sucrose consumption and rate of butanol production by both strains.</p>
							<br>

							<h3>Working principle</h3>
							<p>When liquid media is inoculated with bacteria and the cell population is counted at intervals, it is possible to plot a typical bacterial growth curve that shows the increase in the number of cells over time. Such growth curves show four distinct phases of growth:</p>
							<ul class="b">
								<li style="padding-left: 1em;"> Lag phase: Slow growth or lack of growth due to physiological adaptation of cells to culture conditions or dilution of exoenzymes due to initial low cell densities.</li>
								<li style="padding-left: 1em;"> Log or exponential phase: Optimal growth rates, during which cell numbers double at discrete time intervals known as the mean generation time.</li>
								<li style="padding-left: 1em;"> Stationary phase: Growth (cell division) and death of cells counterbalance each other resulting in no net increase in cell numbers. The reduced growth rate is usually due to a lack of nutrients and/or a buildup of toxic waste constituents.</li>
								<li style="padding-left: 1em;"> Decline or death phase: Death rate exceeds growth rate resulting in a net loss of viable cells.</li>
							</ul>
							<br>
							<p>This is one of the simplest methods used to analyze trends in growth because it uses a spectrophotometer to track changes in the optical density (OD) over time. In other words, as the number of cells in a sample increases, the transmission of light through the sample will decrease.</p>
							<br>
							
							<p>Here we carry out the growth curve in BG-11 + sucrose as a co-culture proxy. This checks if the pCSCX plasmid allows the <i>E. coli</i> to survive when sucrose is the only carbon source available, as BG-11 does not contain any carbon source of its own. The concentration of sucrose used in the experiment is 250 mg/L, which is more than the maximum concentration secreted by the cyanobacteria in Zhang et al, 2020<sup><a href="#ccref3">[3]</a></sup>. After two failed attempts, we carried out the experiment using CoBG-1<sup><a href="#ccref3">[3]</a></sup> from the same paper. The medium contains salts that aid the growth of <i>E. coli</i>.</p>
							<!-- Links need to be inserted in next <p>-->

							<p>The supernatants of the sample taken for OD measurement were reserved for <a href="#cc4">sucrose assay</a>, to determine the rate of sucrose consumption and for <a href="#e5">NMR analysis</a>  to determine the rate of butanol production. Thus, this assay serves three purposes.</p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;"> 1&#176 cultures of KJK01 and pCSCX-KJK01</li>
								<li style="padding-left: 1em;"> BG-11/CoBG-11</li>
								<li style="padding-left: 1em;"> 250 mg/100 mL sucrose solution</li>
								<li style="padding-left: 1em;"> IPTG</li>
								<li style="padding-left: 1em;"> Kan50 and Amp100</li>
								<br>
								<li style="padding-left: 1em;"> Photospectrometer</li>
								<li style="padding-left: 1em;"> Quartz/plastic (disposable) cuvette</li>
								<li style="padding-left: 1em;"> Centrifuge</li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<p> Can be found <a href="https://www.protocols.io/view/e-coli-growth-curve-assay-in-bg-11-sucrose-by5apy2e">here</a></p>
							<a id="cc4"></a><br><hr width=50%><br><br>
							<!-- End of Growth curve of KJK01 and pCSCX-KJK01 in BG-11 + sucrose -->



							<!-- Sucrose assay -->
							<!-- complete -->
							<h2><i>E. coli</i> and cyanobacteria sucrose assay </h2>
							<br><br>
							<h3>Objective</h3>
							<p>To measure the consumption of sucrose by <i>E. coli</i> KJK01 and pCSCX-KJK01 in BG-11.</p>
							<br>

							<h3>Working principle<sup><a href="#ccref4">[4]</a></sup></h3>
							<p>Free D-glucose in the sample extract is determined by conversion to a red coloured quinoneimine dye compound through the action of glucose oxidase and peroxidase at pH 7.4, and employing p-hydroxybenzoic acid and 4-aminoantipyrine. The D-glucose content is estimated by measuring the absorbance of the dye at 510 nm.  </p>
							<br>

							<p>At pH 4.6, sucrose is hydrolysed by the enzyme β-fructosidase to D-glucose and D-fructose. The determination of D-glucose after inversion by β-fructosidase (total D-glucose) is carried out simultaneously according to the principle outlined above. The sucrose content is calculated from the difference of the D-glucose concentrations before and after enzymatic inversion.</p>
							<br>

							<h3>Key Reagents/Apparatus</h3>
							<ul class="a">
								<li style="padding-left: 1em;">Distilled water </li>
								<li style="padding-left: 1em;">D-Glucose standard solution (5 mL, 1.0 mg/mL) in 0.2% (w/v) benzoic acid </li>
								<li style="padding-left: 1em;">GOPOD Reagent Enzymes. Glucose oxidase plus peroxidase and 4-aminoantipyrine. Freeze-dried powder </li>
								<li style="padding-left: 1em;">GOPOD Reagent Buffer. p-hydroxybenzoic acid and sodium azide (0.095% w/v) </li>
								<li style="padding-left: 1em;">β-Fructosidase (invertase) solution </li>
								<li style="padding-left: 1em;">Acetate buffer</li>
								<br>
								<li style="padding-left: 1em;">96 well plate </li>
								<li style="padding-left: 1em;">1 mL Eppendorf tubes </li>
								<li style="padding-left: 1em;">Spectrophotometer set at 510 nm </li>
								<li style="padding-left: 1em;">Vortex mixer </li>
								<li style="padding-left: 1em;">Thermostated water bath </li>
							</ul>
							<br>

							<h3>Protocol</h3>
							<p> Can be found <a href="http://dx.doi.org/10.17504/protocols.io.by5xpy7n">here</a></p>
							<a id="cc5"></a><br><a id="ccref1"></a><br><a id="ccref2"></a><br><a id="ccref3"></a>
							<!-- End of Sucrose assay -->


							<!-- References -->
							<hr>
							<h3 id="reference">References</h3>
							<ol class="a">
								<li style="padding-left: 1em;"><a href="https://www.thermofisher.com/in/en/home/references/gibco-cell-culture-basics/transfection-basics/transfection-methods/electroporation.html">https://www.thermofisher.com/in/en/home/references/gibco-cell-culture-basics/transfection-basics/transfection-methods/electroporation.html</a> </li>
								<li style="padding-left: 1em;">Datsenko, K. A., & Wanner, B. L. (2000). One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proceedings of the National Academy of Sciences of the United States of America, 97(12), 6640–6645. <a href="https://doi.org/10.1073/pnas.120163297"></a>https://doi.org/10.1073/pnas.120163297</li>
								<li style="padding-left: 1em;">Zhang, L., Chen, L., Diao, J. et al. Construction and analysis of an artificial consortium based on the fast-growing cyanobacterium Synechococcus elongatus UTEX 2973 to produce the platform chemical 3-hydroxypropionic acid from CO2. Biotechnol Biofuels 13, 82 (2020). <a href="https://doi.org/10.1186/s13068-020-01720-0"></a>https://doi.org/10.1186/s13068-020-01720-0 </li>
								<li style="padding-left: 1em;"><a href="https://www.megazyme.com/sucrose-d-glucose-assay-kit">https://www.megazyme.com/sucrose-d-glucose-assay-kit</a> </li>
							</ol>

						</div>
						<!-------------------------------- END OF CO CULTURE ----------------------------------------------->
					</div>

				</div>
			</div>

        </div>
    </div>
	<!----------------------------------------------------- END OF CONTENT ---------------------------------------------->




	<!------------------------------------------------------------ SPONSORS BAR ----------------------------------------->
	<div class="footer">
		<div class="footer-wrapper">
			<!-- <div id="not-sponsors" class="footer-wrapper">	 -->
				<div id="Team", class="footer-wrapper">
					<img class="teamlogo" src="assets/Logos/IISER-Pune-iGEM_logo.png" alt="iGEM IISER Pune Logo" width="170">
				</div>
				<div id="socials" class="vertical-stack">
					<div><h3>Team IISER Pune India</h3></div>
					<div id="logos" class ="footer-wrapper ">
						<a href="https://www.instagram.com/igem_iiserp/"><img class="footer-links-socials" src="assets/Logos/Instagram_logo.jpg" alt="Instagram" width="20"></a>
						<a href="https://www.linkedin.com/in/igem-iiser-pune-2021/"><img class="footer-links-socials" src="assets/Logos/linkedin_logo.png" alt="Linkedin" width="30"></a>
						<a href="https://twitter.com/igem_iiserpune"><img class="footer-links-socials" src="assets/Logos/Twitter_logo.png" alt="Twitter" width="35"></a>
						<a href="https://www.facebook.com/igemiiserpune"><img class="footer-links-socials" src="assets/Logos/Facebook_logo.png" alt="Facebook" width="25"></a>
						<a href="https://www.youtube.com/channel/UCyo6NMLvM58sSh_tnxht36w"><img class="footer-links-socials" src="assets/Logos/Youtube_logo.png" alt="Youtube" width="35"></a>
					</div>
				</div>
			<!-- </div> -->
			<div id="sponsors" class="footer-wrapper">
				<div id="main-sponsor", class="">
					<a href="https://www.sailife.com/"><img class="footer-links" src="assets/Logos/Sai_logo.jpg" alt="Sai Life Sciences" width="280"></a>
				</div>
				<div class="vertical-stack">
					<a href="https://sg.idtdna.com/pages"><img class="footer-links" src="assets/Logos/IDT_logo.png" alt="IDT" width="220"></a>
					<a href="https://www.geneious.com/"><img class="footer-links" src="assets/Logos/Geneious_logo.png" alt="Geneious" width="220"></a>
				</div>
				<div class="vertical-stack">
					<div id="horizontal stack">
						<a href="https://www.notion.so/"><img class="footer-links" src="assets/Logos/Notion_logo.png" alt="Notion" width="70"></a>
						<img src="assets/Logos/IISER Pune Logo.png" alt="IISER Pune" width="80" style="padding-left: 1em;">
					</div>
					<a href="https://www.zoho.com/"><img class="footer-links" src="assets/Logos/Zoho_logo.png" alt="Zoho" width="150"></a>
				</div>
			</div>	
		</div>
	</div>
	<!------------------------------------------------- END OF SPONSOR BAR------------------------------------------------> 




	<!-------------------------------------------- ACTUAL SCROLLING BAR------------------------------------------------->
	<div class="header"></div>
    <div class="line" id="scrollIndicator"></div>

	<!-- Gets the screenheight of each page for Scrollbar? -->
	<script>
		var screenheight = window.innerHeight,
		    screenwidth = window.innerWidth;

	</script>


	<!-- Scrollbar stuff again, only @Namasi22 knows :) -->
	<script>
		var body = document.body,
    		html = document.documentElement;

		var pageheight = Math.max( body.scrollHeight, body.offsetHeight, 
                       		   html.clientHeight, html.scrollHeight, html.offsetHeight );
	</script>


	<!-- To make the button appear -->
	<script>
		var backtotopheight=pageheight/4.65
	</script>



    <!------------------------------------------------- back to top button ------------------------------------------->
    <a href="#top"><img src="assets/pngs/Back-to-top.png" id="myBtn" title="Go to top" width="100" height="100" class="footer-links"></a>

	
	<!------------------------------------------- LINKING MAIN JS FILES ----------------------------------------------->
    <script src="js/bootstrap-bundle.js"></script>
	<script src="js/main.js"></script>
	<script src="js/jquery.min.js"></script>




	<!------------------------- Displaying sidebar and content tab depending on the tab press---------------------------->
	<script>
		document.querySelectorAll('button[data-bs-toggle="tab"]').forEach((t,i)=>{
			t.addEventListener('show.bs.tab', function (e) {

				let targetClass = t.dataset.bsTarget;
				var sidepane = document.querySelector('#SidenavTabs ' + targetClass);
				var contentpane = document.querySelector('#myTabContent ' + targetClass);
				var sidesibling = document.querySelector('#SidenavTabs .tab-pane.active');
				var contentsibling = document.querySelector('#myTabContent .tab-pane.active');
				var heightpane = document.querySelector('.tab-content.sidenav');


				// hide 2nd pane sibling
				sidesibling.classList.remove('show');
				sidesibling.classList.remove('active');
				contentsibling.classList.remove('show');
				contentsibling.classList.remove('active');

				// show 2nd pane
				sidepane.classList.add('show');
				sidepane.classList.add('active');
				contentpane.classList.add('show');
				contentpane.classList.add('active');

				if ($('#ecoli-wet').hasClass('active')) {
					heightpane.style.top = '-33%';
				} else if ($('#cyano-wet').hasClass('active')) {
					heightpane.style.top = '-24%';
				} if ($('#coculture-wet').hasClass('active')) {
					heightpane.style.top = '-18%';
				} 

			})  
		})
	</script>
	<!------------------------------------------ End of display script ------------------------------------------>




	<!-- SCRIPT FOR SCROLL AWARE SIDEBAR HIGHLIGHT -->
	<!-- ------------------------------------------------------------------------------------------------------------- -->
	<!-- MAKE SURE YOU CHANGE THE NUMBER OF DIVISIONS -->
	<script>
		
		$(document).ready(function() {
			$(window).scroll(function() {

				if($('#ecoli-wet').hasClass('active')) {
					var scroll_top = $(window).scrollTop();
					var div_one_top = $("#e1").offset().top;
					var div_two_top = $("#e2").offset().top;
					var div_three_top = $("#e3").offset().top;
					var div_four_top = $("#e4").offset().top;
					var div_five_top = $("#e5").offset().top;
					var div_six_top = $("#e6").offset().top;
					var div_seven_top = $("#e7").offset().top;
					var div_eight_top = $("#e8").offset().top;
					var div_nine_top = $("#e9").offset().top;
					var div_ten_top = $("#e10").offset().top;
					var div_eleven_top = $("#e11").offset().top;
		
					if (scroll_top > div_one_top && scroll_top < div_two_top) {
						//Division 1
						if (!$('a[href="#e1"]').hasClass('sidenav-active')) {
							$('a[href="#e1"]').addClass('sidenav-active');
						}
		
						if ($('a[href="#e2"]').hasClass('sidenav-active')) {
							$('a[href="#e2"]').removeClass('sidenav-active');
						}
		
					} else if (scroll_top > div_two_top && scroll_top < div_three_top) {
						// Division 2
						if ($('a[href="#e1"]').hasClass('sidenav-active')) {
							$('a[href="#e1"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#e3"]').hasClass('sidenav-active')) {
							$('a[href="#e3"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#e2"]').hasClass('sidenav-active')) {
							$('a[href="#e2"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_three_top && scroll_top < div_four_top) {
						//Division 3
						if ($('a[href="#e2"]').hasClass('sidenav-active')) {
							$('a[href="#e2"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#e4"]').hasClass('sidenav-active')) {
							$('a[href="#e4"]').removeClass('sidenav-active');
						} 
		
						if (!$('a[href="#e3"]').hasClass('sidenav-active')) {
							$('a[href="#e3"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_four_top && scroll_top < div_five_top) {
						// You are in div 4
						if ($('a[href="#e3"]').hasClass('sidenav-active')) {
							$('a[href="#e3"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#e5"]').hasClass('sidenav-active')) {
							$('a[href="#e5"]').removeClass('sidenav-active');
						} 
		
						if (!$('a[href="#e4"]').hasClass('sidenav-active')) {
							$('a[href="#e4"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_five_top && scroll_top < div_six_top) {
						// You are in div 5
						if ($('a[href="#e4"]').hasClass('sidenav-active')) {
							$('a[href="#e4"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#e6"]').hasClass('sidenav-active')) {
							$('a[href="#e6"]').removeClass('sidenav-active');
						} 
		
						if (!$('a[href="#e5"]').hasClass('sidenav-active')) {
							$('a[href="#e5"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_six_top && scroll_top < div_seven_top) {
						// You are in div 6
						if ($('a[href="#e5"]').hasClass('sidenav-active')) {
							$('a[href="#e5"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#e7"]').hasClass('sidenav-active')) {
							$('a[href="#e7"]').removeClass('sidenav-active');
						} 
		
						if (!$('a[href="#e6"]').hasClass('sidenav-active')) {
							$('a[href="#e6"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_seven_top && scroll_top < div_eight_top) {
						// You are in div 7
						if ($('a[href="#e6"]').hasClass('sidenav-active')) {
							$('a[href="#e6"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#e7"]').hasClass('sidenav-active')) {
							$('a[href="#e7"]').addClass('sidenav-active');
						}

						if ($('a[href="#e8"]').hasClass('sidenav-active')) {
							$('a[href="#e8"]').removeClass('sidenav-active');
						}
					} else if (scroll_top > div_eight_top && scroll_top < div_nine_top) {
						//Division 8
						if ($('a[href="#e7"]').hasClass('sidenav-active')) {
							$('a[href="#e7"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#e8"]').hasClass('sidenav-active')) {
							$('a[href="#e8"]').addClass('sidenav-active');
						}

						if ($('a[href="#e9"]').hasClass('sidenav-active')) {
							$('a[href="#e9"]').removeClass('sidenav-active');
						}
					} else if (scroll_top > div_nine_top && scroll_top < div_ten_top) {
						//Divison 9 
						if ($('a[href="#e8"]').hasClass('sidenav-active')) {
							$('a[href="#e8"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#e9"]').hasClass('sidenav-active')) {
							$('a[href="#e9"]').addClass('sidenav-active');
						}

						if ($('a[href="#e10"]').hasClass('sidenav-active')) {
							$('a[href="#e10"]').removeClass('sidenav-active');
						}
					} else if (scroll_top > div_ten_top && scroll_top < div_eleven_top) {
						if ($('a[href="#e9"]').hasClass('sidenav-active')) {
							$('a[href="#e9"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#e10"]').hasClass('sidenav-active')) {
							$('a[href="#e10"]').addClass('sidenav-active');
						}

						if ($('a[href="#e11"]').hasClass('sidenav-active')) {
							$('a[href="#e11"]').removeClass('sidenav-active');
						}
					} else if (scroll_top > div_eleven_top) {
						if ($('a[href="#e10"]').hasClass('sidenav-active')) {
							$('a[href="#e10"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#e11"]').hasClass('sidenav-active')) {
							$('a[href="#e11"]').addClass('sidenav-active');
						}
					}
				} 
				else if ($('#cyano-wet').hasClass('active')) {
					var scroll_top = $(window).scrollTop();
					var div_one_top = $("#c1").offset().top;
					var div_two_top = $("#c2").offset().top;
					var div_three_top = $("#c3").offset().top;
					var div_four_top = $("#c4").offset().top;
					var div_five_top = $("#c5").offset().top;
					var div_six_top = $("#c6").offset().top;
					var div_seven_top = $("#c7").offset().top;
		
					if (scroll_top > div_one_top && scroll_top < div_two_top) {
						//Division 1
						if (!$('a[href="#c1"]').hasClass('sidenav-active')) {
							$('a[href="#c1"]').addClass('sidenav-active');
						}
		
						if ($('a[href="#c2"]').hasClass('sidenav-active')) {
							$('a[href="#c2"]').removeClass('sidenav-active');
						}
		
					} else if (scroll_top > div_two_top && scroll_top < div_three_top) {
						// Division 2
						if ($('a[href="#c1"]').hasClass('sidenav-active')) {
							$('a[href="#c1"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#c3"]').hasClass('sidenav-active')) {
							$('a[href="#c3"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#c2"]').hasClass('sidenav-active')) {
							$('a[href="#c2"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_three_top && scroll_top < div_four_top) {
						//Division 3
						if ($('a[href="#c2"]').hasClass('sidenav-active')) {
							$('a[href="#c2"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#c4"]').hasClass('sidenav-active')) {
							$('a[href="#c4"]').removeClass('sidenav-active');
						} 
		
						if (!$('a[href="#c3"]').hasClass('sidenav-active')) {
							$('a[href="#c3"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_four_top && scroll_top < div_five_top) {
						// You are in div 4
						if ($('a[href="#c3"]').hasClass('sidenav-active')) {
							$('a[href="#c3"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#c5"]').hasClass('sidenav-active')) {
							$('a[href="#c5"]').removeClass('sidenav-active');
						} 
		
						if (!$('a[href="#c4"]').hasClass('sidenav-active')) {
							$('a[href="#c4"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_five_top && scroll_top < div_six_top) {
						// You are in div 5
						if ($('a[href="#c4"]').hasClass('sidenav-active')) {
							$('a[href="#c4"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#c6"]').hasClass('sidenav-active')) {
							$('a[href="#c6"]').removeClass('sidenav-active');
						} 
		
						if (!$('a[href="#c5"]').hasClass('sidenav-active')) {
							$('a[href="#c5"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_six_top && scroll_top < div_seven_top) {
						// You are in div 6
						if ($('a[href="#c5"]').hasClass('sidenav-active')) {
							$('a[href="#c5"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#c7"]').hasClass('sidenav-active')) {
							$('a[href="#c7"]').removeClass('sidenav-active');
						} 
		
						if (!$('a[href="#c6"]').hasClass('sidenav-active')) {
							$('a[href="#c6"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_seven_top) {
						// You are in div 7
						if ($('a[href="#c6"]').hasClass('sidenav-active')) {
							$('a[href="#c6"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#c7"]').hasClass('sidenav-active')) {
							$('a[href="#c7"]').addClass('sidenav-active');
						}
					}
				}
				else if ($('#coculture-wet').hasClass('active')) {
					var scroll_top = $(window).scrollTop();
					var div_one_top = $("#cc1").offset().top;
					var div_two_top = $("#cc2").offset().top;
					var div_three_top = $("#cc3").offset().top;
					var div_four_top = $("#cc4").offset().top;
					var div_five_top = $("#cc5").offset().top;
		
					if (scroll_top > div_one_top && scroll_top < div_two_top) {
						//Division 1
						if (!$('a[href="#cc1"]').hasClass('sidenav-active')) {
							$('a[href="#cc1"]').addClass('sidenav-active');
						}
		
						if ($('a[href="#cc2"]').hasClass('sidenav-active')) {
							$('a[href="#cc2"]').removeClass('sidenav-active');
						}
		
					} else if (scroll_top > div_two_top && scroll_top < div_three_top) {
						// Division 2
						if ($('a[href="#cc1"]').hasClass('sidenav-active')) {
							$('a[href="#cc1"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#cc3"]').hasClass('sidenav-active')) {
							$('a[href="#cc3"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#cc2"]').hasClass('sidenav-active')) {
							$('a[href="#cc2"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_three_top && scroll_top < div_four_top) {
						//Division 3
						if ($('a[href="#cc2"]').hasClass('sidenav-active')) {
							$('a[href="#cc2"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#cc4"]').hasClass('sidenav-active')) {
							$('a[href="#cc4"]').removeClass('sidenav-active');
						} 
		
						if (!$('a[href="#cc3"]').hasClass('sidenav-active')) {
							$('a[href="#cc3"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_four_top && scroll_top < div_five_top) {
						// You are in div 4
						if ($('a[href="#cc3"]').hasClass('sidenav-active')) {
							$('a[href="#cc3"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#cc4"]').hasClass('sidenav-active')) {
							$('a[href="#cc4"]').addClass('sidenav-active');
						} 

						if ($('a[href="#cc5"]').hasClass('sidenav-active')) {
							$('a[href="#cc5"]').removeClass('sidenav-active');
						}

					} else if (scroll_top > div_five_top) {
						// You are in div 4
						if ($('a[href="#cc4"]').hasClass('sidenav-active')) {
							$('a[href="#cc4"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#cc5"]').hasClass('sidenav-active')) {
							$('a[href="#cc5"]').addClass('sidenav-active');
						} 
					}
				}	
				else if ($('#overview-wet').hasClass('active')) {
					var scroll_top = $(window).scrollTop();
					var div_one_top = $("#o1").offset().top;
					var div_two_top = $("#o2").offset().top;
		
					if (scroll_top > div_one_top && scroll_top < div_two_top) {
						//Division 1
						if (!$('a[href="#o1"]').hasClass('sidenav-active')) {
							$('a[href="#o1"]').addClass('sidenav-active');
						}
		
						if ($('a[href="#o2"]').hasClass('sidenav-active')) {
							$('a[href="#o2"]').removeClass('sidenav-active');
						}
		
					} else if (scroll_top > div_two_top) {
						// Division 2
						if ($('a[href="#o1"]').hasClass('sidenav-active')) {
							$('a[href="#o1"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#o2"]').hasClass('sidenav-active')) {
							$('a[href="#o2"]').addClass('sidenav-active');
						}
					}
				}
			});
		});
		
	</script>	
	<!-------------------------------------------- END OF SCROLL AWARE HIGHLIGHT --------------------------------------->
	


	<!-- Script for the slideshow, wherever needed -->
	<script>
		var total = $('.item').length;
		var actualtotal = 80;
		var currentIndex = $('div.active').index() + 1;
		$('#slidetext').html(currentIndex + '/'  + actualtotal);

		// This triggers after each slide change
		$('.carousel').on('slid.bs.carousel', function () {
  			currentIndex = $('div.active').index() + 1;
 
  		// Now display this wherever you want
  		var text = currentIndex + '/' + actualtotal;
  			$('#slidetext').html(text);
		});

	</script>




</body>
