<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Model</title>
	<link rel="stylesheet" href="css/igem-reset.css">
	<link rel="stylesheet" href="css/SBcss/style.css">
   	<link rel="stylesheet" href="css/styleSB.css">
    <link rel="stylesheet" href="css/DryLabPage.css">
	<style>/* Reset CSS */

		#sideMenu, #top_title, .patrollink, #firstHeading, #home_logo, #sideMenu { display:none; }
		#content { padding:0px; width:100%; margin-top:-7px; margin-left:0px; border:none;}
		body, html {background-color:#212529; width: 100%; height: 100%;}
		#bodyContent h1, #bodyContent h2, #bodyContent h3, #bodyContent h4, #bodyContent h5 { margin-bottom: 0px; }
		#bodyContent a[href ^="https://"], .link-https { padding-right:0px;}

    </style>
<body>
    



	<!-- NAVBAR SECTION -->

	<nav class="navbar fixed-top navbar-expand-lg navbar-light menu">
		<div class="container-fluid nav-container">
		  <a class="navbar-brand" href="#">
			<img src="assets/svgs/synbactory2.svg">
		  </a>
		  <div class="center-expand">
		</div>
		  <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		  <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
		    <ul class="navbar-nav">
		      <li class="nav-item">
			<a class="nav-link active" aria-current="page" href="#">Home</a>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Team
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Team.html">Members</a></li>
			  <li><a class="dropdown-item" href="Attributions.html">Attributions</a></li>
			  <li><a class="dropdown-item" href="Collaborations.html">Collaborations</a></li>
			  <li><a class="dropdown-item" href="Sponsors.html">Sponsors</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Project
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Description.html">Description</a></li>
			  <li><a class="dropdown-item" href="Design.html">Design</a></li>
			  <li><a class="dropdown-item" href="Notebook.html">Notebook</a></li>
			  <li><a class="dropdown-item" href="Contribution.html">Contributions</a></li>
			  <li><a class="dropdown-item" href="Engineering.html">Engineering Success</a></li>
			  <li><a class="dropdown-item" href="Implementation.html">Future Implementation</a></li>
			  <li><a class="dropdown-item" href="Safety.html">Safety</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Wet Lab
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Experiments.html">Experiments</a></li>
			  <li><a class="dropdown-item" href="Constructs.html">Constructs</a></li>
			  <li><a class="dropdown-item" href="Measurement.html">Measurement</a></li>
			  <li><a class="dropdown-item" href="LabNotebook.html">Lab Notebook</a></li>
			  <li><a class="dropdown-item" href="Results.html">Results</a></li>
			  <li><a class="dropdown-item" href="FutureWork.html">Future Work</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Parts
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <!-- <li><a class="dropdown-item" href="#">Overview</a></li> -->
			  <li><a class="dropdown-item" href="BasicParts.html">Basic</a></li>
			  <li><a class="dropdown-item" href="CompositeParts.html">Composite</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Dry Lab
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Model.html">Overview</a></li>
			  <li><a class="dropdown-item" href="EColi.html">E Coli</a></li>
			  <li><a class="dropdown-item" href="Cyanobacteria.html">Cyanobacteria</a></li>
			  <li><a class="dropdown-item" href="Coculture.html">Coculture</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Human Practices
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Human_Practices.html">iHP</a></li>
			  <li><a class="dropdown-item" href="#">Public Engagement</a></li>
			  <li><a class="dropdown-item" href="Communication.html">Education</a></li>
			</ul>
		      </li>
		      <li class="nav-item">
			<a class="nav-link" href="#">Judging Form</a>
		      </li>
		      <!-- <button type="button" class="rounded-pill btn-rounded">Settings</button> -->
		    </ul>
		  </div>
		</div>
	</nav>



    <!-- TITLE SECTION -->
    
    <div class="section-1">
        <h1><br><br><br><br>Dry Lab Overview</h1>
    </div>
        


    <!-- WAVES SPACER -->
	<a id="top"></a>
    <div class="spacer layer1"></div>




    <!-- CONTENT + SIDENAV SECTION -->

    <div class="second-div container-fluid">
        <div class="row">
            <!-- SIDENAV -->
            <div class="col-lg-3 col-md-3">
                <div class="sidenav">
                    <nav>
                        <ul id="sidenav-id">
                            <li class="nav-item"><a class="nav-link" href="#1">Introduction</a></li>
                            <li class="nav-item"><a class="nav-link" href="#2">Flux Balance Analysis</a></li>
                            <li class="nav-item"><a class="nav-link" href="#3">MoMA</a></li>
                            <li class="nav-item"><a class="nav-link" href="#4">FSEOF</a></li>
                            <li class="nav-item"><a class="nav-link" href="#5">OptKnock</a></li>
                            <li class="nav-item"><a class="nav-link" href="#6">SteadyCom</a></li>
                            <li class="nav-item"><a class="nav-link" href="#7">Dynamic FBA</a></li>
							<li class="nav-item"><a class="nav-link" href="#8">References</a></li>
                        </ul>
                    </nav>
                </div>
            </div>


            <!-- CONTENT -->
            <div class="col-lg-9 col-md-9">
                <div class="content">


					<!-- INTRODUCTION -->
                    <h1 id="1">Introduction</h1>
					<p>Mathematical modelling has become an integral part of almost all projects dealing with metabolic engineering. Information starting from the dynamics of our system, to what genetic modifications can further our project can be obtained from the right mathematical models. These models help as starting points for the wet-lab experiments as they give a rough idea of where to target your attention.</p>
					<p>SynBactory being heavily dependent on organism’s growths, behaviours and interactions, along with metabolic networks and its possible manipulations, is a project that can benefit a lot from mathematical models.</p>
					<p><strong>The main objectives were:</strong></p>
					<ol class="a">
						<li style="padding-left:1em">Predict growth rates of our individual organisms in pure cultures</li>
						<li style="padding-left:1em">Predict growth rates under different stresses</li>
						<li style="padding-left:1em">Identify the important interactive metabolites in the co culture</li>
						<li style="padding-left:1em">Identify targets to over express or knockout in each organism for improved yields of sucrose and butanol</li>
					</ol>
					<p>The core technique of our modelling attempts is a constraint based modelling technique called Flux Balance Analysis, in short FBA. We have used the COBRA package in MATLAB for almost all our models.</p>
		


					<!-- FLUX BALANCE ANALYSIS-->
                    <h1 id="2">Flux Balance Analysis</h1>
					<p>FBA is a steady state analysis algorithm that is widely used in metabolic predictions and optimisations. FBA assumes the organism to be in a metabolically steady state i.e., there is no accumulation of metabolites within the organism. To perform FBA we require a metabolic model of the organism. A metabolic model of an organism contains information on all (or most) reactions in an organism, the metabolites involved, the genes present in the organism, the Gene-Reaction-Relation, and a calculated biomass production reaction (growth rate reaction) that gives the growth rate when analysed.</p>
					<p><strong>The core elements of FBA are:</strong></p>
					<ol class="a">
						<li style="padding-left:1em">S Matrix or the stoichiometric matrix - This is a matrix of the order (number of metabolites) * (number of reactions)</li>
						<li style="padding-left:1em">The objective function or c - This is a column vector that holds the information to attain the aim of the analysis, for example biomass optimisation</li>	
						<li style="padding-left:1em">Lower bounds and upper bounds - This is a set of values that help constrain the solution space</li>
						<li style="padding-left:1em">Solution flux space - This is a set of fluxes that give the same objective function value</li>
					</ol>
					<p>In an organism, it is normal to have the same metabolites participating in multiple reactions, and therefore it is safe to assume the number of reactions will be more than the number of metabolites. This can be compared to a system of linear equations with more equations than variables. Without constraints the solution space is infinite, so we introduce constraints in the form of bounds.</p>
					<br>
					<p><strong>Equations of FBA and their solutions</strong></p>
					<p><strong>Constraints:</strong></p>
					<ul class="b">
						<li style="padding-left:1em;">Sv = 0</li>
						<li style="padding-left:1em;">Lower Bounds</li>
						<li style="padding-left:1em;">Upper Bounds</li>
					</ul>
					<br>
					<p><strong>Objective Function:</strong></p>
					<ul class="b">
						<li style="padding-left:1em;">C(transpose)v</li>
					</ul>
					<br>
					<p><strong>Objective:</strong></p>
					<ul class="b">
						<li style="padding-left:1em;">Minimise or Maximise Objective function</li>
					</ul>
					<br>
					<p><strong>Solution:</strong></p>
					<ul class="b">
						<li style="padding-left:1em;">Plausible flux distributions for the above mentioned objective function</li>
					</ul>
					<p>If one has perturbed the metabolic network of the organism, then simple maximisation of growth rate does not give results that agree with experiments. MoMA is a widely used algorithm to identify solution sets that agree most with real life values.</p>



					<!-- MOMA -->
					<h1 id="3">Minimisation of Metabolic Adjustments</h1>
					<p>MoMA is an extension of Flux Balance Analysis that imposes more constraints to make the model as close to reality as possible, albeit it isn’t the best at doing so. The logic of MoMA is very simple - minimise the euclidean distance (L2 norm) between the wild-type flux distribution and the flux distribution of the modified organism in the flux vector space.</p>
					<br>
					<p><strong>The algorithm:</strong></p>
					<ol class="a">
						<li style="padding-left:1em;">Find the wild-type/base strain flux distribution using FBA for maximized growth rates</li>
						<li style="padding-left:1em;">Simulate the deletion, over-expression or any stress</li>
						<li style="padding-left:1em;">Impose an additional constraint that minimizes the L2 norm of the solution and the wild type flux; it would look something like this - ||v_wild - v_mod||^2 is minimised.</li>
						<li style="padding-left:1em;">Run FBA multiple times to find such a v_mod.</li>
					</ol>
					<br>
					<p>Fascinating thing is this algorithm quite literally translates into the name of the technique.</p>
					<p>Here we know what the modifications are that we need to make in order to get the strain we need, but what if we need to find further modifications that would help increase the yield of the product, simple FBA or MoMA will not suffice for this purpose. This is where we start using techniques like FSEOF and OptKnock.</p>



					<!-- FSEOF -->
					<h1 id="4">Flux Scanning based on Enforced Objective Flux (FSEOF)</h1>
		
					<p>This algorithm was developed to identify gene targets that can be amplified for increased production of selected metabolites or even increased growth rates. We have used this technique to identify the reactions and further genes that we can over-express to increase sucrose production in cyanobacteria and butanol production in E. coli.</p>
					<p>To understand FSEOF imagine the whole organism as a big city, the reactions as roads, and the flux through the reactions as the traffic in each road. Now consider evacuating this city as our objective, then the roads which lead to the exits will start carrying more traffic. If we can widen these roads we can in theory get more people out. If evacuating this city was another way of saying - maximise the production of a particular metabolite, then widen the roads can be considered as amplifying the corresponding reactions.</p>
					<br>
					<p>In the form of an algorithm it would look something like this:</p>
					<ol class="a">
						<li style="padding-left:1em;">Find maximum growth under media conditions using FBA </li>
						<li style="padding-left:1em;">Find the maximum production of the metabolite using FBA </li>
						<li style="padding-left:1em;">Constraint the production of the metabolite to some fraction of the maximum metabolite production value </li>
						<li style="padding-left:1em;">Run FBA for maximum growth rate </li>
						<li style="padding-left:1em;">Calculate the product of the metabolite production value for a given constraint and the corresponding growth rate value. This would help rank the target later on </li>
						<li style="padding-left:1em;">Plot how the flux values of each reaction changes as we increase the fractional value of the constraint on the metabolite </li>
					</ol>
					<br>
					<p>Since FBA and all of its extensions work with reactions and not genes we will have to trace the reactions back to its controlling genes and see if those genes are responsible for any other process that could end up killing the organism or hindering its growth if amplified.</p>
			
					
					<!-- OptKnock-->
					<h1 id="5">OptKnock</h1>
					<p>OptKnock is an algorithm that find and suggests which set of reactions to remove from a metabolic network to obtain a mutant that will produce a particular target of interest at a higher rate than the wild-type strain. The logic is pretty simple -</p>
					<ol class="a">
						<li style="padding-left:1em;">You find the set of reactions that you want to get knockouts from </li>
						<li style="padding-left:1em;">Change their bounds to 0 (this is how you simulate deletion in models) either one by one or in pairs or triplets </li>
						<li style="padding-left:1em;">Run FBA for maximised growth </li>
						<li style="padding-left:1em;">Once you have a set of reactions which when deleted increases the production of the target of interest you trace them back to their respective genes </li>
						<li style="padding-left:1em;">Once you’ve identified the genes you eliminate all the synthetic lethals from the set. Synthetic lethals are essential genes that was required by the organism to live </li>
					</ol>
					<br>
					<p>Once you’ve eliminated all the synthetic lethal you have a set of reactions and their corresponding genes which can be knocked out for enhanced production of the metabolite of interest.</p>
			

					<!-- STEADYCOM -->
					<h1 id="6">SteadyCom</h1> <br>
					<p>SteadyCom is an algorithm used to analyze steady state behaviours of co-cultures, microbiomes and host-microbe interactions. Similar to FBA, SteadyCom assumes that there is no accumulation of metabolites and only looks at the fluxes of reactions. Apart from this, the crux of SteadyCom is the assumption that - at steady state (if a steady state exists) the growth rates of all the organisms is the same, so that one organism does not dominate and take over the entire population. Based on the constraints in the model, SteadyCom tries to maximize this community growth rate, and provides results including the biomass ratios of the organisms at steady state growth and fluxes of various reactions of the multi-species metabolic network.
					</p>
					<h3>The Logic:</h3> <br>
					<ol class="a">
						<li style="padding-left:1em;">Create a joint model which has all the reactions of every organism’s GSM and a separate community space to allow for the transfer of common extracellular metabolites between the separate extracellular compartments of the individual models.</li>
						<li style="padding-left:1em;">The algorithm predicts steady state fluxes which maximizes the community growth rate, such that all organisms grow at this rate.</li>
						<li style="padding-left:1em;">The fluxes of reactions out of individual organisms are multiplied by their biomass, so that abundances of organisms are taken into account in the interactions.</li>
					</ol>
					
					<!-- dFBA -->
					<h1 id="7">Dynamic FBA</h1>
					<h3>Method</h3><br>
					<p>Dynamic modelling here involves solving ordinary differential equations involving fluxes in the genome models as the rate/differential terms to produce curves which represent the amounts of metabolites present in the bioreactor as a function of time.
						For example, when we optimize a model using FBA, we get the fluxes of all the reactions that allow the model to optimize its objective function. These fluxes can be thought of as time-derivatives of the concentration of the metabolites in the reaction involved. So, given an initial concentration of a metabolite, the FBA flux gives the derivative of the concentration at that point. Using this derivative, we can calculate the concentration of the metabolite at the next time step, and use this as the new initial concentration. This process can be repeated iteratively within the required time frame. 
						</p><br>
					<h3>The Algorithm:</h3><br>
					<ol class="a">
						<li style="padding-left:1em;padding-top: 0.3em;">Set the flux restrictions of the cyanobacteria uptake rates, of co2 and light, based on concentration of these present in the bioreactor available for uptake. </li>
						<li style="padding-left:1em;padding-top: 0.3em;">Optimize the cyanobacteria model with sucrose production as its objective using FBA. This will naturally make the biomass flux of the cyanobacteria set to its lower bound 0. Biomass flux set to zero, implies that the rate of growth of biomass is zero. Hence, the maximized sucrose flux obtained from this result is the theoretical maximum sucrose yield possible from the model within the given constraints.
						</li>
						<li style="padding-left:1em;padding-top: 0.3em;">It is known that reaching the theoretical maximum yields of sucrose production is not very biologically feasible. So it is assumed that the salt stress provided to force cyanobacteria to produce sucrose, allows it to export sucrose out at some fraction of the maximum sucrose yield calculated above. This is referred to as the percentage productivity of sucrose flux in the cyanobacteria.
						</li>
						<li style="padding-left:1em;padding-top: 0.3em;">This productivity rate, x% of max sucrose flux, is now set as the lower bound for the sucrose export reaction in the cyanobacteria model, and the objective function is changed to biomass flux. An FBA is performed to maximize the biomass flux given this new constraint of sucrose production.</li>
						<li style="padding-left:1em;padding-top: 0.3em;">The final results from this FBA are taken as the fluxes of the cyanobacteria model.</li>
						<li style="padding-left:1em;padding-top: 0.3em;">The same process is repeated for the E. coli model, where the available sucrose in the bioreactor comes from the cyanobacteria flux calculated above. Butanol production is first optimized to find the maximum flux possible, then as above, some productivity for this reaction is set and biomass is optimized. 
						</li>
						<li style="padding-left:1em;padding-top: 0.3em;">All the fluxes from both the models provide the time derivatives of the corresponding metabolites, which are integrated and plotted. </li>
					</ol>
					

					<!-- REFERENCES -->
					<h1 id="8">References</h1>
					<p> ENTER TEXT HERE </p>

					<!-- END OF CONTENT -->

					<section class="header-section view-border">
						<div id="carouselExampleCaptions" class="carousel carousel-dark slide" data-bs-ride="carousel">
							<div class="carousel-indicators">
							  <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
							  <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="1" aria-label="Slide 2"></button>
							  <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="2" aria-label="Slide 3"></button>
							</div>
							<div class="carousel-inner">
							  <div class="carousel-item active">
								<img src="assets/graphs/Biomass vs CO_2.jpg" class="d-block w-100" alt="...">
							  </div>
							  <div class="carousel-item">
								<img src="assets/Graphs/Biomass vs Sucrose.jpg" class="d-block w-100" alt="...">
							  </div>
							  <div class="carousel-item">
								<img src="assets/Graphs/3 Variables.jpg" class="d-block w-100" alt="...">
							  </div>
							</div>
							<button class="carousel-control-prev" type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide="prev">
							  <span class="carousel-control-prev-icon" aria-hidden="true"></span>
							  <span class="visually-hidden">Previous</span>
							</button>
							<button class="carousel-control-next" type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide="next">
							  <span class="carousel-control-next-icon" aria-hidden="true"></span>
							  <span class="visually-hidden">Next</span>
							</button>
							  </div>
							  <h3 id="slidetext"></h3>
					</section>
			</div>
		</div>
    </div>
<!-- SPONSORS BAR -->
	<div class="footer">
		<div class="footer-wrapper">
			<!-- <div id="not-sponsors" class="footer-wrapper">	 -->
				<div id="Team", class="footer-wrapper">
					<img class="teamlogo" src="assets/Logos/IISER-Pune-iGEM_logo.png" alt="iGEM IISER Pune Logo" width="170">
				</div>
				<div id="socials" class="vertical-stack">
					<div><h3>Team IISER Pune India</h3></div>
					<div id="logos" class ="footer-wrapper ">
						<a href="https://www.instagram.com/igem_iiserp/"><img class="footer-links-socials" src="assets/Logos/Instagram_logo.jpg" alt="Instagram" width="20"></a>
						<a href="https://www.linkedin.com/in/igem-iiser-pune-2021/"><img class="footer-links-socials" src="assets/Logos/linkedin_logo.png" alt="Linkedin" width="30"></a>
						<a href="https://twitter.com/igem_iiserpune"><img class="footer-links-socials" src="assets/Logos/Twitter_logo.png" alt="Twitter" width="35"></a>
						<a href="https://www.facebook.com/igemiiserpune"><img class="footer-links-socials" src="assets/Logos/Facebook_logo.png" alt="Facebook" width="25"></a>
						<a href="https://www.youtube.com/channel/UCyo6NMLvM58sSh_tnxht36w"><img class="footer-links-socials" src="assets/Logos/Youtube_logo.png" alt="Youtube" width="35"></a>
					</div>
				</div>
			<!-- </div> -->
			<div id="sponsors" class="footer-wrapper">
				<div id="main-sponsor", class="">
					<a href="https://www.sailife.com/"><img class="footer-links" src="assets/Logos/Sai_logo.jpg" alt="Sai Life Sciences" width="280"></a>
				</div>
				<div class="vertical-stack">
					<a href="https://sg.idtdna.com/pages"><img class="footer-links" src="assets/Logos/IDT_logo.png" alt="IDT" width="220"></a>
					<a href="https://www.geneious.com/"><img class="footer-links" src="assets/Logos/Geneious_logo.png" alt="Geneious" width="220"></a>
				</div>
				<div class="vertical-stack">
					<div id="horizontal stack">
						<a href="https://www.notion.so/"><img class="footer-links" src="assets/Logos/Notion_logo.png" alt="Notion" width="70"></a>
						<img src="assets/Logos/IISER Pune Logo.png" alt="IISER Pune" width="80" style="padding-left: 1em;">
					</div>
					<a href="https://www.zoho.com/"><img class="footer-links" src="assets/Logos/Zoho_logo.png" alt="Zoho" width="150"></a>
				</div>
			</div>	
		</div>
	</div>
	<!-- END OF SPONSOR BAR-->

    <div class="header"></div>
    <div class="line" id="scrollIndicator"></div>

	 <!-- back to top button -->
	 <a href="#top"><img src="assets/svgs/arrow-up-square-fill.svg" id="myBtn" title="Go to top" width="30" height="30"></a>

    <script src="js/bootstrap-bundle.js"></script>
	<script src="js/main.js"></script>
	<script src="js/jquery.min.js"></script>

	<script>
		var screenheight = window.innerHeight,
		    screenwidth = window.innerWidth;

	</script>
	<script>
		var body = document.body,
    		html = document.documentElement;

		var pageheight = Math.max( body.scrollHeight, body.offsetHeight, 
                       		   html.clientHeight, html.scrollHeight, html.offsetHeight );
	</script>
	<script>
		var backtotopheight=pageheight/4.65
	</script>
	<script>
		var total = $('.item').length;
		var actualtotal = 80;
		var currentIndex = $('div.active').index() + 1;
		$('#slidetext').html(currentIndex + '/'  + actualtotal);

		// This triggers after each slide change
		$('.carousel').on('slid.bs.carousel', function () {
  			currentIndex = $('div.active').index() + 1;
 
  		// Now display this wherever you want
  		var text = currentIndex + '/' + actualtotal;
  			$('#slidetext').html(text);
		});

	</script>
	
	

	<!-- SCRIPT FOR SCROLL AWARE SIDEBAR HIGHLIGHT -->
	<script>
		
		$(document).ready(function() {
  			$(window).scroll(function() {
				var scroll_top = $(window).scrollTop();
				var div_one_top = $("#1").offset().top;
				var div_two_top = $("#2").offset().top;
				var div_three_top = $("#3").offset().top;
				var div_four_top = $("#4").offset().top;
				var div_five_top = $("#5").offset().top;
				var div_six_top = $("#6").offset().top;
				var div_seven_top = $("#7").offset().top;
				var div_eight_top = $("#8").offset().top;

				if (scroll_top > div_one_top && scroll_top < div_two_top) {
					//Division 1
					if (!$('a[href="#1"]').hasClass('sidenav-active')) {
						$('a[href="#1"]').addClass('sidenav-active');
					}

					if ($('a[href="#2"]').hasClass('sidenav-active')) {
						$('a[href="#2"]').removeClass('sidenav-active');
					}

				} else if (scroll_top > div_two_top && scroll_top < div_three_top) {
					// Division 2
					if ($('a[href="#1"]').hasClass('sidenav-active')) {
						$('a[href="#1"]').removeClass('sidenav-active');
					}

					if ($('a[href="#3"]').hasClass('sidenav-active')) {
						$('a[href="#3"]').removeClass('sidenav-active');
					}

					if (!$('a[href="#2"]').hasClass('sidenav-active')) {
						$('a[href="#2"]').addClass('sidenav-active');
					}
				} else if (scroll_top > div_three_top && scroll_top < div_four_top) {
					//Division 3
					if ($('a[href="#2"]').hasClass('sidenav-active')) {
						$('a[href="#2"]').removeClass('sidenav-active');
					}

					if ($('a[href="#4"]').hasClass('sidenav-active')) {
						$('a[href="#4"]').removeClass('sidenav-active');
					} 

					if (!$('a[href="#3"]').hasClass('sidenav-active')) {
						$('a[href="#3"]').addClass('sidenav-active');
					}
				} else if (scroll_top > div_four_top && scroll_top < div_five_top) {
					// You are in div 4
					if ($('a[href="#3"]').hasClass('sidenav-active')) {
						$('a[href="#3"]').removeClass('sidenav-active');
					}

					if ($('a[href="#5"]').hasClass('sidenav-active')) {
						$('a[href="#5"]').removeClass('sidenav-active');
					} 

					if (!$('a[href="#4"]').hasClass('sidenav-active')) {
						$('a[href="#4"]').addClass('sidenav-active');
					}
				} else if (scroll_top > div_five_top && scroll_top < div_six_top) {
					// You are in div 5
					if ($('a[href="#4"]').hasClass('sidenav-active')) {
						$('a[href="#4"]').removeClass('sidenav-active');
					}

					if ($('a[href="#6"]').hasClass('sidenav-active')) {
						$('a[href="#6"]').removeClass('sidenav-active');
					} 

					if (!$('a[href="#5"]').hasClass('sidenav-active')) {
						$('a[href="#5"]').addClass('sidenav-active');
					}
				} else if (scroll_top > div_six_top && scroll_top < div_seven_top) {
					// You are in div 6
					if ($('a[href="#5"]').hasClass('sidenav-active')) {
						$('a[href="#5"]').removeClass('sidenav-active');
					}

					if ($('a[href="#7"]').hasClass('sidenav-active')) {
						$('a[href="#7"]').removeClass('sidenav-active');
					} 

					if (!$('a[href="#6"]').hasClass('sidenav-active')) {
						$('a[href="#6"]').addClass('sidenav-active');
					}
				} else if (scroll_top > div_seven_top && scroll_top < div_eight_top) {
					// You are in div 7
					if ($('a[href="#6"]').hasClass('sidenav-active')) {
						$('a[href="#6"]').removeClass('sidenav-active');
					}

					if ($('a[href="#8"]').hasClass('sidenav-active')) {
						$('a[href="#8"]').removeClass('sidenav-active');
					} 

					if (!$('a[href="#7"]').hasClass('sidenav-active')) {
						$('a[href="#7"]').addClass('sidenav-active');
					}
				} else if (scroll_top > div_eight_top) {
					// You are in div 8
					if ($('a[href="#7"]').hasClass('sidenav-active')) {
						$('a[href="#7"]').removeClass('sidenav-active');
					}

					if (!$('a[href="#8"]').hasClass('sidenav-active')) {
						$('a[href="#8"]').addClass('sidenav-active');
					}
				}
  			});
		});
		
	</script>
</body>
