

<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Results</title>
	<link rel="stylesheet" href="css/igem-reset.css">
	<link rel="stylesheet" href="css/SBcss/style.css">
   	<link rel="stylesheet" href="css/styleSB.css">
    	<link rel="stylesheet" href="css/DryLabPage.css">
	<style>/* Reset CSS */

		#sideMenu, #top_title, .patrollink, #firstHeading, #home_logo, #sideMenu { display:none; }
		#content { padding:0px; width:100%; margin-top:-7px; margin-left:0px; border:none;}
		body, html {background-color:#212529; width: 100%; height: 100%;}
		#bodyContent h1, #bodyContent h2, #bodyContent h3, #bodyContent h4, #bodyContent h5 { margin-bottom: 0px; }
		#bodyContent a[href ^="https://"], .link-https { padding-right:0px;}

    </style>
<body>
    

	
	<!-- NAVBAR SECTION -->

	<nav class="navbar fixed-top navbar-expand-lg navbar-light menu">
		<div class="container-fluid nav-container">
		  <a class="navbar-brand" href="#">
			<img src="assets/svgs/synbactory2.svg">
		  </a>
		  <div class="center-expand">
		</div>
		  <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		  <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
		    <ul class="navbar-nav">
		      <li class="nav-item">
			<a class="nav-link active" aria-current="page" href="#">Home</a>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Team
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Team.html">Members</a></li>
			  <li><a class="dropdown-item" href="Attributions.html">Attributions</a></li>
			  <li><a class="dropdown-item" href="Collaborations.html">Collaborations</a></li>
			  <li><a class="dropdown-item" href="Partnership.html">Partnership</a></li>
			  <li><a class="dropdown-item" href="Sponsors.html">Sponsors</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Project
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Description.html">Description</a></li>
			  <li><a class="dropdown-item" href="Design.html">Design</a></li>
			  <li><a class="dropdown-item" href="Notebook.html">Notebook</a></li>
			  <li><a class="dropdown-item" href="Contribution.html">Contributions</a></li>
			  <li><a class="dropdown-item" href="Engineering.html">Engineering Success</a></li>
			  <li><a class="dropdown-item" href="Implementation.html">Future Implementation</a></li>
			  <li><a class="dropdown-item" href="Safety.html">Safety</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Wet Lab
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Experiments.html">Experiments</a></li>
			  <li><a class="dropdown-item" href="Constructs.html">Constructs</a></li>
			  <li><a class="dropdown-item" href="Measurement.html">Measurement</a></li>
			  <li><a class="dropdown-item" href="LabNotebook.html">Lab Notebook</a></li>
			  <li><a class="dropdown-item" href="Results.html">Results</a></li>
			  <li><a class="dropdown-item" href="FutureWork.html">Future Work</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Parts
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <!-- <li><a class="dropdown-item" href="#">Overview</a></li> -->
			  <li><a class="dropdown-item" href="BasicParts.html">Basic</a></li>
			  <li><a class="dropdown-item" href="CompositeParts.html">Composite</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Dry Lab
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Model.html">Overview</a></li>
			  <li><a class="dropdown-item" href="EColi.html"><i>E. coli</i></a></li>
			  <li><a class="dropdown-item" href="Cyanobacteria.html">Cyanobacteria</a></li>
			  <li><a class="dropdown-item" href="Coculture.html">Coculture</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Human Practices
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Human_Practices.html">Integrated Human Practices</a></li>
			  <li><a class="dropdown-item" href="Inclusivity.html">Inclusivity and Accessibility</a></li>
			  <li><a class="dropdown-item" href="Communication.html">Public Engagement and Education</a></li>
			</ul>
		      </li>
		      <li class="nav-item">
			<a class="nav-link" href="https://igem.org/2021_Judging_Form?id=3971">Judging Form</a>
		      </li>
		      <!-- <button type="button" class="rounded-pill btn-rounded">Settings</button> -->
		    </ul>
		  </div>
		</div>
	</nav>



    <!-- TITLE SECTION -->
    
    <div class="section-1">
        <h1><br><br><br><br>Results</h1>
    </div>
        


    <!-- WAVES SPACER -->

    <div class="spacer layer1"></div>
	<a id="top"></a>



    <!-- CONTENT + SIDENAV SECTION -->

	<!-- STYLING FOR FONTS -->
	<style>
		.content h1, .content h2, .content h3, .content h4, .content h5, .content h6{
		color: white !important;
		font-family: ProximaNova !important;
	}
	</style>
	<!-- END OF STYLING FOR FONTS -->


    <div class="second-div container-fluid">
        <div class="row">


			<!-- SIDE NAV BAR -->
			<div class="col-lg-3 col-md-3">

				<div class="tab-content sidenav" id="SidenavTabs" style="top: -25.5%">

					<!-- Ecoli sidebar-->
					<div class="tab-pane fade show active ecoli-wet" id="ecoli-wet-side" role="tabpanel" aria-labelledby="ecoli-wet-res-tab"> 
						
						<div class="sidenav">
							<nav>
								<ul id="sidenav-id">
									<li class="nav-item"><a class="nav-link" href="#e1">PCR and gel electrophoresis for confirmation of butanol producing genes</a></li>
									<li class="nav-item"><a class="nav-link" href="#e2">NMR confirmation of butanol production</a></li>
									<li class="nav-item"><a class="nav-link" href="#e3">Transformation of KJK01 with pCSCX plasmid</a></li>
									<li class="nav-item"><a class="nav-link" href="#e4">Growth curve assay for KJK01 and pCSCX-KJK01</a></li>
									<li class="nav-item"><a class="nav-link" href="#e5">MIC assay of butanol and ethanol for KJK01</a></li>
									<li class="nav-item"><a class="nav-link" href="#e6">Sucrose assay for KJK01 in LB</a></li>
									<li class="nav-item"><a class="nav-link" href="#e7">References</a></li>
								</ul>
							</nav>
						</div>
					</div>
						
					<!-- Cyano Sidebar-->
					<div class="tab-pane fade cyano-wet" id="cyano-wet-side" role="tabpanel" aria-labelledby="cyano-wet-res-tab"> 
						
						<div class="sidenav">
							<nav>
								<ul id="sidenav-id">
									<li class="nav-item"><a class="nav-link" href="#c1">Growth Curve - Ambient carbon dioxide</a></li>
									<li class="nav-item"><a class="nav-link" href="#c2">Sucrose Assay</a></li>
									<li class="nav-item"><A class="nav-link" href="#c3">Growth Curve - High carbon dioxide</A></li>
									<li class="nav-item"><a class="nav-link" href="#c4">References</a></li>
		
								</ul>
							</nav>
						</div>
					</div>

					<!-- Coculture Sidebar-->
					<div class="tab-pane fade coculture-wet" id="coculture-wet-side" role="tabpanel" aria-labelledby="coculture-wet-res-tab"> 
						
						<div class="sidenav">
							<nav>
								<ul id="sidenav-id">
									<li class="nav-item"><a class="nav-link" href="#cc1">Growth curve assay for KJK01 and pCSCX-KJK01 in BG-11 + sucrose</a></li>
									<li class="nav-item"><a class="nav-link" href="#cc2">Transformation of KJK01 with pKD4 plasmid</a></li>
									<li class="nav-item"><a class="nav-link" href="#cc3">Growth curve assay for KJK01 and pCSCX-KJK01 in CoBG-11 + sucrose</a></li>
									<li class="nav-item"><a class="nav-link" href="#cc4">Sucrose assay</a></li>
									<li class="nav-item"><a class="nav-link" href="#cc5">References</a></li>
								</ul>
							</nav>
						</div>
					</div>
				</div>
			</div>
			<!-- END OF SIDENAV BAR -->



            <!-------------------------------------------------- CONTENT ----------------------------------------------->
            <div class="col-lg-9 col-md-9">
                <div class="content">

					<!--------------------------------- STYLING FOR THE TABS ------------------------------------>
					<style>
						.content .nav-link{
							/* how it looks by default */
							color:white !important;
							font-size: 1.5rem;
						}
						.content .nav-link:hover{
							color:black !important;
							background-color: #28c79b;
						}
						.content .nav-link.active{
							color: white;
							background-color: #1f5e4c;
						}
						.content .nav-item .nav-link,
						content .nav-tabs .nav-link {
							-webkit-transition: all 250ms ease 0s;
							-moz-transition: all 250ms ease 0s;
							-o-transition: all 250ms ease 0s;
							-ms-transition: all 250ms ease 0s;
							transition: all 250ms ease 0s;
						}
					</style>
					<!--------------------------------- END OF STYLING FOR THE TABS ------------------------------------>

					<!-- Tabs -->
					<ul class="nav nav-tabs nav-fill" id="myTab" role="tablist">
						<li class="nav-item" role="presentation">
							<button class="nav-link active" id="ecoli-wet-res-tab" data-bs-toggle="tab" data-bs-target=".ecoli-wet" type="button" role="tab" aria-controls="ecoli-wet-res" aria-selected="true"><i>E. coli</i></button>
						</li>
						<li class="nav-item" role="presentation">
							<button class="nav-link" id="cyano-wet-res-tab" data-bs-toggle="tab" data-bs-target=".cyano-wet" type="button" role="tab" aria-controls="cyano-wet-res" aria-selected="false">Cyanobacteria</button>
						</li>
						<li class="nav-item" role="presentation">
							<button class="nav-link" id="coculture-wet-res-tab" data-bs-toggle="tab" data-bs-target=".coculture-wet"  type="button" role="tab" aria-controls="coculture-wet-res" aria-selected="false">Co-culture</button>
						</li>
					<a id="e1"></a> <a id="c1"></a> <a id="cc1"></a>
					</ul>
					<!-- End of Tabs -->

					<div class="tab-content" id="myTabContent">
						<!--------------------------------------- start of e coli ---------------------------------------->
						<div class="tab-pane fade show active ecoli-wet" id="ecoli-wet-res" role="tabpanel" aria-labelledby="ecoli-wet-res-tab">
							<br><br><br>
							<!-- PCR FOR BUTANOL GENES -->
							<h1>PCR and gel electrophoresis for confirmation of butanol producing genes</h1>
							<hr><br><br><br>
							<p>To confirm whether the <i>E. coli</i> KJK01 strain contained the butanol producing genes, we extracted the genomic DNA of the cells and subjected it to PCR amplification using primers we received from Dr Yazdani’s lab. Since the cassette has been added as one single component, verification of any component from the cassette will verify the presence of the butanol producing genes in KJK01<sup><a href="#eref1">[1]</a></sup>. We received primers against the <i>hbd-crt</i> genes. An amplicon of the length of ~750bp from the PCR would verify the presence of those genes.</p>
							<br>


							<!-- GDNA EXTRACTION -->
							<h2>Genomic DNA extraction</h2>
							<br>
							<h4>Expected Result:</h4> 
							<p>Large band above the DNA ladder, as gDNA is very large in size and cannot traverse through the gel</p>
							<br>

							<h4>Result:</h4>
							<ul class="b">
								<li style="padding-left: 1em;">In the first genomic DNA extraction, we obtained fragmented DNA which showed up in multiple bands when we ran our gels with the PCR product.</li>
								<li style="padding-left: 1em;">The second genomic DNA extraction yielded a single band above the DNA ladder along with a large bright band at the bottom of the ladder</li>
								<ul class="c">
									<li style="padding-left: 1em;">The first band is the gDNA of KJK01 and the bright band is probably RNA contamination due to improper addition of RNase.</li>
								</ul>
							</ul>
							<br>
							<div class="graph">
								<img src="assets/Wetlab_Graphs/E_coli/Gels/gDNA.png" width=400>
								<p id="desc">Gel showing gDNA band above DNA ladder. Identical samples loaded next to ladder. Bright band at the bottom might be RNA contamination.</p>
							</div>
							<br><br>
							<!-- END OF GDNA-->



							<!-- PCR amplification of hbd-crt cassette and gel electrophoresis -->
							<h2>PCR amplification of <i>hbd-crt</i> cassette and gel electrophoresis</h2>
							<br>
							<h4>Expected Results:</h4> 
							<p>Amplicon of length ~750bp showing up on the gel</p>
							<br>

							<h4>Results:</h4> 
							<p>The gel failed multiple times. We did not get any bands on the gel.</p>
							<br>

							<div class="graph">
								<img src="assets/Wetlab_Graphs/E_coli/Gels/PCR_amp.png" width=400>
								<p id="desc">Failed PCR gel</p>
							</div>
							<br><br>
							<!-- END OF PCR amplification of hbd-crt cassette and gel electrophoresis -->
							
							

							<!-- TROUBLESHOOTING (READ MORE) -->
							<h2> Troubleshooting</h2>
							<span id="dots">...</span>
							<span id="more" >
								<br>
								<p>Despite running the gel multiple times, we failed to obtain a positive result. The modifications we made to our protocol to get the desired results were -</p>
								<ul class="b">
									<li style="padding-left: 1em;"> Reduce dNTP: Our mentor advised us to reduce the amount of dNTP added because the amount could be deleterious to the reaction. The concentration was reduced to 100-500 ng/µl.</li>
									<li style="padding-left: 1em;"> Reduce gDNA concentration: With the same logic as of dNTP, genomic DNA concentration was reduced to 100 ng/µl.</li>
									<li style="padding-left: 1em;"> Compare primer with negative control: To find whether the primers were binding rightly or non-specifically or binding at all, a test was compared with negative control wherein the negative control was devoid of just the gDNA. If the primer dimer band has reduced intensity in the test, it means that the primer is binding to the DNA. If the band is not of the right size, then it is binding non-specifically.</li>
								</ul>
								<br>
								<!-- END OF TROUBLE SHOOTING-->
								<p>But we were unable to see the primer dimer bands at all in the test lanes.</p>
							</span>
							<button onclick="myFunction()" id="ReadMoreButton">read more</button>
							<br><br>
							


							<!-- GRADIENT PCR-->
							<h2>Gradient PCR</h2><br>
							<p>As none of our troubleshooting worked, we spoke to our PI, Prof Pucadyil, from whose lab we borrowed the Pfu polymerase used in the PCR reaction. He pointed out that we hadn’t used DMSO in any of the previous runs when the protocol especially called for it. We had been following the PCR protocol of another lab. He suggested that we perform a gradient PCR at different concentrations of DMSO and different temperatures.</p>
							<br>

							<p>We obtained 20 amplified products after the PCR</p>
							
							<!-- Table 1 -->
							<div class="table" style="padding-left: 5%;">
								<!-- Headers -->
								<div class="table1-row table-header">
									<div class="cell">
										Conditions
									</div>
									<div class="cell">
										Negative Control
									</div>
									<div class="cell">
										0% DMSO
									</div>
									<div class="cell">
										5% DMSO
									</div>
									<div class="cell">
										10% DMSO
									</div>
								</div>
								<!-- End of headers -->

								<!-- Other rows --->
								<div class="table1-row">
									<div class="cell" data-title="Conditions">
										50&#176C
									</div>
									<div class="cell" data-title="Negative Control">
										Gel 1
									</div>
									<div class="cell" data-title="0% DMSO">
										Gel 2
									</div>
									<div class="cell" data-title="5% DMSO">
										Gel 2
									</div>
									<div class="cell" data-title="10% DMSO">
										Gel 2
									</div>
								</div>

								<div class="table1-row">
									<div class="cell" data-title="Conditions">
										52&#176C
									</div>
									<div class="cell" data-title="Negative Control">
										Gel 1
									</div>
									<div class="cell" data-title="0% DMSO">
										Gel 2
									</div>
									<div class="cell" data-title="5% DMSO">
										Gel 2
									</div>
									<div class="cell" data-title="10% DMSO">
										Gel 2
									</div>
								</div>

								<div class="table1-row">
									<div class="cell" data-title="Conditions">
										54&#176C
									</div>
									<div class="cell" data-title="Negative Control">
										Gel 1
									</div>
									<div class="cell" data-title="0% DMSO">
										Gel 2
									</div>
									<div class="cell" data-title="5% DMSO">
										Gel 2
									</div>
									<div class="cell" data-title="10% DMSO">
										Gel 2
									</div>
								</div>

								<div class="table1-row">
									<div class="cell" data-title="Conditions">
										56&#176C
									</div>
									<div class="cell" data-title="Negative Control">
										Gel 1
									</div>
									<div class="cell" data-title="0% DMSO">
										Gel 2
									</div>
									<div class="cell" data-title="5% DMSO">
										Gel 2
									</div>
									<div class="cell" data-title="10% DMSO">
										Gel 2
									</div>
								</div>

								<div class="table1-row">
									<div class="cell" data-title="Conditions">
										60&#176C
									</div>
									<div class="cell" data-title="Negative Control">
										Gel 1
									</div>
									<div class="cell" data-title="0% DMSO">
										Gel 2
									</div>
									<div class="cell" data-title="5% DMSO">
										Gel 2
									</div>
									<div class="cell" data-title="10% DMSO">
										Gel 2
									</div>
								</div>
							</div>
							<br><br>
							<!-- End of Table -->

							<h4>Result:</h4>
							<br>
							<ul>
								<li style="padding-left: 1em;"> Bands found at ~750bp in the 5% and 10% DMSO bands</li>
								<li style="padding-left: 1em;"> Gel 2 was left on for too long and ran out.</li>
							</ul>
							<br>
							<div class="graph">
								<img src="assets/Wetlab_Graphs/E_coli/Gels/Gradient_PCR.png" width=400>
								<p id="desc">Gel 1. Bright bands seen at ~750 bp in the 5% and 10% DMSO lanes. No band seen in negative control and 0% DMSO lanes.</p>
							</div>
							<br>
							<p>This confirmed the presence of butanol producing genes in KJK01.</p>
							<!-- END OF PCR FOR BUTANOL GENES -->
							<a id="e2"></a><br>


							<!-- NMR FOR BUTANOL -->
							<h1> NMR confirmation of butanol production</h1>
							<hr><br><br><br>
							<p>As PCR confirmation of the presence of the butanol producing genes was taking too long, we simultaneously started culturing KJK01 in Luria–Bertani (LB) broth as well as Terrific Broth (TB). Abdelaal<sup><a href="#eref1">[1]</a></sup> et al cultured the strain in TB as <i>E. coli</i> has been shown to produce maximum butanol in that medium. Making TB is a rather time-consuming process for us, hence we decided to check if we could perform the rest of the quantification of butanol production in LB cultures.</p>
							<br>
							
							<p>We inoculated 6 2&#176 cultures as follows:</p>
							<ul class="b">
								<li style="padding-left: 1em;"> LB (0 IPTG)</li>
								<li style="padding-left: 1em;"> LB (0.1 mM IPTG)</li>
								<li style="padding-left: 1em;"> LB (0.2 mM IPTG)</li>
								<li style="padding-left: 1em;"> TB (0 IPTG)</li>
								<li style="padding-left: 1em;"> TB (0.1 mM IPTG)</li>
								<li style="padding-left: 1em;"> TB (0.2 mM IPTG)</li>
							</ul>
							<br>

							<h4>Expected results:</h4>
							<p>The TB cultures are expected to produce more butanol than the LB cultures. The IPTG induced cultures are expected to produce more butanol than the ones with 0 IPTG.</p>
							<br>

							<h4>Results:</h4>
							<!-- table -->
							<div class="table" style="padding-left: 5%;">
								<!-- Headers -->
								<div class="table1-row table-header">
									<div class="cell">
										Sample
									</div>
									<div class="cell">
										Butanol relative intensity
									</div>
									<div class="cell">
										Standard relative intensity
									</div>
									<div class="cell">
										Concentration of butanol (μM)
									</div>
								</div>
								<!-- End of headers -->

								<!-- Other rows --->
								<div class="table1-row">
									<div class="cell" data-title="Sample">
										TB no IPTG
									</div>
									<div class="cell" data-title="Butanol relative intensity">
										2.189
									</div>
									<div class="cell" data-title="Butanol relative intensity">
										0.6995
									</div>
									<div class="cell" data-title="concentration of butanol (μM)">
										208.6356397
									</div>
								</div>

								<div class="table1-row">
									<div class="cell" data-title="Sample">
										TB 0.1
									</div>
									<div class="cell" data-title="Butanol relative intensity">
										4.379
									</div>
									<div class="cell" data-title="Butanol relative intensity">
										0.5964
									</div>
									<div class="cell" data-title="concentration of butanol (μM)">
										489.5169852
									</div>
								</div>

								<div class="table1-row">
									<div class="cell" data-title="Sample">
										TB 0.2
									</div>
									<div class="cell" data-title="Butanol relative intensity">
										3.972
									</div>
									<div class="cell" data-title="Butanol relative intensity">
										0.5579
									</div>
									<div class="cell" data-title="concentration of butanol (μM)">
										474.6607636
									</div>
								</div>


								<div class="table1-row">
									<div class="cell" data-title="Sample">
										LB no IPTG
									</div>
									<div class="cell" data-title="Butanol relative intensity">
										1.6
									</div>
									<div class="cell" data-title="Butanol relative intensity">
										0.883
									</div>
									<div class="cell" data-title="concentration of butanol (μM)">
										120.806342
									</div>
								</div>


								<div class="table1-row">
									<div class="cell" data-title="Sample">
										LB 0.1			
									</div>
									<div class="cell" data-title="Butanol relative intensity">
										15.14
									</div>
									<div class="cell" data-title="Butanol relative intensity">
										1.157
									</div>
									<div class="cell" data-title="concentration of butanol (μM)">
										872.4146932
									</div>
								</div>


								<div class="table1-row">
									<div class="cell" data-title="Sample">
										LB 0.2			
									</div>
									<div class="cell" data-title="Butanol relative intensity">
										12.8
									</div>
									<div class="cell" data-title="Butanol relative intensity">
										1.14
									</div>
									<div class="cell" data-title="concentration of butanol (μM)">
										748.5754386
									</div>
								</div>
							</div>
							<br><br>
							<!-- end of table -->

							<!-- NMR Pictures-->
							<div style="display:flex; flex-direction: column; justify-content:space-around; padding-left: 3.5%;">
								<div style="margin: 10px; display:flex; flex-direction: row; flex-wrap: wrap; justify-content:space-around;">
									
									<section>
										<img src="assets/Wetlab_Graphs/E_coli/NMR/NMR_LBoverlay_1.png" width=375>
										<p id="desc" style="position: relative; text-align: center; padding-top: 0.3em;">NMR spectrum for LB 2&#176 cultures</p>
									</section>

									<section>
										<img src="assets/Wetlab_Graphs/E_coli/NMR/NMR_LBoverlay_2.png" width=375>
										<p id="desc" style="position: relative; text-align: center;  padding-top: 0.3em; margin-right: 26%;">Characteristic butanol peaks are amplified in 0.1 mM (red) and 0.2 mM (green) IPTG induced cultures</p>	
									</section>
								
								</div>
								<div style="margin: 10px; display:flex; justify-content:center; justify-content: space-around;">
									<section>
										<img src="assets/Wetlab_Graphs/E_coli/NMR/NMR_TBoverlay_1.png" width=375>
										<p id="desc" style="position: relative; text-align: center; padding-top: 0.3em;">NMR spectrum for TB 2&#176 cultures</p>
									</section>

									<section>
										<img src="assets/Wetlab_Graphs/E_coli/NMR/NMR_TBoverlay_2.png" width=375>
										<p id="desc" style="position: relative; text-align: center; padding-top: 0.3em; margin-right: 26%;">Characteristic butanol peaks are amplified in 0.1 mM (red) and 0.2 mM (green) IPTG induced cultures</p>
									</section>
								</div>
							</div>
							<br>
							<!-- END OF NMR PICS-->

							<ul class="b">
								<li style="padding-left: 1em;"> The spectrum contains peaks characteristic of butanol, which indicates that butanol is present in the supernatant and produced by KJK01</li>
								<li style="padding-left: 1em;"> The LB culture produced ~1.5 times the amount of butanol produced by the TB cultures</li>
								<li style="padding-left: 1em;"> The supernatants of cultures that were not induced with IPTG showed little butanol content, suggesting leaky expression of the promoter the genes are under</li>
								<li style="padding-left: 1em;"> The supernatants of cultures induced with 0.1 mM IPTG contained slightly more butanol than the ones induced with 0.2 mM IPTG</li>
								<li style="padding-left: 1em;"> KJK01 has been shown to produce equal amounts of ethanol, but due to time constraints we could not analyse the spectrum for ethanol peaks</li>
							</ul>
							<a id="e3"></a><br>
							<!-- END OF NMR FOR BUTANOL-->




							<!-- TRANSFORMATION OF KJK01-->
							<h1>Transformation of KJK01 with pCSCX plasmid</h1>
							<hr><br><br><br>
							<p>KJK01 was transformed with the pCSCX plasmid that would enable it to survive with sucrose as its sole carbon source.<sup><a href="#eref3">[3]</a></sup> The cells were transformed via electroporation and transformed cells were streaked onto LB agar plates containing Kanamycin and Ampicillin.</p>
							<br>
							
							<h4>Results:</h4>
							<p>After overnight incubation, many colonies were seen on the plates, indicating that transformation was successful. Glycerol stocks were made from these plates immediately.</p>
							<br>
							<div class="graph">
								<img src="assets/Wetlab_Graphs/E_coli/Plates/Tranformation_plate_1.jpeg" width=500>
								<p id="desc">LB agar + Kan<sup>50</sup> + Amp<sup>100</sup> plates showing colonies of transformants</p>
							</div>
							<a id="e4"></a>
							<br>

							<!-- END OF TRANSFORMATION-->


							<!-- Growth curve assay for KJK01 and pCSCX-KJK01 -->
							<h1>Growth curve assay for KJK01 and pCSCX-KJK01 </h1>
							<hr><br><br><br>
							<p>We aimed to grow KJK01 and pCSCX-KJK01 in LB and characterize their growth. Due to lack of time in the lab in accordance with COVID-19 restrictions in India, we could perform only one assay and hence could not calculate the growth rates.</p>
							<br>

							<h4>Expected Result:</h4> 
							<p>We expect a sigmoid growth curve for both strains. pCSCX-KJK01 is expected to grow slower than KJK01 due to the metabolic burden of carrying an extra plasmid.</p>
							<br>
							
							<h4>Results:</h4>
							<br>
							
							<!-- Graphs -->
							<div style="display:flex; flex-direction: column; justify-content:space-around; padding-left: 3.5%;">
								<div style="margin: 10px; display:flex; flex-direction: row; flex-wrap: wrap; justify-content:space-around;">
									
									<section>
										<img src="assets/Wetlab_Graphs/E_coli/Growth/LB_Growthcurve_KJK01.png" width="400px" height="400px">
										<p id="desc" style="position: relative; text-align: center; padding-top: 0.3em;">Growth curve of KJK01 2&#176 culture in LB over 11 hours</p>
									</section>

									<section>
										<img src="assets/Wetlab_Graphs/E_coli/Growth/LB_Growthcurve_KJK01_transformed.png" width="400px" height="400px">
										<p id="desc" style="position: relative; text-align: center;  padding-top: 0.3em;"> Growth curve of pCSCX KJK01 2&#176 culture in LB over 11 hours</p>	
									</section>
								
								</div>
							</div>
							<!-- End of graphs -->

							<br><br>
							<ul class="b">
								<li style="padding-left: 1em;">Both strains grow much slower than WT MG1655, probably due to all the modifications made to their genome </li>
								
								<li style="padding-left: 1em;">pCSCX-KJK01 grew slower than KJK01, as expected </li>
								<ul class="b">
									<li style="padding-left: 1em;">When we fit the data to a logistic growth curve, governed by the equation \({dN \over dt} = {Nr(1 - {N \over k})}\), we found that the difference in the inherent growth rate, \(r\),is small: <i>E. coli</i> KJK01 grew at a rate of 0.680 whereas <i>E. coli</i> pCSCX-KJK01 grew at a rate of 0.596. </li>
									<li style="padding-left: 1em;">However, the carrying capacities differed greatly: \(K_{KJK01}\) was around 6.45 while \(K_{pCSCX-KJK01}\) was only 3.44. </li>
									<li style="padding-left: 1em;">We may interpret this to mean that the medium is incapable of supporting higher populations of the transformant as compared to the wild type. The growth curve at a particular OD can be seen by looking at the slope of the graph at a particular value on the y-axis.</li>
								</ul>

								<li style="padding-left: 1em;">The OD<sub>600</sub> of the KJK01 strain increased rapidly to 3, above which our spectrophotometer did not show any reading, causing us to dilute our samples to 50% of the original concentration </li>
								<ul class="c">
									<li style="padding-left: 1em;">In a certain range, the relationship between OD and cell count follows the Beer-Lambert law and is linear, which is why we can double the reading values obtained after dilution<sup><a href="#eref2">[2]</a></sup></li>
									<li style="padding-left: 1em;">We decided to stop taking readings when the cultures reached an OD600 of ~6, as our spectrophotometer stopped showing readings again</li>
									<li style="padding-left: 1em;">The spectrophotometer in our lab does not give accurate readings above OD600 1, which is why readings from our experiment above that limit are not very reliable </li>
								</ul>
								<li style="padding-left: 1em;">We took 5 readings per sample, but the standard deviation was rather small. This lead to error bars smaller than the point size in our graphs, which is why they are not clearly visible, but are certainly present. </li>
							</ul>
							<a id="e5"></a><br>
							<!-- END OF GROWTH CURVE ASSAY -->


							<!-- MIC Assay -->
							<h1> MIC assay of butanol and ethanol for KJK01</h1>
							<hr><br><br><br>

							<p>We performed a MIC assay of butanol and ethanol to identify the minimum concentrations of these compounds at which the growth of our strain gets inhibited. We performed the assay in three 96 welled plates. We used the wild type <i>E. coli</i> strain MG1655 as a control. Our concentration range was from 0.24 to 60 mg/mL (9 dilutions).</p>
							<br>

							<h4>Expected Results</h4>
							<p>For our wild type strain, we expect a MIC of 7.5 to 15 mg/mL for butanol and above 15 mg/mL for ethanol.<sup><a href="#eref4">[4]</a></sup><sup>,</sup><sup><a href="#eref5">[5]</a></sup> Since we have not made any modifications in KJK01 to increase it’s butanol tolerance, we expect similar results in this strain.</p>
							<br>

							<h4>Results</h4>
							<br>
							<!-- PLATE 1 MIC -->
							<div id="MIC1" class="carousel slide" data-bs-ride="carousel">
								<div class="carousel-indicators" style="bottom:-50px;">
								  <button type="button" data-bs-target="#MIC1" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
								  <button type="button" data-bs-target="#MIC1" data-bs-slide-to="1" aria-label="Slide 2"></button>
								  <button type="button" data-bs-target="#MIC1" data-bs-slide-to="2" aria-label="Slide 3"></button>
								  <button type="button" data-bs-target="#MIC1" data-bs-slide-to="3" aria-label="Slide 4"></button>
								  <button type="button" data-bs-target="#MIC1" data-bs-slide-to="4" aria-label="Slide 4"></button>
								  <button type="button" data-bs-target="#MIC1" data-bs-slide-to="5" aria-label="Slide 4"></button>


								</div>
								<div style= "margin-bottom:50px; display:flex; flex-direction:row; align-items: center;" class="carousel-inner">
									<div class="carousel-item active graph"> 
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_1_MG1655_b.png" alt="Best 5 over-expression targets in one graph" width=800>
									</div>

									<div class="carousel-item graph">
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_1_KJK01_b.png" alt="Next best 5 over-expression targets in one graph" width=800>
									</div>

									<div class="carousel-item graph">
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_1_KJK01_ind_b.png" alt="Third best 5 over-expression targets in one graph" width=800>
									</div>

									<div class="carousel-item graph">
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_1_MG1655_e.png" alt="Fourth best 5 over-expression targets in one graph" width=800>
									</div>

									<div class="carousel-item graph">
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_1_KJK01_e.png" alt="Next best 5 over-expression targets in one graph" width=800>
									</div>

									<div class="carousel-item graph">
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_1_KJK01_ind_e.png" alt="Next best 5 over-expression targets in one graph" width=800>
									</div>

								</div>


								<button class="carousel-control-prev" type="button" data-bs-target="#MIC1" data-bs-slide="prev">
								  <span class="carousel-control-prev-icon" aria-hidden="true"></span>
								  <span class="visually-hidden">Previous</span>
								</button>

								<button class="carousel-control-next" type="button" data-bs-target="#MIC1" data-bs-slide="next">
								  <span class="carousel-control-next-icon" aria-hidden="true"></span>
								  <span class="visually-hidden">Next</span>
								</button>

							</div>
							<div style="margin-top: 20px;display:flex; flex-direction:row; justify-content: center;"></div>
							<p id="desc" style="justify-content: center; display: flex;">MIC Graphs for plate 1</p><br><br><br>
							<!-- END OF PLATE 2 MIC-->


							<!-- PLATE 2 MIC -->
							<div id="MIC2" class="carousel slide" data-bs-ride="carousel">
								<div class="carousel-indicators" style="bottom:-50px;">
								  <button type="button" data-bs-target="#MIC2" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
								  <button type="button" data-bs-target="#MIC2" data-bs-slide-to="1" aria-label="Slide 2"></button>
								  <button type="button" data-bs-target="#MIC2" data-bs-slide-to="2" aria-label="Slide 3"></button>
								  <button type="button" data-bs-target="#MIC2" data-bs-slide-to="3" aria-label="Slide 4"></button>
								  <button type="button" data-bs-target="#MIC2" data-bs-slide-to="4" aria-label="Slide 4"></button>
								  <button type="button" data-bs-target="#MIC2" data-bs-slide-to="5" aria-label="Slide 4"></button>


								</div>
								<div style= "margin-bottom:50px; display:flex; flex-direction:row; align-items: center;" class="carousel-inner">
									<div class="carousel-item active graph"> 
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_2_MG1655_b.png" alt="Best 5 over-expression targets in one graph" width=800>
									</div>

									<div class="carousel-item graph">
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_2_KJK01_b.png" alt="Next best 5 over-expression targets in one graph" width=800>
									</div>

									<div class="carousel-item graph">
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_2_KJK01_ind_b.png" alt="Third best 5 over-expression targets in one graph" width=800>
									</div>

									<div class="carousel-item graph">
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_2_MG1655_e.png" alt="Fourth best 5 over-expression targets in one graph" width=800>
									</div>

									<div class="carousel-item graph">
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_2_KJK01_e.png" alt="Next best 5 over-expression targets in one graph" width=800>
									</div>

									<div class="carousel-item graph">
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_2_KJK01_ind_e.png" alt="Next best 5 over-expression targets in one graph" width=800>
									</div>

								</div>


								<button class="carousel-control-prev" type="button" data-bs-target="#MIC2" data-bs-slide="prev">
								  <span class="carousel-control-prev-icon" aria-hidden="true"></span>
								  <span class="visually-hidden">Previous</span>
								</button>

								<button class="carousel-control-next" type="button" data-bs-target="#MIC2" data-bs-slide="next">
								  <span class="carousel-control-next-icon" aria-hidden="true"></span>
								  <span class="visually-hidden">Next</span>
								</button>

							</div>
							<div style="margin-top: 20px;display:flex; flex-direction:row; justify-content: center;"></div>
							<p id="desc" style="justify-content: center; display: flex;">MIC Graphs for plate 2</p><br><br><br>
							<!-- END OF PLATE 1 MIC-->


							<!-- PLATE 3 MIC -->
							<div id="MIC3" class="carousel slide" data-bs-ride="carousel">
								<div class="carousel-indicators" style="bottom:-50px;">
								  <button type="button" data-bs-target="#MIC3" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
								  <button type="button" data-bs-target="#MIC3" data-bs-slide-to="1" aria-label="Slide 2"></button>
								  <button type="button" data-bs-target="#MIC3" data-bs-slide-to="2" aria-label="Slide 3"></button>
								  <button type="button" data-bs-target="#MIC3" data-bs-slide-to="3" aria-label="Slide 4"></button>
								  <button type="button" data-bs-target="#MIC3" data-bs-slide-to="4" aria-label="Slide 4"></button>
								  <button type="button" data-bs-target="#MIC3" data-bs-slide-to="5" aria-label="Slide 4"></button>


								</div>
								<div style= "margin-bottom:50px; display:flex; flex-direction:row; align-items: center;" class="carousel-inner">
									<div class="carousel-item active graph"> 
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_3_MG1655_b.png" alt="Best 5 over-expression targets in one graph" width=800>
									</div>

									<div class="carousel-item graph">
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_3_KJK01_b.png" alt="Next best 5 over-expression targets in one graph" width=800>
									</div>

									<div class="carousel-item graph">
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_3_KJK01_ind_b.png" alt="Third best 5 over-expression targets in one graph" width=800>
									</div>

									<div class="carousel-item graph">
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_3_MG1655_e.png" alt="Fourth best 5 over-expression targets in one graph" width=800>
									</div>

									<div class="carousel-item graph">
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_3_KJK01_e.png" alt="Next best 5 over-expression targets in one graph" width=800>
									</div>

									<div class="carousel-item graph">
										<img src="assets/Wetlab_Graphs/E_coli/MIC/Plate_3_KJK01_ind_e.png" alt="Next best 5 over-expression targets in one graph" width=800>
									</div>

								</div>


								<button class="carousel-control-prev" type="button" data-bs-target="#MIC3" data-bs-slide="prev">
								  <span class="carousel-control-prev-icon" aria-hidden="true"></span>
								  <span class="visually-hidden">Previous</span>
								</button>

								<button class="carousel-control-next" type="button" data-bs-target="#MIC3" data-bs-slide="next">
								  <span class="carousel-control-next-icon" aria-hidden="true"></span>
								  <span class="visually-hidden">Next</span>
								</button>

							</div>
							<div style="margin-top: 20px;display:flex; flex-direction:row; justify-content: center;"></div>
							<p id="desc" style="justify-content: center; display: flex;">MIC Graphs for plate 3</p><br><br><br>
							<!-- END OF PLATE 3 MIC-->							

							<!-- Table -->
							<div class="table" style="padding-left: 5%;">
								<!-- Headers -->
								<div class="table1-row table-header">
									<div class="cell">
										Strain
									</div>
									<div class="cell">
										Compound
									</div>
									<div class="cell">
										MIC (mg/mL)
									</div>
									<div class="cell">
										Comments
									</div>
								</div>
								<!-- End of headers -->

								<!-- Other rows --->
								<div class="table1-row">
									<div class="cell" data-title="Strain">
										MG1655
									</div>
									<div class="cell" data-title="Compound">
										Butanol
									</div>
									<div class="cell" data-title="MIC (mg/mL)">
										1.87 - 3.75
									</div>
									<div class="cell" data-title="Comments">
										Our three plates do not agree to a single value however, two of our plates show closeby values.
									</div>
								</div>

								<div class="table1-row">
									<div class="cell" data-title="Strain">
										KJK01 uninduced
									</div>
									<div class="cell" data-title="Compound">
										Butanol
									</div>
									<div class="cell" data-title="MIC (mg/mL)">
										1.87
									</div>
									<div class="cell" data-title="Comments">
										We observed contamination in plate 1 for concentrations of 3.75 and 7.5 mg/mL. However, plates 1 and 2 agree on the MIC. The third plate is an outlier.
									</div>
								</div>

								<div class="table1-row">
									<div class="cell" data-title="Strain">
										KJK01 induced
									</div>
									<div class="cell" data-title="Compound">
										Butanol
									</div>
									<div class="cell" data-title="MIC (mg/mL)">
										60
									</div>
									<div class="cell" data-title="Comments">
										We observed a high starting OD in some wells of plate 3. However, 2 of our 3 plates indicate this as our MIC. For confirmation, we will have to perform the MIC assay with a higher concentration range.									</div>
								</div>

								<div class="table1-row">
									<div class="cell" data-title="Strain">
										MG1655
									</div>
									<div class="cell" data-title="Compound">
										Ethanol
									</div>
									<div class="cell" data-title="MIC (mg/mL)">
										60
									</div>
									<div class="cell" data-title="Comments">
										All our plates indicate this as the MIC. We can further refine this by considering a different concentration range.
									</div>
								</div>

								<div class="table1-row">
									<div class="cell" data-title="Strain">
										KJK01 uninduced
									</div>
									<div class="cell" data-title="Compound">
										Ethanol
									</div>
									<div class="cell" data-title="MIC (mg/mL)">
										30
									</div>
									<div class="cell" data-title="Comments">
										We observed contamination of a few wells in plate 3 contamination. However, our other two plates are suggestive of an MIC of 30 mg/mL
									</div>
								</div>

								<div class="table1-row">
									<div class="cell" data-title="Strain">
										KJK01 induced
									</div>
									<div class="cell" data-title="Compound">
										Ethanol
									</div>
									<div class="cell" data-title="MIC (mg/mL)">
										30
									</div>
									<div class="cell" data-title="Comments">
										Two of our three plates agree on this value.
									</div>
								</div>
							</div>
							<!-- End of table -->
							<br><br>

							<p>Our results fall close to the range we expected. We observed low tolerance of our strains to butanol. Our induced strain seemed to show a higher tolerance to butanol than the others which suggests that the genes that produce butanol also help tolerate it. Whatever may be the mechanistic reason, the high tolerance of our butanol producing strain works in our favour.</p>
							<br>

							<p>The ethanol tolerance of our strains is as we had predicted. High ethanol tolerance is advantageous for us since our strain produces equal amounts of butanol and ethanol.</p>

							<a id="e6"></a><br>
							<!-- END OF MIC ASSAY-->


							<!-- Sucrose Assay in LB -->
							<h1>Sucrose assay for KJK01 in LB</h1>
							<hr><br><br><br>

							<p>Three strains of <i>E. coli</i> were grown in LB medium to check their sucrose consumption rate. Using the Megazyme Sucrose/D-Glucose Assay Kit,<sup<a href="#eref6">[6]</a> we were able to quantify the amount of sucrose left in our medium over time. This helped us analyse how much sucrose was being taken up by our culture over a period of 24 hrs. Initial concentration of sucrose supplied to the media was 0.25 g/L. </p>
							<br>

							<p>The strains under study were</p>
							<ul class="b">
								<li style="padding-left: 1em;">KJK01 (negative control)</li>
								<li style="padding-left: 1em;">pCSCX-KJK01 + antibiotics (uninduced)</li>
								<li style="padding-left: 1em;">pCSCX-KJK01 + antibiotics + 0.1 mM IPTG (induced)</li>
							</ul>
							<br>

							<h4>Expected Results</h4>
							<ul class="b">
								<li style="padding-left: 1em;">KJK01 was expected to consume minimal/no sucrose</li>
								<li style="padding-left: 1em;">pCSCX-KJK01, both induced and uninduced, were expected to consume sucrose</li>
							</ul>
							<br>

							<h4>Results</h4>
							<ul class="b">
								<li style="padding-left: 1em;">The <i>E. coli</i> KJK01 consumed most sucrose. Out of the 0.25g/L sucrose which was supplied to the media, only 0.006g/L was left after a period of 24 hours, confirming that the culture had consumed it almost entirely.</li>
								<ul class="c">
									<li style="padding-left: 1em;">We however cannot confirm if the sucrose was consumed by KJK01 or other contaminants, due to lack of antibiotics in the medium</li>
								</ul>

								<li style="padding-left: 1em;">The results for pCSCx-KJK01, induced and uninduced, could not be calculated as our spectrophotometer readings were not reliable</li>
							</ul>
							<br>

							<div class="graph">
								<img src="assets/WetlabPhotos/SucroseAssay_ecoli.png" width="500">
								<p id="desc">Gradual decrease in the red colour of quinoneimine dye shows decreasing sucrose concentration in the media over time.</p>
							</div>
							<br>

							<div class="graph">
								<img src="assets/Wetlab_Graphs/E_coli/Sucroseassay_ecoli_result.png" width="500">
								<p id="desc">Decreasing sucrose content in the media of wild type <i>E. coli</i> KJK01 over a period of 24 hours.</p>
							</div>
							<img src="assets/pngs/Mascot_ecoli.png" width=200 id="graph-bottom-left-ecoli" style="margin-left: 70px; margin-top: -260px;">

							<br>
							<a id="e7"></a><a id="eref1"></a><br><a id="eref2"></a><br><a id="eref3"></a><br><a id="eref4"></a>

							<!--REFERENCES -->
							<hr>
							<h3>References</h3>
							<ol class="a">
								<li style="padding-left: 1em;" id="eref5">Ali Samy Abdelaal, Kamran Jawed, Syed Shams Yazdani, CRISPR/Cas9-mediated engineering of Escherichia coli for n-butanol production from xylose in defined medium, Journal of Industrial Microbiology and Biotechnology, Volume 46, Issue 7, 1 July 2019, Pages 965–975, <a href="https://doi.org/10.1007/s10295-019-02180-8">https://doi.org/10.1007/s10295-019-02180-8</a></li>
								<li style="padding-left: 1em;" id="eref6">Stevenson, K., McVey, A. F., Clark, I. B., Swain, P. S., & Pilizota, T. (2016). General calibration of microbial growth in microplate readers. Scientific reports, 6(1), 1-7. <a href="https://doi.org/10.1038/srep38828">https://doi.org/10.1038/srep38828</a> </li>
								<li style="padding-left: 1em;">Bruschi, M., Boyes, S. J., Sugiarto, H., Nielsen, L. K., & Vickers, C. E. (2012). A transferable sucrose utilization approach for non-sucrose-utilizing Escherichia coli strains. Biotechnology advances, 30(5), 1001–1010. <a href="https://doi.org/10.1016/j.biotechadv.2011.08.019">https://doi.org/10.1016/j.biotechadv.2011.08.019</a> </li>
								<li style="padding-left: 1em;">Wilbanks, B., Trinh, C.T. Comprehensive characterization of toxicity of fermentative metabolites on microbial growth. Biotechnol Biofuels 10, 262 (2017). <a href="https://doi.org/10.1186/s13068-017-0952-4">https://doi.org/10.1186/s13068-017-0952-4</a> </li>
								<li style="padding-left: 1em;">Knoshaug, E. P., & Zhang, M. (2009). Butanol tolerance in a selection of microorganisms. Applied biochemistry and biotechnology, 153(1-3), 13–20. <a href="https://doi.org/10.1007/s12010-008-8460-4">https://doi.org/10.1007/s12010-008-8460-4</a> </li>
								<li style="padding-left: 1em;"><a href="https://www.megazyme.com/sucrose-d-glucose-assay-kit">https://www.megazyme.com/sucrose-d-glucose-assay-kit</a></li>
							</ol>
							<br><br>
							<!-- END OF REFERENCES-->

							<!-- END OF SUCROSE ASSAY IN LB-->
						</div>
						<!-------------------------------------- end of e coli -------------------------------------------->



						<!--------------------------------------- start of cyano ---------------------------------------->
						<div class="tab-pane fade cyano-wet" id="cyano-wet-res" role="tabpanel" aria-labelledby="cyano-wet-res-tab">
							<br><br><br>
							
							<!-- GROWTH CURVE -->
							<h1>Growth curve - Ambient carbon dioxide</h1>
							<hr><br><br><br>

							<p>We aimed to characterize the growth of our <i>S. elongatus</i> UTEX 2973 wild type and engineered strain. The population density of our culture is measured in terms of the optical density/absorbance of the culture measured at 730 nm (a wavelength at which there is no absorbance due to photosynthetic or other pigments)<sup><a href="#cref1">[1]</a></sup>. After setting up the culture, a 1 mL sample was taken every 3 hours and its OD was measured in a biospectrometer.</p>
							<br>

							<p>We carried out this experiment with two replicates of WT and two of the <i>cscB</i>-incorporated strain. The data were fit to a logistic growth curve, and \(K-\) and \(r-\) values were estimated. </p>
							<p>Recall that logistic growth is defined by the equation:</p>
							<p> $$ {dN\over dt} = Nr (1 -  {N \over K})$$</p>
							<br>

							<p>Which can be integrated to yield</p>
							<p>$$ N_t = {K \over {1 + Ae^{-rt}}}$$ where $$A = { N_0 \over {K - N_0}}$$</p>
							<br>
							
							<p>Standard deviation was estimated from the difference between the mean of the two replicates and either value; a smaller source of error was the instrumental error, which was added in quadrature as a correction to the former estimate.</p>
							<br>

							<p>At OD<sub>730</sub> values above 0.6, we began seeing fluctuations in the second decimal digit of our biophotometer reading. As these were larger fluctuations, we began diluting the samples for measurements in the more precise range and calculating the OD of the original culture from our measured value.</p>
							<br>

							<p>The first iteration of this assay was carried out at ambient carbon dioxide levels -  that is, 0.04%. We did not induce the gene <i>cscB</i> with IPTG, nor did we use high light levels. The growth curves resulting from these experiments are provided below:</p>
							<br>

							<ul class="b">
								<li style="padding-left: 1em;"><i>S. elongatus</i> UTEX 2973-WT</li>
								<br>
								<div class="graph">
									<img src="assets/Wetlab_Graphs/Cyanobacteria/Cyano_growthcurve1_WT_logisticinv.png">
								</div>
								<img src="assets/pngs/Mascot_cyano.png" width=200 id="graph-bottom-right-cyano">

								<br><br>

								<li style="padding-left: 1em;"><i>S. elongatus</i> UTEX 2973-<i>cscB</i></li>
								<br>
								<div class="graph">
									<img src="assets/Wetlab_Graphs/Cyanobacteria/Cyano_growthcurve1_cscB_logisticinv.png">
								</div>
								<img src="assets/pngs/Mascot_ecoli.png" width=200 id="graph-bottom-left-ecoli">

							</ul>
							<br>

							<p>As we’ve fitted the data to a logistic model, we cannot speak of a single “doubling time” for a UTEX 2973 culture - the rate of doubling would decrease as the culture approaches its carrying capacity. However, at low ODs, the curve is effectively identical to an exponential graph, and the doubling time can be estimated in that neighborhood, as given by the formula:</p>
							<p> $$ T_{double} = {ln(2) \over r}$$</p>
							<br>

							<p>These doubling times are incredibly slow - 27 hours for the wild type and 63 hours for the <i>cscB</i>-transformant. We encourage you to verify this on the graphs above! According to the literature, <i>S. elongatus</i> UTEX 2973 can be grown at doubling times shorter than two hours!<sup><a href="#cref2">[2]</a></sup></p>
							<br>

							<p>It is relevant to note that the \(K \)-value for the <i>cscB</i> strain is essentially infinite -  in other words, the curve is indistinguishable from an exponential growth curve.</p>
							<a id="c2"></a><br><br>
							<!-- End of Growth Curve-->


							<!-- Sucrose Assay -->
							<h1> Sucrose Assay</h1>
							<hr><br><br><br>
							<p>We used Megazyme’s colorimetric assay to estimate sucrose concentration.<sup><a href="#cref3">[3]</a></sup> The assay kit involves hydrolyzing sucrose to produce glucose and fructose, oxidizing the glucose to produce peroxide, and quantifying the peroxide colorimetrically (which is used up to produce a quinoneimine fluorescent dye). We first prepared standard graphs of absorbance versus sucrose concentration, to identify a range of values within which the two bear a linear relationship. We then used the slope of this curve to estimate the sucrose concentration in our ambient-carbon cultures of <i>S. elongatus</i> UTEX 2973 grown at a light intensity of ~100 mol photons/m2/s. 1mL supernatant samples of the culture were taken and stored at 4℃.</p>
							<br>
							
							<p>In accordance with theory, the wild type produced no sucrose. The <i>cscB</i> strain, however, did produce some sucrose over time. We obtained a maximum titer of 0.09 g/L of sucrose in our <i>cscB</i> strain grown in BG-11, at ambient carbon dioxide levels, without any IPTG induction, over a period of 60 hours. This is significantly lower than the titer obtained with the same strains in Lin et al. (2020), which was around 5 g/L over 72 hours.<sup><a href="#cref4">[4]</a></sup> This is unsurprising, as that study took place under vastly different conditions - the strains were grown at a CO2 concentration of 0.5-0.6%, at a light intensity of 250 mol photons/m2/s, and were induced by 1 mM of IPTG. </p>
							<br>
							<div class="graph">
								<img src="assets/Wetlab_Graphs/Cyanobacteria/SucroseAssay_1.png" width=400>
							</div>
							<img src="assets/pngs/Mascot_cyano.png" width=200 id="graph-bottom-right-cyano" style="margin-left: 600px; margin-top: -190px;">

							<a id="c3"></a><br><br>
							<!-- END OF SUCROSE ASSAY -->


							<!-- HIGH CO2-->
							<h1>Growth curve - High carbon dioxide</h1>
							<hr><br><br><br>

							<p>Next, we grew the <i>cscB</i> expressing strain of UTEX 2973 again while replicating the conditions from Lin et al. (2020), to analyze its growth and sucrose production. Both induced (1mM IPTG) and uninduced (0mM IPTG) cultures were maintained in two replicates each. Measurements were taken every 12 hours, and the standard deviation for each data point was calculated as previously.</p>
							<br>

							<p>However, since our incubator is not equipped to supply and maintain CO2, we resorted to bicarbonate flashing: an inexact technique of simulating higher carbon dioxide concentrations by regular manual addition of an equivalent quantity of bicarbonate ions into the medium<sup><a href="#cref5">[5]</a></sup>. We did not have time to optimise the frequency of bicarbonate flashing, or to experiment with refurbishing the nutrients in the medium. We think it is for these reasons that the cultures frequently turned dull and occasionally crashed in OD.</p>
							<br>

							<p>The noise due to these large fluctuations in OD has severely impacted the goodness-of-fit of our logistic curves. These poor fits are provided below:</p>
							<ul class="b">
								<li style="padding-left: 1em;">WT (negative control)</li>
								<br>
								<div class="graph">
									<img src="assets/Wetlab_Graphs/Cyanobacteria/Cyano_growthcurve_HC_WT_loginv.png">
								</div>
								<img src="assets/pngs/Mascot_cyano.png" width=200 id="graph-bottom-left-cyano">

								<br>

								<li style="padding-left: 1em;"><i>cscB</i>(IPTG+)</li>
								<bR>
								<div class="graph">
									<img src="assets/Wetlab_Graphs/Cyanobacteria/Cyano_growthcurve_cscb_IPTG+.png">
								</div>
								<img src="assets/pngs/Mascot_cyano.png" width=200 id="graph-bottom-right-cyano">
								<br>

								<li style="padding-left: 1em;"><i>cscB</i>(IPTG-)</li>
								<br>
								<div class="graph">
									<img src="assets/Wetlab_Graphs/Cyanobacteria/Cyano_growthcurve_cscb_IPTG-.png">
								</div>
								<img src="assets/pngs/Mascot_ecoli.png" width=200 id="graph-bottom-left-ecoli">

								<br>
							</ul>
							<br>

							<p>One insight from these curves is that growth at low ODs is indeed much higher under high carbon and high light settings - the doubling times in the exponential phase were now 6.9 hrs, 23 hrs and 13.9 hrs for the WT, <i>cscB</i> (IPTG+) and cscB (IPTG-) respectively. The lack of steady growth is concerning and we would like to try and optimise the frequency of butanol flashing to get steadily growing cultures.</p>
							<br>

							<p>Unfortunately, we did not have the time to run the sucrose assay on the supernatants we collected under high carbon and high light conditions as of the Wiki freeze. We expect to see a rise in the sucrose titer as  a consequence of the increased carbon flux in the system.</p>
							<a id="c4"></a><br><a id="cref1"></a><br><a id="cref2"></a><br><a id="cref3"></a><br>
							
							<!-- REFERENCES -->
							<hr>
							<h3>References</h3>
							<ol class="a">
								<li style="padding-left: 1em;" id="cref4">Chioccioli, M., Hankamer, B., & Ross, I. L. (2014). Flow cytometry pulse width data enables rapid and sensitive estimation of biomass dry weight in the microalgae Chlamydomonas reinhardtii and Chlorella vulgaris. PLoS One, 9(5), e97269. </li>
								<li style="padding-left: 1em;" id="cref5">Yu, J., Liberton, M., Cliften, P. F., Head, R. D., Jacobs, J. M., Smith, R. D., ... & Pakrasi, H. B. (2015). <i>Synechococcus elongatus</i> UTEX 2973, a fast growing cyanobacterial chassis for biosynthesis using light and CO 2. Scientific reports, 5(1), 1-10. </li>
								<li style="padding-left: 1em;"><a href="https://www.megazyme.com/sucrose-d-glucose-assay-kit">https://www.megazyme.com/sucrose-d-glucose-assay-kit</a> </li>
								<li style="padding-left: 1em;">Lin, PC., Zhang, F. & Pakrasi, H.B. Enhanced production of sucrose in the fast-growing cyanobacterium <i>Synechococcus elongatus</i> UTEX 2973. Sci Rep 10, 390 (2020). <a href="https://doi.org/10.1038/s41598-019-57319-5">https://doi.org/10.1038/s41598-019-57319-5</a> </li>
								<li style="padding-left: 1em;">Sengupta, A., Pritam, P., Jaiswal, D., Bandyopadhyay, A., Pakrasi, H. B., & Wangikar, P. P. (2020). Photosynthetic Co-Production of Succinate and Ethylene in A Fast-Growing Cyanobacterium, <i>Synechococcus elongatus</i> PCC 11801. Metabolites, 10(6), 250. <a href="https://doi.org/10.3390/metabo10060250"></a></li>
							</ol>
							<br><br>
						</div>
						<!-------------------------------------- end of cyano -------------------------------------------->



						<!--------------------------------------- start of co culture ---------------------------------------->
						<div class="tab-pane fade coculture-wet" id="coculture-wet-res" role="tabpanel" aria-labelledby="coculture-wet-res-tab">
							<br><br><br>
							
							<!-- Growth curve assay for KJK01 and pCSCX-KJK01 in BG-11 + sucrose -->
							<h1>Growth curve assay for KJK01 and pCSCX-KJK01 in BG-11 + sucrose</h1>
							<hr><br><br><br>
							<p>We aimed to grow KJK01 and pCSCX-KJK01 in BG-11 + sucrose and characterize their growth. BG-11 does not have any carbon source<sup><a href="#ccref1">[1]</a></sup> as it is meant for photosynthetic organisms. pCSCX is a plasmid containing the <i>cscabk</i> cluster that helps in the transport and metabolism of sucrose<sup><a href="#ccref2">[2]</a></sup>. In the mixture of BG-11 + sucrose, sucrose is the sole carbon source for <i>E. coli</i>. If the strain manages to grow in this mixture, it means that it has the genes necessary for sucrose consumption.</p>
							<br>
							<p>This experiment served as a proxy for our co-culture, as it indicated that our <i>E. coli</i> strains were able to subsist on plain sucrose. The next step would be to grow the strains in BG-11 + supernatant of the cyanobacterial culture in order to see how the additional compounds secreted by <i>S. elongatus</i> affect the growth of the <i>E. coli</i>, and finally moving on actually culturing the two organisms together.</p>
							<br>
							
							<p>We inoculated 3 cultures:</p>
							<ul class="b">
								<li style="padding-left: 1em;">KJK01 (negative control) </li>
								<li style="padding-left: 1em;">pCSCX-KJK01 + antibiotics (uninduced) </li>
								<li style="padding-left: 1em;">pCSCX-KJK01 + antibiotics + 0.1 mM IPTG (induced) </li>
							</ul>
							<br>


							<p>The supernatants would simultaneously be used for sucrose consumption and NMR quantification assays as well, to obtain sucrose consumption rates and butanol production rates for the three cultures.</p>
							<h4>Expected Results:</h4>
							<ul class="b">
								<li style="padding-left: 1em;">KJK01 (negative control) - This culture is expected to show negligible/no growth and no butanol production. </li>
								<li style="padding-left: 1em;">pCSCX-KJK01 (uninduced) - This culture is expected to show fastest growth and low butanol production. </li>
								<li style="padding-left: 1em;">pCSCX-KJK01 (induced with 0.1 mM IPTG) - This culture is expected to show slow growth and maximum butanol production. </li>
							</ul>
							<br>

							<h4>Results:</h4>
							<p>We repeated this experiment twice and obtained two growth curves for each of the three strains.</p>
							<br>

							<p>Growth Curve 1:</p>

							<!-- 3 graphs -->
							<div style="display:flex; flex-direction: column; justify-content:space-around;">
								<div style="margin: 10px; display:flex; flex-direction: row; flex-wrap: wrap; justify-content:space-around;">
									
									<section>
										<img src="assets/Wetlab_Graphs/Co Culture/Growth_Curve_BG11/BG11_Sucrose_GrowthCurve1_KJK01 _1509.png" width="400px" height="400px">
										<p id="desc" style="position: relative; text-align: center; padding-top: 0.3em;">KJK01 growth curve in BG-11 + sucrose over 24 hours</p>
									</section>

									<section>
										<img src="assets/Wetlab_Graphs/Co Culture/Growth_Curve_BG11/BG11_Sucrose_GrowthCurve1_KJK01Transformed_induced_1509.png" width="400px" height="400px">
										<p id="desc" style="position: relative; text-align: center;  padding-top: 0.3em;">pCSCX KJK01 (uninduced) growth curve in BG-11 + sucrose over 24 hours</p>	
									</section>
								
								</div>
								<div style="margin: 10px; display:flex; justify-content:center;">
									<section>
										<img src="assets/Wetlab_Graphs/Co Culture/Growth_Curve_BG11/BG11_Sucrose_GrowthCurve1_KJK01 Transformed_1509.png" width="400px" height="400px">
										<p id="desc" style="position: relative; text-align: center; padding-top: 0.3em;">pCSCX KJK01 (induced) growth curve in BG-11 + sucrose over 24 hours</p>
									</section>
								</div>
							</div>
							<img src="assets/pngs/Mascot_ecoli.png" width=200 id="graph-bottom-left-ecoli" style="margin-left: 90px; margin-top: -300px;">

							<!-- end of graphs -->
							<br><br>


							<ul class="b">
								<li style="padding-left: 1em;">KJK01 - This culture showed good growth, reaching OD600 0.3 in 24 hours </li>
								<li style="padding-left: 1em;">pCSCX-KJK01 (uninduced) - This culture showed no growth in 24 hours </li>
								<li style="padding-left: 1em;">pCSCX-KJK01 (induced with 0.1 mM IPTG) - This culture showed no growth in 24 hours </li>
							</ul>
							<br>
							<p>This was a rather unexpected result and the complete opposite of what we thought would happen. We suspected that the first culture might have been contaminated, and the contaminants were able to survive on sucrose. We could have confirmed if the KJK01 was growing if we had submitted our supernatants for NMR analysis, but the slots were already full. </p>
							<br>


							<p>Growth Curve 2:</p>
							<p>We repeated the same experiment as in growth curve 1, but took readings every two hours instead of one hour, as taking 6 samples, measuring their OD, centrifuging them and freezing the supernatants every hour was simply not practical. We made sure to UV the hood and clean it thoroughly with 70% ethanol in order to minimize chances of contaminating the wild type culture which did not have any antibiotics.</p>
							
							<div style="display:flex; flex-direction: column; justify-content:space-around;">
								<div style="margin: 10px; display:flex; flex-direction: row; flex-wrap: wrap; justify-content:space-around;">
									
									<section>
										<img src="assets/Wetlab_Graphs/Co Culture/Growth_Curve_BG11/BG11_Sucrose_GrowthCurve1_KJK01_1709.png" width="400px" height="400px">
										<p id="desc" style="position: relative; text-align: center; padding-top: 0.3em;">KJK01 growth curve in BG-11 + sucrose over 40 hours</p>
									</section>

									<section>
										<img src="assets/Wetlab_Graphs/Co Culture/Growth_Curve_BG11/BG11_Sucrose_GrowthCurve1_KJK01_transformed_induced_1709.png" width="400px" height="400px">
										<p id="desc" style="position: relative; text-align: center;  padding-top: 0.3em;">pCSCX KJK01 (uninduced) growth curve in BG-11 + sucrose over 40 hours</p>	
									</section>
								
								</div>
								<div style="margin: 10px; display:flex; justify-content:center;">
									<section>
										<img src="assets/Wetlab_Graphs/Co Culture/Growth_Curve_BG11/BG11_Sucrose_GrowthCurve1_KJK01_Transformed_1_1709.png" width="400px" height="400px">
										<p id="desc" style="position: relative; text-align: center; padding-top: 0.3em;">pCSCX KJK01 (induced) growth curve in BG-11 + sucrose over 40 hours</p>
									</section>
								</div>
							</div>
							<img src="assets/pngs/Mascot_ecoli.png" width=200 id="graph-bottom-left-ecoli" style="margin-left: 90px; margin-top: -300px;">
							<br><br>


							<ul class="b">
								<li style="padding-left: 1em;">KJK01 - This culture showed good growth, reaching OD<sub>600</sub> 0.17 in 40 hours </li>
								<li style="padding-left: 1em;">pCSCX-KJK01 (uninduced) - This culture showed no growth in 40 hours </li>
								<li style="padding-left: 1em;">pCSCX-KJK01 (induced with 0.1 mM IPTG) - This culture showed no growth in 40 hours </li>
							</ul>
							<br>
							
							<a id="cc2"></a><br>
							<!-- END OF GROWTH CURVE IN BG11 + SUCROSE-->


							<!-- TRANSFORMATION E COLI-->
							<h1>Transformation of KJK01 with pKD4 plasmid</h1>
							<hr><br><br><br>
							<p>KJK01 was transformed with the pKD4 plasmid, which is a blank plasmid containing only Kanamycin resistance. This strain would serve as a negative control for our next growth curve, and would show that any sucrose consuming properties that have been conferred upon the KJK01 strain is solely via the pCSCX plasmid. The controls in the previous growth curves did not have the same conditions as the rest of the experiment, and hence could not be called true controls. We chose Kan because the sucrose exporting strains of <i>S. elongatus</i> from Prof Pakrasi’s lab have Kanamycin and Gentamycin resistance <sup><a href="#ccref3">[3]</a></sup></p>
							<br>

							<h4>Results</h4>
							<p>Transformation failed, we did not see any colonies on the LB agar + Kan<sup>50</sup> plates.</p>
							<br>

							<p>A possible reason for this might be because the pKD4 plasmid has an incompatible ori. We were already warned by Dr. Nishad Matange (who lent us the plasmid) that we may not be able to get colonies with pKD4 because it contains oriR𝛄 which requires that require the trans-acting Π protein (the pir gene product) for replication<sup><a href="#ccref4">[4]</a></sup>.</p>
							<br>

							<p>Dr. Gayatri Panaghat gave an alternative to this by suggesting that we perform site directed mutagenesis on our pcscX plasmid and remove the genes coding for <i>cscabk</i> and replace it with a Kan resistance gene, which would serve as a true negative control. However, we did not have the necessary time for ordering primers and carrying out the mutagenesis process. Hence, we proceeded by Dr Pananghat’s second advice, which was to repeat the experiment with a strict check on possible sources of contamination.</p>
							<a id="cc3"></a><br>
							<!-- end of transformation-->


							<!-- Growth curve COBG11 -->
							<h1>Growth curve assay for KJK01 and pCSCX-KJK01 in CoBG-11 + sucrose</h1>
							<hr><br><br><br>

							<p>After our first two growth curves failed, we went back to the Zhang et al<sup><a href="#ccref5">[5]</a></sup> paper to look at their protocol, because we did not have time to optimize our current protocol further. We replaced BG-11 with CoBG-11, a medium created by the authors that contains BG-11 as well as certain salts that help <i>E. coli</i> grow better.<sup><a href="#cc5">[5]</a></sup> We also took readings every 6 hours, as the strains seemed to be slow-growing.</p>
							<br>
							
							<p>We inoculated 3 cultures:</p>
							<ul class="b">
								<li style="padding-left: 1em;"> KJK01 (negative control)</li>
								<li style="padding-left: 1em;">pCSCX-KJK01 (uninduced)</li>
								<li style="padding-left: 1em;">pCSCX-KJK01 + 0.1 mM IPTG (induced)</li>
							</ul>
							<br>
							
							<p>Since the transformation with pKD4 failed, we did not have a valid negative control anymore, hence upon our mentor’s recommendation, we inoculated all cultures without antibiotics.</p>
							<br>

							<div style="display:flex; flex-direction: column; justify-content:space-around;">
								<div style="margin: 10px; display:flex; flex-direction: row; flex-wrap: wrap; justify-content:space-around;">
									
									<section>
										<img src="assets/Wetlab_Graphs/Co Culture/Growth_Curve_BG11/CoBG11_Sucrose_GrowthCurve1_KJK01 .png" width="400px" height="400px">
										<p id="desc" style="position: relative; text-align: center; padding-top: 0.3em;">KJK01 growth curve in CoBG-11 + sucrose over 96 hours</p>
									</section>

									<section>
										<img src="assets/Wetlab_Graphs/Co Culture/Growth_Curve_BG11/CoBG11_Sucrose_GrowthCurve1_KJK01_transformed.png" width="400px" height="400px">
										<p id="desc" style="position: relative; text-align: center;  padding-top: 0.3em;">pCSCX KJK01 (uninduced) growth curve in CoBG-11 + sucrose over 96 hours</p>	
									</section>
								
								</div>
								<div style="margin: 10px; display:flex; justify-content:center;">
									<section>
										<img src="assets/Wetlab_Graphs/Co Culture/Growth_Curve_BG11/CoBG11_Sucrose_GrowthCurve1_KJK01_transformed_induced.png" width="400px" height="400px">
										<p id="desc" style="position: relative; text-align: center; padding-top: 0.3em;">pCSCX KJK01 (induced) growth curve in CoBG-11 + sucrose over 96 hours</p>
									</section>
								</div>
							</div>
							<img src="assets/pngs/Mascot_cyano.png" width=200 id="graph-bottom-right-cyano" style="margin-left: 600px; margin-top: -225px;">



							<h4>Results</h4>
							<ul class="b">
								<li style="padding-left: 1em;"> KJK01 - The culture hit an OD<sub>600</sub> of 0.26 in 4 days</li>
								<li style="padding-left: 1em;">pCSCX-KJK01 (uninduced) - The culture showed no growth in 4 days</li>
								<li style="padding-left: 1em;">pCSCX-KJK01 + 0.1 mM IPTG (induced) - The culture hit an OD<sub>600</sub> of 0.22 in 4 days</li>
							</ul>
							<br>
							
							<p>We cannot tell if the OD is due to contamination or the growth of the culture itself because once again the WT KJK01, which has no sucrose metabolising genes, has managed to grow while the pCSCX-KJK01 (uninduced) culture has not grown. We do however see growth in the pCSCX-KJK01 (induced) culture. </p>
							<br>

							<p>The only way we can confirm if the culture has grown is by analyzing the supernatants to check for the presence of butanol. We have submitted the supernatants for analysis, but unfortunately, the results will not be here before the Wiki freeze deadline. Even if butanol is found in the supernatant we cannot be sure if the <i>E. coli</i> has grown on the sucrose in the medium or some other sugars released by the contaminants due to the absence of antibiotics.</p>
							<br>

							<p>As a part of our future work, we plan to repeat this experiment by using the modified pCSCX plasmid described above as our negative control. We also need to check if the sucrose given to the <i>E. coli</i> is of lower concentration than required. The Zhang et al<sup><a href="#ccref5">[5]</a></sup> paper used concentrations up to 200 mg/L based on the maximum concentration of sucrose secreted by the <i>S. elongatus</i>, but their <i>E. coli</i> had the <i>cscabk</i> genes integrated into the genome. The strains containing the pCSCX plasmid had been grown on media containing 2% sucrose<sup><a href="#ccref2">[2]</a></sup>. Along with the right negative control, we will also repeat this experiment with higher concentrations of sucrose.</p>
							<a id="cc4"></a><br>
							<!-- END OF TRANSFORMATION-->


							<!-- SUCROSE ASSAY -->
							<h1> Sucrose assay</h1>
							<hr><br><br><br>

							<p>Luria broth (LB) is the most commonly used growth medium for <i>E. coli</i>. Considering that we have planned to use BG-11 as our final co-culture media, we decided to grow three strains of <i>E. coli</i> in a medium containing BG-11 + 0.25g/L as a proxy for our co-culture.</p>
							<br>

							<p>Using the Megazyme Sucrose/D-Glucose Assay Kit<sup><A href="#ccref6">[6]</A></sup>, we had planned to quantify the amount of sucrose left in our medium over time. This would help us analyse how much sucrose was being taken up by our culture over a period of 24 hrs.</p>
							<br>

							<h4>Expected Results</h4>
							<ul class="b">
								<li style="padding-left: 1em;">KJK01 was expected to consume sucrose with less rate of consumption </li>
								<li style="padding-left: 1em;">pCSCX-KJK01, both induced and uninduced, were expected to consume sucrose </li>
							</ul>
							<br>

							<h4>Results for Growth curve 1</h4>
							<ul class="b">
								<li style="padding-left: 1em;">KJK01 had 0.009 g/L sucrose left in the medium, having the highest rate of sucrose consumption.</li>
								<li style="padding-left: 1em;">pCSCX-KJK01 + antibiotics (uninduced) had 0.169 g/L sucrose left in the medium, having the lowest rate of sucrose consumption.  </li>
								<li style="padding-left: 1em;">pCSCX-KJK01 + antibiotics + 0.1 mM IPTG (induced) had 0.155 g/L sucrose left in the medium </li>
							</ul>

							<p>We can confirm that the sucrose in the pCSCX-KJK01 cultures were consumed by the strain as there were antibiotics in the medium. We cannot however say the same for the KJK01 cultures.</p>
							<bR>

							<p>The results for Growth curve 2 and 3 are not ready yet</p>
							<br>

							<p>These experiments are just the beginning to confirm that our transformed pCSCX-KJK01 induced strain of <i>E. coli</i> can not only subsist in the BG-11 medium, but can also efficiently consume sucrose present in the medium. </p>
							<br>
							
							<p>Hopefully, as we troubleshoot more, we wish to confirm without doubt that the sucrose consuming ability of pCSCX-KJK01 is due to pCSCX, as well as improve their sucrose uptake efficiency.</p>
							<a id="cc5"></a><br><a id="ccref1"></a><br><a id="ccref2"></a><br><a id="ccref3"></a>
							<!-- end of sucrose assay-->


							<!-- references -->
							<hr>
							<h3>References</h3>
							<ol class="a">
								<li style="padding-left: 1em;" id="ccref4"><a href="https://www.himedialabs.com/TD/M1541.pdf">https://www.himedialabs.com/TD/M1541.pdf</a></li>
								<li style="padding-left: 1em;" id="ccref5">Bruschi, M., Boyes, S. J., Sugiarto, H., Nielsen, L. K., & Vickers, C. E. (2012). A transferable sucrose utilization approach for non-sucrose-utilizing Escherichia coli strains. Biotechnology advances, 30(5), 1001–1010. <a href="https://doi.org/10.1016/j.biotechadv.2011.08.019">https://doi.org/10.1016/j.biotechadv.2011.08.019</a></li>
								<li style="padding-left: 1em;" id="ccref6">Lin, PC., Zhang, F. & Pakrasi, H.B. Enhanced production of sucrose in the fast-growing cyanobacterium <i>Synechococcus elongatus</i> UTEX 2973. Sci Rep 10, 390 (2020). <a href="https://doi.org/10.1038/s41598-019-57319-5">https://doi.org/10.1038/s41598-019-57319-5</a> </li>
								<li style="padding-left: 1em;">Datsenko, K. A., & Wanner, B. L. (2000). One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proceedings of the National Academy of Sciences of the United States of America, 97(12), 6640–6645. <a href="https://doi.org/10.1073/pnas.120163297">https://doi.org/10.1073/pnas.120163297</a></li>
								<li style="padding-left: 1em;">Zhang, L., Chen, L., Diao, J. et al. Construction and analysis of an artificial consortium based on the fast-growing cyanobacterium <i>Synechococcus elongatus</i> UTEX 2973 to produce the platform chemical 3-hydroxypropionic acid from CO2. Biotechnol Biofuels 13, 82 (2020). <a href="https://doi.org/10.1186/s13068-020-01720-0">https://doi.org/10.1186/s13068-020-01720-0</a> </li>
								<li style="padding-left: 1em;"> <a href="https://www.megazyme.com/sucrose-d-glucose-assay-kit">https://www.megazyme.com/sucrose-d-glucose-assay-kit</a></li>
							</ol>
							<br><br>

						</div>
						<!-------------------------------------- end of co culture -------------------------------------------->
					</div>
				</div>
			</div>
        </div>
    </div>



	<!---------------------------------------------- SPONSORS BAR ------------------------------------------------------------>
	<div class="footer">
		<div class="footer-wrapper">
			<!-- <div id="not-sponsors" class="footer-wrapper">	 -->
				<div id="Team", class="footer-wrapper">
					<img class="teamlogo" src="assets/Logos/IISER-Pune-iGEM_logo.png" alt="iGEM IISER Pune Logo" width="170">
				</div>
				<div id="socials" class="vertical-stack">
					<div><h3>Team IISER Pune India</h3></div>
					<div id="logos" class ="footer-wrapper ">
						<a href="https://www.instagram.com/igem_iiserp/"><img class="footer-links-socials" src="assets/Logos/Instagram_logo.jpg" alt="Instagram" width="20"></a>
						<a href="https://www.linkedin.com/in/igem-iiser-pune-2021/"><img class="footer-links-socials" src="assets/Logos/linkedin_logo.png" alt="Linkedin" width="30"></a>
						<a href="https://twitter.com/igem_iiserpune"><img class="footer-links-socials" src="assets/Logos/Twitter_logo.png" alt="Twitter" width="35"></a>
						<a href="https://www.facebook.com/igemiiserpune"><img class="footer-links-socials" src="assets/Logos/Facebook_logo.png" alt="Facebook" width="25"></a>
						<a href="https://www.youtube.com/channel/UCyo6NMLvM58sSh_tnxht36w"><img class="footer-links-socials" src="assets/Logos/Youtube_logo.png" alt="Youtube" width="35"></a>
					</div>
				</div>
			<!-- </div> -->
			<div id="sponsors" class="footer-wrapper">
				<div id="main-sponsor", class="">
					<a href="https://www.sailife.com/"><img class="footer-links" src="assets/Logos/Sai_logo.jpg" alt="Sai Life Sciences" width="280"></a>
				</div>
				<div class="vertical-stack">
					<a href="https://sg.idtdna.com/pages"><img class="footer-links" src="assets/Logos/IDT_logo.png" alt="IDT" width="220"></a>
					<a href="https://www.geneious.com/"><img class="footer-links" src="assets/Logos/Geneious_logo.png" alt="Geneious" width="220"></a>
				</div>
				<div class="vertical-stack">
					<div id="horizontal stack">
						<a href="https://www.notion.so/"><img class="footer-links" src="assets/Logos/Notion_logo.png" alt="Notion" width="70"></a>
						<img src="assets/Logos/IISER Pune Logo.png" alt="IISER Pune" width="80" style="padding-left: 1em;">
					</div>
					<a href="https://www.zoho.com/"><img class="footer-links" src="assets/Logos/Zoho_logo.png" alt="Zoho" width="150"></a>
				</div>
			</div>	
		</div>
	</div>
	<!------------------------------------------ END OF SPONSOR BAR ---------------------------------------------------------> 



	<!--------------------------------------- ACTUAL SCROLLING BAR ----------------------------------------------------->
	<div class="header"></div>
    <div class="line" id="scrollIndicator"></div>

	<!-- Gets the screenheight of each page for Scrollbar? -->
	<script>
		var screenheight = window.innerHeight,
		    screenwidth = window.innerWidth;

	</script>


	<!-- Scrollbar stuff again, only @Namasi22 knows :) -->
	<script>
		var body = document.body,
    		html = document.documentElement;

		var pageheight = Math.max( body.scrollHeight, body.offsetHeight, 
                       		   html.clientHeight, html.scrollHeight, html.offsetHeight );
	</script>


	<!-- To make the button appear -->
	<script>
		var backtotopheight=pageheight/4.65
	</script>


    <!------------------------------------------ back to top button ------------------------------------------------------>
    <a href="#top"><img src="assets/svgs/arrow-up-square-fill.svg" id="myBtn" title="Go to top" width="30" height="30"></a>

	
	<!---------------------------------------- LINKING MAIN JS FILES ----------------------------------------------------->
    <script src="js/bootstrap-bundle.js"></script>
	<script src="js/main.js"></script>
	<script src="js/jquery.min.js"></script>
	<script src="https://2021.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>


	


	<!------------------------- Displaying sidebar and content tab depending on the tab press---------------------------->
	<script>
		document.querySelectorAll('button[data-bs-toggle="tab"]').forEach((t,i)=>{
			t.addEventListener('show.bs.tab', function (e) {

				let targetClass = t.dataset.bsTarget;
				var sidepane = document.querySelector('#SidenavTabs ' + targetClass);
				var contentpane = document.querySelector('#myTabContent ' + targetClass);
				var sidesibling = document.querySelector('#SidenavTabs .tab-pane.active');
				var contentsibling = document.querySelector('#myTabContent .tab-pane.active');
				var heightpane = document.querySelector('.tab-content.sidenav');


				// hide 2nd pane sibling
				sidesibling.classList.remove('show');
				sidesibling.classList.remove('active');
				contentsibling.classList.remove('show');
				contentsibling.classList.remove('active');

				// show 2nd pane
				sidepane.classList.add('show');
				sidepane.classList.add('active');
				contentpane.classList.add('show');
				contentpane.classList.add('active');

				if ($('#ecoli-wet-res').hasClass('active')) {
					heightpane.style.top = '-23.5%';
				} else if ($('#cyano-wet-res').hasClass('active')) {
					heightpane.style.top = '-12%';
				} if ($('#coculture-wet-res').hasClass('active')) {
					heightpane.style.top = '-19%';
				} 

			})  
		})
	</script>
	<!------------------------------------------ End of display script ------------------------------------------>




	<!-- SCRIPT FOR SCROLL AWARE SIDEBAR HIGHLIGHT -->
	<!-- ------------------------------------------------------------------------------------------------------------- -->
	<!-- MAKE SURE YOU CHANGE THE NUMBER OF DIVISIONS -->
	<script>
		
		$(document).ready(function() {
			$(window).scroll(function() {

				if($('#ecoli-wet-res').hasClass('active')) {
					var scroll_top = $(window).scrollTop();
					var div_one_top = $("#e1").offset().top;
					var div_two_top = $("#e2").offset().top;
					var div_three_top = $("#e3").offset().top;
					var div_four_top = $("#e4").offset().top;
					var div_five_top = $("#e5").offset().top;
					var div_six_top = $("#e6").offset().top;
					var div_seven_top = $("#e7").offset().top;
		
					if (scroll_top > div_one_top && scroll_top < div_two_top) {
						//Division 1
						if (!$('a[href="#e1"]').hasClass('sidenav-active')) {
							$('a[href="#e1"]').addClass('sidenav-active');
						}
		
						if ($('a[href="#e2"]').hasClass('sidenav-active')) {
							$('a[href="#e2"]').removeClass('sidenav-active');
						}
		
					} else if (scroll_top > div_two_top && scroll_top < div_three_top) {
						// Division 2
						if ($('a[href="#e1"]').hasClass('sidenav-active')) {
							$('a[href="#e1"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#e3"]').hasClass('sidenav-active')) {
							$('a[href="#e3"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#e2"]').hasClass('sidenav-active')) {
							$('a[href="#e2"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_three_top && scroll_top < div_four_top) {
						//Division 3
						if ($('a[href="#e2"]').hasClass('sidenav-active')) {
							$('a[href="#e2"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#e4"]').hasClass('sidenav-active')) {
							$('a[href="#e4"]').removeClass('sidenav-active');
						} 
		
						if (!$('a[href="#e3"]').hasClass('sidenav-active')) {
							$('a[href="#e3"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_four_top && scroll_top < div_five_top) {
						// You are in div 4
						if ($('a[href="#e3"]').hasClass('sidenav-active')) {
							$('a[href="#e3"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#e5"]').hasClass('sidenav-active')) {
							$('a[href="#e5"]').removeClass('sidenav-active');
						} 
		
						if (!$('a[href="#e4"]').hasClass('sidenav-active')) {
							$('a[href="#e4"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_five_top && scroll_top < div_six_top) {
						// You are in div 5
						if ($('a[href="#e4"]').hasClass('sidenav-active')) {
							$('a[href="#e4"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#e6"]').hasClass('sidenav-active')) {
							$('a[href="#e6"]').removeClass('sidenav-active');
						} 
		
						if (!$('a[href="#e5"]').hasClass('sidenav-active')) {
							$('a[href="#e5"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_six_top && scroll_top < div_seven_top) {
						// You are in div 6
						if ($('a[href="#e5"]').hasClass('sidenav-active')) {
							$('a[href="#e5"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#e7"]').hasClass('sidenav-active')) {
							$('a[href="#e7"]').removeClass('sidenav-active');
						} 
		
						if (!$('a[href="#e6"]').hasClass('sidenav-active')) {
							$('a[href="#e6"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_seven_top) {
						// You are in div 7
						if ($('a[href="#e6"]').hasClass('sidenav-active')) {
							$('a[href="#e6"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#e7"]').hasClass('sidenav-active')) {
							$('a[href="#e7"]').addClass('sidenav-active');
						}
					}
				} 
				else if ($('#cyano-wet-res').hasClass('active')) {
					var scroll_top = $(window).scrollTop();
					var div_one_top = $("#c1").offset().top;
					var div_two_top = $("#c2").offset().top;
					var div_three_top = $("#c3").offset().top;
					var div_four_top = $("#c4").offset().top;

					if (scroll_top > div_one_top && scroll_top < div_two_top) {
						//Division 1
						if (!$('a[href="#c1"]').hasClass('sidenav-active')) {
							$('a[href="#c1"]').addClass('sidenav-active');
						}
		
						if ($('a[href="#c2"]').hasClass('sidenav-active')) {
							$('a[href="#c2"]').removeClass('sidenav-active');
						}
		
					} else if (scroll_top > div_two_top && scroll_top < div_three_top) {
						// Division 2
						if ($('a[href="#c1"]').hasClass('sidenav-active')) {
							$('a[href="#c1"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#c3"]').hasClass('sidenav-active')) {
							$('a[href="#c3"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#c2"]').hasClass('sidenav-active')) {
							$('a[href="#c2"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_three_top && scroll_top < div_four_top) {
						//Division 3
						if ($('a[href="#c2"]').hasClass('sidenav-active')) {
							$('a[href="#c2"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#c4"]').hasClass('sidenav-active')) {
							$('a[href="#c4"]').removeClass('sidenav-active');
						} 
		
						if (!$('a[href="#c3"]').hasClass('sidenav-active')) {
							$('a[href="#c3"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_four_top) {
						// You are in div 4
						if ($('a[href="#c3"]').hasClass('sidenav-active')) {
							$('a[href="#c3"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#c4"]').hasClass('sidenav-active')) {
							$('a[href="#c4"]').addClass('sidenav-active');
						}
					}
				}
				else if ($('#coculture-wet-res').hasClass('active')) {
					var scroll_top = $(window).scrollTop();
					var div_one_top = $("#cc1").offset().top;
					var div_two_top = $("#cc2").offset().top;
					var div_three_top = $("#cc3").offset().top;
					var div_four_top = $("#cc4").offset().top;
					var div_five_top = $("#cc5").offset().top;
		
					if (scroll_top > div_one_top && scroll_top < div_two_top) {
						//Division 1
						if (!$('a[href="#cc1"]').hasClass('sidenav-active')) {
							$('a[href="#cc1"]').addClass('sidenav-active');
						}
		
						if ($('a[href="#cc2"]').hasClass('sidenav-active')) {
							$('a[href="#cc2"]').removeClass('sidenav-active');
						}
		
					} else if (scroll_top > div_two_top && scroll_top < div_three_top) {
						// Division 2
						if ($('a[href="#cc1"]').hasClass('sidenav-active')) {
							$('a[href="#cc1"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#cc3"]').hasClass('sidenav-active')) {
							$('a[href="#cc3"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#cc2"]').hasClass('sidenav-active')) {
							$('a[href="#cc2"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_three_top && scroll_top < div_four_top) {
						//Division 3
						if ($('a[href="#cc2"]').hasClass('sidenav-active')) {
							$('a[href="#cc2"]').removeClass('sidenav-active');
						}
		
						if ($('a[href="#cc4"]').hasClass('sidenav-active')) {
							$('a[href="#cc4"]').removeClass('sidenav-active');
						} 
		
						if (!$('a[href="#cc3"]').hasClass('sidenav-active')) {
							$('a[href="#cc3"]').addClass('sidenav-active');
						}
					} else if (scroll_top > div_four_top && scroll_top < div_five_top) {
						// You are in div 4
						if ($('a[href="#cc3"]').hasClass('sidenav-active')) {
							$('a[href="#cc3"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#cc4"]').hasClass('sidenav-active')) {
							$('a[href="#cc4"]').addClass('sidenav-active');
						} 

						if ($('a[href="#cc5"]').hasClass('sidenav-active')) {
							$('a[href="#cc5"]').removeClass('sidenav-active');
						}
					} else if (scroll_top > div_five_top) {
						// You are in div 4
						if ($('a[href="#cc4"]').hasClass('sidenav-active')) {
							$('a[href="#cc4"]').removeClass('sidenav-active');
						}
		
						if (!$('a[href="#cc5"]').hasClass('sidenav-active')) {
							$('a[href="#cc5"]').addClass('sidenav-active');
						} 
					}
				}	
			});
		});
		
	</script>	
	<!-------------------------------------------- END OF SCROLL AWARE HIGHLIGHT --------------------------------------->



	
	<!-- Script for the slideshow, wherever needed -->
	<script>
		var total = $('.item').length;
		var actualtotal = 80;
		var currentIndex = $('div.active').index() + 1;
		$('#slidetext').html(currentIndex + '/'  + actualtotal);

		// This triggers after each slide change
		$('.carousel').on('slid.bs.carousel', function () {
  			currentIndex = $('div.active').index() + 1;
 
  		// Now display this wherever you want
  		var text = currentIndex + '/' + actualtotal;
  			$('#slidetext').html(text);
		});

	</script>


	<!-- READ MORE SCRIPT-->
	<script>
		function myFunction() {
			var dots = document.getElementById("dots");
			var moreText = document.getElementById("more");
			var btnText = document.getElementById("ReadMoreButton");

			if (dots.style.display === "none") {
				dots.style.display = "inline";
				btnText.innerHTML = "Read more";
				moreText.style.display = "none";
			} else {
				dots.style.display = "none";
				btnText.innerHTML = "Read less";
				moreText.style.display = "inline";
			}
		}
	</script>

</body>
