<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Bootstrap and Sass Testing</title>
	<link rel="stylesheet" href="css/igem-reset.css">
	<link rel="stylesheet" href="css/SBcss/style.css">
    <link rel="stylesheet" href="css/styleSB.css">
    <link rel="stylesheet" href="css/DryLabPage.css">
	<style>/* Reset CSS */

		#sideMenu, #top_title, .patrollink, #firstHeading, #home_logo, #sideMenu { display:none; }
		#content { padding:0px; width:100%; margin-top:-7px; margin-left:0px; border:none;}
		body, html {background-color:#212529; width: 100%; height: 100%;}
		#bodyContent h1, #bodyContent h2, #bodyContent h3, #bodyContent h4, #bodyContent h5 { margin-bottom: 0px; }
		#bodyContent a[href ^="https://"], .link-https { padding-right:0px;}

    </style>
    <style>/* Sidenav */
.sidenav {
      height: auto;
      width: auto;
      padding-top: 60px;
      padding-left: 10px;

      margin-top: 25%;
      margin-bottom: 20px;
      position: -webkit-sticky;
      position: sticky;
      top: 10%!important;
    }



.sidenav a {
      text-align: left;
      color: #cc3333;
      font-size: 2vmin!important;
        font-family: 'Titillium Web'!important;
  font-style: normal !important;
  font-weight: 600 !important;
    }

.sidenav a:hover {
      color: #ffffff;
      background-color: #cc3333;
      border-radius: 20px!important;
    }

@media (max-width: 900px) {
      .sidenav {
                display: none;
                }
    }

    </style>
    <style>/* Second-div */
.second-div {
      background-color: #212529 !important;

    }

    .second-div .content h1 {
      text-align: left !important;
      padding-top: 8%;
      font-family: "Titillium Web" !important;
        font-weight:700 !important;
         font-size: 6vmin !important;

    }

    .second-div .content p {
      padding-left: 5%;
      padding-right: 5%;
      font-family: "Titillium Web" !important;
        font-weight:400 !important;
         font-size: 2.5vmin !important;
    }





            html {
                scroll-behavior: smooth;
                }
                /* body{
                    padding-top:65px;
                    background:#edd1cb;
                } */
	</style>

    <style>/* Other */
ul{
list-style: none !important;

}

ul li{
list-style:none !important;
}

    /* @media (max-width: 768px) {
      .top-illus{
        margin-top:0px;
      }
      
    }*/
body {
            position: relative;
            width: 100%;
            padding-top: 0px;
        }
html {
        scroll-behavior: smooth;
        }
                /* body{
                    padding-top:65px;
                    background:#edd1cb;
                } */
    </style>

	<style>/* .content */
/* ul, li is used in content section */
ul  {
        list-style: none !important;
    }

ul li   {
            list-style:none !important;
        }

	.content h3 {
            font-size: 3vmin;
            font-weight: 600 !important;
        }

	.content a {
            margin: auto;
            text-decoration: none;
            color: greenyellow;
            position: relative;
            transition: all 0.3s cubic-bezier(0.2, 0, 0, 1);
            z-index: 0;
            white-space: nowrap;
        }
    .content a:after {
            content: '';
            
            height: 2px;
            position: absolute;
            bottom: 0;
            right: 0rem;
            left: 0rem;
            background-color: #D3251B;
            transition: all 0.3s cubic-bezier(0.2, 0, 0, 1);
            transform-origin: bottom center;
            z-index: -1;
        }
    .content a:hover {
            color: #fff;
        }
    .content a:hover:after {
            right: 0;
            left: 0;
            height: 100%;
        }

        #content {
            width: 100%;
            margin: 0px;
            padding: 0px;
        }

        /* #top_title {
            display: none;
        } */


        /* .content * {
        } */

    .content p, .content li {
            font-size: 2.5vmin !important;
        }



        /* #dry-lab {
            width: 50%;
            text-align: center;
            display: block;
            justify-content: center;
            margin: auto;
            flex: 1;
        } */

	</style>



<body>
    
	<!-- NAVBAR SECTION -->

	<nav class="navbar fixed-top navbar-expand-lg navbar-light menu">
		<div class="nav-container">
		  <a class="navbar-brand" href="#">
			<img src="assets/svgs/synbactory2.svg">
		  </a>
		  <div class="center-expand">
		</div>
		  <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		  <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
		    <ul class="navbar-nav">
		      <li class="nav-item">
			<a class="nav-link active" aria-current="page" href="#">Home</a>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Team
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="#">Members</a></li>
			  <li><a class="dropdown-item" href="#">Attributions</a></li>
			  <li><a class="dropdown-item" href="#">Collaborations</a></li>
			  <li><a class="dropdown-item" href="#">Sponsors</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Project
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="#">Design</a></li>
			  <li><a class="dropdown-item" href="#">Description</a></li>
			  <li><a class="dropdown-item" href="#">Notebook</a></li>
			  <li><a class="dropdown-item" href="#">Contributions</a></li>
			  <li><a class="dropdown-item" href="#">Engineering Success</a></li>
			  <li><a class="dropdown-item" href="#">Future Implementation</a></li>
			  <li><a class="dropdown-item" href="#">Safety</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Wet Lab
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="#">Experiments</a></li>
			  <li><a class="dropdown-item" href="#">Constructs</a></li>
			  <li><a class="dropdown-item" href="#">Measurement</a></li>
			  <li><a class="dropdown-item" href="#">Lab Notebook</a></li>
			  <li><a class="dropdown-item" href="#">Results</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Parts
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="#">Overview</a></li>
			  <li><a class="dropdown-item" href="#">Basic</a></li>
			  <li><a class="dropdown-item" href="#">Composite</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Dry Lab
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Model.html">Overview</a></li>
			  <li><a class="dropdown-item" href="EColi.html">E Coli</a></li>
			  <li><a class="dropdown-item" href="Cyanobacteria.html">Cyanobacteria</a></li>
			  <li><a class="dropdown-item" href="Coculture.html">Coculture</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Human Practices
			</a>
			<ul class="dropdown-menu" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="#">iHP</a></li>
			  <li><a class="dropdown-item" href="#">Public Engagement</a></li>
			  <li><a class="dropdown-item" href="#">Education</a></li>
			</ul>
		      </li>
		      <li class="nav-item">
			<a class="nav-link" href="#">Judging Form</a>
		      </li>
		      <!-- <button type="button" class="rounded-pill btn-rounded">Settings</button> -->
		    </ul>
		  </div>
		</div>
	</nav>

    <!-- TITLE SECTION -->
    
    <div class="section-1">
        <h1><br><br><br><br>Dry Lab Overview</h1>
    </div>
        
    <!-- WAVES SPACER -->

    <div class="spacer layer1"></div>

    <!-- CONTENT + SIDENAV SECTION -->

    <div class="second-div container-fluid">
        <div class="row">
            <!-- SIDENAV -->
            <div class="col-lg-3 col-md-3">
                <div class="sidenav">
                    <nav>
                        <ul>
                            <li class="nav-item"><a class="nav-link" href="#1">Objectives</a></li>
                            <li class="nav-item"><a class="nav-link" href="#2">Preprocessing</a></li>
                            <li class="nav-item"><a class="nav-link" href="#3">Computational saturated mutagenesis and
                                    scoring</a></li>
                            <li class="nav-item"><a class="nav-link" href="#4">Molecular dynamics simulations</a></li>
                            <li class="nav-item"><a class="nav-link" href="#5">Grafting of inhibitors into
                                    cyclotides</a></li>
                            <li class="nav-item"><a class="nav-link" href="#6">References</a></li>

                        </ul>









                    </nav>

                    <!-- <hr> -->







                </div>
            </div>
            <!-- CONTENT -->
            <div class="col-lg-9 col-md-9">
                <div class="content">
                    <h1 id="1">Objectives</h1>
                    <hr>
                    <ol>
                        <li>
                            <p>To identify and model structures of protein-protein interactions between <em>Plasmodium
                                    falciparum</em> infected erythrocytes and Human endothelial cell-surface receptors
                                that play a pivotal role in the pathology of malaria.</p>
                        </li>
                        <li>
                            <p>In silico design and development of a library of peptide drugs against <em>Plasmodium falciparum</em> using Cyclotide kalata B1 as the drug scaffold.</p>
                        </li>
                        <li>
                            <p>To generate Cyclotide kalata B1 grafted with the peptide of interest and assess its stability.</p>
                        </li>

                    </ol>
                    <br>
                    <h1>Our Approach:</h1>


                    <figure>

                       <img class="img-fluid"  src="https://2020.igem.org/wiki/images/7/7c/T--IISER-Pune-India--model1.png">



                    </figure>



                    <h1 id="2">1) Preprocessing</h1>
                    <hr>
                    <p>Results of this step is given in the <a href="https://2020.igem.org/Team:IISER-Pune-India/Model_Results#1" target="_blank">Model Results</a> subpage.</p>
                    <br>
                    <p>The first and most crucial step in developing any inhibitory drug is to choose the interaction
                        which
                        the inhibitor will target. We started with a thorough literature review to document important
                        protein-protein interactions between <em>P. falciparum</em>-infected RBCs and Human endothelial
                        cells and their receptors that are responsible for the pathology of the disease.We started our search from the PlasmoDB Database <sup>[1]</sup>. As we needed an exhaustive list of interactions, along with their crystal structures, we performed an advanced search on RCSB <sup>[2]</sup> for all Protein-protein interactions between Plasmodium falciparum and Homo sapiens. This gave us a list of <a href="https://github.com/igemsoftware2020/IISER-Pune-India/blob/master/Peptides_against_Malaria/1_Preprocessing_and_scoring/pdb_ids.txt" target="_blank" rel="noopener">444 PDB IDs</a>. We wrote a program for a <a href="https://github.com/igemsoftware2020/IISER-Pune-India/blob/master/Peptides_against_Malaria/1_Preprocessing_and_scoring/RCSB_static_parser.py"
                            target="_blank" rel="noopener">static-web parser</a> that dynamically parsed the RCSB Database to yield all the required information on all the shortlisted PDB IDs.</p>
                    <p>We worked with this database of 444 interactions and narrowed our interests to choose only those interactions which contained the wildtype Human protein and <em>P. falciparum</em>-infected erythrocytes proteins (We did not consider the interactions with artificial antibodies because we were working only on naturally found protein protein interactins). We started our dry lab work with the following five interactions:</p>
                    <br>
                    <div class="table-responsive">
                        <table class="table table-bordered table-width-75">
                            <thead class="text-center">
                                <tr>
                                    <th scope="col">Interaction ID</th>
                                    <th scope="col">PDB ID</th>
                                    <th scope="col">Description</th>
                                    <th scope="col">Type of Interaction</th>
                                    <th scope="col">Experimental Method</th>
                                </tr>
                            </thead>
                            <tbody>
                                <tr>
                                    <th scope="row" class="text-center">1</th>
                                    <td><a href="https://www.rcsb.org/structure/5MZA" target="_blank"
                                            rel="noopener">5MZA</a></td>
                                    <td>The DBL-beta domain of PF11_0521 PfEMP1 bound to human ICAM-1</td>
                                    <td>Cell Adhesion</td>
                                    <td>X-Ray Diffraction</td>
                                </tr>
                                <tr>
                                    <th scope="row" class="text-center">2</th>
                                    <td><a href="https://www.rcsb.org/structure/4U0Q" target="_blank"
                                            rel="noopener">4U0Q</a></td>
                                    <td><em>Plasmodium falciparum</em> reticulocyte-binding protein homologue 5 (PfRH5) bound to
                                        basigin</td>
                                    <td>Signaling Protein</td>
                                    <td>X-Ray Diffraction</td>
                                </tr>
                                <tr>
                                    <th scope="row" class="text-center">3</th>
                                    <td><a href="https://www.rcsb.org/structure/5LGD" target="_blank"
                                            rel="noopener">5LGD</a></td>
                                    <td>The CIDRa domain from MCvar1 PfEMP1 bound to CD36</td>
                                    <td>Cell Adhesion</td>
                                    <td>X-Ray Diffraction</td>
                                </tr>
                                <tr>
                                    <th scope="row" class="text-center">4</th>
                                    <td><a href="https://www.rcsb.org/structure/4V3D" target="_blank"
                                            rel="noopener">4V3D</a></td>
                                    <td>The CIDRa domain from HB3var03 PfEMP1 bound to endothelial protein C receptor
                                    </td>
                                    <td>Signaling Protein</td>
                                    <td>X-Ray Diffraction</td>
                                </tr>
                                <tr>
                                    <th scope="row" class="text-center">5</th>
                                    <td><a href="https://www.rcsb.org/structure/6S8U" target="_blank"
                                            rel="noopener">6S8U</a></td>
                                    <td>PfEMP1 IT4var13 DBLbeta domain bound to ICAM-1</td>
                                    <td>Cell Adhesion</td>
                                    <td>X-Ray Diffraction</td>
                                </tr>
                            </tbody>
                        </table>
                    </div>
                    <br>
                    <p>The above 5 protein crystal structures obtained from RCSB were incomplete (with missing
                        residues and regions). We attempted to computationally model the missing regions and build
                        models using homology modeling (with SWISS-MODEL<sup>[3]</sup>).
                    </p>

                    <h3>How we designed Peptide Inhibitors</h3>
                    <p>The strategy we used to inhibit the formation of the <strong>Host-Parasite protein complex</strong> was to design a peptide that would competitively bind to the Parasite protein, preventing its binding to the host (Human) protein. Therefore, we identified motifs on the Human protein that remain closest to and bind strongly to the parasite protein. We isolated these motifs along with the parasite protein into different structures and considered each one of them as a potential Inhibitory Model. By
                        this method, we generated multiple models each consisting of the Parasite
                        protein and a peptide sequence (6-20 amino acids in length) from the Human protein.</p>
                    <br>
                    <h1 id="3">2) Computational Saturated Mutagenesis and Scoring</h1>
                    <hr>
<p>Results of this step is given in the <a href="https://2020.igem.org/Team:IISER-Pune-India/Model_Results#2"  target="_blank">Model Results</a> subpage.</p>
<br>
                    <h2>Computational Saturated Mutagenesis</h2>
                    <br>
                    <p>Saturation mutagenesis is a protein-engineering technique in which single/multiple codons is substituted with all possible amino acids at a particular position. This is used widely to improve upon certain properties of proteins such as catalytic activity, enantioselectivity, thermostability, and binding affinity.<sup>[4]</sup></p>

                    <p>Here we performed Saturation Mutagenesis on the peptide sidechain of each model to obtain mutated sequences that show better binding affinity than the original peptide. We wrote <a
                            href="https://github.com/igemsoftware2020/IISER-Pune-India/blob/master/Peptides_against_Malaria/1_Preprocessing_and_scoring/preprocessing_functions.py"
                            target="_blank" rel="noopener">a custom script</a> in Python 2.7 that acts as a wrapper function, using the inbuilt functions of UCSF Chimera<sup>[5]</sup> to mutate a specific residue to a different side-chain conformation (rotamer)  or into a different amino acid for each position of the original sequence. This was repeated for all the Inhibitory Models.</p>

                    <p>We used the <b>Dunbrack 2010 smooth backbone-dependent rotamer library</b> and for chain-terminal
                        residues, the Dunbrack 2002 backbone-independent version was used.<sup>[6]</sup> The
                        backbone-dependent rotamer library consists of rotamer frequencies, mean dihedral angles,
                        variances as a function of the backbone dihedral angles and the rotamer probabilities, estimated
                        using kernel density estimation. Bond lengths and angles were taken from the Amber parameter
                        files <code>all*94.lib</code> and hydrogens were not included. Only the sidechain atoms of a rotamer
                        were evaluated.</p>

                    <p>Rotamers were chosen automatically based on the lowest clash score, the most number of Hydrogen-bonds,
                        best fit to an electron density map, and/or highest probability according to the library. For
                        clashes and Hydrogen-bond detection, interactions with other rotamers in the same set and the current
                        residue at that position were disregarded, but all other atoms in the vicinity were included.
                    </p>

                    <h3>Identifying Clashes and Hydrogen bonds</h3>
                    <br>
                    <p>Clashes, which is one of the selection criteria, are those unfavourable interactions that occur when atoms come too close together i.e. are in close contact with each other. The clash parameters such as ‘overlap’ control what can to be considered as a clash. The Van der Waals (VDW) overlap between two atoms <em>i</em> and
                        <em>j</em> is defined as the sum of their VDW radii minus the distance between them and minus an
                        allowance for potentially hydrogen-bonded pairs:</p>

                    <p class="text-center p-5 math-equation"><em>Overlap</em><sub>ij</sub> = r<sub>VDW-i</sub> + r<sub>VDW-j</sub> -
                        d<sub>ij</sub> - <em>allowance</em><sub>ij</sub></p>

                    <p>A cutoff of 0.6 Å was set as the minimum VDW overlap that would count as a clash. A larger
                        positive cutoff restricts the results to more severe clashes. When the VDW overlap was
                        calculated, 0.4 Å was subtracted for atom pairs consisting of possible hydrogen bond donors (or
                        its hydrogen) and possible acceptor as an allowance. An allowance &gt; 0 reflects the observation
                        that atoms sharing a hydrogen bond can come closer to each other than would be expected from
                        their VDW radii. The clash score is calculated as a simple count of the number of clashes.</p>

                    <p>To identify the Hydrogen-bonds, a set of geometric criteria based on a <b>survey of small-molecule
                            crystal structures</b>, as described in<sup>[7]</sup> was used with a tolerance of <strong>0.4 Å</strong> on
                        the distance and <strong>20.0 degrees</strong> on the angular range criteria for each category of Hydrogen-bond. There
                        is an upper bound on distance and one or more angular range criteria for each category of the
                        Hydrogen-bond. It is generally useful to add some relaxation to this criteria because most structures
                        are not as high-resolution as those used in the survey.</p>

                    <br>
                    <h2>Scoring of the mutants</h2>
                    <br>
                    <p>To compare the binding energies of these mutant peptides to the parasite protein, we scored the models obtained after saturation mutagenesis using FoldX, an empirical force
                        field.<sup>[8]</sup> FoldX is a force field developed for the rapid evaluation of stability, folding, and dynamics of
                        proteins to assess the effect of mutations. The different energy terms taken into account in
                        FoldX have been weighted using empirical data from protein engineering experiments, and the
                        predictive power has been tested on a very large set of protein mutants, covering most of the
                        structural environments found in proteins.<sup>[9]</sup></p>

                    <p>We worked with FoldX because it had been validated on protein-peptide complexes, to estimate the
                        change of the protein–peptide-binding energy with respect to the energy of an experimentally
                        resolved complex and also because the use of FoldX in protein design has been extensively
                        reported.<sup>[10]</sup></p>
                        
                    <br>
                    <h3>Repairing and Standardizing Models before scoring</h3>
                    <br>
                    <ul>
                        <li>
                            <p>The 'RepairPDB' function within FoldX was used to perform a quick optimization in native
                                structures<sup>[10]</sup> before scoring the models. 'RepairPDB' identifies those
                                residues which have bad torsion angles, or Van der Waals' clashes, or total energy, and
                                repairs them. First, it looks for all <strong>Asn</strong> (Asparagine), <strong>Gln </strong>(Glutamine) and <strong>His</strong>
                                (Histidine) residues and flips them by 180 degrees.</p>
                        </li>
                        <li>
                            <p>This is done to prevent incorrect rotamer assignment in the structure due to the fact
                                that the electron density of Asn and Gln carboxamide groups is almost symmetrical and
                                the correct placement can only be discerned by calculating the interactions with the
                                surrounding atoms. The same applies to His.</p>
                        </li>

                        <li>
                            <p>A small optimization of the side chains is done to eliminate small Van der Waals’ clashes.
                                This way moving side chains in the final step is prevented. The residues that have very
                                bad energies are identified and mutated and their neighbours to themselves to explore
                                the different rotamer combinations to find the new energy minima.<sup>[8]</sup> We used
                                the default values for the various parameters in the command.</p>
                        <li>
                    </ul>

                    <h3>Assessing Energy Values</h3>

                    <br>
                    <p>The interaction between two molecules is driven by the free energy of binding,
                        (ΔG<sub>binding</sub>), that is directly related to the thermodynamic dissociation constant
                        (K<sub>d</sub>) by the following equation:</p>

                    <p class="text-center p-5 math-equation">ΔG<sub>binding</sub> = -<em>RT</em> ln(<em>K</em><sub>d</sub>)</p>
                    <p>where R is the gas constant (1.9859 cal mol<sup>-1</sup> K<sup>-1</sup>) and T is the temperature
                        in Kelvin.<sup>[9]</sup></p>

                    <p>In order to calculate the free energy of binding of a complex AB, FoldX ‘<b>AnalyseComplex</b>’
                        command was used. It computes the Gibbs energies of the complex (ΔG<sub>AB</sub>) and of the two
                        molecules A and B alone. It operates by unfolding the selected targets and determining the
                        stability of the remaining molecules and then subtracting the sum of the individual energies
                        from the global energy.<sup>[8]</sup> The free energy of folding (the stability) of the protein is broken down into the energy
                        components used by FoldX such as the contribution of backbone Hydrogen-bonds, the contribution of
                        sidechain-sidechain and sidechain-backbone Hydrogen-bonds, Electrostatic interactions, Penalization for
                        burying polar groups, Contribution of hydrophobic groups, Energy penalization due to
                        Van der Waals’ clashes (inter-residue), Entropy cost of fixing the side chain, etc.</p>

                    <p>The interaction energy is then given by:</p>
                    <p class="text-center p-5 math-equation">ΔG<sub>binding</sub> = ΔG<sub>AB</sub> - (ΔG<sub>A</sub> +
                        ΔG<sub>B</sub>)</p>
                    <p>The output contains the ΔG<sub>binding</sub> for each pair of polypeptide chains in the PDB file
                        plus an additional term that reflects the intrachain Van der Waals clashes of residues forming
                        part of the interface.<sup>[9]</sup></p>

                    <p>After scoring all our mutant models (models obtained after performing saturation mutagenesis on
                        the initial models), we sorted the peptide inhibitors according to their Binding/Interaction
                        Energy. We could identify many peptides that had better scores than the wildtype sequence and hence could
                        bind stronger to the Parasite protein. We also created and scored Hybrid
                        peptides for many mutants by selecting the residues with the
                        least Binding Energy at each position and stitched them together to form a peptide of the same
                        length.</p>


                    <br>
                    <h1 id="4">3) Molecular Dynamics Simulations</h1>
                    <hr>
<p>Results of this step is given in the <a href="https://2020.igem.org/Team:IISER-Pune-India/Model_Results#3" target="_blank">Model Results</a> subpage.</p>
                    <br>
                    <h3>Why Molecular Dynamic Simulations are necessary</h3>
                    <br>
                    <p>Simulations can provide the ultimate detail concerning individual particle motions as a function
                        of time and can be used to address specific questions about the properties of a model system.
                        Molecular dynamics can be defined as a computer simulation technique that permits the prediction
                        of the time evolution of a particular interacting system involving the generation of atomic
                        trajectories of a system using numerical integration of Newton’s laws of motion for a specific
                        interatomic potential defined along with appropriate initial and boundary
                        conditions.<sup>[11]</sup> Molecular dynamics simulations are ubiquitous for understanding the
                        physical basis of the structure and function of biological macromolecules.</p>

                    <p>We performed MD Simulations to understand how well the designed inhibitory peptide interacts with
                        the Parasite protein. If the interaction is strong, then it may act as an efficient peptide
                        inhibitor and can slow down the pathogenesis of <em>Plasmodium falciparum</em> malaria. We also plan to
                        graft the inhibitor into a synthetic kalataB1 cyclotide and check its stability under
                        physiological conditions.</p>
                    <br>
                    <h3>GROMACS for performing MD Simulations</h3>
                    <br>
                    <p>For performing the MD Simulations, we have used the GROMACS 2019.1 package.<sup>[12]</sup> We
                        chose the AMBER99SB-ILDN force field as it had a better accord with experimental NMR relaxation
                        data of test protein systems<sup>[13]</sup> and also was used for the MD Simulation of a similar
                        study of peptide-peptide interaction.<sup>[14]</sup> This force field, based on the ff94 force
                        field is commonly associated with the AMBER simulation package is one of the most widely
                        used parameter sets for biomolecular simulation.<sup>[13]</sup> All MD Simulations were
                        performed on the PARAM BRAHMA facility under the National Supercomputing Mission, Government of
                        India at IISER Pune.</p>
                    <h3>Methods</h3>

                    <p>MD runs were performed in triplicates for three best scoring inhibitor models along with their
                        respective Parasite protein. We generated a topology file using the AMBER99SB-ILDN force fields
                        and solvated it using the SPC/E water model.<sup>[15]</sup> A water box of the minimum
                        solute-box distance of 1.0 nm was used to solvate each system. Sodium and Chloride ions were
                        added to achieve charge neutrality by replacing the solvent molecules. The Particle mesh Ewald (PME)
                        sum method<sup>[16]</sup> was used to treat the electrostatic interactions and LINCS
                        algorithm<sup>[17]</sup> to constrain the hydrogen bond length.</p>
                    <p>A time step of 2 fs was used for the integration. The whole system was minimized for 50000 steps
                        or till the maximum force was less than 1000.0 kJ/mol/nm. The system was then brought to 300 K
                        in an NVT ensemble simulation of 100 ps using Velocity Rescaling thermostat.<sup>[18]</sup> The
                        pressure was stabilized in an NPT ensemble for 100 ps using Parrinello-Rahman
                        barostat.<sup>[19]</sup> The entire system was simulated for 100 ns, where temperature and
                        pressure are regulated by Velocity rescaling thermostat<sup>[18]</sup> and Parrinello-Rahman
                        barostat,<sup>[19]</sup> respectively. The snapshots of the structure were taken at every 500 ps
                        starting from zero. The temperature and potential energy of the system were monitored for
                        anomalies.</p>
                    <p>We visualized the trajectory file to see how the inhibitor interacts with the Parasite protein.
                        RMSD plots were made using <code>trjconv</code> tool in GROMACS to verify the stability of the proteins.
                        Furthermore, <b>the distance between centroids of inhibitor and Parasite protein over time and
                            intermolecular hydrogen bond frequency</b> over the triplicate MD runs were plotted and
                        analyzed using custom scripts.</p>

                    <br>

                    <h3>Analysis of Molecular Dynamics Data</h3>

                    <p>An MD simulation is an in-silico experiment with large volumes of data. We wish to interpret this data to learn more about the interaction between our Inhibitor and the <em>P. falciparum</em> protein. To analyse the MD simulations of a protein and its peptide inhibitor, we need to understand two particular aspects in greater detail:</p>

                    <ol>
                        <li>
                            <strong>How does the distance between the centroids of the Protein and peptide evolve over time?</strong>
                            <ol>
                                <li>If it increasing, it means that over-time the peptide flies away from the protein and is not binding effectively.</li>
                                <li>If the distance between the centroid of the two entities remains constant, then over the duration of the simulation, the peptide binds strongly to the protein and can act as a good inhibitor.</li>
                                <li>If the distance decreases, then the peptide moves closer to the Protein core. (However, this is assumed to be unlikely)</li>
                            </ol>

                            <p>To perform these calculations, we wrote the script <a href="https://github.com/igemsoftware2020/IISER-Pune-India/blob/master/Peptides_against_Malaria/2_MD_simulations_and_analysis/PDB_centroid_analyser.py" target="_blank" rel="noopener">PDB_centroid_analyser</a></p> 
                        </li>
                        <li>
                            <strong>Hydrogen Bond Profile Analysis</strong>
                            <p>To understand effective binding, we analyse the number of intermolecular Hydrogen bonds between the Protein and the peptide. This entails calculating the hydrogen bond profile of each snapshot of the MD simulation and to determine which residues and specifically which atom in these residues of the peptide inhibitor form hydrogen bonds with the protein. We also aim to determine the relative abundance of Hydrogen bonds formed over the entire simulation period. To perform these calculations we wrote <a href="https://github.com/igemsoftware2020/IISER-Pune-India/blob/master/Peptides_against_Malaria/2_MD_simulations_and_analysis/PDB_hbond_analyser.py" target="_blank" rel="noopener">PDB_Hbond_analyser.py</a></p>
                        </li>
                    </ol>

                    <br>
                    <h1 id="5">4) Grafting of inhibitors into cyclotides</h1>
                    <hr>
<p>Results of this step is given in the <a href="https://2020.igem.org/Team:IISER-Pune-India/Model_Results#4" target="_blank">Model Results</a> subpage.</p>
<br>
                    <p>The ultimate aim of our project is to graft our designed inhibitors into Cyclotide kalata-B1 and
                        hence synthesise a stable peptide inhibitor for various key interactions involved in the
                        pathogenesis of <em>P. falciparum</em> malaria. So we decided to computationally assess the stability of our
                        grafted cyclotide drug. There were many challenges to this, beginning with the homology modelling of cyclic
                        peptides. Most homology modelling packages give us a linear
                        peptide and there was difficulty in finding a template, so we approached <b>Dr. David Norman,
                            College of Life Sciences, University of Dundee</b>, for help. He provided us with a
                        tutorial<sup>[20]</sup> on the MODELLER<sup>[21]</sup> package and offered help in case we faced
                        any difficulties.</p>

                    <p>The first step in modelling a sequence is aligning our sequence to a template. Since structures
                        identical to our grafted cyclotides could not be found, we used the structure of kalata B1 given
                        in the PDB database with PDB ID 4TTN to get a linear homology model using
                        SWISS-MODEL.<sup>[3]</sup> We used a <a
                            href="https://github.com/igemsoftware2020/IISER-Pune-India/tree/master/Peptides_against_Malaria/2_MD_simulations_and_analysis" target="_blank" rel="noopener">custom script</a> built on top
                        of MODELLER<sup>[21]</sup> to align our target sequence to the model obtained from SWISS-MODEL.
                        The alignment was done using the BLOSUM62 scoring scheme. The alignment file along with the PDB
                        file of the model from SWISS-MODEL was used to generate an N to C backbone circularised.
                        MODELLER gives non circularised peptides as output by default, so we used a <a
                            href="https://github.com/igemsoftware2020/IISER-Pune-India/tree/master/Peptides_against_Malaria/2_MD_simulations_and_analysis" target="_blank" rel="noopener">custom script</a> using a
                        special patch to achieve an end to end circularisation of the backbone.</p>

                    <p>There are a lot of shortcomings for this method. One major problem is that as the hybrid
                        cyclotide was modelled based only on a native kalata B1 backbone, the 3-dimensional structure of
                        the grafted region may not have modelled properly. This may affect the binding affinity of the
                        inhibitor for the Parasite protein. We performed MD simulations on these hybrid cyclotides using
                        the same parameters we used for our peptide binding studies. As the study is still rudimentary, we just observed the RMSD plots with respect to the backbone to get an idea of its stability</p>

                    <br>


			</div>

		</div>

        </div>

    </div>

    <div class="header"></div>
    <div class="line" id="scrollIndicator"></div>

    <script src="js/bootstrap-bundle.js"></script>
	<script src="js/main.js"></script>
	<script src="js/jquery.min.js"></script>

	<script>
		var screenheight = window.innerHeight,
		    screenwidth = window.innerWidth;

	</script>
	<script>
		var body = document.body,
    		html = document.documentElement;

		var pageheight = Math.max( body.scrollHeight, body.offsetHeight, 
                       		   html.clientHeight, html.scrollHeight, html.offsetHeight );
	</script>
	<script>
		var backtotopheight=pageheight/4.65
	</script>
	<script>
		var total = $('.item').length;
		var actualtotal = 80;
		var currentIndex = $('div.active').index() + 1;
		$('#slidetext').html(currentIndex + '/'  + actualtotal);

		// This triggers after each slide change
		$('.carousel').on('slid.bs.carousel', function () {
  			currentIndex = $('div.active').index() + 1;
 
  		// Now display this wherever you want
  		var text = currentIndex + '/' + actualtotal;
  			$('#slidetext').html(text);
		});

	</script>
</body>