

<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Engineering</title>
	<link rel="stylesheet" href="css/igem-reset.css">
	<link rel="stylesheet" href="css/SBcss/style.css">
   	<link rel="stylesheet" href="css/styleSB.css">
    	<link rel="stylesheet" href="css/DryLabPage.css">
	<style>/* Reset CSS */

		#sideMenu, #top_title, .patrollink, #firstHeading, #home_logo, #sideMenu { display:none; }
		#content { padding:0px; width:100%; margin-top:-7px; margin-left:0px; border:none;}
		body, html {background-color:#212529; width: 100%; height: 100%;}
		#bodyContent h1, #bodyContent h2, #bodyContent h3, #bodyContent h4, #bodyContent h5 { margin-bottom: 0px; }
		#bodyContent a[href ^="https://"], .link-https { padding-right:0px;}

    </style>
  
<body>
    
	<!-- NAVBAR SECTION -->

	<nav class="navbar fixed-top navbar-expand-lg navbar-light menu">
		<div class="container-fluid nav-container">
		  <a class="navbar-brand" href="index.html">
			<img src="assets/svgs/synbactory2.svg">
		  </a>
		  <div class="center-expand">
		</div>
		  <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
		    <span class="navbar-toggler-icon"></span>
		  </button>
		  <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
		    <ul class="navbar-nav">
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Team
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Team.html">Members</a></li>
			  <li><a class="dropdown-item" href="Attributions.html">Attributions</a></li>
			  <li><a class="dropdown-item" href="Collaborations.html">Collaborations</a></li>
			  <li><a class="dropdown-item" href="Partnership.html">Partnership</a></li>
			  <li><a class="dropdown-item" href="Sponsors.html">Sponsors</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Project
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Description.html">Description</a></li>
			  <li><a class="dropdown-item" href="Design.html">Design</a></li>
			  <li><a class="dropdown-item" href="https://possible-trapezoid-297.notion.site/11e9906b405543a29361e38a50c81024?v=af10ed5ac15145e8a267a68ed8eb2005">Notebook</a></li>
			  <li><a class="dropdown-item" href="Contribution.html">Contributions</a></li>
			  <li><a class="dropdown-item" href="Engineering.html">Engineering Success</a></li>
			  <li><a class="dropdown-item" href="Implementation.html">Future Implementation</a></li>
			  <li><a class="dropdown-item" href="Safety.html">Safety</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Wet Lab
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Experiments.html">Experiments</a></li>
			  <li><a class="dropdown-item" href="Constructs.html">Constructs</a></li>
			  <li><a class="dropdown-item" href="Measurement.html">Measurement</a></li>
			  <li><a class="dropdown-item" href="LabNotebook.html">Lab Notebook</a></li>
			  <li><a class="dropdown-item" href="Results.html">Results</a></li>
			  <li><a class="dropdown-item" href="FutureWork.html">Future Work</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Parts
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <!-- <li><a class="dropdown-item" href="#">Overview</a></li> -->
			  <li><a class="dropdown-item" href="BasicParts.html">Basic</a></li>
			  <li><a class="dropdown-item" href="CompositeParts.html">Composite</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Dry Lab
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Model.html">Overview</a></li>
			  <li><a class="dropdown-item" href="EColi.html">E Coli</a></li>
			  <li><a class="dropdown-item" href="Cyanobacteria.html">Cyanobacteria</a></li>
			  <li><a class="dropdown-item" href="Coculture.html">Coculture</a></li>
			</ul>
		      </li>
		      <li class="nav-item dropdown">
			<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
			  Human Practices
			</a>
			<ul class="dropdown-menu fade-down" aria-labelledby="navbarDropdown">
			  <li><a class="dropdown-item" href="Human_Practices.html">Integrated Human Practices</a></li>
			  <li><a class="dropdown-item" href="Inclusivity.html">Inclusivity and Accessibility</a></li>
			  <li><a class="dropdown-item" href="Communication.html">Public Engagement and Education</a></li>
			</ul>
		      </li>
		      <li class="nav-item">
			<a class="nav-link" href="https://igem.org/2021_Judging_Form?id=3971">Judging Form</a>
		      </li>
		      <!-- <button type="button" class="rounded-pill btn-rounded">Settings</button> -->
		    </ul>
		  </div>
		</div>
	</nav>

    <!-- TITLE SECTION -->
    
    <div class="section-1">
        <h1><br><br><br><br>Engineering</h1>
    </div>
        
    <!-- WAVES SPACER -->

    <div class="spacer layer1"></div>

    <!-- CONTENT + SIDENAV SECTION -->

<style>
	.content h1, .content h2, .content h3, .content h4, .content h5, .content h6{
	color: white !important;
	font-family: ProximaNova !important;
}
</style>
    <div class="second-div container-fluid">
        <div class="row">
            <!-- SIDENAV -->
            <div class="col-lg-3 col-md-3">
                <div class="sidenav">
                    <nav>
                        <ul id="sidenav-id">
                            <li class="nav-item"><a class="nav-link" href="#1">Expt. 1 - “Choosing Optimal promoters”</a></li>
                            <li class="nav-item"><a class="nav-link" href="#2">Expt 2 - “gDNA extraction and PCR”</a></li>
                            <li class="nav-item"><a class="nav-link" href="#3">Expt 3 - “E. coli in CoBG11”</a></li>
                            <li class="nav-item"><a class="nav-link" href="#4">Expt 4 - “Sucrose Assay”</a></li>
                            <li class="nav-item"><a class="nav-link" href="#5">Expt 5 - “S. elongatus growth curves”</a></li>
                            <li class="nav-item"><a class="nav-link" href="#6">Co-culture modelling cycle</a></li>
                            <li class="nav-item"><a class="nav-link" href="#7">Building the SteadyCom model</a></li>
                            <li class="nav-item"><a class="nav-link" href="#8">Simulating Salt Stress</a></li>
                            <li class="nav-item"><a class="nav-link" href="#9">References</a></li>
                        </ul>
                    </nav>
                </div>
            </div>
            <!-- CONTENT -->

	  <style>
	    a:hover,a:focus{
    text-decoration: none;
    outline: none;
}
.tab .nav-tabs{
    background-color: transparent;
    border: none;
}
.tab .nav-tabs li a{
    color: #fff;
    background: transparent;
    font-size: 19px;
    font-weight: 800;
    letter-spacing: 1px;
    text-align: center;
    text-transform: uppercase;
    padding: 13px 15px 7px;
    margin: 0 0 1px 0;
    border: none;
    border-radius: 0;
    overflow: hidden;
    position: relative;
    z-index: 1;
    transition: all 0.3s ease 0s;
}
.tab .nav-tabs li:last-child a{ margin-right: 0; }
.tab .nav-tabs li a:hover,
.tab .nav-tabs li.active a{
    background: transparent;
    border: none;
}
.tab .nav-tabs li.active a{ color: #fff; }
.tab .nav-tabs li a:before{
    content: "";
    background: #444;
    height: 100%;
    width: 100%;
    transform: perspective(400px) rotateX(75deg);
    transform-origin: bottom center;
    position: absolute;
    top: 0;
    left: 0;
    z-index: -1;
    transition: all 0.5s ease 0s;
}
/* .tab .nav-tabs li.active a:before,
.tab .nav-tabs li a:hover:before{

} */
.tab .tab-content{
    color: #fff;
    background: #333;
    font-size: 16px;
    letter-spacing: 1px;
    line-height: 23px;
    padding: 20px;
    border-radius: 0 0 20px 20px;
    box-shadow: 0 0 20px -5px rgba(0,0,0,0.5);
}
.tab .tab-content h3{
    color: #fff;
    font-size: 24px;
    font-weight: 700;
    text-transform: uppercase;
    letter-spacing: 1px;
    margin: 0 0 7px 0;
}
@media only screen and (max-width: 479px){
    .tab .nav-tabs{
        background-color: #333;
        border-radius: 21px 21px 0 0;
    }
    .tab .nav-tabs li{ width: 100%; }
    .tab .nav-tabs li a{ color: #fff; }
    .tab .tab-content h3{ font-size: 18px; }
}
table{
	border-collapse:collapse;
}
td {
            border-style:solid; 
            border-width:1px; 
            border-color:#fff; 
            padding:10px;
	    text-align: center; 
	    vertical-align: middle;
         }
td[data-cycle="design"]{
	background-color: pink;
	color: black;
}
td[data-cycle="build"]{
	background-color: yellowgreen;
}
td[data-cycle="test"]{
	background-color: blueviolet;
}
td[data-cycle="learn"]{
	background-color: mediumpurple;
}
    </style>

<!--
<div class="row">
	<div class="tab" role="tabpanel">
	    
	    <ul class="nav nav-tabs" role="tablist">
		<li role="presentation" class="active"><a href="#Cycle1_A" aria-controls="Cycle1_A" role="tab" data-toggle="tab">Cycle 1</a></li>
		<li role="presentation"><a href="#Cycle2_A" aria-controls="Cycle2_A" role="tab" data-toggle="tab">Cycle 2</a></li>
		<li role="presentation"><a href="#Cycle3_A" aria-controls="Cycle3_A" role="tab" data-toggle="tab">Cycle 3</a></li>
		<li role="presentation"><a href="#Cycle4_A" aria-controls="Cycle4_A" role="tab" data-toggle="tab">Cycle 4</a></li>
		<li role="presentation"><a href="#Cycle5_A" aria-controls="Cycle5_A" role="tab" data-toggle="tab">Cycle 5</a></li>
	    </ul>
	    
	    <div class="tab-content tabs">
		<div role="tabpanel" class="tab-pane fade in show active" id="Cycle1_A">
		    <h3>Cycle 1</h3>
			<table>
				<tr><td data-cycle="design">Design</td><td></td></tr>
				<tr><td data-cycle="build">Build</td><td></td></tr>
				<tr><td data-cycle="test">Test</td><td></td></tr>
				<tr><td data-cycle="learn">Learn</td><td></td></tr>
			</table>
		</div>
		<div role="tabpanel" class="tab-pane fade" id="Cycle2_A">
		    <h3>Cycle 2</h3>
		    <table>
			<tr><td data-cycle="design">Design</td><td></td></tr>
			<tr><td data-cycle="build">Build</td><td></td></tr>
			<tr><td data-cycle="test">Test</td><td></td></tr>
			<tr><td data-cycle="learn">Learn</td><td></td></tr>
		</table>
		</div>
		<div role="tabpanel" class="tab-pane fade" id="Cycle3_A">
		    <h3>Cycle 3</h3>
		    <table>
			<tr><td data-cycle="design">Design</td><td></td></tr>
			<tr><td data-cycle="build">Build</td><td></td></tr>
			<tr><td data-cycle="test">Test</td><td></td></tr>
			<tr><td data-cycle="learn">Learn</td><td></td></tr>
		</table>
		</div>
		<div role="tabpanel" class="tab-pane fade" id="Cycle4_A">
		    <h3>Cycle 4</h3>
		    <table>
			<tr><td data-cycle="design">Design</td><td></td></tr>
			<tr><td data-cycle="build">Build</td><td></td></tr>
			<tr><td data-cycle="test">Test</td><td></td></tr>
			<tr><td data-cycle="learn">Learn</td><td></td></tr>
		</table>
		</div>
		<div role="tabpanel" class="tab-pane fade" id="Cycle5_A">
			<h3>Cycle 5</h3>
			<table>
				<tr><td data-cycle="design">Design</td><td></td></tr>
				<tr><td data-cycle="build">Build</td><td></td></tr>
				<tr><td data-cycle="test">Test</td><td></td></tr>
				<tr><td data-cycle="learn">Learn</td><td></td></tr>
			</table>
		</div>
	    </div>
	</div>

-->


            <div class="col-lg-9 col-md-9">
                <div class="content"><br><br>
			<p>Due to the second wave of the pandemic in India, our wet lab work was considerably delayed, effectively leaving us with a mere month and a half in the lab. We utilised our time working remotely to devise a thorough plan of experiments, incorporating a Design-Build-Test cycle:</p>
                    <h1 id="1">Experiment 1 - “Choosing Optimal promoters”</h1>
                    <hr><br>
		    
			<div class="row"><!-- CYCLE A -->
				<div class="tab" role="tabpanel">
				    <!-- Nav tabs -->
				    <ul class="nav nav-tabs" role="tablist">
					<li role="presentation" class="active"><a href="#Cycle1_A" aria-controls="Cycle1_A" role="tab" data-toggle="tab">Cycle 1</a></li>
					<li role="presentation"><a href="#Cycle2_A" aria-controls="Cycle2_A" role="tab" data-toggle="tab">Cycle 2</a></li>
				    </ul>
				    <!-- Tab panes -->
				    <div class="tab-content tabs">
					<div role="tabpanel" class="tab-pane fade in show active" id="Cycle1_A">
					    <h3>Cycle 1</h3>
						<table>
							<tr><td data-cycle="design">Design</td><td>Design constructs via overlap PCR of various <em>native</em> promoters - <em>Pcpc560, PcscB m6, PpsbA2, PpsbA3</em> - upstream of fluorescent reporters. Compare their promoter strengths to that of strong constitutive promoter J23119 and the IPTG-inducible <em>lacUV5</em>.</td></tr>
							<tr><td data-cycle="build">Build</td><td>Incorporate constructs via homology directed repair (HDR) into Neutral Site I of the genome using a CRISPR/Cas12a system</td></tr>
							<tr><td data-cycle="test">Test</td><td>Vary light intensity and concentration of carbon dioxide and study fluorescence to characterize promoter activity (strength and response)</td></tr>
							<tr><td data-cycle="learn">Learn</td><td>Select promoters with high activity and easily tunable response.</td></tr>
						</table>
					</div>
					<div role="tabpanel" class="tab-pane fade" id="Cycle2_A">
					    <h3>Cycle 2</h3>
					    <table>
						<tr><td data-cycle="design">Design</td><td>Design constructs with strong/tunable promoters from previous round, with <em>cscB</em>, <em>sps</em> and <em>spp</em></td></tr>
						<tr><td data-cycle="build">Build</td><td>Incorporate <em>cscB</em> is incorporated into Neutral Site I, and&nbsp; <em>sps</em> and <em>spp</em> into Neutral Site III, <em>via </em>HDR using a CRISPR/Cas12a system</td></tr>
						<tr><td data-cycle="test">Test</td><td>Collect supernatant samples over time to estimate sucrose production; use Dr Pakrasi lab’s strains as a positive control.</td></tr>
					</table>
					</div>
				    </div>
				</div>
				</div>
<br><br>
<p>We were unfortunately unable to carry out most of these experiments in the little time remaining to us. We instead acquired sucrose-secreting strains of S. elongatus UTEX 2973 from the Pakrasi lab, to measure their sucrose production rates. We have also acquired everything we need for future experiments according to our original plan.</p>
<br><br>
			<h1 id="2">Experiment 2 - “gDNA extraction and PCR”</h1>
			<hr><br>
			<div class="row"><!-- CYCLE B -->
				<div class="tab" role="tabpanel">
				    
				    <ul class="nav nav-tabs" role="tablist">
					<li role="presentation" class="active"><a href="#Cycle1_B" aria-controls="Cycle1_B" role="tab" data-toggle="tab">Cycle 1</a></li>
					<li role="presentation"><a href="#Cycle2_B" aria-controls="Cycle2_B" role="tab" data-toggle="tab">Cycle 2</a></li>
					<li role="presentation"><a href="#Cycle3_B" aria-controls="Cycle3_B" role="tab" data-toggle="tab">Cycle 3</a></li>
					<li role="presentation"><a href="#Cycle4_B" aria-controls="Cycle4_B" role="tab" data-toggle="tab">Cycle 4</a></li>
					<li role="presentation"><a href="#Cycle5_B" aria-controls="Cycle5_B" role="tab" data-toggle="tab">Cycle 5</a></li>
				    </ul>
				    
				    <div class="tab-content tabs">
					<div role="tabpanel" class="tab-pane fade in show active" id="Cycle1_B">
					    <h3>Cycle 1</h3>
						<table>
							<tr><td data-cycle="design">Design</td><td>To extract genomic DNA from<em> E. coli</em> KJK01 and verify the presence of <em>hbd-crt</em> genes by PCR</td></tr>
							<tr><td data-cycle="build">Build</td><td>Extracted genomic DNA through phenol-chloroform method, conducted PCR to amplify <em>hbd-crt</em>, ran PCR product on an agarose gel</td></tr>
							<tr><td data-cycle="test">Test</td><td>Imaged the gel and observed no bands</td></tr>
							<tr><td data-cycle="learn">Learn</td><td>Possible inhibition of PCR due to excess concentration of dNTPs and gDNA in PCR mix - reduce concentration in subsequent attempt </td></tr>
						</table>
					</div>
					<div role="tabpanel" class="tab-pane fade" id="Cycle2_B">
					    <h3>Cycle 2</h3>
					    <table>
						<tr><td data-cycle="design">Design</td><td>PCR with reduced concentrations of dNTPs and gDNA and quality check for gDNA</td></tr>
						<tr><td data-cycle="build">Build</td><td>The above was executed<br>PCR product and gDNA were run on an agarose gel</td></tr>
						<tr><td data-cycle="test">Test</td><td>Imaged the gel - multiple bands in the well corresponding to gDNA, no bands corresponding to the PCR product appeared </td></tr>
						<tr><td data-cycle="learn">Learn</td><td>Suspected nuclease contamination in gDNA sample</td></tr>
					</table>
					</div>
					<div role="tabpanel" class="tab-pane fade" id="Cycle3_B">
					    <h3>Cycle 3</h3>
					    <table>
						<tr><td data-cycle="design">Design</td><td>Extract gDNA once again with modified protocol</td></tr>
						<tr><td data-cycle="build">Build</td><td>gDNA extracted once again <br><br>

							Proteinase K added first, followed by RNase A after the deactivation of the former<br><br>
							
							Final gDNA resuspension done in autoclaved ddH2O (MilliQ water) instead of Elution Buffer<br><br>
							
							gDNA run on agarose gel
							</td></tr>
						<tr><td data-cycle="test">Test</td><td>Single large band appeared above the highest band of the DNA ladder<br><br>

							Large bright bands present at the bottom of the ladder
							</td></tr>
						<tr><td data-cycle="learn">Learn</td><td>Presence of good quality gDNA verified<br><br>

							Large bright band due to RNA contamination 
							</td></tr>
					</table>
					</div>
					<div role="tabpanel" class="tab-pane fade" id="Cycle4_B">
					    <h3>Cycle 4</h3>
					    <table>
						<tr><td data-cycle="design">Design</td><td>PCR verification of <em>hbd-crt</em> from new gDNA extract </td></tr>
						<tr><td data-cycle="build">Build</td><td>The above was executed along with a negative control that was devoid of the gDNA template <br><br>

							PCR products were run on an agarose gel
							</td></tr>
						<tr><td data-cycle="test">Test</td><td>No bands corresponding to the genes of interest appeared</td></tr>
						<tr><td data-cycle="learn">Learn</td><td>RNA contamination may have led to overestimation of gDNA concentration when measured by NanoDrop. <br><br>
							PCR should be redone with undiluted template gDNA <br><br>

							Add DMSO to PCR Mix to inhibit renaturation of denatured DNA
						</td></tr>
					</table>
					</div>
					<div role="tabpanel" class="tab-pane fade" id="Cycle5_B">
						<h3>Cycle 5</h3>
						<table>
							<tr><td data-cycle="design">Design</td><td>Gradient PCR with varying DMSO concentrations at various annealing temperatures </td></tr>
							<tr><td data-cycle="build">Build</td><td>The above was executed along with a negative control devoid of the gDNA template</td></tr>
							<tr><td data-cycle="test">Test</td><td>Band corresponding to genes of interest at 750 bp appeared</td></tr>
							<tr><td data-cycle="learn">Learn</td><td><em>hbd-crt</em> was amplified during PCR, its presence on the genome of <em>E. coli</em> KJK01 is verified</td></tr>
						</table>
					</div>
				    </div>
				</div>
				</div>
				<br><br>
				<h1 id="3">Experiment 3 - “E. coli in CoBG11”</h1>
			<hr><br>
				<div class="row"><!-- CYCLE C -->
					<div class="tab" role="tabpanel">
					    
					    <ul class="nav nav-tabs" role="tablist">
						<li role="presentation" class="active"><a href="#Cycle1_C" aria-controls="Cycle1_C" role="tab" data-toggle="tab">Cycle 1</a></li>
						<li role="presentation"><a href="#Cycle2_C" aria-controls="Cycle2_C" role="tab" data-toggle="tab">Cycle 2</a></li>
						<li role="presentation"><a href="#Cycle3_C" aria-controls="Cycle3_C" role="tab" data-toggle="tab">Cycle 3</a></li>
					</ul>
					    
					    <div class="tab-content tabs">
						<div role="tabpanel" class="tab-pane fade in show active" id="Cycle1_C">
						    <h3>Cycle 1</h3>
							<table>
								<tr><td data-cycle="design">Design</td><td>Inoculate KJK01 and pCSCX-KJK01 cultures in BG-11 + sucrose to check for growth, sucrose consumption and butanol production. Sucrose concentration used is 250 mg/L, higher than maximum sucrose concentration secreted by cyanobacteria in Zhang et al, 2020.</td></tr>
								<tr><td data-cycle="build">Build</td><td>Inoculate 3 cultures<br>
									
									&bull;&nbsp;&nbsp;KJK01 (without antibiotics)<br>
									&bull;&nbsp;&nbsp;pCSCX-KJK01 (uninduced + antibiotics)<br>
									&bull;&nbsp;&nbsp;pCSCX-KJK01 (induced + antibiotics)
									
									<br>Take hourly OD readings, store supernatants at -20&deg;C and submit for sucrose assay and NMR analysis.</td></tr>
								<tr><td data-cycle="test">Test</td><td>KJK01 showed growth, while both pCSCX-KJK01 strains did not. Maximum sucrose was consumed by KJK01.</td></tr>
								<tr><td data-cycle="learn">Learn</td><td>Suspected contamination in the KJK01 culture. Repeat experiment carefully to minimize all possibilities of contamination.</td></tr>
							</table>
						</div>
						<div role="tabpanel" class="tab-pane fade" id="Cycle2_C">
						    <h3>Cycle 2</h3>
						    <table>
							<tr><td data-cycle="design">Design</td><td>UV laminar hood for 20 minutes, and completely sanitize the insides with 70% ethanol before inoculation and each measurement. Repeat experiment in the same manner as above, taking care to minimize risk of contamination. </td></tr>
							<tr><td data-cycle="build">Build</td><td>Inoculate 3 cultures<br>
								
								&bull;&nbsp;&nbsp;KJK01 (without antibiotics)<br>
								&bull;&nbsp;&nbsp;pCSCX-KJK01 (uninduced + antibiotics)<br>
								&bull;&nbsp;&nbsp;pCSCX-KJK01 (induced + antibiotics)
								
								<br>Take hourly OD readings, store supernatants at -20&deg;C and submit for sucrose assay and NMR analysis.</td></tr>
							<tr><td data-cycle="test">Test</td><td>KJK01 showed growth, while both pCSCX-KJK01 strains did not. </td></tr>
							<tr><td data-cycle="learn">Learn</td><td>Suspect contamination in the KJK01 culture again, possibly due to lack of antibiotics. Were advised by mentors to use a different negative control. Decided to replace BG-11 with CoBG-11, as done by Zhang et al.</td></tr>
						</table>
						</div>
						<div role="tabpanel" class="tab-pane fade" id="Cycle3_C">
						    <h3>Cycle 3</h3>
						    <table>
							<tr><td data-cycle="design">Design</td><td>Use KJK01 transformed with a blank Kan resistance plasmid (pKD4) as a negative control to show that sucrose consumption is solely due to pCSCX plasmid, and to minimize risk of contamination. Use CoBG-11, a medium containing BG-11 and salts that help E. coli grow better. Repeat experiment with same composition of pCSCX cultures.</td></tr>
							<tr><td data-cycle="build">Build</td><td>Transformation failed, but due to lack of time could not repeat it and chose to inoculate all cultures without antibiotics in order to have a true negative control. <br><br>Inoculate 3 cultures in CoBG-11<br>
								
								&bull;&nbsp;&nbsp;KJK01 (without antibiotics)<br>
								&bull;&nbsp;&nbsp;pCSCX-KJK01 (uninduced + antibiotics)<br>
								&bull;&nbsp;&nbsp;pCSCX-KJK01 (induced + without antibiotics)
								
								<br>Take hourly OD readings, store supernatants at -20&deg;C and submit for sucrose assay and NMR analysis.</td></tr>
							<tr><td data-cycle="test">Test</td><td>KJK01 and pCSCX-KJK01 (induced) showed growth, while pCSCX-KJK01 (uninduced) did not. 

								<br><br>Cannot confirm if growth is of our strains or the contaminants due to lack of antibiotics in the medium. Samples have been submitted for sucrose assay and NMR analysis, however results have not arrived yet.
								</td></tr>
							<tr><td data-cycle="learn">Learn</td><td>Will perform site directed mutagenesis on pCSCX plasmid to render cscABK genes non-functional, and transform KJK01 with it. This will serve as a true negative control for the next assay.

								<br><br>CoBG-11 will be used next time as well.
								
								<br><br>Cultures might not have grown due to low concentrations of sucrose having been added. Will repeat with higher concentrations of sucrose, look at <a href="Results.html">Results</a>.
								</td></tr>
						</table>
						</div>
					    </div>
					</div>
					</div>
					<br><br>
					<h1 id="4">Experiment 4 - “Sucrose Assay”</h1>
					<hr><br><br>
					<p>
						For sucrose quantification, we used the Megazyme Sucrose/D-Glucose K-SUCGL kit. The kit uses the enzymes glucose oxidase, peroxidase and β-fructosidase (invertase). 
						<br><br>Sucrose is hydrolyzed by invertase to D-glucose and D-fructose. 
						<br><br>The D-glucose content is determined by conversion to a red-colored quinoneimine dye (by the action of glucose oxidase, peroxidase) and the D-glucose concentration is quantified via absorbance measurements at 510 nm.  To isolate the contribution of the glucose resulting from the hydrolysis of sucrose, we need to subtract the absorbance of an identical sample, untreated by invertase. This absorbance is a measurement of pre-existing free D-glucose in our supernatant. We denote this invertase-free solution as solution A. The solution with the invertase added is correspondingly denoted solution B. 
						<br><br>For every sample, two absorbance measurements must be taken - one of the sample without beta-fructosidase, denoted as solution A, to measure the initial amount of free D-glucose present in the sample, and one of the sample with beta-fructosidase, denoted as solution B, to measure the final amount D-glucose in the sample (this includes any initial amounts of free D-glucose, and any sucrose that got converted to D-glucose by the action of beta-fructosidase). The difference in the absorbance between these two solutions gives the amount of sucrose in the sample.	
						<br><br>The kit requires the sample to contain a sucrose concentration between 0.02 g/L and 0.5 g/L, for which it prescribes dilution instructions based on the estimated amount of sucrose and glucose in the sample.
						<br><br>However, since we are dealing with bacterial culture supernatants containing unknown sucrose concentrations, we would not know how much to dilute it by. In this interest, we decided to plot a standard curve of absorbance vs sucrose concentration.
						<br><br>The absorbance readings increase linearly with sucrose concentration over a range of concentrations, before plateauing out at some value. If a particular sample yields an absorbance in the plateau range, it needs to be diluted down into the linear range before it can be measured again.

					</p>
					<br><br>

<div class="row"><!-- CYCLE D -->
	<div class="tab" role="tabpanel">
	    
	    <ul class="nav nav-tabs" role="tablist">
		<li role="presentation" class="active"><a href="#Cycle1_D" aria-controls="Cycle1_D" role="tab" data-toggle="tab">Cycle 1</a></li>
		<li role="presentation"><a href="#Cycle2_D" aria-controls="Cycle2_D" role="tab" data-toggle="tab">Cycle 2</a></li>
		<li role="presentation"><a href="#Cycle3_D" aria-controls="Cycle3_D" role="tab" data-toggle="tab">Cycle 3</a></li>
	    </ul>
	    
	    <div class="tab-content tabs">
		<div role="tabpanel" class="tab-pane fade in show active" id="Cycle1_D">
		    <h3>Cycle 1</h3>
			<table>
				<tr><td data-cycle="design">Design</td><td>Verify if the Megazyme sucrose D-glucose colorimetric assay kit works with sucrose solutions of known concentration</td></tr>
				<tr><td data-cycle="build">Build</td><td>Prepared sucrose solutions of known concentrations 0.1 g/L and 10 g/L <br><br>Used Megazyme’s kit protocol to verify the concentrations</td></tr>
				<tr><td data-cycle="test">Test</td><td>Solution A gave negative absorbance readings compared to the distilled water blank for both samples instead of the expected absorbance of 0.

					<br><br>Received large error in calculated concentration relative to the prepared concentrations.  
					
					<br><br>Since both samples were diluted to the same  concentration as per the kit’s protocol, they should have given the same absorbance reading for Solution B, which they failed to do so. 
					</td></tr>
				<tr><td data-cycle="learn">Learn</td><td>Don’t use just distilled water as a blank. Use distilled water + acetate buffer as blank for solution A, and distilled water + beta-fructosidase as blank for solution B. 

					<br><br>Biospectrometer may be faulty, use 96 well plate reader. 
					
					<br><br>Prepare fresh samples of sucrose solutions.
					</td></tr>
			</table>
		</div>
		<div role="tabpanel" class="tab-pane fade" id="Cycle2_D">
		    <h3>Cycle 2</h3>
		    <table>
			<tr><td data-cycle="design">Design</td><td>Prepare a standard curve of sucrose concentration versus absorbance. Use the curve to estimate concentration of sucrose from unknown samples/bacterial culture supernatants. </td></tr>
			<tr><td data-cycle="build">Build</td><td>Prepared fresh sucrose solutions of known concentrations: 0, 0.1, 2.5, 5, 7.5, 10, 15, 20 g/L

				<br><br>Performed the enzymatic reactions as per the kit protocol and measured absorbance in a 96 well plate reader. 
				
				<br><br>Plotted absorbance of the samples against their known concentrations.
				</td></tr>
			<tr><td data-cycle="test">Test</td><td>Obtained no discernible trend. </td></tr>
			<tr><td data-cycle="learn">Learn</td><td>Re-prepare sucrose samples with more precision and redo the standard curve. 

				<br><br>Store sucrose solution samples in the refrigerator to prevent any contamination. 
				</td></tr>
		</table>
		</div>
		<div role="tabpanel" class="tab-pane fade" id="Cycle3_D">
		    <h3>Cycle 3</h3>
		    <table>
			<tr><td data-cycle="design">Design</td><td>Re-prepared fresh sucrose samples to  construct a standard curve. </td></tr>
			<tr><td data-cycle="build">Build</td><td>Prepared fresh sucrose solutions of known concentrations: 0, 0.01, 0.05, 0.5, 1, 2.5, 5, 10, 20 g/L 

				<br><br>Performed the enzymatic reactions as per the kit protocol and measured absorbance in a 96 well plate reader. 
				</td></tr>
			<tr><td data-cycle="test">Test</td><td>Plotted absorbance of the samples against their known concentrations. 

				<br><br>Obtained a curve that follows a linear trend at concentrations in the range 0 to 2.5 g/L and then begins to plateau. 
				
				<br><br>The relative error between the calculated concentration of the samples and their known concentrations was fairly constant in the concentration range between 0.05 g/L and 1 g/L.
				</td></tr>
			<tr><td data-cycle="learn">Learn</td><td>The linear concentration range we obtained corresponds to the kit’s prescribed sucrose concentration range in which prior dilution is unnecessary.  

				<br><br>Thus, we have determined the sucrose concentration range in which the concentration can be measured with minimal and constant error - 0.05 to 1 g/L
				</td></tr>
		</table>
		</div>
	    </div>
	</div>
</div>
<br><br>
<h1 id="5">Experiment 5 - “S. elongatus growth curves”</h1>
<hr><br><br>

<div class="row"><!-- CYCLE E -->
	<div class="tab" role="tabpanel">
	    
	    <ul class="nav nav-tabs" role="tablist">
		<li role="presentation" class="active"><a href="#Cycle1_E" aria-controls="Cycle1_E" role="tab" data-toggle="tab">Cycle 1</a></li>
		<li role="presentation"><a href="#Cycle2_E" aria-controls="Cycle2_E" role="tab" data-toggle="tab">Cycle 2</a></li>
		<li role="presentation"><a href="#Cycle3_E" aria-controls="Cycle3_E" role="tab" data-toggle="tab">Cycle 3</a></li>
		<li role="presentation"><a href="#Cycle4_E" aria-controls="Cycle4_E" role="tab" data-toggle="tab">Cycle 4</a></li>
		<li role="presentation"><a href="#Cycle5_E" aria-controls="Cycle5_E" role="tab" data-toggle="tab">Cycle 5</a></li>
	    </ul>
	    
	    <div class="tab-content tabs">
		<div role="tabpanel" class="tab-pane fade in show active" id="Cycle1_E">
		    <h3>Cycle 1</h3>
			<table>
				<tr><td data-cycle="design">Design</td><td>Acclimate <em>S. elongatus </em>UTEX 2973-WT and UTEX 2973-<em>cscB</em> to 150 mM NaCl</td></tr>
				<tr><td data-cycle="build">Build</td><td>Inoculate WT and cscB cultures in BG11 with 150 mM NaCl and grow overnight until the culture is lime green and reaches an OD greater than 0.8</td></tr>
				<tr><td data-cycle="test">Test</td><td>Culture grows very slowly, does not grow healthy overnight</td></tr>
				<tr><td data-cycle="learn">Learn</td><td>Cultures pause their growth under salt stress for an acclimatory period. </td></tr>
			</table>
		</div>
		<div role="tabpanel" class="tab-pane fade" id="Cycle2_E">
		    <h3>Cycle 2</h3>
		    <table>
			<tr><td data-cycle="design">Design</td><td>Continue acclimation till maximum OD and inoculate new 100 mL cultures with 150 mM NaCl of OD in range 0.1 to 0.2</td></tr>
			<tr><td data-cycle="build">Build</td><td>cscB reached an OD of 0.6, while WT crossed 0.8. The former was pelleted down and inoculated into a new culture, while the latter could be transferred directly without pelleting. Two replicates of each were maintained</td></tr>
			<tr><td data-cycle="test">Test</td><td>The cscB cultures entered stasis and took a while to grow, while the WT cultures temporarily decreased in OD.</td></tr>
			<tr><td data-cycle="learn">Learn</td><td>Maintain similar treatment for both cultures to have comparable initial ODs.</td></tr>
		</table>
		</div>
		<div role="tabpanel" class="tab-pane fade" id="Cycle3_E">
		    <h3>Cycle 3</h3>
		    <table>
			<tr><td data-cycle="design">Design</td><td>Take measurements of OD and supernatant concentration of sucrose every three hours</td></tr>
			<tr><td data-cycle="build">Build</td><td>2 1mL-aliquots were taken from each culture for each measurement. The absorbance at 730 nm was measured for one aliquot. The other one was centrifuged to separate and store the supernatant for sucrose assay use.</td></tr>
			<tr><td data-cycle="test">Test</td><td>The growth rates were stunningly slow. Fitting the data to a logistic curve, we got doubling times of 27 and 36 hours in the exponential phase(given by T<sub>double</sub>=ln(2)/r). This is much slower than the minimum according to the literature (in the range of 1.5 hours).<br><br>
				The  sucrose produced by the cscB strain was 0.13 g/L while the literature reports a yield of 5 g/L in the same time.
			</td></tr>
			<tr><td data-cycle="learn">Learn</td><td>We decided to optimise the growth conditions for the culture to high carbon (0.5-0.6% as opposed to 0.04% of ambient CO2) and high light (150 &mu;Em<sup>-2</sup>s<sup>-1</sup>).</td></tr>
		</table>
		</div>
		<div role="tabpanel" class="tab-pane fade" id="Cycle4_E">
		    <h3>Cycle 4</h3>
		    <table>
			<tr><td data-cycle="design">Design</td><td>Prepare more BG-11 and then add bicarbonate to the medium</td></tr>
			<tr><td data-cycle="build">Build</td><td>We made the medium and autoclaved it</td></tr>
			<tr><td data-cycle="test">Test</td><td>The medium did not have a steady pH, crossing beyond 8.3 into 9+</td></tr>
			<tr><td data-cycle="learn">Learn</td><td>We need to stabilise the pH with TES buffer ( as recommended by Dr Michelle Liberton). As the CoBG11 medium optimised for co-culture with E. coli by Zhang et al, 2020 includes the TES buffer, 150 mM of NaCl and some additional NH<sub>4</sub>Cl, we decided to use it for the OD curve.[1]</td></tr>
		</table>
		</div>
		<div role="tabpanel" class="tab-pane fade" id="Cycle5_E">
			<h3>Cycle 5</h3>
			<table>
				<tr><td data-cycle="design">Design</td><td>We repeat the WT and cscB growth curves in CoBG11 with bicarbonate, controlling for the presence or absence of IPTG</td></tr>
				<tr><td data-cycle="build">Build</td><td>Acclimated the two strains, then inoculated one WT culture in 100mL CoBG11, along with 4 cscB cultures, of which two were induced with IPTG</td></tr>
				<tr><td data-cycle="test">Test</td><td>The cultures grew very strangely, growing greener and duller in unpredictable ways, perhaps because we have not optimised the frequency of bicarbonate flashing. However, the OD increased faster, reaching 1.8 in the case of one of the uninduced cscB cultures. At one point, the cells aggregated in the cscB cultures when they were removed from the incubator for a measurement, but the aggregates later disappeared.<br><br>
					The sucrose assays for these were ongoing as of the wiki freeze.</td></tr>
				<tr><td data-cycle="learn">Learn</td><td>We need to optimise the bicarbonate flashing frequency.</td></tr>
			</table>
		</div>
	    </div>
	</div>
	</div>

	<br><br><br>
	<h1 id="6">Co-culture modelling cycle</h1>
	<hr><br><br>

	<div class="row"><!-- CYCLE F -->
		<div class="tab" role="tabpanel">
		    
		    <ul class="nav nav-tabs" role="tablist">
			<li role="presentation" class="active"><a href="#Cycle1_F" aria-controls="Cycle1_F" role="tab" data-toggle="tab">Cycle 1</a></li>
		
		    </ul>
		    
		    <div class="tab-content tabs">
			<div role="tabpanel" class="tab-pane fade in show active" id="Cycle1_F">
			    <h3>Cycle 1</h3>
				<table>
					<tr><td data-cycle="design">Design</td><td>The design includes two modeling techniques - dynamic and steady state models, within which similar parameters could be varied and their effects could be analyzed.</td></tr>
					<tr><td data-cycle="build">Build</td><td>Steady state model - A joint model of the co-culture was built using the individual genome scale models of the cyanobacteria and the E. coli. *
						<br>Dynamic model - built by adapting the dynamic model made by the Toulouse INSA-UPS team. 
						</td></tr>
					<tr><td data-cycle="test">Test</td><td>Varying parameters such as sucrose productivity of cyanobacteria, butanol productivity of E. coli, O2 level in the consortium and CO2 uptake, to see its effects on the butanol yield and the population dynamics of the organisms.</td></tr>
					<tr><td data-cycle="learn">Learn</td><td>The two models, based on different assumptions, provide similar results on strategies to optimize butanol production by the co-culture.</td></tr>
				</table>
			</div>
			
		    </div>
		</div>
		</div>
<br><br>
		<h1 id="7">Building the SteadyCom model</h1>
		<hr><br><br>
		<div class="row"><!-- CYCLE G -->
			<div class="tab" role="tabpanel">
			    
			    <ul class="nav nav-tabs" role="tablist">
				<li role="presentation" class="active"><a href="#Cycle1_G" aria-controls="Cycle1_G" role="tab" data-toggle="tab">Cycle 1</a></li>
				<li role="presentation"><a href="#Cycle2_G" aria-controls="Cycle2_G" role="tab" data-toggle="tab">Cycle 2</a></li>
				<li role="presentation"><a href="#Cycle3_G" aria-controls="Cycle3_G" role="tab" data-toggle="tab">Cycle 3</a></li>
				<li role="presentation"><a href="#Cycle4_G" aria-controls="Cycle4_G" role="tab" data-toggle="tab">Cycle 4</a></li>
				<li role="presentation"><a href="#Cycle5_G" aria-controls="Cycle5_G" role="tab" data-toggle="tab">Cycle 5</a></li>
			    </ul>
			    
			    <div class="tab-content tabs">
				<div role="tabpanel" class="tab-pane fade in show active" id="Cycle1_G">
				    <h3>Cycle 1</h3>
					<table>
						<tr><td data-cycle="design">Design</td><td>The steady state model was designed to be built using the SteadyCom algorithm. This requires making a joint model of certain specifications built from the individual genome scale models. </td></tr>
						<tr><td data-cycle="build">Build</td><td>The joint model was built using CreateMultispeciesmodel.m function in the COBRA Toolbox from modified versions of iML1515 and iSyu683.</td></tr>
						<tr><td data-cycle="test">Test</td><td>SteadyCom was run on the joint model. However, the results showed that the co-culture was infeasible.</td></tr>
						<tr><td data-cycle="learn">Learn</td><td>To troubleshoot and understand the specific issue for this infeasibility, cultures of just E. coli and cyanobacteria were individually built with CreatMultispeciesmodel.m function and analysed with SteadyCom. 
							<br>Results showed that the issue was in the cyanobacteria model due to lack of standardization of reaction names and the format of sink reactions. This prevented the CreateMultispeciesmodel.m function from identifying the reactions correctly.
							</td></tr>
					</table>
				</div>
				<div role="tabpanel" class="tab-pane fade" id="Cycle2_G">
				    <h3>Cycle 2</h3>
				    <table>
					<tr><td data-cycle="design">Design</td><td>Reaction names were changed in the genome scale model and were manually curated into the joint model by us.</td></tr>
					<tr><td data-cycle="build">Build</td><td>SteadyCom was run using the new joint model.</td></tr>
					<tr><td data-cycle="test">Test</td><td>SteadyCom showed a successful steady state growth of both organisms with the production of butanol.</td></tr>
					<tr><td data-cycle="learn">Learn</td><td>Sink reactions in the individual species that take metabolites from the intracellular compartments are not recognized by the CreateMultispeciesModel function and hence not included in the joint model and have to be manually added. </td></tr>
				</table>
				</div>
			    </div>
			</div>
			</div>
<br><br>
<h1 id="8">Simulating Salt Stress</h1>
		<hr><br><br>
			<div class="row"><!-- CYCLE H -->
				<div class="tab" role="tabpanel">
				    
				    <ul class="nav nav-tabs" role="tablist">
					<li role="presentation" class="active"><a href="#Cycle1_H" aria-controls="Cycle1_H" role="tab" data-toggle="tab">Cycle 1</a></li>
					<li role="presentation"><a href="#Cycle2_H" aria-controls="Cycle2_H" role="tab" data-toggle="tab">Cycle 2</a></li>
					<li role="presentation"><a href="#Cycle3_H" aria-controls="Cycle3_H" role="tab" data-toggle="tab">Cycle 3</a></li>
					<li role="presentation"><a href="#Cycle4_H" aria-controls="Cycle4_H" role="tab" data-toggle="tab">Cycle 4</a></li>
					<li role="presentation"><a href="#Cycle5_H" aria-controls="Cycle5_H" role="tab" data-toggle="tab">Cycle 5</a></li>
				    </ul>
				    
				    <div class="tab-content tabs">
					<div role="tabpanel" class="tab-pane fade in show active" id="Cycle1_H">
					    <h3>Cycle 1</h3>
						<table>
							<tr><td data-cycle="design">Design</td><td>The design includes running FBA on a GSM model to obtain values that agree with either literature or our wet lab results. 
								<br>Media composition determined from previous experimental data in literature.
								</td></tr>
							<tr><td data-cycle="build">Build</td><td>GSM model of cyanobacteria S. elongatus UTEX 2973 obtained  from Dr. Costas Manaras’ Lab (Pennsylvania)</td></tr>
							<tr><td data-cycle="test">Test</td><td>Compare values of the simulation to literature values under similar media composition.
								<br>Values obtained were slightly off from the values in (Abernathy et al)
								</td></tr>
							<tr><td data-cycle="learn">Learn</td><td>MoMA is required to traverse all possibilities and narrow down on more realistic values</td></tr>
						</table>
					</div>
					<div role="tabpanel" class="tab-pane fade" id="Cycle2_H">
					    <h3>Cycle 2</h3>
					    <table>
						<tr><td data-cycle="design">Design</td><td>Running MoMA on the GSM model for cyanobacteria </td></tr>
						<tr><td data-cycle="build">Build</td><td>Organism (model) is put under 150mM salt stress for varying values of CO_2 uptake rates.
							<br>Salt stress is simulated by coupling biomass and and sucrose accumulation (song et al)</td></tr>
						<tr><td data-cycle="test">Test</td><td>Compare values for growth rate with Abernathy et al (2017) and Song et al (2016)</td></tr>
						<tr><td data-cycle="learn">Learn</td><td>The model is good enough to produce results from literature increasing our confidence in the viability of the model.</td></tr>
					</table>
					</div>
				    </div>
				</div>
				</div>
				<br><br><br><h1 id="8">The Grand Cycle</h1><br><br>

				<h3 style="text-decoration: underline;"><i>Notation:</i></h3>
		    <p>In this write-up, we use the following shorthand terms:<br>
			<ul>
				<li>By <i>sucrose productivity</i> we denote the rate of sucrose production from S. elongatus in mmol gDW<sup>-1</sup> hr<sup>-1</sup>  </li>
				<li>By <i>butanol productivity</i> we denote the rate of butanol production from E. coli KJK01and its transformants in mmol gDW<sup>-1</sup> hr<sup>-1</sup>.</li>
			</ul>
			</p>
			<div class="row"><!-- CYCLE I -->
				<div class="tab" role="tabpanel">
				    
				    <ul class="nav nav-tabs" role="tablist">
					<li role="presentation" class="active"><a href="#Cycle1_I" aria-controls="Cycle1_I" role="tab" data-toggle="tab">Cycle 1</a></li>
					<li role="presentation"><a href="#Cycle2_I" aria-controls="Cycle2_I" role="tab" data-toggle="tab">Cycle 2</a></li>
					<li role="presentation"><a href="#Cycle3_I" aria-controls="Cycle3_I" role="tab" data-toggle="tab">Cycle 3</a></li>
				    </ul>
				    
				    <div class="tab-content tabs">
					<div role="tabpanel" class="tab-pane fade in show active" id="Cycle1_I">
					    <h3>Cycle 1</h3>
						<table>
							<tr><td data-cycle="design">Design</td><td>Based on the shortlist of overexpression and deletion targets from our strain optimization modelling, we will design experiments to actually conduct these overexpression and deletions in S. elongatus and E. coli.</td></tr>
							<tr><td data-cycle="build">Build</td><td>We will then execute these experiments and measure the sucrose and butanol productivity of all the modified strains</td></tr>
							<tr><td data-cycle="test">Test</td><td>We will compare this with the original strains to see if the knockout/deletion has made a significant positive impact</td></tr>
							<tr><td data-cycle="learn">Learn</td><td>Those modified strains which give the highest results of increased sucrose and butanol productivity will be selected to be used in the future</td></tr>
						</table>
					</div>
					<div role="tabpanel" class="tab-pane fade" id="Cycle2_I">
					    <h3>Cycle 2</h3>
					    <table>
						<tr><td data-cycle="design">Design</td><td>Our co-culture modelling predicted that increasing both butanol productivity of E. coli and Sucrose productivity of cyanobacteria will result in a better yield of butanol as a whole from the co culture. However this must be tested in the wet lab. We will design experiments to compare the butanol yield of the co culture with the optimized strains from the previous step to the butanol yield of a co culture of the base strains. </td></tr>
						<tr><td data-cycle="build">Build</td><td>We will then execute these experiments and measure the butanol yield from both co cultures</td></tr>
						<tr><td data-cycle="test">Test</td><td>We will compare the butanol yields and see if using the optimized strains with higher sucrose and butanol productivity gives a higher butanol yield in co culture.</td></tr>
						<tr><td data-cycle="learn">Learn</td><td>Based on the results, we will know if the results of our co culture modelling holds and that if increasing sucrose and butanol productivity in fact does increase the co culture butanol yield</td></tr>
					</table>
					</div>
					<div role="tabpanel" class="tab-pane fade" id="Cycle3_I">
					    <h3>Cycle 3</h3>
					    <table>
						<tr><td data-cycle="design">Design</td><td>Our co culture modelling also predicted that extracting oxygen from the co culture would increase the butanol yield. These results must also be tested in the wetlab. We will design experiments to test the butanol yield from the co culture at different oxygen levels. </td></tr>
						<tr><td data-cycle="build">Build</td><td>We will execute these experiments and measure the butanol yields at varying oxygen levels</td></tr>
						<tr><td data-cycle="test">Test</td><td>We will identify the oxygen level at which butanol yield is maximum</td></tr>
						<tr><td data-cycle="learn">Learn</td><td>Based on the above, we will know if the result from drylab that a microaerobic setting for E. coli will give us the highest butanol yield. </td></tr>
					</table>
					</div>
				    </div>
				</div>
				</div>
                        <!-- <p>
                            <ul class="a">
                                <li style="padding-left:1em;padding-top: 0.3em;">Purify DNA and PCR-verify presence of genes in all strains/plasmids.</li>
                                <li style="padding-left:1em;padding-top: 0.3em;">Revive ordered bacteria from stocks. Streak new plates and prepare stocks.</li>
                                <li style="padding-left:1em;padding-top: 0.3em;">Pellet down KJK01 and check the supernatant for butanol via NMR.</li>
                                <li style="padding-left:1em;padding-top: 0.3em;">Make E. coli competent for transformation</li>
                                <li style="padding-left:1em;padding-top: 0.3em;">Transform E. coli with pCscX. </li>
                                <li style="padding-left:1em;padding-top: 0.3em;">Take growth measurements of our E. coli over time.</li>
                                <li style="padding-left:1em;padding-top: 0.3em;">Use MegaZyme’s sucrose assay kit to test for sucrose consumption by KJK01 (sample successive supernatants) over time.</li>
                                <li style="padding-left:1em;padding-top: 0.3em;">Grow S. elongatus UTEX 2973 (from Pakrasi lab) and test for sucrose production.</li>
                                <li style="padding-left:1em;padding-top: 0.3em;">Genomically integrate native promoters and carry out fluorescence measurement assays for promoter activity, under high and low light intensities, and high and low carbon dioxide concentrations.</li>
                            </ul><br>
                            Due to the second wave of the pandemic in India, our wet lab work was considerably delayed, effectively leaving us with a mere month and a half in the lab. We utilised our time working remotely to devise a thorough plan of experiments, incorporating a Design-Test-Build cycle:
                        </p> -->

			<h1 id="9">References</h1><br>
			<ol>
				<li>Zhang, L., Chen, L., Diao, J., Song, X., Shi, M., & Zhang, W. (2020). Construction and analysis of an artificial consortium based on the fast-growing cyanobacterium Synechococcus elongatus UTEX 2973 to produce the platform chemical 3-hydroxypropionic acid from CO.</li>
			</ol>
			</div>

		</div>

        </div>

    </div>
<!-- SPONSORS BAR -->
	<div class="footer">
		<div class="footer-wrapper">
			<!-- <div id="not-sponsors" class="footer-wrapper">	 -->
				<div id="Team", class="footer-wrapper">
					<img class="teamlogo" src="assets/Logos/IISER-Pune-iGEM_logo.png" alt="iGEM IISER Pune Logo" width="170">
				</div>
				<div id="socials" class="vertical-stack">
					<div><h3>Team IISER Pune India</h3></div>
					<div id="logos" class ="footer-wrapper ">
						<a href="https://www.instagram.com/igem_iiserp/"><img class="footer-links-socials" src="assets/Logos/Instagram_logo.jpg" alt="Instagram" width="20"></a>
						<a href="https://www.linkedin.com/in/igem-iiser-pune-2021/"><img class="footer-links-socials" src="assets/Logos/linkedin_logo.png" alt="Linkedin" width="30"></a>
						<a href="https://twitter.com/igem_iiserpune"><img class="footer-links-socials" src="assets/Logos/Twitter_logo.png" alt="Twitter" width="35"></a>
						<a href="https://www.facebook.com/igemiiserpune"><img class="footer-links-socials" src="assets/Logos/Facebook_logo.png" alt="Facebook" width="25"></a>
						<a href="https://www.youtube.com/channel/UCyo6NMLvM58sSh_tnxht36w"><img class="footer-links-socials" src="assets/Logos/Youtube_logo.png" alt="Youtube" width="35"></a>
					</div>
				</div>
			<!-- </div> -->
			<div id="sponsors" class="footer-wrapper">
				<div id="main-sponsor", class="">
					<a href="https://www.sailife.com/"><img class="footer-links" src="assets/Logos/Sai_logo.jpg" alt="Sai Life Sciences" width="280"></a>
				</div>
				<div class="vertical-stack">
					<a href="https://sg.idtdna.com/pages"><img class="footer-links" src="assets/Logos/IDT_logo.png" alt="IDT" width="220"></a>
					<a href="https://www.geneious.com/"><img class="footer-links" src="assets/Logos/Geneious_logo.png" alt="Geneious" width="220"></a>
				</div>
				<div class="vertical-stack">
					<div id="horizontal stack">
						<a href="https://www.notion.so/"><img class="footer-links" src="assets/Logos/Notion_logo.png" alt="Notion" width="70"></a>
						<img src="assets/Logos/IISER Pune Logo.png" alt="IISER Pune" width="80" style="padding-left: 1em;">
					</div>
					<a href="https://www.zoho.com/"><img class="footer-links" src="assets/Logos/Zoho_logo.png" alt="Zoho" width="150"></a>
				</div>
			</div>	
		</div>
	</div>
	<!-- END OF SPONSOR BAR-->

    <div class="header"></div>
    <div class="line" id="scrollIndicator"></div>

    <script src="js/bootstrap-bundle.js"></script>
	<script src="js/main.js"></script>
	<script src="js/jquery.min.js"></script>

	<script>
		var screenheight = window.innerHeight,
		    screenwidth = window.innerWidth;

	</script>
	<script>
		var body = document.body,
    		html = document.documentElement;

		var pageheight = Math.max( body.scrollHeight, body.offsetHeight, 
                       		   html.clientHeight, html.scrollHeight, html.offsetHeight );
	</script>
	<script>
		var backtotopheight=pageheight/4.65
	</script>
	<script>
		var total = $('.item').length;
		var actualtotal = 80;
		var currentIndex = $('div.active').index() + 1;
		$('#slidetext').html(currentIndex + '/'  + actualtotal);

		// This triggers after each slide change
		$('.carousel').on('slid.bs.carousel', function () {
  			currentIndex = $('div.active').index() + 1;
 
  		// Now display this wherever you want
  		var text = currentIndex + '/' + actualtotal;
  			$('#slidetext').html(text);
		});

	</script>
	<script>    
        $(function(){
            $('#sidenav-content a.tab-link').click(function(){
                var target = $(this).attr('href');
                $('#main-card-header a[href="' + target+ '"]').tab('show');
                
                $('html, body').animate({
                    scrollTop: $('#scroll-placeholder').offset().top - 70
                }, 200);
            });
        });
	</script>
    	<script>
            $("ul.nav-tabs a").click(function (e) {
      e.preventDefault();  
        $(this).tab('show');
    });
    
        </script>
</body>
